--- rukrakia\KOREAN\RU_ELBER.tra	Thu Oct 04 14:59:23 2007
+++ C:\BWP Patchstudio\patched files\rukrakia\KOREAN\RU_ELBER.tra	Fri Oct 03 04:56:14 2014
@@ -18,4 +18,7 @@
 @17   = ~~
 @18   = ~~
 @19   = ~~
-@20   = ~~
\ No newline at end of file
+@20   = ~~
+@21   = ~라이젠의 무덤?
+
+엘버라는 자는 젠킨스가 말했던 대로 앰케스란의 여관에 있었다. 내가 그를 추궁하거나 그렇지 않거나 어느 쪽이든 그는 달가워 하지 않는 듯 보였다. 그의 품에 라이젠의 무덤의 위치가 적혀진 지도가 있었고 나는 선택을 해야만 한다.~
