--- rukrakia\KOREAN\rukra25p.tra	Thu Aug 16 14:03:37 2007
+++ C:\BWP Patchstudio\patched files\rukrakia\KOREAN\rukra25p.tra	Fri Oct 03 05:04:09 2014
@@ -17,4 +17,4 @@
 @16   = ~아아, 농담이 아니군요! 저는 제 사랑을 믿었는데...~
 @17   = ~원망은 하지 않을게요... 다만... 아니에요. 갈게요. *훌쩍* 간다구요...~
 @18   = ~농담이 아니었다니... 사랑이 식어버린 건 아니죠, <GABBER>? 그럴거라구 믿을게요.~
-@19   = ~거기라면 안전하긴 하지만... 사랑이 식어버린 건 아니죠, <GABBER>? 그럴거라구 믿을게요.~
\ No newline at end of file
+@19   = ~거기라면 안전하긴 하지만... 사랑이 식어버린 건 아니죠, <GABBER>? 그럴거라구 믿을게요.~
