--- rukrakia\KOREAN\rukra25j.tra	Sat Sep 29 06:06:25 2007
+++ C:\BWP Patchstudio\patched files\rukrakia\KOREAN\rukra25j.tra	Fri Oct 03 05:03:30 2014
@@ -230,4 +230,4 @@
 @229  = ~디먼 프린스 데모고르곤이라... 그 존재가 악마의 왕자였군요. 그럼 오드렌은 지금쯤...~
 @230  = ~어떠한 결정을 내리든지 저는 반대하지 않아요, <GABBER>. 다만 저는 그 악마의 왕자와 진지한 면담을 해보고 싶군요.~
 @231  = ~푸훗! 당신에게 흡정검을 꽂아넣는 것도 영광이겠군요, 악마 공작!~
-@232  = ~누가 학살당하는지 두고 보면 알겠지.~
\ No newline at end of file
+@232  = ~누가 학살당하는지 두고 보면 알겠지.~
