--- NeJ3\dlg\DENTGRD.d	Sat Dec 16 05:10:58 2006
+++ C:\BWP Patchstudio\patched files\NeJ3\dlg\DENTGRD.d	Wed Apr 10 10:24:30 2019
@@ -1,24 +1,24 @@
 BEGIN ~DENTGRD~
 
 IF ~Global("Kerish_Look","GLOBAL", 0)~ THEN BEGIN 0
-  SAY ~Not know you.  What you want here?~
-  IF ~~ THEN REPLY ~I’m here to see your leader.~ GOTO 2
-  IF ~~ THEN REPLY ~Nothing.  Farewell.~ GOTO 3
+  SAY @0
+  IF ~~ THEN REPLY @1 GOTO 2
+  IF ~~ THEN REPLY @2 GOTO 3
 END
 
 IF ~Global("Kerish_Look","GLOBAL", 1)~ THEN BEGIN 1
-  SAY ~Kerish let you walk.  He not let me speak to you.  Go now.~
+  SAY @3
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~You want see Kerish?  He leader.~
-  IF ~~ THEN REPLY ~Yes, take me to see Kerish.~ DO ~StartCutSceneMode()
+  SAY @4
+  IF ~~ THEN REPLY @5 DO ~StartCutSceneMode()
 StartCutScene("wtgcut2")~ EXIT
-  IF ~~ THEN REPLY ~No, I have changed my mind.  Farewell.~ GOTO 3
+  IF ~~ THEN REPLY @6 GOTO 3
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~Then you leave.  Go now!~
-  IF ~~ THEN REPLY ~Very well, farewell.~ DO ~StartCutScene("wtgcut1")~ EXIT
+  SAY @7
+  IF ~~ THEN REPLY @8 DO ~StartCutScene("wtgcut1")~ EXIT
 END
