--- NeJ3\dlg\DGNOMESL.d	Tue Mar 25 01:44:54 2014
+++ C:\BWP Patchstudio\patched files\NeJ3\dlg\DGNOMESL.d	Wed Apr 10 10:24:38 2019
@@ -1,51 +1,51 @@
 BEGIN ~DGNOMESL~
 
 IF ~RandomNum(10,1)~ THEN BEGIN 0
-  SAY ~This is hell.  Urdlen and his Worm Realm have nothing on Marketh's mines.~
+  SAY @0
   IF ~~ THEN EXIT
 END
 
 IF ~RandomNum(10,2)~ THEN BEGIN 1
-  SAY ~Move along, stranger.  You'll only find death down here.~
+  SAY @1
   IF ~~ THEN EXIT
 END
 
 IF ~RandomNum(10,3)~ THEN BEGIN 2
-  SAY ~You picked the wrong place to poke your nose.  Get out of here before Marketh and his goons find you.~
+  SAY @2
   IF ~~ THEN EXIT
 END
 
 IF ~RandomNum(10,4)~ THEN BEGIN 3
-  SAY ~Another day, another ton of gemstones dumped in the lap of a madman.~
+  SAY @3
   IF ~~ THEN EXIT
 END
 
 IF ~RandomNum(10,5)~ THEN BEGIN 4
-  SAY ~I used to like mining, you know...~
+  SAY @4
   IF ~~ THEN EXIT
 END
 
 IF ~RandomNum(10,6)~ THEN BEGIN 5
-  SAY ~It's said that some of our people escaped from Dorn's and are safe somewhere.  Tarnelm in the Oubliette may know more.~
+  SAY @5
   IF ~~ THEN EXIT
 END
 
 IF ~RandomNum(10,7)~ THEN BEGIN 6
-  SAY ~The mad mage allied with Marketh has bull-headed men and umber hulks serving him.  When we fail, the hulks rip us to pieces.~
+  SAY @6
   IF ~~ THEN EXIT
 END
 
 IF ~RandomNum(10,8)~ THEN BEGIN 7
-  SAY ~Elementals are being destroyed nearby.  I can feel the ground cry out when the earth elementals are harmed.~
+  SAY @7
   IF ~~ THEN EXIT
 END
 
 IF ~RandomNum(10,9)~ THEN BEGIN 8
-  SAY ~No!  No!  Don't hit me!  The crossed hands, the bloody wrists!  No more!~
+  SAY @8
   IF ~~ THEN EXIT
 END
 
 IF ~RandomNum(10,10)~ THEN BEGIN 9
-  SAY ~Stay away from the Artisan's District and Malavon's Dome.  Shikata's salamanders are kind compared to the monsters there.~
+  SAY @9
   IF ~~ THEN EXIT
 END
