--- NeJ3\dlg\DJORILBG.d	Sat Dec 16 05:53:20 2006
+++ C:\BWP Patchstudio\patched files\NeJ3\dlg\DJORILBG.d	Wed Apr 10 10:24:43 2019
@@ -1,32 +1,32 @@
 BEGIN ~DJORILBG~
 
 IF ~NumTimesTalkedTo(0)~ THEN BEGIN 0
-  SAY ~I have better things to do than talk with you.  You want something?~
-  IF ~~ THEN REPLY ~I知 here to see your leader.~ GOTO 2
-  IF ~~ THEN REPLY ~I would like to ask you some questions.~ GOTO 4
-  IF ~~ THEN REPLY ~No, farewell.~ EXIT
+  SAY @0
+  IF ~~ THEN REPLY @1 GOTO 2
+  IF ~~ THEN REPLY @2 GOTO 4
+  IF ~~ THEN REPLY @3 EXIT
 END
 
 IF ~NumTimesTalkedToGT(0)~ THEN BEGIN 1
-  SAY ~Why are you bothering me again?  Go away!~
-  IF ~~ THEN REPLY ~I知 here to see your leader.~ GOTO 2
-  IF ~~ THEN REPLY ~I would like to ask you some questions.~ GOTO 4
-  IF ~~ THEN REPLY ~Very well, farewell.~ EXIT
+  SAY @4
+  IF ~~ THEN REPLY @1 GOTO 2
+  IF ~~ THEN REPLY @2 GOTO 4
+  IF ~~ THEN REPLY @5 EXIT
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~See the throne over there and the guy in front of it?  Need another hint?~
-  IF ~~ THEN REPLY ~No, I think understand quite well.  Farewell.~ EXIT
-  IF ~~ THEN REPLY ~I do not like your attitude.  I think I need to teach you some manners.~ GOTO 3
+  SAY @6
+  IF ~~ THEN REPLY @7 EXIT
+  IF ~~ THEN REPLY @8 GOTO 3
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~Ha!  I知 going to enjoy this more than you can imagine.  Die, scum!~
-  IF ~~ THEN REPLY ~So be it!~ DO ~Enemy()~ EXIT
+  SAY @9
+  IF ~~ THEN REPLY @10 DO ~Enemy()~ EXIT
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~Ha!  I don稚 care.  Go bother someone else with your questions.~
-  IF ~~ THEN REPLY ~I知 here to see your leader.~ GOTO 2
-  IF ~~ THEN REPLY ~I see.  Farewell.~ EXIT
+  SAY @11
+  IF ~~ THEN REPLY @1 GOTO 2
+  IF ~~ THEN REPLY @12 EXIT
 END
