--- NeJ3\dlg\DKERISH.d	Sun Mar 23 18:28:24 2014
+++ C:\BWP Patchstudio\patched files\NeJ3\dlg\DKERISH.d	Wed Apr 10 10:24:44 2019
@@ -1,418 +1,266 @@
 BEGIN ~DKERISH~
 
 IF ~NumTimesTalkedTo(0)~ THEN BEGIN 0
-  SAY ~I greet you.  I am Kerish, leader of the salamanders...  Who are you?~
-  IF ~~ THEN REPLY ~I am a simple traveler. ~ DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 2
-  IF ~~ THEN REPLY ~An adventurer.~ DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 3
-  IF ~~ THEN REPLY ~Me?  I am an explorer who chanced upon this place. ~ DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 4
+  SAY @0
+  IF ~~ THEN REPLY @1 DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 2
+  IF ~~ THEN REPLY @2 DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 3
+  IF ~~ THEN REPLY @3 DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 4
 END
 
 IF ~NumTimesTalkedToGT(0)~ THEN BEGIN 1
-  SAY ~You have returned.  What can I do for you?~
-  IF ~~ THEN REPLY ~I have some questions for you.~ DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 6
+  SAY @4
+  IF ~~ THEN REPLY @5 DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 6
   IF ~Global("Get_Key","GLOBAL", 3)
-Global("Got_KeyDug", "GLOBAL", 0)~ THEN REPLY ~I want that key that unlocks the exit to the surface.~ DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 19
-  IF ~Global("Kill_Vera","GLOBAL", 1)~ THEN REPLY ~I said I would think about your request to kill the slave leader.  I have decided to do it.~ DO ~SetGlobal("Kill_Vera","GLOBAL", 3)
-SetGlobal("Kerish_Talk","GLOBAL", 0)~ UNSOLVED_JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We agreed to do this.~ GOTO 21
-  IF ~Global("Kill_Vera","GLOBAL", 2)~ THEN REPLY ~I have changed my mind about killing the slave leader for you - I’ll do it.~ DO ~SetGlobal("Kill_Vera","GLOBAL", 3)
-SetGlobal("Kerish_Talk","GLOBAL", 0)~ UNSOLVED_JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We agreed to do this.~ GOTO 21
-  IF ~Global("Kill_Vera","GLOBAL", 3)~ THEN REPLY ~You asked me to kill the slave leader for you.~ DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 22
-  IF ~Global("Return_Slaves","GLOBAL", 1)~ THEN REPLY ~Earlier you asked me to return the slaves to you or slay them.  I have decided to do it.~ DO ~SetGlobal("Return_Slaves","GLOBAL", 3)
-SetGlobal("Kerish_Talk","GLOBAL", 0)~ UNSOLVED_JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We agreed to do this.~ GOTO 21
-  IF ~Global("Return_Slaves","GLOBAL", 2)~ THEN REPLY ~I have changed my mind about returning the slaves to you or slaying them - I’ll do it.~ DO ~SetGlobal("Return_Slaves","GLOBAL", 3)
-SetGlobal("Kerish_Talk","GLOBAL", 0)~ UNSOLVED_JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We agreed to do this.~ GOTO 21
-  IF ~Global("Return_Slaves","GLOBAL", 3)~ THEN REPLY ~You asked me to either return the slaves to you or slay them.~ DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 27
-  IF ~~ THEN REPLY ~Nothing for now.  Farewell.~ DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ EXIT
+Global("Got_KeyDug", "GLOBAL", 0)~ THEN REPLY @6 DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 19
+  IF ~Global("Kill_Vera","GLOBAL", 1)~ THEN REPLY @7 DO ~SetGlobal("Kill_Vera","GLOBAL", 3)
+SetGlobal("Kerish_Talk","GLOBAL", 0)~ UNSOLVED_JOURNAL @8 GOTO 21
+  IF ~Global("Kill_Vera","GLOBAL", 2)~ THEN REPLY @9 DO ~SetGlobal("Kill_Vera","GLOBAL", 3)
+SetGlobal("Kerish_Talk","GLOBAL", 0)~ UNSOLVED_JOURNAL @8 GOTO 21
+  IF ~Global("Kill_Vera","GLOBAL", 3)~ THEN REPLY @10 DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 22
+  IF ~Global("Return_Slaves","GLOBAL", 1)~ THEN REPLY @11 DO ~SetGlobal("Return_Slaves","GLOBAL", 3)
+SetGlobal("Kerish_Talk","GLOBAL", 0)~ UNSOLVED_JOURNAL @12 GOTO 21
+  IF ~Global("Return_Slaves","GLOBAL", 2)~ THEN REPLY @13 DO ~SetGlobal("Return_Slaves","GLOBAL", 3)
+SetGlobal("Kerish_Talk","GLOBAL", 0)~ UNSOLVED_JOURNAL @12 GOTO 21
+  IF ~Global("Return_Slaves","GLOBAL", 3)~ THEN REPLY @14 DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ GOTO 27
+  IF ~~ THEN REPLY @15 DO ~SetGlobal("Kerish_Talk","GLOBAL", 0)~ EXIT
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~A traveler...?  Forgive me, but I doubt you are a traveler.  No simple vagabond could journey here unscathed...  No, you are more than you seem.  A spy, perhaps?  Come to check on me?  Yes... he has been known to hire those of your race.  Why has he sent you?~
-  IF ~~ THEN REPLY ~I *am* a traveler.  However, if you choose to believe otherwise...~ GOTO 5
-  IF ~~ THEN REPLY ~Who is it that you think I work for, Kerish?~ GOTO 25
-  IF ~~ THEN REPLY ~I simply want to be allowed to look around.~ GOTO 18
+  SAY @16
+  IF ~~ THEN REPLY @17 GOTO 5
+  IF ~~ THEN REPLY @18 GOTO 25
+  IF ~~ THEN REPLY @19 GOTO 18
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~An adventurer...?  Forgive me, but I doubt you are an adventurer.  No seeker of fortune could journey here unscathed...  No, you are more than you seem.  A spy, perhaps?  Come to check on me?  Yes... he has been known to hire those of your race.  Why has he sent you?~
-  IF ~~ THEN REPLY ~I *am* an adventurer.  However, if you choose to believe otherwise...~ GOTO 5
-  IF ~~ THEN REPLY ~Who is it that you think I work for, Kerish?~ GOTO 25
-  IF ~~ THEN REPLY ~I simply want to be allowed to look around.~ GOTO 18
+  SAY @20
+  IF ~~ THEN REPLY @21 GOTO 5
+  IF ~~ THEN REPLY @18 GOTO 25
+  IF ~~ THEN REPLY @19 GOTO 18
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~An explorer...?  Forgive me, but I doubt you are an explorer.  No seeker of the unknown could journey here unscathed...  No, you are more than you seem.  A spy, perhaps?  Come to check on me?  Yes... he has been known to hire those of your race.  Why has he sent you?~
-  IF ~~ THEN REPLY ~I *am* an explorer.  However, if you choose to believe otherwise...~ GOTO 5
-  IF ~~ THEN REPLY ~Who is it that you think I work for, Kerish?~ JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the leader of the frost salamanders, believes that we work for someone named Marketh. We decided to do nothing to dissuade him of that belief. It could prove useful.~ GOTO 25
-  IF ~~ THEN REPLY ~I simply want to be allowed to look around.~ GOTO 18
+  SAY @22
+  IF ~~ THEN REPLY @23 GOTO 5
+  IF ~~ THEN REPLY @18 JOURNAL @24 GOTO 25
+  IF ~~ THEN REPLY @19 GOTO 18
 END
 
 IF ~~ THEN BEGIN 5
-  SAY ~Very well...  Play your game of deception, but know you this.   I know your master.  In time I will have an answer.  If not from you, then him.  So tell me, why are you here?~
-  IF ~~ THEN REPLY ~I would ask you some questions.~ GOTO 6
-  IF ~~ THEN REPLY ~I simply want to be allowed to look around.~ GOTO 18
+  SAY @25
+  IF ~~ THEN REPLY @26 GOTO 6
+  IF ~~ THEN REPLY @19 GOTO 18
 END
 
 IF ~~ THEN BEGIN 6
-  SAY ~Then ask.~
-  IF ~~ THEN REPLY ~Who are you again?~ GOTO 7
-  IF ~~ THEN REPLY ~What are you doing here?~ GOTO 8
-  IF ~~ THEN REPLY ~I have come seeking the one in charge.~ DO ~SetGlobal("Joril_Kill_Dad","GLOBAL", 1)~ JOURNAL ~Wyrm’s Tooth.
-
-We have learned that Joril Frostbeard is the lieutenant of Wyrm’s Tooth Glacier. Apparently he took over the position by killing his father, Kreg.~ GOTO 10
-  IF ~~ THEN REPLY ~Why are there dead bodies strewn about here?~ JOURNAL ~Wyrm’s Tooth.
-
-Apparently there has been a slave revolt among the frost salamanders. A few managed to escape but most died in the attempt.~ GOTO 12
-  IF ~Global("Get_Key","GLOBAL", 3)Global("Got_KeyDug", "GLOBAL", 0)~ THEN REPLY ~I want that key that unlocks the exit to the surface.~ GOTO 19
-  IF ~~ THEN REPLY ~I must leave now. Farewell.~ GOTO 18
+  SAY @27
+  IF ~~ THEN REPLY @28 GOTO 7
+  IF ~~ THEN REPLY @29 GOTO 8
+  IF ~~ THEN REPLY @30 DO ~SetGlobal("Joril_Kill_Dad","GLOBAL", 1)~ JOURNAL @31 GOTO 10
+  IF ~~ THEN REPLY @32 JOURNAL @33 GOTO 12
+  IF ~Global("Get_Key","GLOBAL", 3)Global("Got_KeyDug", "GLOBAL", 0)~ THEN REPLY @6 GOTO 19
+  IF ~~ THEN REPLY @34 GOTO 18
 END
 
 IF ~~ THEN BEGIN 7
-  SAY ~I am Kerish, leader of the salamanders... as you well know. ~
-  IF ~~ THEN REPLY ~I have some other questions for you.~ GOTO 6
-  IF ~~ THEN REPLY ~I must be going.  Farewell for now.~ GOTO 18
+  SAY @35
+  IF ~~ THEN REPLY @36 GOTO 6
+  IF ~~ THEN REPLY @37 GOTO 18
 END
 
 IF ~~ THEN BEGIN 8
-  SAY ~What do you mean?  I am awaiting orders as instructed and nothing else...  Why do you ask?  Has someone reported otherwise?  Am I suspected of something?~
-  IF ~~ THEN REPLY ~You seem nervous.  Why?~ GOTO 9
-  IF ~~ THEN REPLY ~No, I have some more questions for you.~ GOTO 6
-  IF ~~ THEN REPLY ~I must be going.  Farewell for now.~ GOTO 18
+  SAY @38
+  IF ~~ THEN REPLY @39 GOTO 9
+  IF ~~ THEN REPLY @40 GOTO 6
+  IF ~~ THEN REPLY @37 GOTO 18
 END
 
 IF ~~ THEN BEGIN 9
-  SAY ~Of course I am nervous.  I know who you serve and therefore have good reason to be.  Can we get on with your questioning?~
-  IF ~~ THEN REPLY ~Yes, I do have more questions.~ GOTO 6
-  IF ~~ THEN REPLY ~I must be going.  Farewell for now.~ GOTO 18
+  SAY @41
+  IF ~~ THEN REPLY @42 GOTO 6
+  IF ~~ THEN REPLY @37 GOTO 18
 END
 
 IF ~~ THEN BEGIN 10
-  SAY ~Ah...  I see...  Well, Kreg Frostbeard is dead.  His son, Joril, now rules...  I did nothing to aid the son in killing his father, I assure you... ~
-  IF ~~ THEN REPLY ~Why should I care if you aided Joril in the killing?~ DO ~SetGlobal("Fool_Joril","GLOBAL", 1)~ JOURNAL ~Wyrm’s Tooth.
-
-We have discovered that the *master* chooses his lieutenants personally. Joril has caused a controversy by slaying his father for the position. Perhaps we can use this knowledge against him somehow.~ GOTO 26
-  IF ~~ THEN REPLY ~Very well.  Where can I find Joril?~ GOTO 11
-  IF ~~ THEN REPLY ~I must be going.  Farewell for now.~ GOTO 18
+  SAY @43
+  IF ~~ THEN REPLY @44 DO ~SetGlobal("Fool_Joril","GLOBAL", 1)~ JOURNAL @45 GOTO 26
+  IF ~~ THEN REPLY @46 GOTO 11
+  IF ~~ THEN REPLY @37 GOTO 18
 END
 
 IF ~~ THEN BEGIN 11
-  SAY ~In the ice caverns to the southwest, of course.  The fool sits upon his father’s throne with impudence.  Seek him there.~
-  IF ~~ THEN REPLY ~You deny aiding Joril in the killing of his father.  Why should I care if you did?~ DO ~SetGlobal("Fool_Joril","GLOBAL", 1)~ JOURNAL ~Wyrm’s Tooth.
-
-We have discovered that the *master* chooses his lieutenants personally. Joril has caused a controversy by slaying his father for the position. Perhaps we can use this knowledge against him somehow.~ GOTO 26
-  IF ~~ THEN REPLY ~I have some more questions for you.~ GOTO 6
-  IF ~~ THEN REPLY ~I must be going.  Farewell for now.~ GOTO 18
+  SAY @47
+  IF ~~ THEN REPLY @48 DO ~SetGlobal("Fool_Joril","GLOBAL", 1)~ JOURNAL @45 GOTO 26
+  IF ~~ THEN REPLY @49 GOTO 6
+  IF ~~ THEN REPLY @37 GOTO 18
 END
 
 IF ~~ THEN BEGIN 12
-  SAY ~The slaves... they revolted.  We had to slay most of them, but a few managed to escape to the level below.~
-  IF ~~ THEN REPLY ~How did they manage to escape you?~ JOURNAL ~Wyrm’s Tooth.
-
-It seems that the frost salamanders are sensitive to heat. We may be able to use that weakness against them.~ GOTO 13
-  IF ~~ THEN REPLY ~I must be going.  Farewell for now.~ GOTO 18
+  SAY @50
+  IF ~~ THEN REPLY @51 JOURNAL @52 GOTO 13
+  IF ~~ THEN REPLY @37 GOTO 18
 END
 
 IF ~~ THEN BEGIN 13
-  SAY ~The room below suddenly became too hot to bear.  We had to flee back here or perish.  In the confusion that followed the slaves bolted for the stairs.  A few made it... but their "freedom" will not last long.  They are trapped down there. ~
-  IF ~~ THEN REPLY ~Trapped?  How so?~ JOURNAL ~Wyrm’s Tooth.
-
-The slaves that escaped the frost salamanders fled to a room beneath the museum. Kerish believes them to be trapped there with no means of escape and no supplies. If this is the case then they will surely perish.~ GOTO 14
-  IF ~~ THEN REPLY ~I must be going.  Farewell for now.~ GOTO 18
+  SAY @53
+  IF ~~ THEN REPLY @54 JOURNAL @55 GOTO 14
+  IF ~~ THEN REPLY @37 GOTO 18
 END
 
 IF ~~ THEN BEGIN 14
-  SAY ~The exit to the surface is locked, so they have nowhere to run to.  Also, they have no supplies nor do they have the means to gather any since I have trolls patrolling the glacier...  Eventually they will starve or return to us.  All we have to do is wait...  Hmm.  It occurs to me that I could use your aid in this matter.~
-  IF ~~ THEN REPLY ~What is it that you think I can do for you?~ GOTO 15
-  IF ~~ THEN REPLY ~Those trolls you mentioned attacked me on the way here.~ GOTO 30
-  IF ~~ THEN REPLY ~I must be going.  Farewell for now.~ GOTO 18
+  SAY @56
+  IF ~~ THEN REPLY @57 GOTO 15
+  IF ~~ THEN REPLY @58 GOTO 30
+  IF ~~ THEN REPLY @37 GOTO 18
 END
 
 IF ~~ THEN BEGIN 15
-  SAY ~The slaves are of no consequence to me - they will return to us, or they will die.  However... the leader of this revolt has made me look bad in the eyes of my people.  For this he must pay.  Go to these slaves and gain their confidence.  Once you gain that confidence, find their leader and slay him.~
-  IF ~Alignment(Protagonist,3)~ THEN REPLY ~Very well, I will do it.  It sounds like fun.~ DO ~SetGlobal("Kill_Vera","GLOBAL", 3)~ UNSOLVED_JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We agreed to do this.~ GOTO 16
-  IF ~~ THEN REPLY ~No, I will not do your dirty work.~ DO ~SetGlobal("Kill_Vera","GLOBAL", 2)~ JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We informed him that we would not do his dirty work for him.~ GOTO 17
-  IF ~~ THEN REPLY ~I will consider it.~ DO ~SetGlobal("Kill_Vera","GLOBAL", 1)~ JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We informed him that we would consider the matter.~ GOTO 17
+  SAY @59
+  IF ~Alignment(Protagonist,3)~ THEN REPLY @60 DO ~SetGlobal("Kill_Vera","GLOBAL", 3)~ UNSOLVED_JOURNAL @8 GOTO 16
+  IF ~~ THEN REPLY @61 DO ~SetGlobal("Kill_Vera","GLOBAL", 2)~ JOURNAL @62 GOTO 17
+  IF ~~ THEN REPLY @63 DO ~SetGlobal("Kill_Vera","GLOBAL", 1)~ JOURNAL @64 GOTO 17
 END
 
 IF ~~ THEN BEGIN 16
-  SAY ~Then I will be in your debt.  Go now.  Perform this deed and then return to me.~
-  IF ~~ THEN REPLY ~Very well, but I have more questions for you.~ GOTO 6
-  IF ~~ THEN REPLY ~Then I will say farewell.~ GOTO 18
+  SAY @65
+  IF ~~ THEN REPLY @66 GOTO 6
+  IF ~~ THEN REPLY @67 GOTO 18
 END
 
 IF ~~ THEN BEGIN 17
-  SAY ~Very well, take some time to think about this.  Come to me if you change your mind.~
-  IF ~~ THEN REPLY ~Very well, but I have more questions for you.~ GOTO 6
-  IF ~~ THEN REPLY ~Then I will say farewell.~ GOTO 18
+  SAY @68
+  IF ~~ THEN REPLY @66 GOTO 6
+  IF ~~ THEN REPLY @67 GOTO 18
 END
 
 IF ~~ THEN BEGIN 18
-  SAY ~Very well, you are my guest.  Feel free to look around.  You will find nothing out of place here.~
-  IF ~~ THEN REPLY ~Then I will say farewell for now.~ DO ~SetGlobal("Kerish_Look","GLOBAL", 1)~ EXIT
+  SAY @69
+  IF ~~ THEN REPLY @70 DO ~SetGlobal("Kerish_Look","GLOBAL", 1)~ EXIT
 END
 
 IF ~~ THEN BEGIN 19
-  SAY ~I have the key you speak of, but I am reluctant to give it up.  There are circumstances that...~
-  IF ~~ THEN REPLY ~Yes, I am aware of the slaves.  I will take precautions to prevent their escape.~ DO ~GiveItem("KeyDug", Protagonist)~ JOURNAL ~Wyrm’s Tooth.
-
-We asked Kerish, the frost salamander leader, for the key to the surface exit. He agreed to give it to us.~ GOTO 20
-  IF ~~ THEN REPLY ~I am aware of the slave issue.  Trust me to handle the situation properly.~ DO ~GiveItem("KeyDug", Protagonist)~ JOURNAL ~Wyrm’s Tooth.
-
-We asked Kerish, the frost salamander leader, for the key to the surface exit. He agreed to give it to us.~ GOTO 20
+  SAY @71
+  IF ~~ THEN REPLY @72 DO ~GiveItem("KeyDug", Protagonist)~ JOURNAL @73 GOTO 20
+  IF ~~ THEN REPLY @74 DO ~GiveItem("KeyDug", Protagonist)~ JOURNAL @73 GOTO 20
 END
 
 IF ~~ THEN BEGIN 20
-  SAY ~Very well.  See that you do.  If those slaves escape, I will hold you personally responsible.  Here is the key.~
-  IF ~~ THEN REPLY ~Thank you.  I must be going now.  Farewell.~ DO ~AddexperienceParty(10500)
+  SAY @75
+  IF ~~ THEN REPLY @76 DO ~AddexperienceParty(10500)
 SetGlobal("Got_KeyDug", "GLOBAL", 1)~ GOTO 18
 END
 
 IF ~~ THEN BEGIN 21
-  SAY ~Then I will be in your debt.  Go now.  Perform this deed and then return to me.~
-  IF ~~ THEN REPLY ~Very well.  Farewell.~ EXIT
+  SAY @65
+  IF ~~ THEN REPLY @77 EXIT
 END
 
 IF ~~ THEN BEGIN 22
-  SAY ~Yes.  Have you performed this task?~
+  SAY @78
   IF ~OR(2)
 Global("Vera_Dead", "GLOBAL",1)
-Global("SPRITE_IS_DEADVERA","GLOBAL",1)~ THEN REPLY ~Yes. It was a woman named Vera. She is dead.~ DO ~SetGlobal("Kill_Vera","GLOBAL", 4)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We agreed to do this.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the leader of the frost salamanders, believes that we work for someone named Marketh. We decided to do nothing to dissuade him of that belief. It could prove useful.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Apparently there has been a slave revolt among the frost salamanders. A few managed to escape but most died in the attempt.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-The slaves that escaped the frost salamanders fled to a room beneath the museum. Kerish believes them to be trapped there with no means of escape and no supplies. If this is the case then they will surely perish.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We informed him that we would not do his dirty work for him.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We informed him that we would consider the matter.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We asked Kerish, the frost salamander leader, for the key to the surface exit. He agreed to give it to us.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would do nothing of the sort.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would consider the matter.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We agreed to do this.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We informed Gareth that the frost salamanders asked us to return him and the other slaves. He said he would rather die than return to slavery and attacked us.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We met a man named Gareth who claims to be an escaped slave of a group of frost salamanders.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-According to Gareth, frost salamanders are sensitive to heat. Perhaps we could use this knowledge against them.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We have learned that the slave leader is a woman named Vera.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Gareth has asked us to retrieve a key. The key unlocks the door to the passage leading to the surface. He believes it is in the possession of the salamanders. We have agreed to do this.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Gareth has asked us to retrieve a key, which will unlock the door to the passage leading to the surface. He believes the key is the possession of the salamanders. We told him we were not willing to fight a building full of frost salamanders for this key.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Gareth has asked us to retrieve a key, which will unlock the door to the passage leading to the surface. He believes the key is the possession of the salamanders. We informed him that we would think about this.%)
-AddexperienceParty(10500)~ SOLVED_JOURNAL ~Wyrm’s Tooth.
-
-We told Kerish that Vera, the slave leader, is dead. He believes his honor has been restored and offered us his gratitude.~ GOTO 23
-  IF ~Global("Heard_Vera_Dead","GLOBAL", 1)~ THEN REPLY ~I have spoken with the slaves as you asked.  They say that their leader died in the escape attempt.~ DO ~SetGlobal("Kill_Vera","GLOBAL",4)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We agreed to do this.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the leader of the frost salamanders, believes that we work for someone named Marketh. We decided to do nothing to dissuade him of that belief. It could prove useful.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Apparently there has been a slave revolt among the frost salamanders. A few managed to escape but most died in the attempt.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-The slaves that escaped the frost salamanders fled to a room beneath the museum. Kerish believes them to be trapped there with no means of escape and no supplies. If this is the case then they will surely perish.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We informed him that we would not do his dirty work for him.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We informed him that we would consider the matter.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We asked Kerish, the frost salamander leader, for the key to the surface exit. He agreed to give it to us.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would do nothing of the sort.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would consider the matter.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We agreed to do this.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We informed Gareth that the frost salamanders asked us to return him and the other slaves. He said he would rather die than return to slavery and attacked us.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We met a man named Gareth who claims to be an escaped slave of a group of frost salamanders.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-According to Gareth, frost salamanders are sensitive to heat. Perhaps we could use this knowledge against them.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We have learned that the slave leader is a woman named Vera.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Gareth has asked us to retrieve a key. The key unlocks the door to the passage leading to the surface. He believes it is in the possession of the salamanders. We have agreed to do this.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Gareth has asked us to retrieve a key, which will unlock the door to the passage leading to the surface. He believes the key is the possession of the salamanders. We told him we were not willing to fight a building full of frost salamanders for this key.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Gareth has asked us to retrieve a key, which will unlock the door to the passage leading to the surface. He believes the key is the possession of the salamanders. We informed him that we would think about this.%)
-AddexperienceParty(10500)~ SOLVED_JOURNAL ~Wyrm’s Tooth.
-
-We told Kerish that the slaves believe their leader to be dead. He seemed satisfied with that.~ GOTO 24
-  IF ~Global("Vera_Dead", "GLOBAL", 0)~ THEN REPLY ~Not yet.  I just wanted to let you know that I am still working on it.  Farewell for now.~ EXIT
+Global("SPRITE_IS_DEADVERA","GLOBAL",1)~ THEN REPLY @79 DO ~SetGlobal("Kill_Vera","GLOBAL", 4)
+EraseJournalEntry(@8)
+EraseJournalEntry(@24)
+EraseJournalEntry(@33)
+EraseJournalEntry(@55)
+EraseJournalEntry(@62)
+EraseJournalEntry(@64)
+EraseJournalEntry(@73)
+EraseJournalEntry(@80)
+EraseJournalEntry(@81)
+EraseJournalEntry(@12)
+EraseJournalEntry(@82)
+EraseJournalEntry(@83)
+EraseJournalEntry(@84)
+EraseJournalEntry(@85)
+EraseJournalEntry(@86)
+EraseJournalEntry(@87)
+EraseJournalEntry(@88)
+AddexperienceParty(10500)~ SOLVED_JOURNAL @89 GOTO 23
+  IF ~Global("Heard_Vera_Dead","GLOBAL", 1)~ THEN REPLY @90 DO ~SetGlobal("Kill_Vera","GLOBAL",4)
+EraseJournalEntry(@8)
+EraseJournalEntry(@24)
+EraseJournalEntry(@33)
+EraseJournalEntry(@55)
+EraseJournalEntry(@62)
+EraseJournalEntry(@64)
+EraseJournalEntry(@73)
+EraseJournalEntry(@80)
+EraseJournalEntry(@81)
+EraseJournalEntry(@12)
+EraseJournalEntry(@82)
+EraseJournalEntry(@83)
+EraseJournalEntry(@84)
+EraseJournalEntry(@85)
+EraseJournalEntry(@86)
+EraseJournalEntry(@87)
+EraseJournalEntry(@88)
+AddexperienceParty(10500)~ SOLVED_JOURNAL @91 GOTO 24
+  IF ~Global("Vera_Dead", "GLOBAL", 0)~ THEN REPLY @92 EXIT
 END
 
 IF ~~ THEN BEGIN 23
-  SAY ~You have done well.  By acting under my command you have restored my honor in the eyes of my people.  You have my gratitude.~
-  IF ~~ THEN REPLY ~It was a pleasure.  Is there another matter I can help you with?~ GOTO 28
+  SAY @93
+  IF ~~ THEN REPLY @94 GOTO 28
 END
 
 IF ~~ THEN BEGIN 24
-  SAY ~Hmm.  Very well...  The slaves have enjoyed their freedom long enough.  Bring them here before me or slay them.  I care not which.~
-  IF ~~ THEN REPLY ~I will go to the slaves and tell them you want them to return.  Farewell.~ DO ~SetGlobal("Return_Slaves","GLOBAL", 3)~ UNSOLVED_JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We agreed to do this.~ EXIT
-  IF ~~ THEN REPLY ~I will do nothing of the sort.  Farewell.~ DO ~SetGlobal("Return_Slaves","GLOBAL", 2)~ JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would do nothing of the sort.~ EXIT
-  IF ~~ THEN REPLY ~I will think about it.  Farewell.~ DO ~SetGlobal("Return_Slaves","GLOBAL", 1)~ JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would consider the matter.~ EXIT
+  SAY @95
+  IF ~~ THEN REPLY @96 DO ~SetGlobal("Return_Slaves","GLOBAL", 3)~ UNSOLVED_JOURNAL @12 EXIT
+  IF ~~ THEN REPLY @97 DO ~SetGlobal("Return_Slaves","GLOBAL", 2)~ JOURNAL @80 EXIT
+  IF ~~ THEN REPLY @98 DO ~SetGlobal("Return_Slaves","GLOBAL", 1)~ JOURNAL @81 EXIT
 END
 
 IF ~~ THEN BEGIN 25
-  SAY ~Marketh would be my guess.  Would you care to confirm that belief?~
-  IF ~~ THEN REPLY ~No, not really.  Believe what you will...~ GOTO 5
+  SAY @99
+  IF ~~ THEN REPLY @100 GOTO 5
 END
 
 IF ~~ THEN BEGIN 26
-  SAY ~Hmph.  It is well known that the *master* chooses his lieutenants himself.  Joril’s actions could be deemed... treason.  I want it understood by you and your master that I had no part in the matter...~
-  IF ~~ THEN REPLY ~Point taken.  Where can I find Joril?~ GOTO 11
-  IF ~~ THEN REPLY ~Very well, let me ask you something else.~ GOTO 6
+  SAY @101
+  IF ~~ THEN REPLY @102 GOTO 11
+  IF ~~ THEN REPLY @103 GOTO 6
 END
 
 IF ~~ THEN BEGIN 27
-  SAY ~Yes... I see no slaves with you.  Have you slain them instead?~
-  IF ~Global("Gareth_Dead", "GLOBAL", 1)~ THEN REPLY ~Yes.  They preferred death to returning here.~ DO ~SetGlobal("Return_Slaves","GLOBAL", 4)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We agreed to do this.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the leader of the frost salamanders, believes that we work for someone named Marketh. We decided to do nothing to dissuade him of that belief. It could prove useful.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Apparently there has been a slave revolt among the frost salamanders. A few managed to escape but most died in the attempt.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-The slaves that escaped the frost salamanders fled to a room beneath the museum. Kerish believes them to be trapped there with no means of escape and no supplies. If this is the case then they will surely perish.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We informed him that we would not do his dirty work for him.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to find the leader of the slave revolt and slay him. We informed him that we would consider the matter.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We asked Kerish, the frost salamander leader, for the key to the surface exit. He agreed to give it to us.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would do nothing of the sort.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would consider the matter.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We agreed to do this.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We informed Gareth that the frost salamanders asked us to return him and the other slaves. He said he would rather die than return to slavery and attacked us.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We met a man named Gareth who claims to be an escaped slave of a group of frost salamanders.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-According to Gareth, frost salamanders are sensitive to heat. Perhaps we could use this knowledge against them.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-We have learned that the slave leader is a woman named Vera.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Gareth has asked us to retrieve a key. The key unlocks the door to the passage leading to the surface. He believes it is in the possession of the salamanders. We have agreed to do this.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Gareth has asked us to retrieve a key, which will unlock the door to the passage leading to the surface. He believes the key is the possession of the salamanders. We told him we were not willing to fight a building full of frost salamanders for this key.%)
-EraseJournalEntry(%Wyrm’s Tooth.
-
-Gareth has asked us to retrieve a key, which will unlock the door to the passage leading to the surface. He believes the key is the possession of the salamanders. We informed him that we would think about this.%)
-AddexperienceParty(10500)~ SOLVED_JOURNAL ~Wyrm’s Tooth.
-
-We told Kerish that the slaves are dead. He offered us his gratitude.~ GOTO 29
-  IF ~Global("Gareth_Dead", "GLOBAL", 0)~ THEN REPLY ~Not yet.  I just wanted to let you know that I am still working on it.  Farewell for now.~ EXIT
+  SAY @104
+  IF ~Global("Gareth_Dead", "GLOBAL", 1)~ THEN REPLY @105 DO ~SetGlobal("Return_Slaves","GLOBAL", 4)
+EraseJournalEntry(@8)
+EraseJournalEntry(@24)
+EraseJournalEntry(@33)
+EraseJournalEntry(@55)
+EraseJournalEntry(@62)
+EraseJournalEntry(@64)
+EraseJournalEntry(@73)
+EraseJournalEntry(@80)
+EraseJournalEntry(@81)
+EraseJournalEntry(@12)
+EraseJournalEntry(@82)
+EraseJournalEntry(@83)
+EraseJournalEntry(@84)
+EraseJournalEntry(@85)
+EraseJournalEntry(@86)
+EraseJournalEntry(@87)
+EraseJournalEntry(@88)
+AddexperienceParty(10500)~ SOLVED_JOURNAL @106 GOTO 29
+  IF ~Global("Gareth_Dead", "GLOBAL", 0)~ THEN REPLY @92 EXIT
 END
 
 IF ~~ THEN BEGIN 28
-  SAY ~Hmm.  Yes...  The slaves have enjoyed their freedom long enough.  Bring them here before me or slay them.  I care not which.~
-  IF ~~ THEN REPLY ~I will go to the slaves and tell them you want them to return.  Farewell.~ DO ~SetGlobal("Return_Slaves","GLOBAL", 3)~ UNSOLVED_JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We agreed to do this.~ EXIT
-  IF ~~ THEN REPLY ~I will do nothing of the sort.  Farewell.~ DO ~SetGlobal("Return_Slaves","GLOBAL", 2)~ JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would do nothing of the sort.~ EXIT
-  IF ~~ THEN REPLY ~I will think about it.  Farewell.~ DO ~SetGlobal("Return_Slaves","GLOBAL", 1)~ JOURNAL ~Wyrm’s Tooth.
-
-Kerish, the frost salamander leader, has asked us to return the escaped slaves to him or slay them. We informed him that we would consider the matter.~ EXIT
+  SAY @107
+  IF ~~ THEN REPLY @96 DO ~SetGlobal("Return_Slaves","GLOBAL", 3)~ UNSOLVED_JOURNAL @12 EXIT
+  IF ~~ THEN REPLY @97 DO ~SetGlobal("Return_Slaves","GLOBAL", 2)~ JOURNAL @80 EXIT
+  IF ~~ THEN REPLY @98 DO ~SetGlobal("Return_Slaves","GLOBAL", 1)~ JOURNAL @81 EXIT
 END
 
 IF ~~ THEN BEGIN 29
-  SAY ~You have done well.  You have my gratitude.~
-  IF ~~ THEN REPLY ~It was a pleasure.  Farewell.~ EXIT
+  SAY @108
+  IF ~~ THEN REPLY @109 EXIT
 END
 
 IF ~~ THEN BEGIN 30
-  SAY ~My apologies.  They are not the most intelligent or observant of creatures.  It is possible that they mistook you for an escaped slave.~
-  IF ~~ THEN REPLY ~I see... You mentioned needing my help. What is it that you think I can do for you?~ GOTO 15
-  IF ~~ THEN REPLY ~I will be going now. Farewell.~ GOTO 18
+  SAY @110
+  IF ~~ THEN REPLY @111 GOTO 15
+  IF ~~ THEN REPLY @112 GOTO 18
 END
