--- NeJ3\dlg\FECOOK.d	Sat Feb 17 20:18:26 2007
+++ C:\BWP Patchstudio\patched files\NeJ3\dlg\FECOOK.d	Wed Apr 10 10:24:56 2019
@@ -2,17 +2,17 @@
 
 IF WEIGHT #1 ~True()
 ~ THEN BEGIN 0
-  SAY ~Ya, I don't like no snot nosed brats causin' trouble here.  You better scram, 'fore I give you a whippin'.~
+  SAY @0
   IF ~~ THEN EXIT
 END
 
 IF WEIGHT #0 ~StateCheck(Myself,STATE_CHARMED)
 ~ THEN BEGIN 1
-  SAY ~Only stuff I know about, is cooking, baking and how ta run a kitchen.~
+  SAY @1
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~Get out or be thrown out!~
+  SAY @2
   IF ~~ THEN EXIT
 END
