--- NeJ3\baf\LDFLEEZ1.BAF	Mon Nov 17 01:54:03 2014
+++ C:\BWP Patchstudio\patched files\NeJ3\baf\LDFLEEZ1.BAF	Wed Apr 10 10:33:52 2019
@@ -23,7 +23,7 @@
 	Global("SPRITE_IS_DEADFlozem","GLOBAL",1)
 THEN
 	RESPONSE #100
-		DisplayStringHead(Myself,~You killed my brother! You'll pay for that!~)
+		DisplayStringHead(Myself,@0)
 		Enemy()
 		Continue()
 END
@@ -61,7 +61,7 @@
 THEN
 	RESPONSE #100
 		SetGlobal("MAP_GROUP_HOSTILE10","MYAREA",1)
-		DisplayStringHead(Myself,~Flozem! Get over here!~)
+		DisplayStringHead(Myself,@1)
 		Enemy()
 		Continue()
 END
