--- NeJ3\baf\UDDSRMC.baf	Sat Dec 09 22:30:50 2006
+++ C:\BWP Patchstudio\patched files\NeJ3\baf\UDDSRMC.baf	Wed Apr 10 10:34:09 2019
@@ -3,7 +3,7 @@
 	Entered([ANYTHING])
 THEN
 	RESPONSE #100
-		DisplayString(Myself,~You hear a loud click.~)
+		DisplayString(Myself,@0)
 		ReallyForceSpell(LastTrigger,TRAP_LIGHTNING_BOLT)
 		SetGlobal("AR6010_TRAP_STATE","GLOBAL",0)
 END
@@ -14,5 +14,5 @@
 THEN
 	RESPONSE #100
 		SetGlobal("AR6010_TRAP_STATE","GLOBAL",3)
-		DisplayString(Myself,~You hear a loud click, followed by eleven similar sounding clicks echoing throughout the room.  Finally, you hear a loud clattering coming from the round table room.~)
+		DisplayString(Myself,@1)
 END
