--- NeJ3\baf\UDDSRMB.baf	Sat Dec 09 22:29:10 2006
+++ C:\BWP Patchstudio\patched files\NeJ3\baf\UDDSRMB.baf	Wed Apr 10 10:34:08 2019
@@ -3,7 +3,7 @@
 	Entered([ANYTHING])
 THEN
 	RESPONSE #100
-		DisplayString(Myself,~You hear a loud click.~)
+		DisplayString(Myself,@0)
 		ReallyForceSpell(LastTrigger,TRAP_LIGHTNING_BOLT)
 		SetGlobal("AR6010_TRAP_STATE","GLOBAL",0)
 END
@@ -15,5 +15,5 @@
 	RESPONSE #100
 		SetGlobal("AR6010_TRAP_STATE","GLOBAL",2)
 		SetGlobal("PUZZLE_B_DISABLED","GLOBAL",1)
-		DisplayString(Myself,~You hear a loud click, followed by eleven similar sounding clicks echoing throughout the room.~)
+		DisplayString(Myself,@1)
 END
