--- ascension\lang\german\mel_immortal.tra	Sat May 21 11:06:40 2022
+++ C:\BWP Patchstudio\patched files\ascension\lang\german\mel_immortal.tra	Tue Feb 07 08:06:52 2023
@@ -1,3 +1,3 @@
 @274 = ~Verflucht! Das Weib kehrt zu Bhaals Essenz zurück, um sich zu heilen! Es muss einen Weg geben, sie noch mehr zu schwächen, oder wir werden sie nie besiegen!~
-@398 = ~Melissan stellt ihre göttlichen Energien an der Quelle wieder her und bewahrt so ihre Unsterblichkeit.  Ihr müsst einen Weg finden, sie noch mehr zu schwächen, oder Ihr werdet sie nie besiegen können.~
+@398 = ~Melissan stellt ihre göttlichen Energien an der Quelle wieder her und bewahrt so ihre Unsterblichkeit. Ihr müsst einen Weg finden, sie noch mehr zu schwächen, oder Ihr werdet sie nie besiegen können.~
 @1018 = ~Melissan stellt immer noch ihre unsterblichen Energien an der Quelle wieder her! Es muss einen Weg geben, sie noch mehr zu schwächen, oder wir werden sie nie besiegen!~
