--- ascension\lang\german\mel_banter.tra	Sat May 21 11:06:40 2022
+++ C:\BWP Patchstudio\patched files\ascension\lang\german\mel_banter.tra	Tue Feb 07 08:08:20 2023
@@ -1 +1 @@
-@630 = ~Was--?  Sterblich!  Nein, das kann nicht sein!  Ich bin fast wieder sterblich!!  Ich habe zuviel Macht verloren!!  Dafür werdet Ihr bezahlen!!~
+@630 = ~Was? Sterblich! Nein, das kann nicht sein! Ich bin fast wieder sterblich!! Ich habe zuviel Macht verloren!! Dafür werdet Ihr bezahlen!!~ 
