--- mapnames\setup-mapnames.tp2	Wed Dec 15 04:28:24 2010
+++ C:\BWP Patchstudio\patched files\mapnames\setup-mapnames.tp2	Sat Sep 08 18:28:41 2012
@@ -274,6 +274,8 @@
 
 AUTO_TRA ~mapnames/%s~
 LANGUAGE ~English~ ~english~ ~mapnames/english/setup.tra~
+LANGUAGE ~Russian~ ~russian~ ~mapnames/russian/setup.tra~
+LANGUAGE ~German~ ~german~ ~mapnames/german/setup.tra~
 
 //! Sword Coast Map Labels ////////////////////////////////////////////
 BEGIN @1 //Sword Coast Map Labels
