--- NeJ2\baf\VPCutIP4.baf	Mon Jan 19 16:47:38 2015
+++ C:\BWP Patchstudio\patched files\NeJ2\baf\VPCutIP4.baf	Wed Apr 10 10:32:17 2019
@@ -5,7 +5,7 @@
 		CutSceneId(Player4)
 		SmallWait(7)
 		DestroyItem("shadbfak")
-		DisplayStringHead(Player4,~The scroll has been warded and disintegrated with a considerable force irreversibly decreasing your dexterity.~)
+		DisplayStringHead(Player4,@0)
 		PlaySound("EFF_P19")
 		Wait(2)
 		FadeToColor([30.0],0)
