--- NeJ2\baf\VPCutIP1.baf	Mon Jan 19 16:47:36 2015
+++ C:\BWP Patchstudio\patched files\NeJ2\baf\VPCutIP1.baf	Wed Apr 10 10:32:15 2019
@@ -5,7 +5,7 @@
 		CutSceneId(Player1)
 		SmallWait(7)
 		DestroyItem("shadbfak")
-		DisplayStringHead(Player1,~The scroll has been warded and disintegrated with a considerable force irreversibly decreasing your dexterity.~)
+		DisplayStringHead(Player1,@0)
 		PlaySound("EFF_P19")
 		Wait(1)
 		FadeToColor([30.0],0)
