--- NeJ2\baf\VPCutIP6.baf	Mon Jan 19 16:47:42 2015
+++ C:\BWP Patchstudio\patched files\NeJ2\baf\VPCutIP6.baf	Wed Apr 10 10:32:18 2019
@@ -5,7 +5,7 @@
 		CutSceneId(Player6)
 		SmallWait(7)
 		DestroyItem("shadbfak")
-		DisplayStringHead(Player6,~The scroll has been warded and disintegrated with a considerable force irreversibly decreasing your dexterity.~)
+		DisplayStringHead(Player6,@0)
 		PlaySound("EFF_P19")
 		Wait(2)
 		FadeToColor([30.0],0)
