--- NeJ2\baf\VPCutIP2.baf	Mon Jan 19 16:47:36 2015
+++ C:\BWP Patchstudio\patched files\NeJ2\baf\VPCutIP2.baf	Wed Apr 10 10:32:16 2019
@@ -5,7 +5,7 @@
 		CutSceneId(Player2)
 		SmallWait(7)
 		DestroyItem("shadbfak")
-		DisplayStringHead(Player2,~The scroll has been warded and disintegrated with a considerable force irreversibly decreasing your dexterity.~)
+		DisplayStringHead(Player2,@0)
 		PlaySound("EFF_P19")
 		Wait(2)
 		FadeToColor([30.0],0)
