--- NeJE\dlg\BAERIE.d	Sat Dec 01 17:38:43 2018
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\BAERIE.d	Tue Apr 09 19:34:49 2019
@@ -33,177 +33,177 @@
 APPEND ~BAERIE~
 
 IF ~~ THEN BEGIN TS449
-  SAY ~Yes... yes. Sorry. I did not mean to disturb anyone. I was just thinking of my people...~
+  SAY @0
   IF ~~ THEN EXTERN ~BKACHI~ 46
 END
 
 IF ~~ THEN BEGIN TS450
-  SAY ~My native lands... my native element is the sky. I miss it too. I can see it from here, from the hard ground and I yearn to be embraced by soft warm air. Kachiko, if you only knew how wonderful it is to be able to fly...~
+  SAY @1
   IF ~~ THEN EXTERN ~BKACHI~ 47
 END
 
 IF ~~ THEN BEGIN TS451
-  SAY ~I never thought that anyone could be envious of my fate...~
+  SAY @2
   IF ~~ THEN EXTERN ~BKACHI~ 48
 END
 
 IF ~~ THEN BEGIN TS452
-  SAY ~Me... strong? Kachiko, nobody ever called me strong before... In fact, they usually call me a freak... I shall think about what you said.~
+  SAY @3
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN TS453
-  SAY ~(*sighs*)~
+  SAY @4
   IF ~~ THEN EXTERN ~BVALYGA~ TS508
 END
 
 IF ~~ THEN BEGIN TS454
-  SAY ~I was just thinking of my parents and how I used to fly with them as a little girl. Oh Valygar, you must know what its like to be ripped away from a life you loved.~
+  SAY @5
   IF ~~ THEN EXTERN ~BVALYGA~ TS509
 END
 
 IF ~~ THEN BEGIN TS455
-  SAY ~I'm sorry Valygar I didn't mean to awaken painful memories for you.~
+  SAY @6
   IF ~~ THEN EXTERN ~BVALYGA~ TS510
 END
 
 IF ~~ THEN BEGIN TS456
-  SAY ~I... I do not know... It is all so sad... I feel so... lonely.~
+  SAY @7
   IF ~~ THEN EXTERN ~BVALYGA~ TS512
 END
 
 IF ~~ THEN BEGIN TS457
-  SAY ~I miss Uncle Quayle.~
+  SAY @8
   IF ~~ THEN EXTERN ~BVALYGA~ TS513
 END
 
 IF ~~ THEN BEGIN TS458
-  SAY ~I... do not want to go back. I do not... I do not miss the circus. That was a cruel place where... everyone was ready to do... just about anything for a round of applause. I miss... my Uncle. I wish... I wish I could meet someone as kind and... caring as him... during our travels. Someone who'd care for me... like you care for <CHARNAME>.~
+  SAY @9
   IF ~~ THEN EXTERN ~BVALYGA~ TS514
 END
 
 IF ~~ THEN BEGIN TS459
-  SAY ~(*blush*) I did not... mean you... I must have put it the wrong way, as usual. I... know that you and <CHARNAME>... Well, I do not... know... I just thought that... Oh, that's so embarrassing! I am so clumsy...~
+  SAY @10
   IF ~~ THEN EXTERN ~BVALYGA~ TS515
 END
 
 IF ~~ THEN BEGIN TS460
-  SAY ~My face is not... not sour!~
+  SAY @11
   IF ~~ THEN EXTERN ~BSIME~ 5
 END
 
 IF ~~ THEN BEGIN TS461
-  SAY ~You mean... you mean I look... ugly?~
+  SAY @12
   IF ~~ THEN EXTERN ~BSIME~ 6
 END
 
 IF ~~ THEN BEGIN TS462
-  SAY ~What's wrong with my... my face?~
+  SAY @13
   IF ~~ THEN EXTERN ~BSIME~ 7
 END
 
 IF ~~ THEN BEGIN TS463
-  SAY ~Oh...*sniffs*~
+  SAY @14
   IF ~~ THEN EXTERN ~BSIME~ 8
 END
 
 IF ~~ THEN BEGIN TS464
-  SAY ~I... I... was trying not to fall down...~
+  SAY @15
   IF ~~ THEN EXTERN ~BSIME~ 10
 END
 
 IF ~~ THEN BEGIN TS465
-  SAY ~For me... for me it is, Sime. Avariel are not accustomed to prolonged walks like this. Avariel fly...~
+  SAY @16
   IF ~~ THEN EXTERN ~BSIME~ 11
 END
 
 IF ~~ THEN BEGIN TS466
-  SAY ~In a cage... where I could not lay down, let alone walk. I... I have never walked this much in my whole life.~
+  SAY @17
   IF ~~ THEN EXTERN ~BSIME~ 12
 END
 
 IF ~~ THEN BEGIN TS467
-  SAY ~I... I know... Do not... do not tell anyone, please?~
+  SAY @18
   IF ~~ THEN EXTERN ~BSIME~ 13
 END
 
 IF ~~ THEN BEGIN TS468
-  SAY ~They'd laugh... They always laugh...~
+  SAY @19
   IF ~~ THEN EXTERN ~BSIME~ 14
 END
 
 IF ~~ THEN BEGIN TS469
-  SAY ~Thank you, Sime. Now if only someone could clean this place up... It is so... dirty and dusty!~
+  SAY @20
   IF ~~ THEN EXTERN ~BSIME~ 15
 END
 
 IF ~~ THEN BEGIN TS470
-  SAY ~You... do not need to check me. I... do what I want.~
+  SAY @21
   IF ~~ THEN EXTERN ~BSIME~ 563
 END
 
 IF ~~ THEN BEGIN TS471
-  SAY ~It is empty.~
+  SAY @22
   IF ~~ THEN EXTERN ~BSIME~ 564
 END
 
 IF ~~ THEN BEGIN TS472
-  SAY ~Here, catch it if you can... (*She throws the vial to Sime and tries to run away, crying).~
+  SAY @23
   IF ~~ THEN EXTERN ~BSIME~ 565
 END
 
 IF ~~ THEN BEGIN TS473
-  SAY ~I... do not need talking to, Sime. I have enough of that from <CHARNAME>.~
+  SAY @24
   IF ~~ THEN EXTERN ~BSIME~ 572
 END
 
 IF ~~ THEN BEGIN TS474
-  SAY ~Help me? You... want to help me? How can you help me? Can you make <CHARNAME> to love me?~
+  SAY @25
   IF ~~ THEN EXTERN ~BSIME~ 573
 END
 
 IF ~~ THEN BEGIN TS475
-  SAY ~You do not... understand.~
+  SAY @26
   IF ~~ THEN EXTERN ~BSIME~ 574
 END
 
 IF ~~ THEN BEGIN TS476
-  SAY ~You... are wrong. I will never... love anyone, but <CHARNAME>!~
+  SAY @27
   IF ~~ THEN EXTERN ~BSIME~ 575
 END
 
 IF ~~ THEN BEGIN NEJ449
-  SAY ~(*Giggles*) What a funny thing to think Leina. No, it is no ritual. I like to stand and feel the wind beneath my arms and imagine it is lifting me into the air as though they were my wings. You see Leina, the only way I will ever fly again is in my dreams.~
+  SAY @28
   IF ~~ THEN EXTERN ~BLEINA~ 197
 END
 
 IF ~~ THEN BEGIN NEJ450
-  SAY ~To soar to the clouds, to feel the wind beneath you as you leave the ground is an experience that cannot be described, it has to be felt. I try to capture the memories, the feelings again and again but each time it becomes a little harder. I fear I am slowly losing that which I am.~
+  SAY @29
   IF ~~ THEN EXTERN ~BLEINA~ 198
 END
 
 IF ~~ THEN BEGIN NEJ451
-  SAY ~But I don’t want to lose any of it! I am Avariel with or without my wings and to be Avariel is to know and to feel the freedom that which comes from soaring with the eagles and walking on the clouds. If I lose that what do I become and where do I belong?~
+  SAY @30
   IF ~~ THEN EXTERN ~BLEINA~ 199
 END
 
 IF ~~ THEN BEGIN NEJ452
-  SAY ~If I could walk on grass it would not be so bad but we tramp more often than not the hard rough road which hurt my feet, and as for the rain, it is always so cold and wet. I hate the way the coldness of it seeps through to my skin!~
+  SAY @31
   IF ~~ THEN GOTO NEJ453
 END
 
 IF ~~ THEN BEGIN NEJ453
-  SAY ~(*Tears well in her eyes as she speaks*) I am Avariel, I was born to fly, to be one with the wind, not to trudge through mud!~
+  SAY @32
   IF ~~ THEN EXTERN ~BLEINA~ 201
 END
 
 IF ~~ THEN BEGIN NEJ454
-  SAY ~I... I will just watch Leina. My legs really hurt from all the walking we have done today.~
+  SAY @33
   IF ~~ THEN EXTERN ~BLEINA~ PCF2Leina26a
 END
 
 IF ~~ THEN BEGIN NEJ455
-  SAY ~I’ll get used to walking first before I try dancing thank you Leina.~
+  SAY @34
 	IF ~Gender(Player1,FEMALE)~ THEN EXTERN ~BLEINA~ PCF2Leina2a
   IF ~Gender(Player1,MALE)~ THEN EXTERN ~BLEINA~ PCM4Leina2b
 	IF ~InParty("Yoshimo")~ THEN EXTERN ~BYOSHIM~ NEJ9
@@ -223,7 +223,7 @@
 END
 
 IF ~~ THEN BEGIN LeiTheat
-  SAY ~That look, are you... are you sure it’s a good idea?~
+  SAY @35
  IF ~!IsValidForPartyDialogue("Anomen")
 !IsValidForPartyDialogue("Cernd")
 !IsValidForPartyDialogue("HaerDalis")
@@ -239,7 +239,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Oh enough with your horse play! Leina always wants a hot bath let’s get the jump on her first this time. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @36 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Anomen")
 !IsValidForPartyDialogue("Cernd")
 !IsValidForPartyDialogue("HaerDalis")
@@ -255,7 +255,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Sssssh. Now listen Leina always wants a bath after a battle and she hasn’t said anything yet so let’s do it first.~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @37 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Anomen")
 !IsValidForPartyDialogue("Cernd")
 !IsValidForPartyDialogue("HaerDalis")
@@ -271,7 +271,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Hey now, listen Leina is always the first to want a bath after a stinkin’ battle so let’s ask her first for a fluffy towel. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @38 GOTO LeiThea1
 	IF ~InParty("Taffic")
 !Dead("Taffic")~ THEN EXTERN ~BTAFFIC~ LeiTheat
 	IF ~InParty("Melora")
@@ -307,7 +307,7 @@
 END
 
 IF ~~ THEN BEGIN LeiThea3
-  SAY ~Leina... You are talking about Leina, aren’t you?~
+  SAY @39
  IF ~!IsValidForPartyDialogue("Anomen")
 !IsValidForPartyDialogue("Cernd")
 !IsValidForPartyDialogue("HaerDalis")
@@ -323,7 +323,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Oh enough with your horse play! Leina always wants a hot bath let’s get the jump on her first this time. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @36 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Anomen")
 !IsValidForPartyDialogue("Cernd")
 !IsValidForPartyDialogue("HaerDalis")
@@ -339,7 +339,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Sssssh. Now listen Leina always wants a bath after a battle and she hasn’t said anything yet so let’s do it first.~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @37 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Anomen")
 !IsValidForPartyDialogue("Cernd")
 !IsValidForPartyDialogue("HaerDalis")
@@ -355,7 +355,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Hey now, listen Leina is always the first to want a bath after a stinkin’ battle so let’s ask her first for a fluffy towel. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @38 GOTO LeiThea1
 	IF ~InParty("Taffic")
 !Dead("Taffic")~ THEN EXTERN ~BTAFFIC~ LeiThea3
 	IF ~InParty("Melora")
@@ -391,7 +391,7 @@
 END
 
 IF ~~ THEN BEGIN LeiThea4
-  SAY ~I don’t know... I like Leina and some of your jokes are really bad.~
+  SAY @40
  IF ~!IsValidForPartyDialogue("Anomen")
 !IsValidForPartyDialogue("Cernd")
 !IsValidForPartyDialogue("HaerDalis")
@@ -407,7 +407,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Oh enough with your horse play! Leina always wants a hot bath let’s get the jump on her first this time. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @36 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Anomen")
 !IsValidForPartyDialogue("Cernd")
 !IsValidForPartyDialogue("HaerDalis")
@@ -423,7 +423,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Sssssh. Now listen Leina always wants a bath after a battle and she hasn’t said anything yet so let’s do it first.~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @37 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Anomen")
 !IsValidForPartyDialogue("Cernd")
 !IsValidForPartyDialogue("HaerDalis")
@@ -439,7 +439,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Hey now, listen Leina is always the first to want a bath after a stinkin’ battle so let’s ask her first for a fluffy towel. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @38 GOTO LeiThea1
 	IF ~InParty("Taffic")
 !Dead("Taffic")~ THEN EXTERN ~BTAFFIC~ LeiThea4
 	IF ~InParty("Melora")
@@ -475,7 +475,7 @@
 END
 
 IF ~~ THEN BEGIN LeiThea1
-  SAY ~Oh, we shall be first this time and on her own territory. What will she say!~
+  SAY @41
 	IF ~True()~ THEN EXTERN ~BLEINA~ TheaQ151
 	IF ~InParty("Taffic")
 !Dead("Taffic")~ THEN EXTERN ~BTAFFIC~ LeiThea1
@@ -512,7 +512,7 @@
 END
 
 IF ~~ THEN BEGIN LeiThea2
-  SAY ~Oh, you have made a song about me! I never dreamt of such a thing, it is so much more exciting than the circus.~
+  SAY @42
 	IF ~True()~ THEN EXTERN ~BLEINA~ TheaQ161
 	IF ~InParty("Viconia")
 !Dead("Viconia")~ THEN EXTERN ~BVICONI~ LeiThea2
