--- NeJE\dlg\BALTH.d	Fri Apr 20 01:27:35 2018
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\BALTH.d	Tue Apr 09 19:34:51 2019
@@ -14,38 +14,38 @@
 APPEND ~BALTH~
 
 IF ~~ THEN BEGIN KR0
-	SAY ~Are you going to tell me what my duty is, paladin?~
+	SAY @0
   		IF ~~ THEN EXTERN ~KELDO25J~ Bz1
 END 
 
 IF ~~ THEN BEGIN KR1
-	SAY ~Very well. I am listening to you.~
+	SAY @1
   		IF ~Global("VP_KrRomanceActive","GLOBAL",2)~ THEN EXTERN ~KELDO25J~ Bz2
   		IF ~!Global("VP_KrRomanceActive","GLOBAL",2)~ THEN EXTERN ~KELDO25J~ Bz2n
 END 
 
 IF ~~ THEN BEGIN KR2
-	SAY ~(Throwing up his eyes) Your bride? You want to tell me that...~
+	SAY @2
   		IF ~~ THEN EXTERN ~KELDO25J~ Bz3
 END 
 
 IF ~~ THEN BEGIN KR3
-	SAY ~She does not want to take her father's throne?~
+	SAY @3
   		IF ~~ THEN EXTERN ~KELDO25J~ Bz4
 END 
 
 IF ~~ THEN BEGIN KR4
-	SAY ~But why? I have an absolute trust in myself that I will kill her and then I'll kill myself.~
+	SAY @4
   		IF ~~ THEN EXTERN ~KELDO25J~ Bz5
 END 
 
 IF ~~ THEN BEGIN KR5
-	SAY ~I am ready to die, paladin!~
+	SAY @5
   		IF ~~ THEN EXTERN ~KELDO25J~ Bz6
 END 
 
 IF ~~ THEN BEGIN KR6
-  SAY ~Death is a necessity, paladin. Further conversation is pointless. I have a task which I must complete.~
+  SAY @6
   IF ~~ THEN DO ~Shout(ALERT)
 Enemy()
 EraseJournalEntry(67720)
