--- NeJE\dlg\BJAHEIR.D	Thu Apr 19 22:39:37 2018
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\BJAHEIR.D	Tue Apr 09 19:35:01 2019
@@ -1,340 +1,339 @@
 APPEND ~BJAHEIR~
 
 IF ~~ THEN BEGIN TS461
-  SAY ~You were pretty insistent on killing Yoshimo. Now you two act like the closest of friends. I find it suspicious.~
+  SAY @0
   IF ~~ THEN EXTERN ~BKACHI~ 50
 END
 
 IF ~~ THEN BEGIN TS462
-  SAY ~No. My first impression never failed me. But tell me child, is it a blush I see on your cheeks when you speak of Yoshimo?~
+  SAY @1
   IF ~~ THEN EXTERN ~BKACHI~ 51
 END
 
 IF ~~ THEN BEGIN TS463
-  SAY ~Well, you may be hot, you may be ill or you may be in love. I assume the latter, since the weather is mild and you are a strong and healthy person.~
+  SAY @2
   IF ~~ THEN EXTERN ~BKACHI~ 52
 END
 
 IF ~~ THEN BEGIN TS464
-  SAY ~Oh, then it is true...~
+  SAY @3
   IF ~~ THEN EXTERN ~BKACHI~ 53
 END
 
 IF ~~ THEN BEGIN TS465
-  SAY ~And we were not arguing.~
+  SAY @4
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN TS466
-  SAY ~Your katana is in satisfactory condition. Your soul truly is dull and requires polishing. I have heard you say to <CHARNAME> that you would not wed and have children. I find a claim like that...~
+  SAY @5
   IF ~~ THEN EXTERN ~BVALYGA~ TS368
 END
 
 IF ~~ THEN BEGIN TS467
-  SAY ~Fool man. Talking of a thing like that with a girl, who never knew her mother and whose stepfather was killed at her very eyes! She have never known a joy of growing in a proper family, she had never been told of love her parents shared and that led to her birth... You do not solve your problems by complaining to her and you traumatize her with your unhealthy views on family building.~
+  SAY @6
   IF ~~ THEN EXTERN ~BVALYGA~ TS369
 END
 
 IF ~~ THEN BEGIN TS468
-  SAY ~I think you can. I suggest that you choose another, happier things to discuss with her, than your life story. By the way, you are getting to know <CHARNAME> very well, Valygar. I'd say you are actively seeking her friendship. I wonder why?~
+  SAY @7
   IF ~~ THEN EXTERN ~BVALYGA~ TS370
 END
 
 IF ~~ THEN BEGIN TS469
-  SAY ~I speak my mind freely and I care of my friends. If someone is unhappy with that... well that's their problem, not mine.~
+  SAY @8
   IF ~~ THEN EXTERN ~BVALYGA~ TS372
 END
 
 IF ~~ THEN BEGIN TS470
-  SAY ~I had been entrusted with the care of <CHARNAME> long ago by Gorion Spellwielder, a man whom I respected deeply. I would never back away from my promise to him. <CHARNAME>'s own qualities are exceptional, but as any child she requires guidance. And that I am willing to give her.~
+  SAY @9
   IF ~~ THEN EXTERN ~BVALYGA~ TS373
 END
 
 IF ~~ THEN BEGIN TS471
-  SAY ~O what can ail thee, knight-at-arms
-Alone and palely loitering?~
+  SAY @10
   IF ~~ THEN EXTERN ~BVALYGA~ TS375
 END
 
 IF ~~ THEN BEGIN TS472
-  SAY ~Yes, I am fine, thank you. Your question had reminded me what my husband Khalid told me when we first met. He said that he just found and lost the world. Then he read me these lines and asked if the faery daughter has a healing for his heart... Oh, Khalid...~
+  SAY @11
   IF ~~ THEN EXTERN ~BVALYGA~ TS376
 END
 
 IF ~~ THEN BEGIN TS473
-  SAY ~It will not help you this time, Valygar. I know precisely what your sickness is, but I am not the one who can heal it. You are in love, Valygar. Shall I give you the name as well, or you know it yourself?~
+  SAY @12
   IF ~~ THEN EXTERN ~BVALYGA~ TS377
 END
 
 IF ~~ THEN BEGIN TS474
-  SAY ~Drop my guard? With the likes of you in the group? I am not mad enough.~
+  SAY @13
   IF ~~ THEN EXTERN ~BSIME~ 132
 END
 
 IF ~~ THEN BEGIN TS475
-  SAY ~I am not easily offended. But I have seen your type oft enough to be weary, Sime Linvail. I have heard you talking to men. Your words are honey, but your heart is cold.  You called yourself a hunter, as I recall. Indeed you are. But the heads of men are your trophies.~
+  SAY @14
   IF ~~ THEN EXTERN ~BSIME~ 133
 END
 
 IF ~~ THEN BEGIN TS476
-  SAY ~You may think so, Sime, but around women like you there is always some turmoil. You seek drama and thrill, and you break hearts without looking back.~
+  SAY @15
   IF ~~ THEN EXTERN ~BSIME~ 134
 END
 
 IF ~~ THEN BEGIN TS477
-  SAY ~And what would you know of potions?~
+  SAY @16
   IF ~~ THEN EXTERN ~BSIME~ 136
 END
 
 IF ~~ THEN BEGIN TS478
-  SAY ~More talents. Aren't we fortunate to have someone so learned and well-traveled amongst us! But Sime, Tethyr is far away from Icewind Dale and we have found an acceptable substitute.~
+  SAY @17
   IF ~~ THEN EXTERN ~BSIME~ 137
 END
 
 IF ~~ THEN BEGIN TS479
-  SAY ~Jealous? No, I am not. Why would I? I am only pitying <CHARNAME>. The empty-headed toy like you would not give anything. You are not capable to give and instill the true deep feelings, warm and tender, of the sort my Khalid and I shared.~
+  SAY @18
   IF ~~ THEN EXTERN ~BSIME~ 581
 END
 
 IF ~~ THEN BEGIN TS480
-  SAY ~Child, are you going just to sit there and listen your mouth agape, to some foolish girl to shame your oldest friend?~
+  SAY @19
   IF ~~ THEN EXTERN ~BSIME~ 582
 END
 
 IF ~~ THEN BEGIN TS481
-  SAY ~I? A loyal companion? So that's how you feel about me? I shall remember.~
+  SAY @20
   IF ~~ THEN EXTERN ~BSIME~ 583
 END
 
 IF ~~ THEN BEGIN TS482
-  SAY ~Oh, now I hear a man grown, not a boy!~
+  SAY @21
   IF ~~ THEN EXTERN ~BSIME~ 584
 END
 
 IF ~~ THEN BEGIN TS483
-  SAY ~I think that I shall use it then.~
+  SAY @22
   IF ~~ THEN EXTERN ~BSIME~ 586
 END
 
 IF ~~ THEN BEGIN NEJ461
-  SAY ~It is of Khalid I think Hrothgar. I have not seen him since we were taken and we have never been apart since our pairing. I cannot rid myself of the thoughts of him still suffering whilst I am free.~
+  SAY @23
   IF ~~ THEN EXTERN ~BHROTH~ 49
 END
 
 IF ~~ THEN BEGIN NEJ462
-  SAY ~You do not understand Hrothgar, Khalid is a man with a gentle heart, he is not afraid to stand and fight but he would prefer that he did not have to. He took on the guardianship of <CHARNAME> when <PRO_HISHER> foster father died because I convinced him it was our duty to do so. He followed because he believed in me, Hrothgar and now he is lost, still in the clutches of that insane mage and I walk free!~ 
+  SAY @24 
   IF ~~ THEN EXTERN ~BHROTH~ 50
 END
 
 IF ~~ THEN BEGIN NEJ463
-  SAY ~Maybe not Hrothgar, but that does not lessen the responsibility that I feel and now, being here in this strange time, I fear I will never see or touch the face of the man I love again. I would hate that he thinks I have abandoned him.~
+  SAY @25
   IF ~~ THEN EXTERN ~BHROTH~ 51
 END
 
 IF ~~ THEN BEGIN NEJ464
-  SAY ~I hear what it is you are saying Hrothgar and I thank you for what you are trying to do, but I will not rest easy until Khalid walks once more by my side.~
+  SAY @26
   IF ~~ THEN EXTERN ~BHROTH~ 52
 END
 
 IF ~~ THEN BEGIN NEJ465
-  SAY ~Be careful how you answer him <CHARNAME>. Balance between past and present is a very slender thread and is easily frayed.~
-  IF ~~ THEN REPLY ~And don't you think that our being here hasn't already done that Jaheira?~ GOTO NEJ466
-  IF ~~ THEN REPLY ~I believe Hrothgar was speaking to me Jaheira, were you never taught that it is rude to butt into other peoples conversations!~ GOTO NEJ466
-  IF ~~ THEN REPLY ~So we have two <CHARNAME>s in the group do we? Which one were you addressing Hrothgar?~ GOTO NEJ466
+  SAY @27
+  IF ~~ THEN REPLY @28 GOTO NEJ466
+  IF ~~ THEN REPLY @29 GOTO NEJ466
+  IF ~~ THEN REPLY @30 GOTO NEJ466
 END
 
 IF ~~ THEN BEGIN NEJ466
-  SAY ~Just a gentle reminder <CHARNAME>, nothing more. I fear that our presence here has already put balance between past and future in a precarious position. We can not afford to take chances either by deed or by word.~
+  SAY @31
   IF ~~ THEN EXTERN ~BHROTH~ 55
 END
 
 IF ~Global("VP_BMinsc","LOCALS",2)~ THEN BEGIN NEJ467
-  SAY ~Phew... Minsc, whatever is that smell?  It smells like something has died in your pocket.~
+  SAY @32
   IF ~~ THEN DO ~SetGlobal("VP_BMinsc","LOCALS",3)~ EXTERN ~BMINSC~ NEJ122
 END
 
 IF ~~ THEN BEGIN NEJ468
-  SAY ~Well something is. I suggest you give Boo a prod and see if he still lives, hamsters have very short lives. Being a ranger you do know this Minsc?~  
+  SAY @33  
   IF ~~ THEN EXTERN ~BMINSC~ NEJ123
 END
 
 IF ~~ THEN BEGIN NEJ469
-  SAY ~Perhaps you should give him a prod anyway, just to be sure.~
+  SAY @34
   IF ~~ THEN EXTERN ~BMINSC~ NEJ124
 END
 
 IF ~~ THEN BEGIN NEJ470
-  SAY ~Food store!? You mean you allow that rodent to store food in your pocket?~
+  SAY @35
   IF ~~ THEN EXTERN ~BMINSC~ NEJ127
 END
 
 IF ~~ THEN BEGIN NEJ471
-  SAY ~His insides? Oh, you mean his instincts, but surely he knows that as long as he has Minsc to feed him, he has no need to store his food?~
+  SAY @36
   IF ~~ THEN EXTERN ~BMINSC~ NEJ128
 END
 
 IF ~~ THEN BEGIN NEJ472
-  SAY ~The word is *hibernate* Minsc, maybe you should tell this rodent of yours that whereas it may feel cold outside your pocket, he will always be warm and snug inside your pocket. There will be no need for him to hibernate. He might also like to know that hibernating creatures have no need of food whilst they sleep, only when they wake up again, so he can clear his smelly store out and give all our noses a rest.~
+  SAY @37
   IF ~~ THEN EXTERN ~BMINSC~ NEJ129
 END
 
 IF ~~ THEN BEGIN NEJ473
-  SAY ~Then you best be prepared to walk a good distance from the rest of the group until he does decide.~
+  SAY @38
   IF ~~ THEN EXIT
 END
 
 IF ~Global("VP_BDar","LOCALS",2)~ THEN BEGIN NEJ474
-  SAY ~Dar, I am tired of your sideways glances. If you wish to say something to me, say it and stop acting coy.~
+  SAY @39
   IF ~~ THEN DO ~SetGlobal("VP_BDar","LOCALS",3)~ EXTERN ~BDAR~ 58
 END
 
 IF ~~ THEN BEGIN NEJ475
-  SAY ~Your glances do not offend me, but if I bother you in some way I would rather you told me instead of looking at me when you think I do not notice. I find it unnerving to be scrutinised so.~
+  SAY @40
   IF ~~ THEN EXTERN ~BDAR~ 59
 END
 
 IF ~~ THEN BEGIN NEJ476
-  SAY ~I thank you for your compliment Dar, but tell me, why should I turn away from you, I sense no malice in your eye or in your heart?~
+  SAY @41
   IF ~~ THEN EXTERN ~BDAR~ 60
 END
 
 IF ~~ THEN BEGIN NEJ477
-  SAY ~Why should my being a druid cause me to be offended by you? In case you had not noticed, I am of mixed race also, neither true elf nor true human.~
+  SAY @42
   IF ~~ THEN EXTERN ~BDAR~ 61
 END
 
 IF ~~ THEN BEGIN NEJ478
-  SAY ~Nature does not plan a lot of things Dar. The instinct to mate and pass on ones genes is so strong in all her creatures that even nature cannot control all of her children. She understands this and never finds the offspring of these unions abhorrent or offensive to her sight. In her eyes, the child can never be blamed for the indiscretions of the parents.~
+  SAY @43
   IF ~~ THEN EXTERN ~BDAR~ 62
 END
 
 IF ~~ THEN BEGIN NEJ479
-  SAY ~You have been forced to seek your way in the world alone, scorned and driven from any civilised contact, it is no wonder you doubt your place in the scheme of things. Know that there will always be those who judge by what they see, but there will also be those who look deeper into the heart as the mother teaches all her servants to do and they are the ones that you must look to for your learning.~ 
+  SAY @44 
   IF ~~ THEN EXTERN ~BDAR~ 63
 END
 
 IF ~Global("VP_BPC","LOCALS",2)~ THEN BEGIN NEJ480
-  SAY ~What are we doing here <CHARNAME>? We already know how these events are to end and we should not be here.~
-  IF ~~ THEN REPLY ~I can’t explain it Jaheira, but I have this inexplicable feeling that for now this is where I should be.~ DO ~SetGlobal("VP_BPC","LOCALS",3)~ GOTO NEJ481
-  IF ~~ THEN REPLY ~My life has been one of many twists, you yourself have been witness to this Jaheira, but it has never taken me anywhere without reason.~ DO ~SetGlobal("VP_BPC","LOCALS",3)~ GOTO NEJ481
-  IF ~~ THEN REPLY ~Hrothgar is a brave man Jaheira, but he can not do this alone, he needs my... our aid.~ DO ~SetGlobal("VP_BPC","LOCALS",3)~ GOTO NEJ482 
-  IF ~~ THEN REPLY ~We are here now and I stand by my decision to to help Hrothgar regardless of how things turn out.~ DO ~SetGlobal("VP_BPC","LOCALS",3)~ GOTO NEJ483
+  SAY @45
+  IF ~~ THEN REPLY @46 DO ~SetGlobal("VP_BPC","LOCALS",3)~ GOTO NEJ481
+  IF ~~ THEN REPLY @47 DO ~SetGlobal("VP_BPC","LOCALS",3)~ GOTO NEJ481
+  IF ~~ THEN REPLY @48 DO ~SetGlobal("VP_BPC","LOCALS",3)~ GOTO NEJ482 
+  IF ~~ THEN REPLY @49 DO ~SetGlobal("VP_BPC","LOCALS",3)~ GOTO NEJ483
 END
 
 IF ~~ THEN BEGIN NEJ481
-  SAY ~That may be so, but this mission is a waste of our time. We can not interfere with these events and as such we should have continued our own journey. We should have taken the portal at Kuldahar and left the past to the past. I fear I should not have been so hasty in agreeing to abandon Khalid, now with the portal lost I know not when or even if I will see him again.~
-  IF ~~ THEN REPLY ~Has it not occurred to you that maybe... just maybe... I am here to alter that which happened? It is possible for a mistake to have been made in the past and I am here to right that mistake.~ GOTO NEJ484
-  IF ~~ THEN REPLY ~So far I have had little choice in my direction and I believe it is the case here. I may not always like where fate leads me but if we were meant to have taken the portal, then she would have ensured that we had.~ GOTO NEJ485
-  IF ~~ THEN REPLY ~We had no idea what lay beyond that portal Jaheira. It could just as easily have been another of Irenicus’s making, hoping to snare us in another hell dimension somewhere. No, it is my belief that fate prevented us from taking that portal for a reason.~ GOTO NEJ485
-  IF ~~ THEN REPLY ~Stop worrying about the past and the future. What will be will be and no amount of worrying on your part will change that.~ GOTO NEJ486
-  IF ~~ THEN REPLY ~A decision was made Jaheira and I know I have made the right choice. Call it instinct, call it gut feeling, whatever, this is where we are meant to be.~ GOTO NEJ487
+  SAY @50
+  IF ~~ THEN REPLY @51 GOTO NEJ484
+  IF ~~ THEN REPLY @52 GOTO NEJ485
+  IF ~~ THEN REPLY @53 GOTO NEJ485
+  IF ~~ THEN REPLY @54 GOTO NEJ486
+  IF ~~ THEN REPLY @55 GOTO NEJ487
 END
 
 IF ~~ THEN BEGIN NEJ482
-  SAY ~You have no grasp of what I am saying do you? Hrothgar’s fate was sealed in the past and your aid must not alter the outcome of these events. Our future would have been much safer if we had taken the portal at Kuldahar, it was a mistake to come here.~
-  IF ~~ THEN REPLY ~Has it not occurred to you that maybe... just maybe... I am here to alter that which happened? It is possible for a mistake to have been made in the past and I am here to right that mistake.~ GOTO NEJ484
-  IF ~~ THEN REPLY ~So far I have had little choice in my direction and I believe it is the case here. I may not always like where fate leads me but if we were meant to have taken the portal, then she would have ensured that we had.~ GOTO NEJ485
-  IF ~~ THEN REPLY ~We had no idea what lay beyond that portal Jaheira. It could just as easily have been another of Irenicus’s making, hoping to snare us in another hell dimension somewhere. No, it is my belief that fate prevented us from taking that portal for a reason.~ GOTO NEJ485
-  IF ~~ THEN REPLY ~Stop worrying about the past and the future. What will be will be and no amount of worrying on your part will change that.~ GOTO NEJ486
-  IF ~~ THEN REPLY ~A decision was made Jaheira and I know I have made the right choice. Call it instinct, call it gut feeling, whatever, this is where we are meant to be.~ GOTO NEJ487
+  SAY @56
+  IF ~~ THEN REPLY @51 GOTO NEJ484
+  IF ~~ THEN REPLY @52 GOTO NEJ485
+  IF ~~ THEN REPLY @53 GOTO NEJ485
+  IF ~~ THEN REPLY @54 GOTO NEJ486
+  IF ~~ THEN REPLY @55 GOTO NEJ487
 END
 
 IF ~~ THEN BEGIN NEJ483
-  SAY ~It is noble of you to want to aid Hrothgar, but what has been must be allowed to be. Have I not said so before? You can not... must not alter what fate has decreed. If by coming here you think to do this, then we should have taken the portal at Kuldahar, for the future will surely be in jeopardy if you succeed in this foolishness.~
-  IF ~~ THEN REPLY ~Has it not occurred to you that maybe... just maybe... I am here to alter that which happened? It is possible for a mistake to have been made in the past and I am here to right that mistake.~ GOTO NEJ484
-  IF ~~ THEN REPLY ~So far I have had little choice in my direction and I believe it is the case here. I may not always like where fate leads me but if we were meant to have taken the portal, then she would have ensured that we had.~ GOTO NEJ485
-  IF ~~ THEN REPLY ~We had no idea what lay beyond that portal Jaheira. It could just as easily have been another of Irenicus’s making, hoping to snare us in another hell dimension somewhere. No, it is my belief that fate prevented us from taking that portal for a reason.~ GOTO NEJ485
-  IF ~~ THEN REPLY ~Stop worrying about the past and the future. What will be will be and no amount of worrying on your part will change that.~ GOTO NEJ486
-  IF ~~ THEN REPLY ~A decision was made Jaheira and I know I have made the right choice. Call it instinct, call it gut feeling, whatever, this is where we are meant to be.~ GOTO NEJ487
+  SAY @57
+  IF ~~ THEN REPLY @51 GOTO NEJ484
+  IF ~~ THEN REPLY @52 GOTO NEJ485
+  IF ~~ THEN REPLY @53 GOTO NEJ485
+  IF ~~ THEN REPLY @54 GOTO NEJ486
+  IF ~~ THEN REPLY @55 GOTO NEJ487
 END
 
 IF ~~ THEN BEGIN NEJ484
-  SAY ~Surely you don’t believe that any more than I do <CHARNAME>? If you have convinced yourself that this is true and you intend acting on your belief, then I fear that the future we knew may not be the future that we will eventually return to. You should take the time to think long and hard on what we may come to lose by any rash and emotional actions you choose to take here.~
-  IF ~~ THEN REPLY ~I cannot help what you feel or believe. I have put my trust in Hrothgar and his is the path we now follow. That is... until we find a way home.~ GOTO NEJ489
-  IF ~~ THEN REPLY ~Hrothgar has assured me that my actions here will not affect my future, so I see no reason not to continue aiding him in his mission.~ GOTO NEJ489
-  IF ~~ THEN REPLY ~Whatever, circumstances being what they are, we have very little choice now, as Hrothgar is our only means of leaving here safely.~ GOTO NEJ489
+  SAY @58
+  IF ~~ THEN REPLY @59 GOTO NEJ489
+  IF ~~ THEN REPLY @60 GOTO NEJ489
+  IF ~~ THEN REPLY @61 GOTO NEJ489
 END
 
 IF ~~ THEN BEGIN NEJ485
-  SAY ~That is rubbish <CHARNAME>, all this talk about fate lending a hand! We make our own choices in this life. Sometimes we are forced by others to do their will, but when we are offered a choice, that is exactly what it is... a choice, and we have to take responsibility for our decision for it is ours alone.~
+  SAY @62
   IF ~~ THEN GOTO NEJ488
 END
 
 IF ~~ THEN BEGIN NEJ486
-  SAY ~I worry with good reason <CHARNAME>. When balance in time is disturbed, whether by accident or by design then a price will have to be paid somewhere or by someone... possibly even by both. I fear that it will be the future that pays the price, along with all  its people who live there now if you interfere with events here.~
-  IF ~~ THEN REPLY ~I cannot help what you feel or believe. I have put my trust in Hrothgar and his is the path we now follow. That is... until we find a way home.~ GOTO NEJ489
-  IF ~~ THEN REPLY ~Hrothgar has assured me that my actions here will not affect my future, so I see no reason not to continue aiding him in his mission.~ GOTO NEJ489
-  IF ~~ THEN REPLY ~Whatever, circumstances being what they are, we have very little choice now, as Hrothgar is our only means of leaving here safely.~ GOTO NEJ489
+  SAY @63
+  IF ~~ THEN REPLY @59 GOTO NEJ489
+  IF ~~ THEN REPLY @60 GOTO NEJ489
+  IF ~~ THEN REPLY @61 GOTO NEJ489
 END
 
 IF ~~ THEN BEGIN NEJ487
-  SAY ~All of my instincts and my knowledge as a druid tells me that we should not be in this place. All things have their own time and place in the greater scheme of things and this is not ours! I can not help but fear that irrevocable harm will be done if we continue on our present path. I may not be the child of a god, and maybe your instincts are more honed than mine for this reason, but who is to say that we should trust your instincts over mine <CHARNAME>?~
-  IF ~~ THEN REPLY ~I cannot help what you feel or believe. I have put my trust in Hrothgar and his is the path we now follow. That is... until we find a way home.~ GOTO NEJ489
-  IF ~~ THEN REPLY ~Hrothgar has assured me that my actions here will not affect my future, so I see no reason not to continue aiding him in his mission.~ GOTO NEJ489
-  IF ~~ THEN REPLY ~Whatever, circumstances being what they are, we have very little choice now, as Hrothgar is our only means of leaving here safely.~ GOTO NEJ489
+  SAY @64
+  IF ~~ THEN REPLY @59 GOTO NEJ489
+  IF ~~ THEN REPLY @60 GOTO NEJ489
+  IF ~~ THEN REPLY @61 GOTO NEJ489
 END
 
 IF ~~ THEN BEGIN NEJ488
-  SAY ~	You chose not to take the portal and we chose to follow you. You could have just as easily have made the decision to take the portal and we would probably still have chosen to follow you. You chose our path, not fate and now we have to live with the consequences of that choice.~
-  IF ~~ THEN REPLY ~I cannot help what you feel or believe. I have put my trust in Hrothgar and his is the path we now follow. That is... until we find a way home.~ GOTO NEJ489
-  IF ~~ THEN REPLY ~Hrothgar has assured me that my actions here will not affect my future, so I see no reason not to continue aiding him in his mission.~ GOTO NEJ489
-  IF ~~ THEN REPLY ~Whatever, circumstances being what they are, we have very little choice now, as Hrothgar is our only means of leaving here safely.~ GOTO NEJ489
+  SAY @65
+  IF ~~ THEN REPLY @59 GOTO NEJ489
+  IF ~~ THEN REPLY @60 GOTO NEJ489
+  IF ~~ THEN REPLY @61 GOTO NEJ489
 END
 
 IF ~~ THEN BEGIN NEJ489
-  SAY ~I don’t have the same faith in the man as you <CHARNAME>. The manner in which he brought us to this place is such that we have no way of knowing the road back... a manner which incidently he quite deftly got out of explaining.~
+  SAY @66
   IF ~~ THEN GOTO NEJ490
 END  
   
 IF ~~ THEN BEGIN NEJ490
-  SAY ~We are now dependent on his staying alive in order to get us out of here, and you can guarantee that he knows this and he is more than likely relying on this to stop us questioning him too closely. Anything happens to Hrothgar and we could be wandering these cold wastelands, forever lost in time and searching for a way home. Trust him if you want, but I shall watch him closely.~ 
+  SAY @67 
 	IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN NEJ491
-  SAY ~Now is not a good time to treat us to a song child!~
+  SAY @68
   IF ~~ THEN EXTERN ~BLEINA~ 185
 END
 
 IF ~~ THEN BEGIN NEJ492
-  SAY ~That was all! We tread a perilous road, not knowing what lies ahead and you let your harp alert our presence to all a sundry. Why can’t you put the thing in your backpack child, lessen the chances of it happening again?~
+  SAY @69
   IF ~~ THEN EXTERN ~BLEINA~ 186
 END
 
 IF ~~ THEN BEGIN NEJ493
-  SAY ~For a half elf you are still a child. You may think your time on the road and your experiences of the world entitles you to be seen as grown up, but your years, or should that be your lack of them, mark you as a child.~
+  SAY @70
   IF ~~ THEN EXTERN ~BLEINA~ 187
 END
 
 IF ~~ THEN BEGIN NEJ494
-  SAY ~My life has been fraught with responsibilities that you know nothing of child! Let me tell you of another I know with the same know it all and insolent attitude that you exhibit. You have probably heard <CHARNAME> speak of Imoen, she too thought she was all grown up. I wonder how her smart mouth is helping her cope with Irenicus now?~
+  SAY @71
   IF ~~ THEN EXTERN ~BLEINA~ 188
 END
 
 IF ~~ THEN BEGIN NEJ495
-  SAY ~I am sorry child, that remark about Imoen was unfair, I admit. It’s true my turns blood cold when I think what that poor child is being subjected to by that sadistic bastard. As for why she speaks as she does... it is because she is a child and enjoys rebelling against her elders. After all, is that not the real reason you ended up in an unsavoury marriage, to spite your parents?~
+  SAY @72
   IF ~~ THEN EXTERN ~BLEINA~ 189
 END
 
 IF ~~ THEN BEGIN NEJ496
-  SAY ~I will try to remember that, but it will be easier to do so if you have lost that insolent attitude of yours by the time we next speak.~
+  SAY @73
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN NEJ497
-  SAY ~Khalid wasn’t much of a talker but when we danced... words were not important He said all he had to say with his eyes and the way he held me. I will never dance again.~
+  SAY @74
   IF ~~ THEN EXTERN ~BLEINA~ PCF2Leina36a
 END
 
 IF ~~ THEN BEGIN NEJ498
-  SAY ~(*To herself*) Would you approve dear heart, I wonder?~
+  SAY @75
   IF ~Gender(Player1,FEMALE)~ THEN EXTERN ~BLEINA~ PCF2Leina2a
   IF ~Gender(Player1,MALE)~ THEN EXTERN ~BLEINA~ PCM4Leina2b
 	IF ~InParty("Yoshimo")~ THEN EXTERN ~BYOSHIM~ NEJ9
 END
 
 IF ~~ THEN BEGIN NEJ499
-  SAY ~(*Clucking her tongue and raising her voice*) Please <CHARNAME> this is a holy place... Gorion would turn in his grave if he could hear you carrying on so!~
+  SAY @76
 	IF ~True()~ THEN EXTERN ~BLEINA~ ClericQuest17
 	IF ~InParty("Taffic")
 !Dead("Taffic")~ THEN EXTERN ~BTAFFIC~ 8
@@ -347,7 +346,7 @@
 END
 
 IF ~~ THEN BEGIN NEJ500
-	SAY ~Yes well... I doubt they expect to see it walking back to greet them!~
+	SAY @77
 	IF ~True()~ THEN EXTERN ~BLEINA~ ClericQuest21
 	IF ~InParty("Hrothgar")
 !Dead("Hrothgar")~ THEN EXTERN ~BHROTH~ 88
@@ -356,7 +355,7 @@
 END
 
 IF ~~ THEN BEGIN LeiSpher1
-	SAY ~Leina can we not grow up? A guessing game does not make a good impression, but if it will get us out of this sooner then a lodge.~
+	SAY @78
 	IF ~True()~ THEN EXTERN ~BLEINA~ Sphere18
 	IF ~InParty("Yoshimo")
 !Dead("Yoshimo")~ THEN EXTERN ~BYOSHIM~ LeiSpher1
@@ -367,7 +366,7 @@
 END
 
 IF ~~ THEN BEGIN LeiSpher2
-	SAY ~Child will you make up your mind? We will be here for a fortnight if you keep changing the size.~
+	SAY @79
 	IF ~True()~ THEN EXTERN ~BLEINA~ Sphere19
 	IF ~InParty("Yoshimo")
 !Dead("Yoshimo")~ THEN EXTERN ~BYOSHIM~ LeiSpher2
@@ -378,7 +377,7 @@
 END
 
 IF ~~ THEN BEGIN LeiSpher3
-	SAY ~(*Rolling her eyes and clicking her tongue*) Tsk... did I say a fortnight?~
+	SAY @80
 	IF ~True()~ THEN EXTERN ~BLEINA~ Sphere20
 	IF ~InParty("Yoshimo")
 !Dead("Yoshimo")~ THEN EXTERN ~BYOSHIM~ LeiSpher3
@@ -389,12 +388,12 @@
 END
 
 IF ~~ THEN BEGIN LeiSpher4
-	SAY ~Yes what you have said reminds me of a book Khalid, found in the library at Candlekeep. He was laughing very hard over this bard’s tale about some man who had a box that was bigger on the inside then the outside.~
+	SAY @81
 	IF ~~ THEN EXTERN ~BLEINA~ Sphere26 
 END
 
 IF ~~ THEN BEGIN LeiTheat
-  SAY ~I have seen some of your ideas in the past, perhaps it would be wise to sleep on it before you act on this one!~
+  SAY @82
  IF ~!IsValidForPartyDialogue("Jan")
 !IsValidForPartyDialogue("Keldorn")
 !IsValidForPartyDialogue("Korgan")
@@ -405,7 +404,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Oh enough with your horse play! Leina always wants a hot bath let’s get the jump on her first this time. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @83 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Jan")
 !IsValidForPartyDialogue("Keldorn")
 !IsValidForPartyDialogue("Korgan")
@@ -416,7 +415,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Sssssh. Now listen Leina always wants a bath after a battle and she hasn’t said anything yet so let’s do it first.~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @84 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Jan")
 !IsValidForPartyDialogue("Keldorn")
 !IsValidForPartyDialogue("Korgan")
@@ -427,7 +426,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Hey now, listen Leina is always the first to want a bath after a stinkin’ battle so let’s ask her first for a fluffy towel. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @85 GOTO LeiThea1
 	IF ~InParty("Taffic")
 !Dead("Taffic")~ THEN EXTERN ~BTAFFIC~ LeiTheat
 	IF ~InParty("Melora")
@@ -453,7 +452,7 @@
 END
 
 IF ~~ THEN BEGIN LeiThea3
-  SAY ~(*Smiles to herself*) Do you really think so <CHARNAME>, do you really think so?~
+  SAY @86
  IF ~!IsValidForPartyDialogue("Jan")
 !IsValidForPartyDialogue("Keldorn")
 !IsValidForPartyDialogue("Korgan")
@@ -464,7 +463,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Oh enough with your horse play! Leina always wants a hot bath let’s get the jump on her first this time. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @83 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Jan")
 !IsValidForPartyDialogue("Keldorn")
 !IsValidForPartyDialogue("Korgan")
@@ -475,7 +474,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Sssssh. Now listen Leina always wants a bath after a battle and she hasn’t said anything yet so let’s do it first.~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @84 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Jan")
 !IsValidForPartyDialogue("Keldorn")
 !IsValidForPartyDialogue("Korgan")
@@ -486,7 +485,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Hey now, listen Leina is always the first to want a bath after a stinkin’ battle so let’s ask her first for a fluffy towel. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @85 GOTO LeiThea1
 	IF ~InParty("Taffic")
 !Dead("Taffic")~ THEN EXTERN ~BTAFFIC~ LeiThea3
 	IF ~InParty("Melora")
@@ -512,7 +511,7 @@
 END
 
 IF ~~ THEN BEGIN LeiThea4
-  SAY ~(*Sighing*) Here was I thinking, that with all that gone before, maybe, just maybe you had grown up a little, <CHARNAME>.~
+  SAY @87
  IF ~!IsValidForPartyDialogue("Jan")
 !IsValidForPartyDialogue("Keldorn")
 !IsValidForPartyDialogue("Korgan")
@@ -523,7 +522,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Oh enough with your horse play! Leina always wants a hot bath let’s get the jump on her first this time. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @83 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Jan")
 !IsValidForPartyDialogue("Keldorn")
 !IsValidForPartyDialogue("Korgan")
@@ -534,7 +533,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Sssssh. Now listen Leina always wants a bath after a battle and she hasn’t said anything yet so let’s do it first.~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @84 GOTO LeiThea1
  IF ~!IsValidForPartyDialogue("Jan")
 !IsValidForPartyDialogue("Keldorn")
 !IsValidForPartyDialogue("Korgan")
@@ -545,7 +544,7 @@
 !IsValidForPartyDialogue("Hrothgar")
 !IsValidForPartyDialogue("Dar")
 !IsValidForPartyDialogue("Melora")
-!IsValidForPartyDialogue("Taffic")~ THEN REPLY ~Hey now, listen Leina is always the first to want a bath after a stinkin’ battle so let’s ask her first for a fluffy towel. What say you?~ GOTO LeiThea1
+!IsValidForPartyDialogue("Taffic")~ THEN REPLY @85 GOTO LeiThea1
 	IF ~InParty("Taffic")
 !Dead("Taffic")~ THEN EXTERN ~BTAFFIC~ LeiThea4
 	IF ~InParty("Melora")
@@ -571,7 +570,7 @@
 END
 
 IF ~~ THEN BEGIN LeiThea1
-  SAY ~Tsk! All of this for what... to see who gets in first and suggests a bath? Silvanus protect me from the whims of children!~
+  SAY @88
 	IF ~True()~ THEN EXTERN ~BLEINA~ TheaQ151
 	IF ~InParty("Taffic")
 !Dead("Taffic")~ THEN EXTERN ~BTAFFIC~ LeiThea1
@@ -598,7 +597,7 @@
 END
 
 IF ~~ THEN BEGIN LeiThea2
-  SAY ~Girl, get that silly notion out of your mind. I have no wish to be part of a bard’s romantic tale.~
+  SAY @89
 	IF ~True()~ THEN EXTERN ~BLEINA~ TheaQ161
 	IF ~InParty("Viconia")
 !Dead("Viconia")~ THEN EXTERN ~BVICONI~ LeiThea2
