--- NeJE\dlg\BMAZZY2.D	Wed Nov 17 13:22:00 2004
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\BMAZZY2.D	Tue Apr 09 19:35:18 2019
@@ -1,32 +1,32 @@
 APPEND ~BMAZZY25~
 
 IF ~~ THEN BEGIN TS68
-  SAY ~That was a long time ago, Valygar. I believe that you said you would sleep on it. Well you have had much sleep since then, yet this is the first time you have brought the subject up.~
+  SAY @0
   IF ~~ THEN EXTERN ~BVALYG25~ TS112
 END
 
 IF ~~ THEN BEGIN TS69
-  SAY ~And you do not want to join with me. That is fine.~
+  SAY @1
   IF ~~ THEN EXTERN ~BVALYG25~ TS113
 END
 
 IF ~~ THEN BEGIN TS70
-  SAY ~Of course I will. I am a fair leader and I have no trouble recruiting. And there is something more I wanted to say, Valygar.~
+  SAY @2
   IF ~~ THEN EXTERN ~BVALYG25~ TS114
 END
 
 IF ~~ THEN BEGIN TS71
-  SAY ~Valygar, I love you both and I am sure you will be very happy together. I think I will hear one day some tales about you two.~
+  SAY @3
   IF ~~ THEN EXTERN ~BVALYG25~ TS115
 END
 
 IF ~~ THEN BEGIN TS72
-  SAY ~It's not at all like you to be so indecisive, Valygar.~
+  SAY @4
   IF ~~ THEN EXTERN ~BVALYG25~ TS116
 END
 
 IF ~~ THEN BEGIN TS73
-  SAY ~There is nothing greater than love, Valygar.~
+  SAY @5
   IF ~~ THEN EXTERN ~BVALYG25~ TS117
 END
 END
