--- NeJE\dlg\DPOMAB.D	Sat Dec 10 19:09:15 2016
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\DPOMAB.D	Tue Apr 09 19:35:52 2019
@@ -2,98 +2,98 @@
 
 IF ~NumTimesTalkedTo(0)
 Global("Know_Balance", "GLOBAL", 0)~ THEN BEGIN 0 // from:
-  SAY ~What... by the Nine Hells... what are you lot doing back here? Thieves, barbarians come I demand some answers now?~ 
-  IF ~~ THEN REPLY ~Barbarians?  I think you're mistaken, friend.~  GOTO 1
-  IF ~~ THEN REPLY ~I'd watch where you wag that tongue of yours, shopkeeper. I am not in the best of moods.~  GOTO 2
-  IF ~~ THEN REPLY ~Right. I'd like to see what you have for sale.~  GOTO 4
-  IF ~~ THEN REPLY ~I am only taking a closer look at your wares back here, checking the quality nothing more. Now what else do you have to sell?~  GOTO 4
+  SAY @0 
+  IF ~~ THEN REPLY @1  GOTO 1
+  IF ~~ THEN REPLY @2  GOTO 2
+  IF ~~ THEN REPLY @3  GOTO 4
+  IF ~~ THEN REPLY @4  GOTO 4
 END
 
 IF ~~ THEN BEGIN 1 // from: 0.0
-  SAY ~Am I now?  I do not think so. All of you northerners are the same to me. Smelly and barbaric. ~ 
-  IF ~~ THEN REPLY ~Keep pushing it... I'll show you just how barbaric I can be.~  GOTO 2
-  IF ~~ THEN REPLY ~Northerners?  I take it you are not from around here.~  GOTO 3
-  IF ~~ THEN REPLY ~Right. I'd like to see what you have for sale.~  GOTO 4
-  IF ~~ THEN REPLY ~Northerners? I take it we are far away from the south.~  GOTO 16
+  SAY @5 
+  IF ~~ THEN REPLY @6  GOTO 2
+  IF ~~ THEN REPLY @7  GOTO 3
+  IF ~~ THEN REPLY @3  GOTO 4
+  IF ~~ THEN REPLY @8  GOTO 16
 END
 
 IF ~~ THEN BEGIN 2 // from: 15.1 15.0 12.2 12.1 12.0 11.2 11.0 4.2 4.1 4.0 1.0 0.1
-  SAY ~How dare you speak to me in such a manner!  Do you know who I am?  ~ 
-  IF ~~ THEN REPLY ~No, but I'm sure you're going to tell me.~  GOTO 3
-  IF ~~ THEN REPLY ~Yes, you are a pompous, little man who runs a little shop... somewhere I have no idea.~  GOTO 3
-  IF ~~ THEN REPLY ~I don't care who you are... I just want to buy something!~  GOTO 3
+  SAY @9 
+  IF ~~ THEN REPLY @10  GOTO 3
+  IF ~~ THEN REPLY @11  GOTO 3
+  IF ~~ THEN REPLY @12  GOTO 3
 END
 
 IF ~~ THEN BEGIN 3 // from: 2.2 2.1 2.0 1.1
-  SAY ~Hmph!  I am Pomab Ak'azmhir, Royal Diplomatic Envoy of Calimshan and Appointed Overseer of the Northern Caravan Routes!~ 
-  IF ~CheckStatGT(Protagonist, 14, CHR)~ THEN REPLY ~Forgive me, your lordship. I had no idea.~  DO ~SetGlobal("Know_Pomab", "GLOBAL", 1)~ GOTO 8
-  IF ~CheckStatLT(Player1, 15, CHR)~ THEN REPLY ~Forgive me, your lordship. I had no idea.~  DO ~SetGlobal("Know_Pomab", "GLOBAL", 1)~ GOTO 5
-  IF ~~ THEN REPLY ~Sure... and I'm the King of Mithril Hall. Ha!~  DO ~SetGlobal("Know_Pomab", "GLOBAL", 1)~ GOTO 5
-  IF ~~ THEN REPLY ~Oh, I see. Your appearance as a lowly shopkeeper is just a clever disguise... to throw off any would be assassins. Am I right?~  DO ~SetGlobal("Know_Pomab", "GLOBAL", 1)~ GOTO 5
+  SAY @13 
+  IF ~CheckStatGT(Protagonist, 14, CHR)~ THEN REPLY @14  DO ~SetGlobal("Know_Pomab", "GLOBAL", 1)~ GOTO 8
+  IF ~CheckStatLT(Player1, 15, CHR)~ THEN REPLY @14  DO ~SetGlobal("Know_Pomab", "GLOBAL", 1)~ GOTO 5
+  IF ~~ THEN REPLY @15  DO ~SetGlobal("Know_Pomab", "GLOBAL", 1)~ GOTO 5
+  IF ~~ THEN REPLY @16  DO ~SetGlobal("Know_Pomab", "GLOBAL", 1)~ GOTO 5
 END
 
 IF ~~ THEN BEGIN 4 // from: 1.2 0.2
-  SAY ~And I would like to see some coin before I go to the trouble of showing you my wares. I have no patience for those who are "just browsing."  ~ 
-  IF ~~ THEN REPLY ~There's no need to be rude. I assure you, I intend to buy. Now, may I see your wares?~  GOTO 2
-  IF ~~ THEN REPLY ~Look you toad, I have gold and I need supplies. Are you going to sell to me or not?~  GOTO 2
-  IF ~~ THEN REPLY ~Maybe it would be easier if I cut your greedy little heart out and just take what I want. What do you think about that?~  DO ~ReputationInc(-1)~ GOTO 2
+  SAY @17 
+  IF ~~ THEN REPLY @18  GOTO 2
+  IF ~~ THEN REPLY @19  GOTO 2
+  IF ~~ THEN REPLY @20  DO ~ReputationInc(-1)~ GOTO 2
 END
 
 IF ~~ THEN BEGIN 5 // from: 16.0 3.3 3.2 3.1
-  SAY ~Your poor attempt at sarcasm is an obvious sign of your lowly birth. I'll have you know that I am third cousin to the Pasha himself!  Not to mention a royal courtier in good standing.~
-  IF ~~ THEN REPLY ~I'm thinking maybe your brain has been addled. Cabin fever, perhaps?~  GOTO 7
-  IF ~~ THEN REPLY ~My apologies. I did not mean to offend.~  GOTO 8
-  IF ~~ THEN REPLY ~Whatever. Can I just buy something, please?~  DO ~StartStore("EHPomab", Protagonist)~ EXIT
+  SAY @21
+  IF ~~ THEN REPLY @22  GOTO 7
+  IF ~~ THEN REPLY @23  GOTO 8
+  IF ~~ THEN REPLY @24  DO ~StartStore("EHPomab", Protagonist)~ EXIT
 END
 
 IF ~~ THEN BEGIN 6 // from: 5.0
-  SAY ~The Pasha... uh... asked me to accept this post of Overseer of the Northern Caravan Routes, as a personal favor.~ 
-  IF ~~ THEN REPLY ~I see. Likely this "post" is naught but a convenient excuse for the Pasha to get rid of a royal pain in his arse.~  GOTO 7
-  IF ~~ THEN REPLY ~Oh, you mean you were banished?~  GOTO 7
-  IF ~~ THEN REPLY ~Interesting. Tell me "Overseer," did you ever hear of such a post before the Pasha offered it to you?~  GOTO 9
+  SAY @25 
+  IF ~~ THEN REPLY @26  GOTO 7
+  IF ~~ THEN REPLY @27  GOTO 7
+  IF ~~ THEN REPLY @28  GOTO 9
 END
 
 IF ~~ THEN BEGIN 7 // from: 6.1 6.0 5.1
-  SAY ~I would not expect someone of *your* station to understand such matters. Now, if you do not mind... buy something or leave!~ 
-  IF ~~ THEN REPLY ~Fine. What do you have for sale?~  DO ~StartStore("EHPomab", Player1)~ EXIT
-  IF ~~ THEN REPLY ~As you wish. Farewell, your Royalness.~  EXIT
+  SAY @29 
+  IF ~~ THEN REPLY @30  DO ~StartStore("EHPomab", Player1)~ EXIT
+  IF ~~ THEN REPLY @31  EXIT
 END
 
 IF ~~ THEN BEGIN 8 // from: 5.2 3.0
-  SAY ~Yes, well... where I am from, one would normally be beaten for such insolence. Be thankful that I am merciful man. Now, what can I do for you?~ 
-  IF ~~ THEN REPLY ~Other than die a horrible, painfully slow death... you can do nothing for me. Farewell.~  EXIT
-  IF ~~ THEN REPLY ~I'd like to see what you have for sale.~  DO ~StartStore("EHPomab", Player1)~ EXIT
-  IF ~~ THEN REPLY ~Never mind. Farewell.~  EXIT
+  SAY @32 
+  IF ~~ THEN REPLY @33  EXIT
+  IF ~~ THEN REPLY @34  DO ~StartStore("EHPomab", Player1)~ EXIT
+  IF ~~ THEN REPLY @35  EXIT
 END
 
 IF ~~ THEN BEGIN 9 // from: 6.2
-  SAY ~Well... I... er... I grow tired of your ridiculous questions!  I have many obligations to tend to, so I suggest you quit wasting my time. Either buy something or get out!~ 
-  IF ~~ THEN REPLY ~Fine. What do you have for sale?~  DO ~StartStore("EHPomab", Protagonist)~ EXIT
-  IF ~~ THEN REPLY ~As you wish. Farewell, your Royalness.~  EXIT
+  SAY @36 
+  IF ~~ THEN REPLY @30  DO ~StartStore("EHPomab", Protagonist)~ EXIT
+  IF ~~ THEN REPLY @31  EXIT
 END
 
 IF ~NumTimesTalkedTo(0)
 Global("Supply_Quest", "GLOBAL", 1)
 Global("Know_Balance", "GLOBAL", 0)~ THEN BEGIN 10 // from:
-  SAY ~What is this?  More barbarians come to my shop?  No doubt with nothing to barter with but more wolf pelts and polished stones. Very well... let us get this over with. What do you want?~ 
-  IF ~PartyHasItem("LIST")~ THEN REPLY ~Actually, Hrothgar sent me. I am to deliver this supply list and outfit myself for the expedition to Kuldahar.~  DO ~TakePartyItem("LIST")
+  SAY @37 
+  IF ~PartyHasItem("LIST")~ THEN REPLY @38  DO ~TakePartyItem("LIST")
 SetGlobal("Supply_Quest", "GLOBAL", 2)
 AddExperienceParty(3500)~ GOTO 11
-  IF ~!PartyHasItem("LIST")~ THEN REPLY ~Actually, Hrothgar sent me. I was supposed to deliver a supply list and outfit myself for the expedition to Kuldahar, but I seem to have misplaced the list.~  GOTO 12
+  IF ~!PartyHasItem("LIST")~ THEN REPLY @39  GOTO 12
 END
 
 IF ~~ THEN BEGIN 11 // from: 10.0
-  SAY ~Very well give it here. I do not remember signing you up for the expedition, though. As the official financier of the expedition, the decision of who goes and who stays will be up to me, not that oaf Hrothgar. Do you understand?~ 
-  IF ~~ THEN REPLY ~I hate to break this to you, but Hrothgar is leading the expedition. You are just a shopkeeper... and a rude one at that.~  GOTO 2
-  IF ~~ THEN REPLY ~I see. As the "official financier" of the expedition, will you be coming along with us to Kuldahar?~  GOTO 15
-  IF ~~ THEN REPLY ~Whatever, little man. Just show me what you have for sale.~  GOTO 2
+  SAY @40 
+  IF ~~ THEN REPLY @41  GOTO 2
+  IF ~~ THEN REPLY @42  GOTO 15
+  IF ~~ THEN REPLY @43  GOTO 2
 END
 
 IF ~~ THEN BEGIN 12 // from: 10.1
-  SAY ~You lost the supply list?  Well, I suggest you go and find it. I cannot afford to have such incompetents on my expedition. As the official leader and chief financier of the expedition, the decision of who goes and who stays will be up to me, not that oaf Hrothgar. Do you understand?~ 
-  IF ~~ THEN REPLY ~I thought Hrothgar was leading the expedition. You're just a shopkeeper... and a rude one at that.~  GOTO 2
-  IF ~~ THEN REPLY ~*You're* leading the expedition?  Great. We're all gonna die.~  GOTO 2
-  IF ~~ THEN REPLY ~Whatever, little man. Just show me what you have for sale.~  GOTO 2
+  SAY @44 
+  IF ~~ THEN REPLY @45  GOTO 2
+  IF ~~ THEN REPLY @46  GOTO 2
+  IF ~~ THEN REPLY @43  GOTO 2
 END
 
 IF ~NumTimesTalkedToGT(0)
@@ -102,30 +102,30 @@
 !InParty("Erevain")
 Dead("Erevain")
 Global("VP_Talk_to_Erevain","LOCALS",1)~ THEN BEGIN 13 // from:
-  SAY ~You again. Why must you continuously pester me?  I have much to do and little time to trifle with the likes of you.~ 
+  SAY @47 
   IF ~Global("Supply_Quest", "GLOBAL", 1)
-PartyHasItem("LIST")~ THEN REPLY ~I have come to deliver the supply list from Hrothgar.~  DO ~TakePartyItem("LIST")
+PartyHasItem("LIST")~ THEN REPLY @48  DO ~TakePartyItem("LIST")
 SetGlobal("Supply_Quest", "GLOBAL", 2)
 AddExperienceParty(3500)~ GOTO 14
-  IF ~~ THEN REPLY ~I would like to buy something.~  DO ~StartStore("EHPomab", Player1)~ EXIT
-  IF ~~ THEN REPLY ~Sorry to disturb you. Farewell.~  EXIT
+  IF ~~ THEN REPLY @49  DO ~StartStore("EHPomab", Player1)~ EXIT
+  IF ~~ THEN REPLY @50  EXIT
 END
 
 IF ~~ THEN BEGIN 14 // from: 13.0
-  SAY ~Very well, give it here. Now, be on your way. The expedition will be departing shortly. It would be a shame if they left without you.~ 
-  IF ~~ THEN REPLY ~One moment. I would like to buy something.~  DO ~StartStore("EHPomab", Player1)~ EXIT
-  IF ~~ THEN REPLY ~Whatever, Pomab. Farewell.~  EXIT
+  SAY @51 
+  IF ~~ THEN REPLY @52  DO ~StartStore("EHPomab", Player1)~ EXIT
+  IF ~~ THEN REPLY @53  EXIT
 END
 
 IF ~~ THEN BEGIN 15 // from: 11.1
-  SAY ~Of course not. Someone has to stay behind to run this town. Without my leadership, the people of this town would not last through the winter.~ 
-  IF ~~ THEN REPLY ~Leadership?  It looks like you can barely run your own shop, let alone the whole town.~  GOTO 2
-  IF ~~ THEN REPLY ~Whatever, little man. Just show me what you have for sale.~  GOTO 2
-  IF ~~ THEN REPLY ~Right. Farewell, Pomab.~  EXIT
+  SAY @54 
+  IF ~~ THEN REPLY @55  GOTO 2
+  IF ~~ THEN REPLY @43  GOTO 2
+  IF ~~ THEN REPLY @56  EXIT
 END
 
 IF ~~ THEN BEGIN 16 // from: 1.3
-  SAY ~I said you're barbarians. And of course you're far away from the south.~ 
+  SAY @57 
   IF ~~ THEN DO ~SetGlobal("Know_Pomab","GLOBAL",1)~ GOTO 5
 END
 
@@ -134,21 +134,21 @@
 Global("VP_Talk_to_Erevain","LOCALS",0)
 InParty("Erevain")
 !Dead("Erevain")~ THEN BEGIN 17
-  SAY ~You again. Why must you continuously pester me?  I have much to do and little time to trifle with the likes of you.~ 
+  SAY @47 
   IF ~~ THEN EXTERN ~EREVAINJ~ 11
 END
 
 IF ~~ THEN BEGIN 18
-  SAY ~A pretentious comment from an elf so far up North near the Spine of the World. I have a royal purpose here. You merely have a dangerous wanderlust.~
+  SAY @58
   IF ~~ THEN EXTERN ~EREVAINJ~ 12
 END
 
 IF ~~ THEN BEGIN 19
-  SAY ~Yes, I miss the sand between my toes, the Memnon wines, but most of all the women. There's not a single trollop let alone a harem anywhere up here!~
+  SAY @59
   IF ~~ THEN EXTERN ~EREVAINJ~ 13
 END
 
 IF ~~ THEN BEGIN 20
-  SAY ~Perhaps... (Sigh)... Perhaps.~
+  SAY @60
   IF ~~ THEN DO ~SetGlobal("VP_Talk_to_Erevain","LOCALS",1)~ EXIT
 END
