--- NeJE\dlg\DREDTOE.D	Mon Jan 19 15:54:40 2015
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\DREDTOE.D	Tue Apr 09 19:35:53 2019
@@ -1,73 +1,73 @@
 BEGIN ~DREDTOE~ 87289 // non-zero flags may indicate non-pausing dialogue
 
 IF ~NumTimesTalkedTo(0)~ THEN BEGIN 0 // from:
-  SAY ~Move out of way!  ~ 
-  IF ~~ THEN REPLY ~Whoa... hold on a second.  I have a few questions for you.~  GOTO 1
-  IF ~~ THEN REPLY ~How rude.  I think it's time someone taught you some manners.~  GOTO 2
-  IF ~~ THEN REPLY ~Whatever you say, big fella.  The road is yours.  Farewell.~  DO ~SetGlobal("Red_Escape","GLOBAL",1)~ EXIT
+  SAY @0 
+  IF ~~ THEN REPLY @1  GOTO 1
+  IF ~~ THEN REPLY @2  GOTO 2
+  IF ~~ THEN REPLY @3  DO ~SetGlobal("Red_Escape","GLOBAL",1)~ EXIT
 END
 
 IF ~~ THEN BEGIN 1 // from: 0.0
-  SAY ~Red Toe has no time to speak with puny peoples.  Must run away.~ 
-  IF ~CheckStatLT(Player1,10,INT)~ THEN REPLY ~Red Toe?  Who is Red Toe?~  GOTO 3
-  IF ~CheckStatGT(Player1,9,INT)~ THEN REPLY ~What kind of name is Red Toe?~  GOTO 4
-  IF ~~ THEN REPLY ~Run away from what?~  GOTO 5
-  IF ~~ THEN REPLY ~As you wish.  Farewell.~  DO ~SetGlobal("Red_Escape","GLOBAL",1)~ EXTERN ~HROTHJ~ 118
+  SAY @4 
+  IF ~CheckStatLT(Player1,10,INT)~ THEN REPLY @5  GOTO 3
+  IF ~CheckStatGT(Player1,9,INT)~ THEN REPLY @6  GOTO 4
+  IF ~~ THEN REPLY @7  GOTO 5
+  IF ~~ THEN REPLY @8  DO ~SetGlobal("Red_Escape","GLOBAL",1)~ EXTERN ~HROTHJ~ 118
 END
 
 IF ~~ THEN BEGIN 2 // from: 0.1
-  SAY ~Red Toe no need *man-nurz.*  Only needs to run away.  Now move out of way or Red Toe squish you all.~ 
-  IF ~~ THEN REPLY ~Run away from what?~  GOTO 5
-  IF ~CheckStatLT(Player1,10,INT)~ THEN REPLY ~Red Toe?  Who is Red Toe?~  GOTO 3
-  IF ~CheckStatGT(Player1,9,INT)~ THEN REPLY ~What kind of name is Red Toe?~  GOTO 4
-  IF ~~ THEN REPLY ~You're not too bright, are you?~  GOTO 8
-  IF ~~ THEN REPLY ~Sorry, but the world will be a much better place without you in it.  Hold still... this will only hurt for a moment.~  DO ~Enemy()
+  SAY @9 
+  IF ~~ THEN REPLY @7  GOTO 5
+  IF ~CheckStatLT(Player1,10,INT)~ THEN REPLY @5  GOTO 3
+  IF ~CheckStatGT(Player1,9,INT)~ THEN REPLY @6  GOTO 4
+  IF ~~ THEN REPLY @10  GOTO 8
+  IF ~~ THEN REPLY @11  DO ~Enemy()
 Attack(Player1)~ EXIT
 END
 
 IF ~~ THEN BEGIN 3 // from: 2.1 1.0
-  SAY ~Me Red Toe.  Great verbeeg warrior.~ 
-  IF ~~ THEN REPLY ~What kind of name is Red Toe?~  GOTO 4
-  IF ~~ THEN REPLY ~If you're such a great warrior, why are you fleeing this place in such a hurry?~  GOTO 5
-  IF ~~ THEN REPLY ~It's a pleasure to meet you Red Toe.  Now, it's going to be an even greater pleasure to kill you.~  DO ~Enemy()
+  SAY @12 
+  IF ~~ THEN REPLY @6  GOTO 4
+  IF ~~ THEN REPLY @13  GOTO 5
+  IF ~~ THEN REPLY @14  DO ~Enemy()
 Attack(Player1)~ EXIT
 END
 
 IF ~~ THEN BEGIN 4 // from: 3.0 2.2 1.1
-  SAY ~When Red Toe was child, he drops big rock on foot.  Rock smashes foot and makes him yell.  Other verbeeg laugh and call him Red Toe.  ~ 
-  IF ~~ THEN REPLY ~I see.  Why are you in such a hurry?  What are you running from?~  GOTO 5
-  IF ~~ THEN REPLY ~You're not too bright, are you?~  GOTO 8
+  SAY @15 
+  IF ~~ THEN REPLY @16  GOTO 5
+  IF ~~ THEN REPLY @10  GOTO 8
 END
 
 IF ~~ THEN BEGIN 5 // from: 4.0 3.1 2.0 1.2
-  SAY ~Red Toe running from holy place.  He runs so he no die like others.~ 
-  IF ~~ THEN REPLY ~Holy Place?  You mean the temple over yonder?~  GOTO 6
-  IF ~~ THEN REPLY ~What others?  What are you talking about?~  GOTO 7
+  SAY @17 
+  IF ~~ THEN REPLY @18  GOTO 6
+  IF ~~ THEN REPLY @19  GOTO 7
 END
 
 IF ~~ THEN BEGIN 6 // from: 8.1 7.0 5.0
-  SAY ~Yes.  Red Toe was guard for friend priests in holy place.  But then bad priests come.  Begin killing friend priests and other verbeeg guards with hurtful magic.  But Red Toe smart.  He falls on ground and makes bad priests think he's dead.  When they gone, Red Toe gets up and runs from holy place.~ 
-  IF ~~ THEN REPLY ~Tell me about these "bad priests."  Why did they attack the temple?~  GOTO 9
-  IF ~~ THEN REPLY ~You're nothing but a cowardly deserter... you don't deserve to live.  Die!~  DO ~Enemy()
+  SAY @20 
+  IF ~~ THEN REPLY @21  GOTO 9
+  IF ~~ THEN REPLY @22  DO ~Enemy()
 Attack(Player1)~ EXIT
-  IF ~~ THEN REPLY ~Pretty smart... for a big barbarian ass.  Off you go then, but don't let me catch you anywhere near Kuldahar.~  DO ~SetGlobal("Red_Escape","GLOBAL",1)~ EXTERN ~HROTHJ~ 118
+  IF ~~ THEN REPLY @23  DO ~SetGlobal("Red_Escape","GLOBAL",1)~ EXTERN ~HROTHJ~ 118
 END
 
 IF ~~ THEN BEGIN 7 // from: 8.0 5.1
-  SAY ~Others.  Inside holy place.  Red Toe not go back there.  Not safe for verbeeg anymore.~ 
-  IF ~~ THEN REPLY ~Holy Place?  You mean the temple over yonder?~  GOTO 6
-  IF ~~ THEN REPLY ~No place is safe for cowardly deserters while we're around. Time for you to die, brute!~  DO ~Enemy()
+  SAY @24 
+  IF ~~ THEN REPLY @18  GOTO 6
+  IF ~~ THEN REPLY @25  DO ~Enemy()
 Attack(Player1)~ EXIT
-  IF ~~ THEN REPLY ~Very well.  Off you go then, but don't let me catch you anywhere near Kuldahar.~  DO ~SetGlobal("Red_Escape","GLOBAL",1)~ EXTERN ~HROTHJ~ 118
+  IF ~~ THEN REPLY @26  DO ~SetGlobal("Red_Escape","GLOBAL",1)~ EXTERN ~HROTHJ~ 118
 END
 
 IF ~~ THEN BEGIN 8 // from: 4.1 2.3
-  SAY ~Red Toe very smart!  That why he not dead like others inside holy place.  That why he runs now.~ 
-  IF ~~ THEN REPLY ~What others?  What are you talking about?~  GOTO 7
-  IF ~~ THEN REPLY ~Holy Place?  You mean the temple over yonder?~  GOTO 6
+  SAY @27 
+  IF ~~ THEN REPLY @19  GOTO 7
+  IF ~~ THEN REPLY @18  GOTO 6
 END
 
 IF ~NumTimesTalkedToGT(0)~ THEN BEGIN 9 // from: 6.0
-  SAY ~No!  Red Toe done talking!  Now he runs.  Runs away before bad priests come back and kill him too!~ 
+  SAY @28 
   IF ~~ THEN DO ~SetGlobal("Red_Escape","GLOBAL",1)~ EXTERN ~HROTHJ~ 118
 END
