--- NeJE\dlg\Durbin.d	Thu Mar 29 18:15:30 2018
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\Durbin.d	Tue Apr 09 19:35:54 2019
@@ -1,116 +1,116 @@
 BEGIN ~DURBIN~
 
 IF ~NumTimesTalkedTo(0)~ THEN BEGIN D1
-	SAY ~Well met friends, I am Durbin Vizenge, I am indebted to you for rescuing me for I was surely about to be devoured by those creatures.~
+	SAY @0
 	IF ~~ THEN DO ~SetGlobal("VP_Know_Durbin","GLOBAL",1)~ GOTO D1a
 END
 
 IF ~~ THEN BEGIN D1a
-	SAY ~I fear I don’t have anything I can give you in the way of a reward but if you look around I am sure you will find trinkets of value that those beasts have taken from other not so fortunate souls.~
+	SAY @1
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",0)
 Global("FirkraagDeal","GLOBAL",0)
 Global("Firmag01Move","GLOBAL",0)
-GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY ~Tell me Durbin, do you also have business with Lord Firkraag or was it he who locked you in here with these beasts?~ GOTO D2
+GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY @2 GOTO D2
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",0)
 Global("FirkraagDeal","GLOBAL",0)
 Global("Firmag01Move","GLOBAL",0)
-GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY ~What have we here? Seeing your predicament you certainly can’t be another of Lord Firkraag’s spies sent to spy on me.~ GOTO D2
+GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY @3 GOTO D2
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",0)
 Global("FirkraagDeal","GLOBAL",0)
 Global("Firmag01Move","GLOBAL",0)
-GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY ~Then you’d better hope you have information worth trading for your life!~ GOTO D17
+GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY @4 GOTO D17
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",0)
 Global("FirkraagDeal","GLOBAL",0)
 Global("Firmag01Move","GLOBAL",0)
-GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY ~Well met to you Durbin. If you could tell me where Lord Firkraag is holding the child of Garren Windspear, I’ll deem that payment enough?~ GOTO D3
+GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY @5 GOTO D3
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",0)
 Global("FirkraagDeal","GLOBAL",0)
 Global("Firmag01Move","GLOBAL",0)
-GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY ~Well met to you Durbin. If you could tell me where Lord Firkraag  is hiding out, I’ll deem that payment enough?~ GOTO D5
+GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY @6 GOTO D5
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",0)
 Global("FirkraagDeal","GLOBAL",0)
 Global("Firmag01Move","GLOBAL",0)
 InParty("Sharteel")
 !Dead("Sharteel")
-GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY ~I have no need for payment... it’s all in a days work for an adventurer. You are free to go.~ EXTERN ~SHARTJ~ durbin1
-	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",1)~ THEN REPLY ~If you’ve come looking for Firkraag, you’re too late my friend... he’s dead.~ GOTO D19
-	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",1)~ THEN REPLY ~What have we here? If you’re a servant of the dragon, consider yourself relieved of your duties... he’s dead!~ GOTO D19
+GlobalLT("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY @7 EXTERN ~SHARTJ~ durbin1
+	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",1)~ THEN REPLY @8 GOTO D19
+	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",1)~ THEN REPLY @9 GOTO D19
 	IF ~OR(3)
 Global("FirkraagDeal","GLOBAL",1)
 GlobalGT("Firmag01Move","GLOBAL",0)
-Global("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY ~Then maybe I’ll take what little you do have in payment.~ GOTO D23
+Global("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY @10 GOTO D23
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",1)
 OR(3)
 Global("FirkraagDeal","GLOBAL",1)
 GlobalGT("Firmag01Move","GLOBAL",0)
 Global("VP_AngelosLegacy","GLOBAL",4)
 InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~I have no need for payment... it’s all in a days work for an adventurer. You are free to go.~ EXTERN ~SHARTJ~ durbin17
+!Dead("Sharteel")~ THEN REPLY @7 EXTERN ~SHARTJ~ durbin17
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",0)
 OR(3)
 Global("FirkraagDeal","GLOBAL",1)
 GlobalGT("Firmag01Move","GLOBAL",0)
 Global("VP_AngelosLegacy","GLOBAL",4)
 InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~I have no need for payment... it’s all in a days work for an adventurer. You are free to go.~ EXTERN ~SHARTJ~ durbin18
+!Dead("Sharteel")~ THEN REPLY @7 EXTERN ~SHARTJ~ durbin18
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",0)
 OR(3)
 Global("FirkraagDeal","GLOBAL",1)
 GlobalGT("Firmag01Move","GLOBAL",0)
-Global("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY ~So what are you, another of the dragon’s servants? I don’t see him rushing to save your skin from the Wolfweres.~ GOTO D25
+Global("VP_AngelosLegacy","GLOBAL",4)~ THEN REPLY @11 GOTO D25
 END
 
 IF ~~ THEN BEGIN D2
-	SAY ~I was looking for yo... ahem... intruders in the other room. Unfortunately for me, I didn’t notice when those beasts were thrown in and the door locked behind them.~
+	SAY @12
 =
-	~I have been in the service of Lord Firkraag for many years now, an honour you couldn’t possibly begin to understand. However, I don’t flatter myself as being important in my Lord’s eyes. Certainly not important enough for him to notice I was missing.~
+	@13
 =
-	~If in time he discovered he was a servant or two down, he’d merely replace them thinking they’d perished sometime over the years. So you see, until you arrived, I was resigned to ending up in the bellies of those beasts. Now, may I go?~
-	IF ~~ THEN REPLY ~I’ll consider it if you tell me where Garren Windspear’s child is being held?~ GOTO D3 //reply to D2 & D18
-	IF ~~ THEN REPLY ~Not before you tell me where this Lord Firkraag of yours is hiding out!~ GOTO D5 //reply to D2 & D18
-	IF ~~ THEN REPLY ~I don’t think so. I can’t have you warning your master that we are closing in on him now can I?~ GOTO D7 //reply to D2 & D18
+	@14
+	IF ~~ THEN REPLY @15 GOTO D3 //reply to D2 & D18
+	IF ~~ THEN REPLY @16 GOTO D5 //reply to D2 & D18
+	IF ~~ THEN REPLY @17 GOTO D7 //reply to D2 & D18
 END
 
 IF ~~ THEN BEGIN D3
-	SAY ~(*Smiles*) The child... of course. My Lord knew you’d come, so I doubt he’ll mind me telling you now... Go back the way you have just come and cross the wooden bridge, I warn you it is well guarded. There is a room on your right which leads down to another and that is where you will find the child.~
+	SAY @18
 = 
-	~Even if you overcome the guards I don’t believe my Lord would have entrusted the key to the child’s cell to them. For that you must confront my Lord himself... if you dare. Now if that is all, I’ll take my leave of you.~
-	IF ~~ THEN REPLY ~One more question, can you tell me where this Lord Firkraag of yours is hiding out?~ GOTO D4
-	IF ~~ THEN REPLY ~I don’t think so. I can’t have you warning your master that we are closing in on him now can I?~ GOTO D7
+	@19
+	IF ~~ THEN REPLY @20 GOTO D4
+	IF ~~ THEN REPLY @17 GOTO D7
 	IF ~InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~Get out of here, but I warn you, if you stand with your master against me I’ll not hesitate in taking the life I have just saved.~ EXTERN ~SHARTJ~ durbin11
+!Dead("Sharteel")~ THEN REPLY @21 EXTERN ~SHARTJ~ durbin11
 END
 
 IF ~~ THEN BEGIN D4
-	SAY ~Yes, I’m sure my Lord wouldn’t mind my disclosing his whereabouts, not now you are so close. My Lord Firkraag’s personal quarters are on the lower floor of this place. The entrance to my Lord’s quarters is in the room where the child is imprisoned.~
+	SAY @22
 = 
-	~Turn left as you walk into this room and there you will find steps leading to the lower floor. 	He almost certainly knows you are here and is expecting you. I hope you are as prepared for him. Now if that is all, I’ll take my leave of you.~
-	IF ~~ THEN REPLY ~I don’t think so. I can’t have you warning your master that we are closing in on him now can I?~ GOTO D7
+	@23
+	IF ~~ THEN REPLY @17 GOTO D7
 	IF ~InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~Get out of here, but I warn you, if you stand with your master against me I’ll not hesitate in taking the life I have just saved.~ EXTERN ~SHARTJ~ durbin11
+!Dead("Sharteel")~ THEN REPLY @21 EXTERN ~SHARTJ~ durbin11
 END
 
 IF ~~ THEN BEGIN D5
-	SAY ~(*Smiles*) I doubt you would have gotten this far if he hadn’t meant for you to, so I can safely say he wouldn’t mind my disclosing his whereabouts, now that you are so close.~
+	SAY @24
 =
-	~My Lord Firkraag’s personal quarters are on the lower floor of this place. Go back the way you have just come and cross the wooden bridge, I warn you it is well guarded. There is a room on your right which leads down to another.~
+	@25
 =
-	~Turn left as you walk into this room and there you will find steps leading to the lower floor. He almost certainly knows you are here and is expecting you. I hope you are as prepared for him. Now if that is all...~
-	IF ~~ THEN REPLY ~One more question, can you tell me where Garren Windspear’s child is being held?~ GOTO D6
-	IF ~~ THEN REPLY ~I don’t think so. I can’t have you warning your master that we are closing in on him now can I?~ GOTO D7
+	@26
+	IF ~~ THEN REPLY @27 GOTO D6
+	IF ~~ THEN REPLY @17 GOTO D7
 	IF ~InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~Get out of here, but I warn you, if you stand with your master against me I’ll not hesitate in taking the life I have just saved.~ EXTERN ~SHARTJ~ durbin11
+!Dead("Sharteel")~ THEN REPLY @21 EXTERN ~SHARTJ~ durbin11
 END
 
 IF ~~ THEN BEGIN D6
-	SAY ~The child... of course. You will find the child in the room that holds the entrance to my Lord’s private quarters. My Lord will have set guards around the child and even if you manage to overcome them, I don’t believe he would have entrusted the key to the child’s cell to them. For that you must confront my Lord himself... if you dare. Now if that is all...~
-	IF ~~ THEN REPLY ~I don’t think so. I can’t have you warning your master that we are closing in on him now can I?~ GOTO D7
+	SAY @28
+	IF ~~ THEN REPLY @17 GOTO D7
 	IF ~InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~Get out of here, but I warn you, if you stand with your master against me I’ll not hesitate in taking the life I have just saved.~ EXTERN ~SHARTJ~ durbin11
+!Dead("Sharteel")~ THEN REPLY @21 EXTERN ~SHARTJ~ durbin11
 END
 
 IF ~~ THEN BEGIN D7
-	SAY ~(*Sighs*) I can not hope to best you in a fight, and anyway I am tired. I die knowing that I have lived a much longer life than I would have, had I not sworn my fealty to my master. I trust that when you meet him you do nothing foolish and treat him with respect, as that may be the only way to you leaving here alive.~
+	SAY @29
 	IF ~OR(2)
 !InParty("Sharteel")
 Dead("Sharteel")~ THEN DO ~EscapeArea()~ EXIT
@@ -119,131 +119,131 @@
 END
 
 IF ~~ THEN BEGIN D8
-	SAY ~I’ll have you know that I been in the service of Lord Firkraag for many years now, an honour you couldn’t possibly begin to understand.~
+	SAY @30
 =
-	~I was looking for yo... intruders in the other room when, unfortunately for me, I didn’t notice that those beasts had been thrown in and the door locked behind them. I don’t flatter myself as being important in my Lord’s eyes. Certainly not important enough for him to notice I was missing.~
+	@31
 =
-	~So you see, until you arrived, I was resigned to ending up in the bellies of those beasts. Now, may I go?~
+	@32
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin2
 END
 
 IF ~~ THEN BEGIN D9
-	SAY ~If it’s information you want then I know very little. I wasn’t privy to his Lordships dealings. He is very private, solitary even, preferring to use his servants as his eyes and ears, but I can tell you, of the few visitors who ventured here, even fewer left.~
+	SAY @33
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin3
 END
 
 IF ~~ THEN BEGIN D10
-	SAY ~I may be old but I am not senile young woman! (*His brow furrows*) Dossan you say... yes, there was something a while back now. What was it?~
+	SAY @34
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin4
 END
 
 IF ~~ THEN BEGIN D11
-	SAY ~Threatening gestures won’t get you anywhere. You just have to be patient as it takes a little longer these days for my brain to recall past incidents that were of little consequence to me.~
+	SAY @35
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin5
 END
 
 IF ~~ THEN BEGIN D12
-	SAY ~(*Looking into Shar-Teel’s eyes, the old man shakes his head*) Sadly, oft’ an evil heart is masked by a beautiful face. I have come across many such faces but none with eyes that shine with the lust for blood in the way that yours do. A cruel heart surely beats within your breast.~
+	SAY @36
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin6
 END
 
 IF ~~ THEN BEGIN D13
-	SAY ~That won’t be necessary, things are coming back to me. I can’t be certain... I wasn’t here when the young man dropped by, but it is said that there are certain items my Lord keeps hidden well away from his usual hoard...~
+	SAY @37
 =
-	~Items, that for personal reasons, he didn’t want falling into the hands of those who seek to destroy him... or so I have heard. Could these be what you seek?~
+	@38
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin7
 END
 
 IF ~~ THEN BEGIN D14
-	SAY ~Alas, you have made this journey for nothing. They are no longer here. Sometime ago a servant had discovered his Lordship’s hidden cache. As it so happen, a short while later he subsequently failed in his service to Lord Firkraag, and fearing for his life he absconded with the lot.~
+	SAY @39
 =
-	~I remember the day well, his Lordship’s anger was such that this whole place shook and I feared it would topple to the ground... as he raged in search of the thief but he was long gone.~
+	@40
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin8
 END
 
 IF ~~ THEN BEGIN D15
-	SAY ~(*Squirming away from her sword*) Please, you’ll not learn anything if you kill me. I beg you, give me your word that you’ll set me free and I’ll tell you what I know?~
+	SAY @41
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin9
 END
 
 IF ~~ THEN BEGIN D16
-	SAY ~There... there have been reported sightings of him in the area of Umar Hills but he always manages to disappear before anyone can get close enough to apprehend him. With each new sighting and failure of capture that reached his Lordship’s ears, he could be heard raging and cursing enough to bring the walls down around us all. I... I swear, that is the truth.~
+	SAY @42
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin10
 END
 
 IF ~~ THEN BEGIN D17
-	SAY ~(*Sighs*) So, this rescue is just a reprieve. I escape the bellies of the beasts only to find myself at the mercy of another type of beast. Ah well, I was already resigned to my fate, do as you will.~
-	IF ~~ THEN REPLY ~Do you know where the coward Lord Firkraag is hiding out?~ GOTO D5
-	IF ~~ THEN REPLY ~How did you come to be locked in with those beasts?~ GOTO D2
-	IF ~~ THEN REPLY ~Lord Firkraag throw you to the wolves did he? Now what did you do to deserve that I wonder?~ GOTO D18
+	SAY @43
+	IF ~~ THEN REPLY @44 GOTO D5
+	IF ~~ THEN REPLY @45 GOTO D2
+	IF ~~ THEN REPLY @46 GOTO D18
 END
 
 IF ~~ THEN BEGIN D18
-	SAY ~I’ll have you know that I been in the service of Lord Firkraag for many years now and I was looking for yo... ahem... intruders in the other room.~
+	SAY @47
 =
-	~Unfortunately for me, I didn’t notice when those beasts were thrown in and the door locked behind them. I don’t flatter myself as being important in my Lord’s eyes. Certainly not important enough for him to notice I was missing.~
+	@48
 =
-	~So you see, until you arrived, I was resigned to ending up in the bellies of those beasts. Now, may I go?~
-	IF ~~ THEN REPLY ~I’ll consider it if you tell me where Garren Windspear’s child is being held?~ GOTO D3 //reply to D2 & D18
-	IF ~~ THEN REPLY ~Not before you tell me where this Lord Firkraag of yours is hiding out!~ GOTO D5 //reply to D2 & D18
-	IF ~~ THEN REPLY ~I don’t think so. I can’t have you warning your master that we are closing in on him now can I?~ GOTO D7 //reply to D2 & D18
+	@32
+	IF ~~ THEN REPLY @15 GOTO D3 //reply to D2 & D18
+	IF ~~ THEN REPLY @16 GOTO D5 //reply to D2 & D18
+	IF ~~ THEN REPLY @17 GOTO D7 //reply to D2 & D18
 END
 
 IF ~~ THEN BEGIN D19
-	SAY ~(*Shaking his head sadly*) I have long expected this day but it doesn’t make it any less painful. I knew that there would come a day when his arrogance would be his downfall. That he would, one day, challenge one too many young adventurers and one of those adventurers would be his death.~
+	SAY @49
 = 
-	~Still, it is a sad day when so magnificent and ancient a creature such as he dies. The knowledge he carried, gathered over the centuries... lost forever. Forgive me, I must go mourn my Lord.~
+	@50
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin12
 END
 
 IF ~~ THEN BEGIN D20
-	SAY ~(*Angrily*) What... my Lord lies dead and you question me about trinkets when you have clearly stolen all that he prized! You and your kind are no better than vultures... taking from the dead!~
+	SAY @51
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin13
 END
 
 IF ~~ THEN BEGIN D21
-	SAY ~(*Sneers*) Bah, why should I? You’ve already taken enough of my Lord’s possessions to satisfy your greed, not to mention the name you’ll have made for yourself when you tell your tale of how you slaughtered one of nature’s mightiest and most ancient of creatures! Isn’t that enough to satisfy your lust for glory?~
+	SAY @52
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin14
 END
 
 IF ~~ THEN BEGIN D21a
-	SAY ~(*He pales and starts squirming at the touch of her sword*) Please, you’ll not learn anything if you kill me. I beg you, give me your word that you’ll set me free and I’ll tell you what I know?~
+	SAY @53
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin15
 END
 
 IF ~~ THEN BEGIN D22
-	SAY ~There... there have been reported sightings of him in the area of Umar Hills but he always manages to disappear before anyone can get close enough to apprehend him. With each new sighting and failure of capture that reached his Lordship’s ears, he could be heard raging and cursing enough to bring the walls down around us all. I... I swear, that is the truth.~
+	SAY @42
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin16
 END
 
 IF ~~ THEN BEGIN D23
-	SAY ~(*Sighs*) So, this rescue is just a reprieve. I escape the bellies of the beasts only to find myself at the mercy of another type of beast. Ah well, I was already resigned to my fate, do as you will.~
+	SAY @43
 	IF ~OR(2)
 !InParty("Sharteel")
 Dead("Sharteel")~ THEN DO ~EscapeArea()~ EXIT
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",1)
 InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~I have no need for payment... it’s all in a days work for an adventurer. You are free to go.~ EXTERN ~SHARTJ~ durbin17
+!Dead("Sharteel")~ THEN REPLY @7 EXTERN ~SHARTJ~ durbin17
 	IF ~Global("SPRITE_IS_DEADfirkra02","GLOBAL",0)
 InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~I have no need for payment... it’s all in a days work for an adventurer. You are free to go.~ EXTERN ~SHARTJ~ durbin18
+!Dead("Sharteel")~ THEN REPLY @7 EXTERN ~SHARTJ~ durbin18
 END
 
 IF ~~ THEN BEGIN D24
-	SAY ~(*Haughtily*) You’ve met my Lord, seen his magnificence for yourself, felt the power surrounding him and yet you still live. I am guessing my Lord Firkraag has some future plans for you... you’ll get nothing from me.~
+	SAY @54
 	IF ~~ THEN EXTERN ~SHARTJ~ durbin19
 END
 
 IF ~~ THEN BEGIN D25
-	SAY ~I am and have been in the service of Lord Firkraag for many years now, an honour you couldn’t possibly begin to understand. However, I don’t flatter myself as being important in my Lord’s eyes. Certainly not important enough for him to notice I was missing.~
+	SAY @55
 =
-	~If in time he discovered he was a servant or two down, he’d merely replace them thinking they’d perished sometime over the years. So you see, until you arrived, I was resigned to ending up in the bellies of those beasts. Now, may I go?~
+	@14
 	IF ~OR(2)
 !InParty("Sharteel")
-Dead("Sharteel")~ THEN REPLY ~I’ve no quarrel with you, begone.~ DO ~EscapeArea()~ EXIT
+Dead("Sharteel")~ THEN REPLY @56 DO ~EscapeArea()~ EXIT
 	IF ~InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~I’ve no quarrel with you, begone.~ EXTERN ~SHARTJ~ durbin20
+!Dead("Sharteel")~ THEN REPLY @56 EXTERN ~SHARTJ~ durbin20
 	IF ~InParty("Sharteel")
-!Dead("Sharteel")~ THEN REPLY ~I don’t think so. I’m coming back for the dragon and dealing with you will leave me ones less servant hindering my goal.~ EXTERN ~SHARTJ~ durbin20
-	IF ~~ THEN REPLY ~(*Lie*) Your lord lays dead below, struck down by my hand. What say you now?~ GOTO D19
+!Dead("Sharteel")~ THEN REPLY @57 EXTERN ~SHARTJ~ durbin20
+	IF ~~ THEN REPLY @58 GOTO D19
 END
