--- NeJE\bdash\ThirdP\baf\eAR0311.baf	Sun Aug 26 22:39:03 2018
+++ C:\BWP Patchstudio\patched files\NeJE\bdash\ThirdP\baf\eAR0311.baf	Tue Apr 09 19:41:22 2019
@@ -20,7 +20,7 @@
 		SetGlobal("AranKeyJournalEntry","AR0311",1)
 		EraseJournalEntry(34183)
 		EraseJournalEntry(34187)
-		AddJournalEntry(~Chapter 3 Throughout my time in Athkatla and my adventures in and around the city, I have seen hints of some great struggle between the Shadow Thieves and some other source... a guild filled with vampires and perhaps even greater evils. I have decided to invest in none of them, and this is by far preferable to receiving 'help' from either vampires or thieves. I have assassinated Gaelan Bayle and obtained a key from the lair of the Shadowmaster of the Shadow Thieves, who controls the entirety of this powerful organization: Aran Linvail. As ominous as that may sound, I have now little other choice but to plan an assault on Aran's lair to find any clue in my search for Irenicus.~,INFO) 
+		AddJournalEntry(@0,INFO) 
 
 END
 
