--- NeJE\dlg\Govan.d	Mon Jan 19 15:54:58 2015
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\Govan.d	Tue Apr 09 19:36:06 2019
@@ -1,108 +1,104 @@
 BEGIN ~Govan~
 
 IF ~NumTimesTalkedTo(0)~ THEN BEGIN 0
-  SAY ~(*hic*) Who are you, and by what right do you interrupt a man enjoying quiet ale or two with his friends?~
-  IF ~~ THEN REPLY ~Govan Duvaine? You are Govan Duvaine, are you not?~ DO ~SetGlobal("VP_MetGovan","GLOBAL",1)~ GOTO 1
-  IF ~~ THEN REPLY ~I have business with one, Govan Duvaine? From the description given me by Samuel Thunderburp I think I have found my man.~ DO ~SetGlobal("VP_MetGovan","GLOBAL",1)~ GOTO 1
+  SAY @0
+  IF ~~ THEN REPLY @1 DO ~SetGlobal("VP_MetGovan","GLOBAL",1)~ GOTO 1
+  IF ~~ THEN REPLY @2 DO ~SetGlobal("VP_MetGovan","GLOBAL",1)~ GOTO 1
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~I could be him. (*hic*) Depends on who is asking and what you want!~
-  IF ~~ THEN REPLY ~I am looking for a girl goes by the name of Leina. I believe that is a name known to you?~ GOTO 2
-  IF ~~ THEN REPLY ~Who I am, is not important, what I want is! Tell me what you know of a girl named Leina?~ GOTO 2
+  SAY @3
+  IF ~~ THEN REPLY @4 GOTO 2
+  IF ~~ THEN REPLY @5 GOTO 2
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~I know lots of girls but none important enough to remember by name. Leina you say, pretty name. If she is as fetching as her name, perhaps I should be looking her up too!~
-  IF ~~ THEN REPLY ~I am not here to play games Duvaine. Here read this and then tell me you do not know any Leina.~ DO ~TakePartyItem("GovanLet")~ GOTO 3
-  IF ~~ THEN REPLY ~I am under the impression you have already *looked* her up, or at least attempted to, Duvaine. Here, is this not your handiwork?~ DO ~TakePartyItem("GovanLet")~ GOTO 3
-  IF ~~ THEN REPLY ~I am not amused by your smart answers Duvaine. This letter here tells me you know this Leina that I speak of.~ DO ~TakePartyItem("GovanLet")~ GOTO 3
+  SAY @6
+  IF ~~ THEN REPLY @7 DO ~TakePartyItem("GovanLet")~ GOTO 3
+  IF ~~ THEN REPLY @8 DO ~TakePartyItem("GovanLet")~ GOTO 3
+  IF ~~ THEN REPLY @9 DO ~TakePartyItem("GovanLet")~ GOTO 3
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~Oh, that Leina, stuck up wench! The name Duvaine not good enough for her and her no more than a minstrel singing to keep herself off the streets.~
+  SAY @10
   IF ~~ THEN DO ~GiveItem("GovanLet",[PC])~ GOTO 4
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~Mind you, with her talent, or should that be lack of talent, she would make more if she were on the streets.~
+  SAY @11
   IF ~IfValidForPartyDialog("Melora")~ THEN EXTERN ~MELORAJ~ 2
   IF ~!IfValidForPartyDialog("Melora")~ THEN GOTO 5
 END
 
 IF ~~ THEN BEGIN 5
-  SAY ~If you want to speak to that one, The Five Flagons is where you will find her.~
-  IF ~~ THEN REPLY ~Would I be right in thinking that your intentions towards this Leina were, how shall I put it, less than honourable, only she saw right through you? Made you angry did it?~ GOTO 6
-  IF ~~ THEN REPLY ~This Leina clearly recognised you for the cad you are and spurned your advances. How does that make you feel Duvaine?~ GOTO 6
-  IF ~~ THEN REPLY ~Poor spoilt, little rich boy, not used to being turned down by the ladies! Tell me Duvaine, what did you do when your words and flowers failed to seduce her?~ GOTO 6
+  SAY @12
+  IF ~~ THEN REPLY @13 GOTO 6
+  IF ~~ THEN REPLY @14 GOTO 6
+  IF ~~ THEN REPLY @15 GOTO 6
 END
 
 IF ~~ THEN BEGIN 6
-  SAY ~Not that it is any business of yours what my intentions are towards Leina, but I would have married her if she had said yes. In fact, I fully intend to ask her again.~
+  SAY @16
   IF ~~ THEN GOTO 7
 END
 
 IF ~~ THEN BEGIN 7
-  SAY ~She is pretty enough and a man could do worse. Besides, her father is an influential man, it will be a fine pairing between our families. Now, maybe it is time you told me what your interest in Leina is?~
-  IF ~~ THEN REPLY ~She seems to have up and disappeared without a word to a soul. Your letter and *gift* were found in her room and by the crumpled state of them it was clear your attentions were not welcome.~ GOTO 8
-  IF ~~ THEN REPLY ~She is missing Duvaine and from what I see of you, you are not the sort to take kindly when a lady says no to you.~ GOTO 8
-  IF ~~ THEN REPLY ~I have no interested in your affairs or your intentions towards this girl, but she is missing and the barkeep of The Five Flagons has offered me a generous reward if I can find her. So tell me what did you do with her when she refused your proposal, Duvaine?~ GOTO 12
+  SAY @17
+  IF ~~ THEN REPLY @18 GOTO 8
+  IF ~~ THEN REPLY @19 GOTO 8
+  IF ~~ THEN REPLY @20 GOTO 12
 END
 
 IF ~~ THEN BEGIN 8
-  SAY ~I know nothing of her disappearance! All I did was send a messenger with the letters and the flowers, I never even saw her myself.~
+  SAY @21
   IF ~~ THEN GOTO 9
 END
 
 IF ~~ THEN BEGIN 9
-  SAY ~When he returned with her refusal, I thought to leave it for a few days then try my luck again. I was not the only person to think this marriage convenient you know.~
+  SAY @22
   IF ~~ THEN GOTO 10
 END  
   
 IF ~~ THEN BEGIN 10
-  SAY ~It was Leina's father who suggested it to me, he had some notion that if she married me it would put a stop to her wanderings.~
+  SAY @23
   IF ~~ THEN GOTO 11
 END   
   
 IF ~~ THEN BEGIN 11
-  SAY ~He also knows she refused me because I sent the messenger to tell him, but I also sent word that I would not be giving up.~
+  SAY @24
   IF ~~ THEN GOTO 14
 END
 
 IF ~~ THEN BEGIN 12
-  SAY ~Now hold on a minute! I am not that short of female company to let one refusal get to me! When the messenger I had sent to Leina with my proposal returned with her refusal, my intentions were to wait a few days and try my luck again.~
+  SAY @25
   IF ~~ THEN GOTO 13
 END  
 
 IF ~~ THEN BEGIN 13
-  SAY ~The marriage was her father's idea you know, not mine! He had some notion that by marrying Leina off to me it would stop her wanderings. He also knew of her refusal because I sent the messenger with word that Leina had refused, but I also said to tell him that I would not be giving up.~
+  SAY @26
   IF ~~ THEN GOTO 14
 END
 
 IF ~~ THEN BEGIN 14
-  SAY ~Maybe he is the one you should be speaking to and not me!~
-  IF ~~ THEN REPLY ~I may just do that Duvaine. Where might I find this father of Leina?~ GOTO 15
-  IF ~~ THEN REPLY ~I will, Duvaine, but if I do not get the answers I expect from him, I will return with more questions for you. Now where do I find this man?~ GOTO 15
-  IF ~~ THEN REPLY ~Tell me where to find this man, Duvaine, and you had better hope I find Leina safe there with him or I will be back to deal with you further.~ GOTO 15
+  SAY @27
+  IF ~~ THEN REPLY @28 GOTO 15
+  IF ~~ THEN REPLY @29 GOTO 15
+  IF ~~ THEN REPLY @30 GOTO 15
 END
 
 IF ~~ THEN BEGIN 15
-  SAY ~Old man Vulova! I thought everyone knew where to find him. You new to the city? No matter, he has an estate over in the government district. His business takes him away a lot, but he still keeps the house maintained for when he is in Athkatla.~
+  SAY @31
   IF ~~ THEN GOTO 16
 END
   
 IF ~~ THEN BEGIN 16
-  SAY ~Since the death of his wife, Leina's mother, he spends less and less time there. I think he keeps it more as a shrine to her than a home.~
-  IF ~~ THEN REPLY ~Then I will pay this Vulova a visit. In the meantime you had not better get any ideas of not being here should I return, because I will find you and believe me, you will wish you had stayed here when I do!~ UNSOLVED_JOURNAL ~Trouble at The Five Flagons Inn.
-
-I found Govan Duvaine in The Copper Coronet... he strikes me as a typical spoilt little rich boy, not used to being said no to. Whereas he admits to writing the letter, he says he knows nothing of her whereabouts... he suggests I speak with her father. I will find him at the Vulova estate in the government district... apparently it was he who suggested Duvaine should court his daughter in order to keep her in Athkatla.~ GOTO 17
-  IF ~~ THEN REPLY ~I will speak with Vulova and for your sake, I hope that I find this Leina is indeed with her father, and that I find her safe!~ UNSOLVED_JOURNAL ~Trouble at The Five Flagons Inn.
-
-I found Govan Duvaine in The Copper Coronet... he strikes me as a typical spoilt little rich boy, not used to being said no to. Whereas he admits to writing the letter, he says he knows nothing of her whereabouts... he suggests I speak with her father. I will find him at the Vulova estate in the government district... apparently it was he who suggested Duvaine should court his daughter in order to keep her in Athkatla.~ GOTO 17
+  SAY @32
+  IF ~~ THEN REPLY @33 UNSOLVED_JOURNAL @34 GOTO 17
+  IF ~~ THEN REPLY @35 UNSOLVED_JOURNAL @34 GOTO 17
 END
 
 IF ~~ THEN BEGIN 17
-  SAY ~So good luck in meeting Vulova and finding the girl.~
+  SAY @36
   IF ~~ THEN DO ~SetGlobal("VP_ResqueLeina","GLOBAL",8)~ EXIT
 END 
 
