--- NeJE\dlg\HROTHP.D	Sat Sep 22 19:27:05 2018
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\HROTHP.D	Tue Apr 09 19:36:19 2019
@@ -1,51 +1,51 @@
 BEGIN ~HROTHP~
 
 IF ~Global("KickedOut","LOCALS",1)~ THEN BEGIN 0
-  SAY ~Well met my friend. Is it coincidence that bring you past my way or are you seeking my company on the road again?~ [HROT_L11]
-  IF ~~ THEN REPLY ~Well met to you Hrothgar. It most certainly is not coincidence that I pass this way. I am here to ask if you will rejoin with the group, I would welcome your company once more?~ GOTO 5
-  IF ~~ THEN REPLY ~I have no need for you in the group as of yet Hrothgar, but seeing you stood there I thought to come over and greet you my friend.~ GOTO 4
+  SAY @0 
+  IF ~~ THEN REPLY @1 GOTO 5
+  IF ~~ THEN REPLY @2 GOTO 4
 END
 
 IF ~Global("KickedOut","LOCALS",0)~ THEN BEGIN 1
-  SAY ~I am not at all certain that this is the appropriate time for us to separate, <CHARNAME>. There is still a lot we can achieve together. Will you not think again?~ [HROT_L01]
-  IF ~~ THEN REPLY ~You are right of course Hrothgar, we will achieve more if we continue together. Let us get back to the road, together.~ GOTO 3
-  IF ~~ THEN REPLY ~I do not need to think again Hrothgar. I have made my decision and I expect you to respect it.~ GOTO 7
+  SAY @3 
+  IF ~~ THEN REPLY @4 GOTO 3
+  IF ~~ THEN REPLY @5 GOTO 7
 END
 
 IF ~HappinessLT(Myself,-299)~ THEN BEGIN 2
-  SAY ~I have tolerated your evil and sickening ways long enough <CHARNAME>. You have been a great help to me but I can no longer remain with this group for fear of being seen to condone your actions.~ [HROT_L07]
-  IF ~~ THEN REPLY ~I see no need to be other than I am Hrothgar. I am the same person I was when you elected to have me join you in Easthaven. Did you think to change me?~ GOTO 11
-  IF ~~ THEN REPLY ~I am what I am Hrothgar. You were pleased enough to have my company when we left Easthaven, why choose to judge me now?~ GOTO 12
-  IF ~~ THEN REPLY ~I made no pretense as to what my goals were when first me met Hrothgar. Is it my fault you chose to overlook them for the sake of getting your expedition together?~ GOTO 13
+  SAY @6 
+  IF ~~ THEN REPLY @7 GOTO 11
+  IF ~~ THEN REPLY @8 GOTO 12
+  IF ~~ THEN REPLY @9 GOTO 13
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~Then together it is <CHARNAME>, let us continue as we were.~ [HROT_L02]
+  SAY @10 
   IF ~~ THEN DO ~SetGlobal("KickedOut","LOCALS",0)
 SetGlobal("VP_Break_NeJ_Plot","GLOBAL",0)
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~I had hoped your appearance meant we were to travel again <CHARNAME>, but I see that is not your wish. My time grows short in this place and I cannot say whether you will find me here again or not. So once again it is fare thee well and safe journey my friend.~ [HROT_L13]
+  SAY @11 
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN 5
-  SAY ~And I would welcome the chance to walk your road again <CHARNAME>. Come, you can tell me all that has been happening in my absence as we travel.~ [HROT_L12]
+  SAY @12 
   IF ~~ THEN DO ~SetGlobal("KickedOut","LOCALS",0)
 SetGlobal("VP_Break_NeJ_Plot","GLOBAL",0)
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN 6
-  SAY ~As you wish.  I will not remain here long, however.  A safe journey to you, friend.~
+  SAY @13
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN 7
-  SAY ~If this is farewell I shall continue alone and hopefully find my way back to my home. I will remain here but only for a brief time should you have need to find for me again. We may meet at some future time and we may not. I wish you luck in your search for your destiny <CHARNAME>. Fare thee well.~ [HROT_L03]
-  IF ~Global("VP_In_IWD_Part1","GLOBAL",1)~ THEN REPLY ~You may as well continue your own path Hrothgar. I cannot see me having a need to seek you out again any time soon. Farewell.~ GOTO 8
+  SAY @14 
+  IF ~Global("VP_In_IWD_Part1","GLOBAL",1)~ THEN REPLY @15 GOTO 8
   IF ~Global("VP_In_IWD_Part1","GLOBAL",0)
 !AreaCheck("AR0301")
 !AreaCheck("AR0302")
@@ -67,34 +67,34 @@
 !AreaCheck("AR1303")
 !AreaCheck("AR0604")
 !AreaCheck("AR0605")
-!AreaCheck("AR0606")~ THEN REPLY ~Then it is here that I shall look should I have need of you again Hrothgar. Farewell my friend.~ GOTO 9
+!AreaCheck("AR0606")~ THEN REPLY @16 GOTO 9
   IF ~OR(5)
 AreaCheck("AR0601")
 AreaCheck("AR0602")
 AreaCheck("AR0603")
 Global("VP_In_IWD_Part1","GLOBAL",0)
-Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY ~It would be best if you were to make your way to one of the inns or taverns Hrothgar, it would be quicker for me to find you there than here.~ GOTO 10
+Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY @17 GOTO 10
   IF ~OR(3)
 Global("VP_In_IWD_Part1","GLOBAL",1)
 Global("VP_OnIsland","GLOBAL",1)
-Global("VP_InHalruaa","GLOBAL",1)~ THEN REPLY ~If you can, try and wait here. I‘ll pick up you later.~ GOTO 9
+Global("VP_InHalruaa","GLOBAL",1)~ THEN REPLY @18 GOTO 9
 END
 
 IF ~~ THEN BEGIN 8
-  SAY ~No one can ever be sure of what or whom he will have need of come tomorrow. I wish you luck on your road to your destiny <CHARNAME>. For now, I say fare thee well and keep safe.~ [HROT_L05]
+  SAY @19 
   IF ~~ THEN DO ~SetGlobal("KickedOut","LOCALS",1)
 EscapeAreaMove("TT1004",777,300,0)~ EXIT
 END
 
 IF ~~ THEN BEGIN 9
-  SAY ~I wish you luck on your road to your destiny <CHARNAME> but do not leave it too late to look me up should you have need. For now, I say fare thee well and keep safe my friend.~ [HROT_L04]
+  SAY @20 
   IF ~~ THEN DO ~SetGlobal("KickedOut","LOCALS",1)
 SetGlobalTimer("VP_Leaving_Timer","LOCALS",ONE_DAY)
 SetGlobal("VP_Break_NeJ_Plot","GLOBAL",1)~ EXIT
 END
 
 IF ~~ THEN BEGIN 10
-  SAY ~No matter where I wait <CHARNAME>, it will only be for a brief time, but if you say an inn or tavern will be more convenient, then that is where you will find me. Just do not leave it too late before you return. I wish you luck in your search for your destiny <CHARNAME>. For now I say fare thee well and keep safe my friend.~ [HROT_L06]
+  SAY @21 
   IF ~~ THEN  DO ~SetGlobal("KickedOut","LOCALS",1)
 SetGlobalTimer("VP_Leaving_Timer","LOCALS",TEN_DAYS)
 SetGlobal("VP_Break_NeJ_Plot","GLOBAL",1)
@@ -102,16 +102,16 @@
 END
 
 IF ~~ THEN BEGIN 11
-  SAY ~No <CHARNAME>, I did not think to change you. Not at first anyway, because then I did not realise how ruthless you were. Now as I continue to see the truth of your cruel nature, and how you revel in it, I cannot see a change forth coming anytime soon. My honour will not allow me to follow this evil path of yours any longer. So it is fare thee well <CHARNAME> and remember, we will all be judged by our deeds in time.~ [HROT_L08]
+  SAY @22 
   IF ~~ THEN DO ~EscapeArea()~ EXIT
 END
 
 IF ~~ THEN BEGIN 12
-  SAY ~I am not judging you <CHARNAME>, that will come later and from a far higher authority than I. I was mistaken when I thought I knew you when first I met you. Oh, I knew you were ruthless <CHARNAME>, but I had no idea how cruel your nature really was. It is because of the wickedness that I see in you now that my honour decrees I leave this group immediately. So I say fare thee well and you would do well to remember my words <CHARNAME>.~ [HROT_L09]
+  SAY @23 
   IF ~~ THEN DO ~EscapeArea()~ EXIT
 END
   
 IF ~~ THEN BEGIN 13
-  SAY ~It is true, I knew you joined my expedition to Kuldahar for your own gains but I had no idea how ruthless you were <CHARNAME>. The fact that I overlooked your cruel nature in the beginning does not mean I have to continue to watch your wickedness anymore. My honour will not allow me to follow this evil path of yours any longer. So it is fare thee well <CHARNAME> and remember, we will all be judged by our deeds in time.~ [HROT_L10]
+  SAY @24 
   IF ~~ THEN DO ~EscapeArea()~ EXIT
 END  
