--- NeJE\dlg\KAYL2.d	Sat Nov 04 23:54:07 2017
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\KAYL2.d	Tue Apr 09 19:36:30 2019
@@ -26,7 +26,5 @@
 EraseJournalEntry(34246)
 ReputationInc(1)
 TakePartyItem("misc6i")
-GiveItemCreate("brac20",LastTalkedToBy,1,0,0)~ SOLVED_JOURNAL ~Seek out and investigate the Fallen Paladins.
-
-I haven't killed the rogue paladin, Anarg, but let him and his group of Fallen Paladins leave Athkatla. Nevertheless, I've returned Anarg's Cup to Ryan Trawl and had to lie to him to increase my good reputation, along with a significant amount of Sir Ryan's goodwill.~ EXIT
+GiveItemCreate("brac20",LastTalkedToBy,1,0,0)~ SOLVED_JOURNAL @0 EXIT
 END
