--- NeJE\dlg\KeldornM.d	Thu Mar 29 18:51:39 2018
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\KeldornM.d	Tue Apr 09 19:36:34 2019
@@ -1,23 +1,23 @@
 BEGIN ~KELDORNM~
 
 IF ~NumTimesTalkedTo(0)~ THEN BEGIN 0
-	SAY ~(For a while he stares at you with indifference in his eyes, not recognising you. Then suddenly recognition of whom it is that he sees in front of him flashes in his eyes) <CHARNAME>... you... you have died also? Ah, I failed to aid you... on the contrary, it appears that have I lured you to your doom. Forgive me my lady... forgive me, I have failed you!~ [ifyougo]
-		IF ~~ THEN REPLY ~(Though you try to suppress them, tears start falling down your cheeks) No, my friend, I am alive! Almighty Kelemvor led me to you and here I am. I beg you, please come with me, it's not your time to... to retire. I can't... I don't want to leave you in this dreadful place!~ GOTO 1
-		IF ~~ THEN REPLY ~Keldorn, my friend, my mentor! Do not blame yourself for the things that have not yet happened. I am alive, look, but without your help I really will die. I need your firm hand, your light spirit, your wisdom. Please, Keldorn, do not think that you are of no use!~ GOTO 2
-		IF ~~ THEN REPLY ~That is all you are regretting? You do not remember anything else? You do not remember that you have friends who need you, a mother who loves you and cares for you? Tell me, how will I look into her eyes... how will I explain to her that her son, the last count Firecam, has chosen eternal peace over his duties to his family? What words am I to find, Keldorn?~ GOTO 3
+	SAY @0 
+		IF ~~ THEN REPLY @1 GOTO 1
+		IF ~~ THEN REPLY @2 GOTO 2
+		IF ~~ THEN REPLY @3 GOTO 3
 END
 
 IF ~~ THEN BEGIN 1
-	SAY ~You are alive? Of course you are alive, you are crying. Don't, my lady, don't cry for me, I feel so right here, so peaceful. I am so happy that you came, I can say my last farewell to you. I swear <CHARNAME>, I will miss you, you are the last ray of light in my life.~
-		IF ~~ THEN REPLY ~(Choking back the tears) What r-ray? You don't remember a thing, do you Keldorn?!~ GOTO 9
-		IF ~~ THEN REPLY ~You remember how I cried on your chest, and you were saying it was too early to mourn you? (More softly) You... you said that I was the nicest girl you had ever met?~ GOTO 4
-		IF ~~ THEN REPLY ~You have forgotten about the war? You have forgotten about the siege of your own castle? What will your subjects think of its owner... that he left them at their most desperate hour? What if those creatures win? They will kill all, wipe out all in the search for that damned ring. They do not know that it is already in the hands of Kelemvor!~ GOTO 5
-		IF ~~ THEN REPLY ~You haven't even asked how it is I come to be here? Why don't you ask me how you died? Do you not wonder what has become of the ring of Cyric that you were trusted to keep and protect? I can answer you. I put on the ring and found myself in Baator, I talked to the guardian there, I went to Sigil, and Lord Kelemvor took the ring from me.~ GOTO 6
-		IF ~~ THEN REPLY ~Do you remember how you died Keldorn? A priest of Cyric cursed you with an unremovable curse to deprive the army of their commander! Yes, I have power enough to withstand the fight and enough dexterity not to get myself killed, but I lack your knowledge and experience to continue the defence. Anyway, your people will see that you are no longer with them them and they... they will be completely and utterly desolated, Keldorn!~ GOTO 10
+	SAY @4
+		IF ~~ THEN REPLY @5 GOTO 9
+		IF ~~ THEN REPLY @6 GOTO 4
+		IF ~~ THEN REPLY @7 GOTO 5
+		IF ~~ THEN REPLY @8 GOTO 6
+		IF ~~ THEN REPLY @9 GOTO 10
 END
 
 IF ~~ THEN BEGIN 2
-	SAY ~(With a sigh) You still need mentors, my child. Am I the only one who can teach you the sense and wisdom of this world? This is of course is my duty... then let it be as you wish.~
+	SAY @10
 		IF ~~ THEN DO ~SetGlobal("VP_KrRomanceActive","GLOBAL",3)
 SetGlobal("VP_Kr_CyricPlot","GLOBAL",9)
 ClearAllActions()
@@ -26,75 +26,75 @@
 END
 
 IF ~~ THEN BEGIN 3
-	SAY ~The family was always the last priority for me as you may remember, and my mother will understand perfectly. You have nothing to worry about.~
+	SAY @11
 		IF ~~ THEN DO ~SetGlobal("VP_KrRomanceActive","GLOBAL",3)
 SetGlobal("VP_Kr_CyricPlot","GLOBAL",9)
 ClearAllActions()
 StartCutSceneMode()
-StartCutScene("KRCut13")~ JOURNAL ~I had a vision: defence is breached, goblins and giants conquer Firecam's castle and kill everyone within. Keldorn is dead. I have only a void left in my soul. But I know that I must at least save Imoen. I must resist the dark power that is raging inside me, in memory of Keldorn. To him I own that much... Now, back to Athkatla.~ EXIT
+StartCutScene("KRCut13")~ JOURNAL @12 EXIT
 END
 
 IF ~~ THEN BEGIN 4
-	SAY ~(Looking at you... his eyes not blinking) I... I do remember. How you sat beside me guarding my sleep and trying to conceal it. I remember now, <CHARNAME>! You are right, I had forgotten. This water is the source of oblivion, it strips us of all sufferings without looking into its reasons.~
+	SAY @13
 		IF ~~ THEN GOTO K3
 END
 
 IF ~~ THEN BEGIN 5
-	SAY ~Yea gods, I had not thought of that! How did Kelemvor get hold of the ring?~
-		IF ~~ THEN REPLY ~(Quietly) I...I  put it on to save you, then I was in Baator, I talked to the guardian there, and I was returned to Sigil... This is a long story, Keldorn!~ GOTO 7
+	SAY @14
+		IF ~~ THEN REPLY @15 GOTO 7
 END
 
 IF ~~ THEN BEGIN 6
-	SAY ~Why did you put on the ring?!~
-		IF ~~ THEN REPLY ~To save you!~ GOTO 7
+	SAY @16
+		IF ~~ THEN REPLY @17 GOTO 7
 END
 
 IF ~~ THEN BEGIN 7
-	SAY ~(Shaking his head) Baator, guardian, Kelemvor... I can not take all of this in right now. However, you are right my lady, I need to return! You have done so much alone whilst I was away.~
+	SAY @18
 		IF ~~ THEN EXTERN ~KELEMVOR~ K3
 END
 
 IF ~~ THEN BEGIN 8
-	SAY ~Then it looks as though I have deserted the battlefield? Ah, how awry! You are right, we must finish that which we started. How could I leave you now! Come, my dear friend, we must move swiftly.~
+	SAY @19
 		IF ~~ THEN EXTERN ~KELEMVOR~ K3
 END
 
 IF ~~ THEN BEGIN 9
-	SAY ~No... I do not remember anything. I am very tired, <CHARNAME>, I've lost my family, my friends. No, I have nothing to return to. Please do not try to persuade me, do not disturb my peace. It galls me to have to admit that I failed to aid you, but such was the will of Gods. Farewell, <CHARNAME>, and please my lady... do not cry, I beg you.~
+	SAY @20
 		IF ~~ THEN DO ~SetGlobal("VP_KrRomanceActive","GLOBAL",3)
 SetGlobal("VP_Kr_CyricPlot","GLOBAL",9)
 ClearAllActions()
 StartCutSceneMode()
-StartCutScene("KRCut13")~ JOURNAL ~I had a vision: defence is breached, goblins and giants conquer Firecam's castle and kill everyone within. Keldorn is dead. I have only a void left in my soul. But I know that I must at least save Imoen. I must resist the dark power that is raging inside me, in memory of Keldorn. To him I own that much... Now, back to Athkatla.~ EXIT
+StartCutScene("KRCut13")~ JOURNAL @12 EXIT
 END
 
 IF ~~ THEN BEGIN 10
-	SAY ~Please, don't cry my lady.~
-		IF ~~ THEN REPLY ~(Your tears burn your cheeks with rage) I... I have gotten used to fighting beside you and you knowing that you will always cover my back... So tell me, why are you letting them win?! Why?!~ GOTO 8
+	SAY @21
+		IF ~~ THEN REPLY @22 GOTO 8
 END
 
 IF ~~ THEN BEGIN K3
-	SAY ~I have told you that it is not within my power to forsake your care, and I will repeat it... I still can not forsake it anymore than I may forsake you.~
+	SAY @23
 =
-~(Giving you his hand) I will go with you!~
+@24
 		IF ~~ THEN EXTERN ~KELEMVOR~ K3
 END
 
 IF ~~ THEN BEGIN K4
-	SAY ~No, I do not agree! A young girl must not sacrifice herself for my sake... for the sake of an old man! No my lady, I will stay here. You must not pay for my stupidity!~
+	SAY @25
 		IF ~~ THEN EXTERN ~KELEMVOR~ K5
 END
 
 IF ~~ THEN BEGIN K5
-	SAY ~I will become one of the fallen!~
+	SAY @26
 		IF ~~ THEN EXTERN ~KELEMVOR~ K6
 END
 
 IF ~~ THEN BEGIN K6
-	SAY ~If you are hesitating, then I am not. I will stay Lord Kelemvor, and my decision is final.~
+	SAY @27
 		IF ~~ THEN DO ~SetGlobal("VP_KrRomanceActive","GLOBAL",3)
 SetGlobal("VP_Kr_CyricPlot","GLOBAL",9)
 ClearAllActions()
 StartCutSceneMode()
-StartCutScene("KRCut13")~ JOURNAL ~I had a vision: defence is breached, goblins and giants conquer Firecam's castle and kill everyone within. Keldorn is dead. I have only a void left in my soul. But I know that I must at least save Imoen. I must resist the dark power that is raging inside me, in memory of Keldorn. To him I own that much... Now, back to Athkatla.~ EXIT
+StartCutScene("KRCut13")~ JOURNAL @12 EXIT
 END
