--- NeJE\dlg\LadyAlia.d	Sat Jan 30 23:44:01 2016
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\LadyAlia.d	Tue Apr 09 19:36:41 2019
@@ -3,26 +3,26 @@
 IF WEIGHT #0 ~NumTimesTalkedTo(0)
 Global("VP_Kr_TheWar","GLOBAL",8)
 Global("VP_Talk_To_Keldorn","LOCALS",1)~ THEN BEGIN 0
-	SAY ~Keldorn, my darling boy! How glad I am to see you... and you, my sweet <CHARNAME>. Keldorn has written of you many times, telling me how he could not have borne his sorrow without you. Those poor, sweet innocent girls...~
+	SAY @0
 =
-~(For a moment, a shadow passes over lady Alia's face as she recalls her grand daughters' fate... then she forced herself to smile at you and continues)~
+@1
 =
-~You know, my dear child, there was a young cleric from temple of Kelemvor asking for you. He left a package for you saying that it was very urgent. Marrius was very anxious that this should be delivered directly into your hands.~
+@2
 		IF ~~ THEN DO ~SetGlobal("VP_Kr_TheWar","GLOBAL",9)~ EXTERN ~KELDORJ~ kelemring_st0
 END
 
 IF WEIGHT #-1 ~Global("VP_Talk_To_Keldorn","LOCALS",2)~ THEN BEGIN 1
-	SAY ~I am so happy for you both! I never dared hope that I would live to see such a joyous day... come, let me give you both a hug!~
+	SAY @3
 		IF ~~ THEN EXTERN ~KELDORJ~ Alia0
 END
 
 IF ~~ THEN BEGIN 2
-	SAY ~No, not in the least! You made the right decision in choosing not to wait. Now I will not need to worry so much over you.~
+	SAY @4
 		IF ~~ THEN EXTERN ~KELDORJ~ Alia1
 END	
 	
 IF ~~ THEN BEGIN 3
-	SAY ~Because you will have a loving and caring wife beside you. Marriage will bring trust in each other, thus making you both stronger.~
+	SAY @5
 		IF ~~ THEN DO ~SetGlobal("VP_Talk_To_Keldorn","LOCALS",3)
 SetGlobal("VP_Castle_Banters","GLOBAL",1)
 ClearAllActions()
@@ -31,6 +31,6 @@
 END
 
 IF ~~ THEN BEGIN 4
-	SAY ~Keldorn! The ladies can manage on their own. Come, I need you son.~
+	SAY @6
 		IF ~~ THEN EXTERN ~FRIDA~ 3
 END	
