--- NeJE\dlg\LEONA.d	Sat Jan 23 20:38:58 2016
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\LEONA.d	Tue Apr 09 19:36:46 2019
@@ -1,27 +1,27 @@
 APPEND ~LEONA~
 
 IF ~~ THEN BEGIN KR0
-  SAY ~You left us and our mother for this... whore!~
+  SAY @0
   IF ~~ THEN EXTERN ~KELDO25J~ LoveOne1
 END
 
 IF ~~ THEN BEGIN KR1
-  SAY ~How dare you patronize us, daddy! You never thought about us, we were indifferent to you, you.~
+  SAY @1
   IF ~~ THEN EXTERN ~KELDO25J~ LoveOne2
 END  
   
 IF ~~ THEN BEGIN KR2
-  SAY ~Mommy?! So, you have remembered her? You have been away from home for months, but when Sir William began to watch after us, you came and made him leave! I wish Sir William were our father, not you. It is good that we were killed, and don't have to witness you marrying this...~
+  SAY @2
   IF ~~ THEN EXTERN ~VESPER~ KR1
 END   
   
 IF ~~ THEN BEGIN KR3
-  SAY ~You couldn't find our murderers, and you didn't even avenge us.~
+  SAY @3
   IF ~~ THEN EXTERN ~KELDO25J~ LoveOne3
 END    
   
 IF ~~ THEN BEGIN KR4
-  SAY ~It is not true! You were there, but you came back because of her! You did everything for her! You left us to die because of her.~
+  SAY @4
    IF ~~ THEN EXTERN ~KELDO25J~ LoveOne4
 END  
 END
