--- NeJE\dlg\LOVEONE6.D	Sat Feb 02 19:30:25 2008
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\LOVEONE6.D	Tue Apr 09 19:36:47 2019
@@ -1,26 +1,26 @@
 BEGIN ~LOVEONE6~
 
 IF ~NumTimesTalkedTo(0)~ THEN BEGIN 0
-  SAY ~Sime Linvail... I was afraid that I would never see you again.~
+  SAY @0
   IF ~~ THEN EXTERN ~SIMEJ~ TSTOB1
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~What is there to understand? I came to ask you why did you kill me?~
+  SAY @1
   IF ~~ THEN EXTERN ~SIMEJ~ TSTOB2
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~Recklessness... Darling, I only became reckless, when I understood that you would give your heart to anyone, but me.~
+  SAY @2
   IF ~~ THEN EXTERN ~SIMEJ~ TSTOB3
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~But you did not love me. Funny how it still hurts to see you with another <PRO_MANWOMAN>... just like it did in life.~
+  SAY @3
   IF ~~ THEN EXTERN ~SIMEJ~ TSTOB4
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~And I loved you madly. I loved you so much that I did not want to live if you did not return my feelings... so I chose to fight as I was a berserker... sooner or later I knew that the death will find me...~
+  SAY @4
   IF ~~ THEN EXTERN ~SIMEJ~ TSTOB5
 END
