--- NeJE\dlg\MAZZYJ.d	Sat Apr 07 16:20:43 2018
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\MAZZYJ.d	Tue Apr 09 19:36:50 2019
@@ -19,7 +19,7 @@
 APPEND ~MAZZYJ~
 
 IF ~~ THEN BEGIN NEJ205
-  SAY ~We should find this Govan and tell him of Leina's disappearance. If he loves her as deeply as his words imply, he will be distraught at such news and will surely know how to go about a search of this city.~
+  SAY @0
   IF ~True()~ THEN DO ~SetGlobal("VP_NPC_LeinaRoom","GLOBAL",1)~ EXIT
   IF ~InParty("Hrothgar")~ THEN EXTERN ~HROTHJ~ 47
   IF ~InParty("Valygar")~ THEN EXTERN ~VALYGARJ~ NEJ157
@@ -29,7 +29,7 @@
 END
 
 IF ~~ THEN BEGIN NEJ206
-  SAY ~I have sworn to help you find your Imoen, <CHARNAME>, and Mazzy Fenton does not go back on her word. If you travel to Ulcaster, then so do I my friend.~
+  SAY @1
   IF ~True()~ THEN EXTERN ~HROTHJ~ 60
   IF ~InParty("Jaheira")
 !Dead("Jaheira")~ THEN EXTERN ~JAHEIRAJ~ NEJ546
@@ -55,7 +55,7 @@
 END
 
 IF ~~ THEN BEGIN NEJ207
-  SAY ~This is a wondrous thing to be able to do. To travel to the past and undo what has already come to pass! (*sighs*) Oh Patrick, if only...~
+  SAY @2
   IF ~True()~ THEN EXTERN ~TILORN~ 19
   IF ~InParty("Minsc")
 !Dead("Minsc")~ THEN EXTERN ~MINSCJ~ NEJ261
@@ -74,7 +74,7 @@
 END
 
 IF ~~ THEN BEGIN tafficuh
-  SAY ~My friend, why could you not have been content? Being small is no disgrace, by deed alone it is possible to become 10ft. tall in the eyes of others.~
+  SAY @3
  IF ~True()~ THEN EXTERN ~TAFFICP~ umberhk3 
  IF ~InParty("Minsc")
 !Dead("Minsc")~ THEN EXTERN ~MINSCJ~ tafficuh
@@ -93,7 +93,7 @@
 END
 
 IF ~~ THEN BEGIN Kr_WarTk
-  SAY ~I will join the troops, <CHARNAME>, that is where my place is.~
+  SAY @4
   	IF ~True()~ THEN DO ~ClearAllActions()
 StartCutSceneMode()
 StartCutScene("KRCut04")~ EXIT
@@ -105,28 +105,28 @@
 END
 
 IF ~~ THEN BEGIN Kelem0
-  SAY ~How is this? <CHARNAME> has killed me and I could do nothing to prevent it...  not even a single scratch did I leave! Oh, Arvoreen, what kind of strength...~
-  	IF ~~ THEN REPLY ~Mazzy! Mazzy my dearest friend... can you ever forgive me? I swear I could not control myself!~ GOTO Kelem1
-  	IF ~~ THEN REPLY ~Sounds as though she is jealous of me. How nice.~ GOTO Kelem2
+  SAY @5
+  	IF ~~ THEN REPLY @6 GOTO Kelem1
+  	IF ~~ THEN REPLY @7 GOTO Kelem2
 END
 
 IF ~~ THEN BEGIN Kelem1
-  SAY ~Forgive? There's nothing to forgive. You could not help yourself and I would have done the same.~
-  	IF ~~ THEN REPLY ~You... you would have put the ring on?~ GOTO Kelem3
+  SAY @8
+  	IF ~~ THEN REPLY @9 GOTO Kelem3
 END
 
 IF ~~ THEN BEGIN Kelem2
-  SAY ~Jealous? Oh no. Why do I need strength that is uncontrollable, unpredictable? It's just that I had a chance to ascertain that Gods are truly invincible.~
-  	IF ~~ THEN REPLY ~Did you have any doubts?~ GOTO Kelem5
+  SAY @10
+  	IF ~~ THEN REPLY @11 GOTO Kelem5
 END
 
 IF ~~ THEN BEGIN Kelem3
-  SAY ~What's the ring got to do with it, <CHARNAME>? If our situations were reversed, I would never have forgiven myself if I had not tried all possible means to save my... my friend.~
-  	IF ~~ THEN REPLY ~Mazzy, you are right. How could I have ever doubted you!~ GOTO Kelem4
+  SAY @12
+  	IF ~~ THEN REPLY @13 GOTO Kelem4
 END
 
 IF ~~ THEN BEGIN Kelem4
-  SAY ~That said, let us hurry and leave this place!~
+  SAY @14
 	IF ~True()~ THEN EXTERN ~KELEMVOR~ 4A
   	IF ~InParty("Edwin")~ THEN EXTERN ~EDWINJ~ Kelem0
   	IF ~InParty("Kachiko")~ THEN EXTERN ~KACHIJ~ Kelem0
@@ -138,7 +138,7 @@
 END
 
 IF ~~ THEN BEGIN Kelem5
-  SAY ~I was wondering are you going to take me from here or what?~
-  	IF ~~ THEN REPLY ~Yes, of course. Come dearest friend Mazzy. We need to save Keldorn.~ GOTO Kelem4
+  SAY @15
+  	IF ~~ THEN REPLY @16 GOTO Kelem4
 END
 END
