--- NeJE\dlg\NALIAJ.d	Sat Dec 08 05:47:11 2018
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\NALIAJ.d	Tue Apr 09 19:36:56 2019
@@ -61,7 +61,7 @@
 APPEND ~NALIAJ~
 
 IF ~~ THEN BEGIN NEJ320
-  SAY ~I know the name on this letter. He is the son of one of Athkatla's influential families. My father has... did speak of the Duvaine's often, but I have no idea where it is in Athkatla they live.~
+  SAY @0
   IF ~True()~ THEN DO ~SetGlobal("VP_NPC_LeinaRoom","GLOBAL",1)~ EXIT
   IF ~InParty("Hrothgar")~ THEN EXTERN ~HROTHJ~ 47
   IF ~InParty("Valygar")~ THEN EXTERN ~VALYGARJ~ NEJ157
@@ -75,7 +75,7 @@
 END
 
 IF ~~ THEN BEGIN NEJ321
-  SAY ~I fail to see how visiting a ruin is helping the needy of Athkatla! Still I suppose there must needy people everywhere looking for aid, we may be of some help to someone on our journey to this Ulcaster.~
+  SAY @1
   IF ~True()~ THEN EXTERN ~HROTHJ~ 60
   IF ~InParty("Jaheira")
 !Dead("Jaheira")~ THEN EXTERN ~JAHEIRAJ~ NEJ546
@@ -103,7 +103,7 @@
 END
 
 IF ~~ THEN BEGIN NEJ322
-  SAY ~A wizards academy. No matter that this one is in the past, I just never dreamt I would get to see such a place!~
+  SAY @2
   IF ~True()~ THEN EXTERN ~TILORN~ 19
   IF ~InParty("Minsc")
 !Dead("Minsc")~ THEN EXTERN ~MINSCJ~ NEJ261
@@ -124,58 +124,58 @@
 END
 
 IF ~~ THEN BEGIN NEJ323
-  SAY ~I’m glad to be of help.~
+  SAY @3
   IF ~~ THEN DO ~SetGlobal("VP_Nalia_EnchQuest","GLOBAL",1)~ EXTERN ~XANH~ 23
 END
 
 IF ~~ THEN BEGIN NEJ324
-  SAY ~If we had the time I would like nothing better than to learn from a master, but they are specialists and I doubt they would tutor a mage generalist.~
+  SAY @4
   IF ~~ THEN EXTERN ~XANH~ 37
 END
 
 IF ~~ THEN BEGIN NEJ325
-  SAY ~I’m glad to be of help.~
+  SAY @3
   IF ~~ THEN DO ~SetGlobal("VP_Nalia_AbjQuest","GLOBAL",1)~ EXTERN ~TUPHIN~ 7
 END
 
 IF ~~ THEN BEGIN NEJ326
-  SAY ~I’m glad to be of help.~
+  SAY @3
   IF ~~ THEN DO ~SetGlobal("VP_Nalia_InvQuest","GLOBAL",1)~ EXTERN ~TUTHEOD~ 26
 END
 
 IF ~~ THEN BEGIN NEJ327
-  SAY ~I’m glad to be of help.~
+  SAY @3
   IF ~~ THEN DO ~SetGlobal("VP_Nalia_InvQuest","GLOBAL",1)~ EXTERN ~TUTHEOD~ 29
 END
 
 IF ~~ THEN BEGIN NEJ328
-  SAY ~I’m glad to be of help.~
+  SAY @3
   IF ~~ THEN DO ~SetGlobal("VP_Nalia_IllQuest","GLOBAL",1)~ EXTERN ~TUVASHON~ 17
 END
 
 IF ~~ THEN BEGIN NEJ329
-  SAY ~At last a chance to do more.~
+  SAY @5
   IF ~~ THEN DO ~SetGlobal("VP_Nalia_ConQuest","GLOBAL",1)~ EXTERN ~TUMORD~ 15
 END
 
 IF ~~ THEN BEGIN NEJ330
-  SAY ~Sorry, but I truly thought I was capable of more.~
-  IF ~~ THEN REPLY ~Do we need an intelligence check or something here? All you had to do was to read a couple of scrolls... you can read can’t you?~ GOTO NEJ331
-  IF ~!Class(Player1,MAGE_ALL)~ THEN REPLY ~I don’t understand a lot about magic but surely it has to be easier to read scrolls then trying to remember spells that have been memorised?~ GOTO NEJ331
+  SAY @6
+  IF ~~ THEN REPLY @7 GOTO NEJ331
+  IF ~!Class(Player1,MAGE_ALL)~ THEN REPLY @8 GOTO NEJ331
 END
 
 IF ~~ THEN BEGIN NEJ331
-  SAY ~Magic can be very unpredictable whether it’s read from parchment or a memorised spell. It takes but the slightest wrong move by a finger, a miss pronunciation of a word and the spell will lost or worse altered and the results are as I have said unpredictable.~
+  SAY @9
   IF ~~ THEN DO ~ActionOverride("TuMord",EscapeArea())~ EXIT
 END
 
 IF ~~ THEN BEGIN NEJ332
-  SAY ~I always hoped to be given the chance to do more.~
+  SAY @10
   IF ~~ THEN DO ~SetGlobal("VP_Nalia_TraQuest","GLOBAL",1)~ EXTERN ~TUMALV~ 18
 END
 
 IF ~~ THEN BEGIN NEJ333
-  SAY ~Dead is dead! He will never come back so what does it matter?~
+  SAY @11
 	IF ~True()~ THEN EXIT
 	IF ~InParty("Korgan")
 !Dead("Korgan")~ THEN EXTERN ~KORGANJ~ NEJ224
@@ -196,55 +196,55 @@
 END
 
 IF ~~ THEN BEGIN abjurer2
-  SAY ~I got it wrong, sorry, <CHARNAME>.~
-  IF ~~ THEN REPLY ~You call yourself a mage? Thanks to your incompetence, girl, we have lost some valuable prizes.~ GOTO abjurer3
-  IF ~~ THEN REPLY ~It happens, we win some we lose some, I am sure you gave it your best shot.~ GOTO abjurer4
+  SAY @12
+  IF ~~ THEN REPLY @13 GOTO abjurer3
+  IF ~~ THEN REPLY @14 GOTO abjurer4
 END
   
 IF ~~ THEN BEGIN abjurer3
-  SAY ~That’s unfair! Let me remind you that you can’t lose what you never had.~
+  SAY @15
   IF ~~ THEN EXTERN ~TUPHIN~ npcfail
 END
 	
 IF ~~ THEN BEGIN abjurer4
-  SAY ~You’re very gracious, <CHARNAME>.~
+  SAY @16
   IF ~~ THEN EXTERN ~TUPHIN~ npcfail
 END
 
 IF ~~ THEN BEGIN tranfail
-  SAY ~I got it wrong, sorry, <CHARNAME>.~
-  IF ~~ THEN REPLY ~You failed, a simple kill the monster and open the chest task! It isn’t as if you haven’t done it before.~ GOTO tranfai2
-  IF ~~ THEN REPLY ~It’s not that important. What is important is that Malvolio got you out safely.~ GOTO tranfai3
+  SAY @12
+  IF ~~ THEN REPLY @17 GOTO tranfai2
+  IF ~~ THEN REPLY @18 GOTO tranfai3
 END
 
 IF ~~ THEN BEGIN tranfai2
-  SAY ~The last time, if I recall correctly, <CHARNAME>, I had some muscle to back me up and that situation wasn’t dependent solely on magic.~
+  SAY @19
   IF ~~ THEN EXTERN ~TUMALV~ 24
 END  
 
 IF ~~ THEN BEGIN tranfai3
-  SAY ~Thank you for that, <CHARNAME>, but I am still disappointed that I didn’t do better.~
+  SAY @20
   IF ~~ THEN EXTERN ~TUMALV~ 24
 END
 
 IF ~~ THEN BEGIN invfail
-  SAY ~I got it wrong, sorry <CHARNAME>.~
-  IF ~~ THEN REPLY ~That was abysmal! With all the spells you have at your disposal, I can not believe that you couldn’t take care of a few pesky pests!~ GOTO invfail2
-  IF ~~ THEN REPLY ~It happens. I was probably expecting too much letting you go into a situation you hadn’t been given time to prepare for.~GOTO invfail3
+  SAY @21
+  IF ~~ THEN REPLY @22 GOTO invfail2
+  IF ~~ THEN REPLY @23GOTO invfail3
 END
 
 IF ~~ THEN BEGIN invfail2
-  SAY ~It was an unfair test <CHARNAME>, using me as an exterminator and not telling me the extent of the infestation. I did the best I could under such circumstances.~
+  SAY @24
   IF ~~ THEN EXTERN ~TUTHEOD~ failtest  
 END
 
 IF ~~ THEN BEGIN invfail3
-  SAY ~How many fights give us time to prepare <CHARNAME>? There are only so many situations a mage can handle on her own and then only if she has a wide enough selection of spells at her disposal.~
+  SAY @25
   IF ~~ THEN EXTERN ~TUTHEOD~ failtest  
 END
 
 IF ~~ THEN BEGIN tafficuh
-  SAY ~I... I cannot look at you Taffic without remembering that it was creatures such as you have become that helped the trolls burrow into my home and brought about the death of my father.~
+  SAY @26
  IF ~True()~ THEN EXTERN ~TAFFICP~ umberhk3 
  IF ~InParty("Minsc")
 !Dead("Minsc")~ THEN EXTERN ~MINSCJ~ tafficuh
@@ -265,23 +265,23 @@
 END
 
 IF ~~ THEN BEGIN vashon
-  SAY ~I am sorry, I know that I should have been able to have done more.~
-  IF ~~ THEN REPLY ~If I had a gold piece for each time I heard sorry for an excuse, I would be rich beyond your imaginings. Sorry is never an excuse for failure.~ GOTO vashon3
-  IF ~~ THEN REPLY ~No matter, you couldn’t have second guessed Vashon’s test, no one could... not even if you’d been a diviner.~ GOTO vashon4
+  SAY @27
+  IF ~~ THEN REPLY @28 GOTO vashon3
+  IF ~~ THEN REPLY @29 GOTO vashon4
 END
   
 IF ~~ THEN BEGIN vashon3
-  SAY ~See at as you like. It was not an excuse, it was an apology for no reason other than me feeling badly over letting you down.~
+  SAY @30
   IF ~~ THEN EXTERN ~TUVASHON~ npcfail
 END
 	
 IF ~~ THEN BEGIN vashon4
-  SAY ~Still, a mage should always try to be prepared for the unknown element in any situation she must face. I will remember that in future.~
+  SAY @31
   IF ~~ THEN EXTERN ~TUVASHON~ npcfail
 END
 
 IF ~~ THEN BEGIN Kr_WarTk
-  SAY ~We must all help, <CHARNAME>.~
+  SAY @32
   	IF ~True()~ THEN DO ~ClearAllActions()
 StartCutSceneMode()
 StartCutScene("KRCut04")~ EXIT
@@ -297,13 +297,13 @@
 END
 
 IF ~~ THEN BEGIN Kelem0
-  SAY ~Where am I? It's so cold here, so dreadfully cold!~
-	IF ~~ THEN REPLY ~You are dead, Nalia. I have killed you... you have to believe that it was unintentional. Please forgive me dearest Nalia... forgive me!~ GOTO Kelem1
-	IF ~~ THEN REPLY ~Nalia, that is a strange question! You are dead yes, but you are not going to let so little a thing upset you are you?~ GOTO Kelem2
+  SAY @33
+	IF ~~ THEN REPLY @34 GOTO Kelem1
+	IF ~~ THEN REPLY @35 GOTO Kelem2
 END
 
 IF ~~ THEN BEGIN Kelem1
-  SAY ~You have killed me? Ah yes, I remember. You put on the ring of Cyric and it made you insane. I do not blame you, <CHARNAME>, I can see that it was no fault of yours... please take me away from here!~
+  SAY @36
 	IF ~True()~ THEN EXTERN ~KELEMVOR~ 4A
   	IF ~InParty("Edwin")~ THEN EXTERN ~EDWINJ~ Kelem0
   	IF ~InParty("Kachiko")~ THEN EXTERN ~KACHIJ~ Kelem0
@@ -319,13 +319,13 @@
 END
 
 IF ~~ THEN BEGIN Kelem2
-  SAY ~You have killed me, and you call it a little thing?~
-	IF ~~ THEN REPLY ~Nalia... you know how it happened. It was that damned ring of Cyric! Please, I beg of you let us leave this place.~ GOTO Kelem3 
-	IF ~~ THEN REPLY ~You wanted adventure! You should have learnt to move quicker. The battle field is not a ball-room, it is all *quick quick quick*. There is no time for *quick, quick, slow* you know!~ GOTO Kelem4
+  SAY @37
+	IF ~~ THEN REPLY @38 GOTO Kelem3 
+	IF ~~ THEN REPLY @39 GOTO Kelem4
 END
 
 IF ~~ THEN BEGIN Kelem3
-  SAY ~Yes, let's go.~
+  SAY @40
 	IF ~True()~ THEN EXTERN ~KELEMVOR~ 4A
   	IF ~InParty("Edwin")~ THEN EXTERN ~EDWINJ~ Kelem0
   	IF ~InParty("Kachiko")~ THEN EXTERN ~KACHIJ~ Kelem0
@@ -341,7 +341,7 @@
 END
 
 IF ~~ THEN BEGIN Kelem4
-  SAY ~What the joke! But... yes. Let's go.~
+  SAY @41
 	IF ~True()~ THEN EXTERN ~KELEMVOR~ 4A
   	IF ~InParty("Edwin")~ THEN EXTERN ~EDWINJ~ Kelem0
   	IF ~InParty("Kachiko")~ THEN EXTERN ~KACHIJ~ Kelem0
