--- NeJE\dlg\PPSAEM.d	Fri Aug 12 22:01:32 2016
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\PPSAEM.d	Tue Apr 09 19:37:04 2019
@@ -5,45 +5,45 @@
 
 IF WEIGHT #7 ~Global("Start_Island","GLOBAL",1)
 Global("Saemon_Mistake","GLOBAL",0)~ THEN BEGIN TS3
-  SAY ~Ahem... dear <SIRMAAM> we had arrived it seems... ~
-  IF ~~ THEN REPLY ~Where? ~ DO ~SetGlobal("Saemon_Mistake","GLOBAL",1)~ GOTO TS4
+  SAY @0
+  IF ~~ THEN REPLY @1 DO ~SetGlobal("Saemon_Mistake","GLOBAL",1)~ GOTO TS4
 END
 
 IF ~~ THEN BEGIN TS4
-  SAY ~Uhm, that’s not Brinnlaw, I am positive... Maybe it is some part of Kara-Tur. You have couple locals among your ranks they would know better than I.~
+  SAY @2
   IF ~~ THEN EXTERN ~YOSHJ~ TS168
 END
 
 IF ~~ THEN BEGIN TS5
-  SAY ~We’d have to wait for the night to fall, to be able to tell our position from the stars. Meanwhile, the crew will inspect the ship and repair what we can. The journey ahead of us might be long and harsh.~
-  IF ~Global("ThiefGroup","GLOBAL",0)~ THEN REPLY ~Then I suggest we go ashore and see if we can find some provisions or any clues to where we are.~ DO ~SetGlobal("Island_War","GLOBAL",1)~ EXIT
+  SAY @3
+  IF ~Global("ThiefGroup","GLOBAL",0)~ THEN REPLY @4 DO ~SetGlobal("Island_War","GLOBAL",1)~ EXIT
   IF ~!InParty("Sime")
-Global("ThiefGroup","GLOBAL",1)~ THEN REPLY ~Then I suggest we go ashore and see if we can find some provisions or any clues to where we are. What about you, my trusty advisor?~ DO ~SetGlobal("Island_War","GLOBAL",1)~ EXTERN ~PPSIME~ TS0
+Global("ThiefGroup","GLOBAL",1)~ THEN REPLY @5 DO ~SetGlobal("Island_War","GLOBAL",1)~ EXTERN ~PPSIME~ TS0
   IF ~InParty("Sime")
-Global("ThiefGroup","GLOBAL",1)~ THEN REPLY ~Then I suggest we go ashore and see if we can find some provisions or any clues to where we are. What about you, my trusty advisor?~ DO ~SetGlobal("Island_War","GLOBAL",1)~ EXTERN ~SIMEJ~ 4
-  IF ~Global("ThiefGroup","GLOBAL",2)~ THEN REPLY ~Then I suggest we go ashore and see if we can find some provisions or any clues to where we are. What about you, Bodhi?~ DO ~SetGlobal("Island_War","GLOBAL",1)~ EXTERN ~PPBODHI3~ TS23
+Global("ThiefGroup","GLOBAL",1)~ THEN REPLY @5 DO ~SetGlobal("Island_War","GLOBAL",1)~ EXTERN ~SIMEJ~ 4
+  IF ~Global("ThiefGroup","GLOBAL",2)~ THEN REPLY @6 DO ~SetGlobal("Island_War","GLOBAL",1)~ EXTERN ~PPBODHI3~ TS23
 END
 
 IF WEIGHT #8 ~GlobalLT("Kachiko_Saved","GLOBAL",2)
 GlobalGT("Island_War","GLOBAL",0)~ THEN BEGIN TS6
-  SAY ~We need to cut more timber to repair the ship. Please, come back later.~
+  SAY @7
   IF ~~ THEN EXIT
 END
 
 IF WEIGHT #9 ~Global("VP_OnIsland","GLOBAL",1)
 Global("End_Island","GLOBAL",2)~ THEN BEGIN TS29
-  SAY ~The good news is that the ship is ready to sail. The bad news is that we are far to the south from Athkatla, and I am not sure if we will see Amn’s shores again.~
+  SAY @8
 =
-~I have no knowledge of these waters and I am reluctant to start out without more information... What is your command, my <PRO_LADYLORD>?~
-  IF ~~ THEN REPLY ~Well, I’d trade you my good news! I got a map, showing a magical portal, which can teleport our ship back to our lands, and a wardstone, which makes possible to restore our time-space continuum.~ DO ~SetGlobal("End_Island","GLOBAL",3)~ GOTO TS30
+@9
+  IF ~~ THEN REPLY @10 DO ~SetGlobal("End_Island","GLOBAL",3)~ GOTO TS30
 END
 
 IF ~~ THEN BEGIN TS30
-  SAY ~Magical portal? Time wardstone? Sounds just as good as the Fog of Fate spell... Let me see the sea charts you get...~
+  SAY @11
 =
-~Hmm, looks like it was made with skill. I have a feeling it’s genuine.~
+@12
 =
-~I suggest that we go to Athkatla and then sail to Brinnlaw in a more traditional way, because the area miles around Brinnlaw is heavily warded against magical travels... Rig up, crew!~
+@13
   IF ~Global("ThiefGroup","GLOBAL",0)~ THEN DO ~StartCutSceneMode()
 StartCutScene("Cut41krS")~ EXIT
   IF ~Global("ThiefGroup","GLOBAL",1)
