--- NeJE\bdash\Language\deutsch\replace.d	Thu Sep 29 17:40:28 2005
+++ C:\BWP Patchstudio\patched files\NeJE\bdash\Language\deutsch\replace.d	Tue Apr 09 19:34:40 2019
@@ -1,15 +1,15 @@
 //Ersetzt die Tokens von BRDBYEND.d
 
-REPLACE_SAY ~BRDBYEND~ 0 ~Willkommen in Ulgoth Bart, <SIRMAAM>! Es gehen hier neuerdings komische Geschichten um.~
-REPLACE_SAY ~BRDBYEND~ 4 ~Nehmt Euch nur de Zeit die Ihr braucht, redet mit den Leuten. Das ist alles was wir hoffen, alles was wir verlangen konnen. Schoner Tag noch <SIRMAAM>~
-REPLACE_SAY ~BRDBYEND~ 5 ~Und, wie laufen die Dinge Eurer Meinung nach hier so <SIRMAAM>?~
-REPLACE_SAY ~BRDBYEND~ 6 ~Also, was habt Ihr mir zum Sagen, <SIRMAAM>?~
-REPLACE_SAY ~BRDBYEND~ 7 ~Ist schon klar, <SIRMAAM>.  Wenn’s nur der Umhang ist, dann verstehe ich auch nicht warum.~
-REPLACE_SAY ~BRDBYEND~ 8 ~Ist schon klar, <SIRMAAM>. Gut, dass Ihr den Umhang zuruckgebracht habt.~
-REPLACE_SAY ~BRDBYEND~ 9 ~Gut, dass er weg ist, <SIRMAAM>. Er kroch schon unsere Armel hoch .~
-REPLACE_SAY ~BRDBYEND~ 13 ~Danke, dass Du das geklart hast <SIRMAAM>. Du wirst bei uns so was, wie ein Held sein. Die Leute haben schon uber Euch geredet.~
-REPLACE_SAY ~BRDBYEND~ 14 ~Willst uns sonst noch was erzahlen <SIRMAAM>?~
-REPLACE_SAY ~BRDBYEND~ 15 ~Eine gute Idee, <SIRMAAM>. I mein, wir sollten ihm sich selbst uberlassen. ~ 
-REPLACE_SAY ~BRDBYEND~ 17 ~Gut, dass Ihr Euch darum gekummert habt <SIRMAAM>, ansonsten hatten wir einen riesigen Arger am Hals.~
-REPLACE_SAY ~BRDBYEND~ 20 ~Ich wurde ihm auch nicht trauen, <SIRMAAM>. Du hast eine gute Entscheidung getroffen.~
-REPLACE_SAY ~BRDBYEND~ 22 ~Nun, <SIRMAAM>, Ich denke damit ist alles erledigt. Danke fur Eure Hilfe. Der Herzog gab mir diese Schriftrolle. Damit konnt Ihr Euch aus dem Palast teleportieren. Bitte schon. Und Danke noch mal.~
+REPLACE_SAY ~BRDBYEND~ 0 @0
+REPLACE_SAY ~BRDBYEND~ 4 @1
+REPLACE_SAY ~BRDBYEND~ 5 @2
+REPLACE_SAY ~BRDBYEND~ 6 @3
+REPLACE_SAY ~BRDBYEND~ 7 @4
+REPLACE_SAY ~BRDBYEND~ 8 @5
+REPLACE_SAY ~BRDBYEND~ 9 @6
+REPLACE_SAY ~BRDBYEND~ 13 @7
+REPLACE_SAY ~BRDBYEND~ 14 @8
+REPLACE_SAY ~BRDBYEND~ 15 @9 
+REPLACE_SAY ~BRDBYEND~ 17 @10
+REPLACE_SAY ~BRDBYEND~ 20 @11
+REPLACE_SAY ~BRDBYEND~ 22 @12
