--- NeJE\dlg\SOLA.D	Wed Dec 29 17:11:40 2004
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\SOLA.D	Tue Apr 09 19:37:18 2019
@@ -1,14 +1,11 @@
 BEGIN ~SOLA~
 
 IF ~~ THEN BEGIN 0
-  SAY ~Love him, Sime... His bite is deadly and sweet as almonds. Nothing is more tender than death...~
+  SAY @0
   IF ~~ THEN EXTERN ~SOLAUF~ 0
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~Following each other 'til the last warmth came,
-From our two hearts, two vast torches will pour
-The reflection of their double flames
-On our two spirits, these twin mirrors.~
+  SAY @1
   IF ~~ THEN EXTERN ~SOL~ 1
 END
