--- NeJE\dlg\TuJiri.d	Mon Jan 19 15:56:18 2015
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\TuJiri.d	Tue Apr 09 19:37:31 2019
@@ -1,119 +1,115 @@
 BEGIN ~TuJiri~
 
 IF ~NumberOfTimesTalkedTo(0)~ THEN BEGIN 0
-  SAY ~Good day to you. I am Jiri, can I assist you in any way?~
-  IF ~~ THEN REPLY ~I am <GABBER>. Well met, Jiri, could you please tell me where I would find whoever is in charge here?~ JOURNAL ~Halruaa‘s Internal Affairs.
-
-I have successfully infiltrated the academy of Halruaa... there was a moment when the guards on the main gate stopped me... I was sure I had been discovered before even set foot onto academy grounds, but the wizard only wanted to be sure I was aware of Halruaa’s rules concerning dress and weaponry. Thank goodness Tilorn prepared me... A young scholar by the name of Jiri directed me to the main building... and what an impressive building it is...  as rich and as imposing as any temple that I have seen in my travels!~ GOTO 1
-  IF ~~ THEN REPLY ~Hello, Jiri, I am <GABBER>. What are all these buildings here?~ JOURNAL ~Halruaa‘s Internal Affairs.
-
-I have successfully infiltrated the academy of Halruaa... there was a moment when the guards on the main gate stopped me... I was sure I had been discovered before even set foot onto academy grounds, but the wizard only wanted to be sure I was aware of Halruaa’s rules concerning dress and weaponry. Thank goodness Tilorn prepared me... A young scholar by the name of Jiri directed me to the main building... and what an impressive building it is...  as rich and as imposing as any temple that I have seen in my travels!~ GOTO 2
+  SAY @0
+  IF ~~ THEN REPLY @1 JOURNAL @2 GOTO 1
+  IF ~~ THEN REPLY @3 JOURNAL @2 GOTO 2
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~The main Academy is the place you that you need to be. That is the impressive building in front of us.~
-  IF ~~ THEN REPLY ~Then that is where I am heading, thank you, Jiri.~ EXIT
+  SAY @4
+  IF ~~ THEN REPLY @5 EXIT
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~These smaller buildings here are what makes up the living quarters of Halruaa and that impressive building is the main Academy, which incidently, is where you need to be as a new arrival.~
-  IF ~~ THEN REPLY ~Then that is where I am heading, thank you, Jiri.~ EXIT
+  SAY @6
+  IF ~~ THEN REPLY @5 EXIT
 END
 
 IF ~!NumberOfTimesTalkedTo(0)
 Global("VP_Brial_Search","LOCALS",1)~ THEN BEGIN 3
-  SAY ~Good day to you, <GABBER>.~
+  SAY @7
   IF ~~ THEN EXIT
 END
 
 IF ~!NumberOfTimesTalkedTo(0)
 Global("VP_Brial_Search","LOCALS",0)
 PartyHasItem("TuLette4")~ THEN BEGIN 4
-  SAY ~Well hello again <GABBER>. Now that you have had time to settle in, what do you think of our academy here?~
-  IF ~~ THEN REPLY ~It’s good to see you again Jiri. I am certainly impressed by what I have seen so far. Tell me Jiri, who is Brial and where might I find her?~ GOTO 5
-  IF ~~ THEN REPLY ~Jiri, you couldn’t have timed your appearance better. I am looking for Brial, have you any idea where I might find her?~ GOTO 5
-  IF ~~ THEN REPLY ~Hello Jiri. There is certainly a lot more than meets the eye here at Halruaa... a lot more. I have been asked to seek out a tutor by the name of Brial but I can’t seem to find her?~ GOTO 16
+  SAY @8
+  IF ~~ THEN REPLY @9 GOTO 5
+  IF ~~ THEN REPLY @10 GOTO 5
+  IF ~~ THEN REPLY @11 GOTO 16
 END
 
 IF ~~ THEN BEGIN 5
-  SAY ~Brial... why she’s one of the senior staff of the academy and a very accomplished mage. She doesn’t tutor as much as she used to, in fact she’s spending more and more time in the company of Halruaa’s elite these days. What could you possibly want with her?~
-  IF ~~ THEN REPLY ~I have a message for her from RayVon.~ GOTO 6
-  IF ~~ THEN REPLY ~She has some information that is vital to me, I need to find her quickly.~ GOTO 11
-  IF ~~ THEN REPLY ~She sent a message that she wanted to see me but the messenger forgot to tell me where to find her.~ GOTO 11
+  SAY @12
+  IF ~~ THEN REPLY @13 GOTO 6
+  IF ~~ THEN REPLY @14 GOTO 11
+  IF ~~ THEN REPLY @15 GOTO 11
 END
 
 IF ~~ THEN BEGIN 6
-  SAY ~So RayVon has you running errands for him now... I would stay out of his way if you can. Once he picks his *gopher* for the term, he’ll run you ragged going from one end of the academy to the other on insignificant errands.~
+  SAY @16
   IF ~~ THEN GOTO 7
 END
   
 IF ~~ THEN BEGIN 7
-  SAY ~He always picks on a new student because they are afraid to say no to him. He had me running in circles until I stood up to him, you should do the same.~
+  SAY @17
   IF ~~ THEN GOTO 8
 END  
   
 IF ~~ THEN BEGIN 8
-  SAY ~As for taking a message to Brial, you would be wise to give it to another tutor to pass on. The seniors have their apartments in the main academy building but entrance to the floor where they are located, is strictly forbidden to all students, any student found there is dealt with very severely.~
-  IF ~~ THEN REPLY ~Thank you Jiri. I’ll think on what you have said.~ GOTO 9
-  IF ~~ THEN REPLY ~I would prefer to give her the message myself so please, I’ll ask that you tell no one, although I am sure access will be granted me once I say that I come from RayVon.~ GOTO 9 
+  SAY @18
+  IF ~~ THEN REPLY @19 GOTO 9
+  IF ~~ THEN REPLY @20 GOTO 9 
 END
 
 IF ~~ THEN BEGIN 9
-  SAY ~Just be careful <CHARNAME>, the mages here are pretty up tight and are being very tight lipped at the moment. It is not for us students to question the affairs of the masters but with all this tension about, if you are discovered in their private quarters, whatever you say your reason is, it will surely be misconstrued.~
+  SAY @21
   IF ~~ THEN GOTO 10
 END  
   
 IF ~~ THEN BEGIN 10
-  SAY ~Take my advice, give the message to another tutor to pass on. Now, I must take my leave of you. Farewell and enjoy your time at the academy... it is a great place to learn and has much to offer.~
+  SAY @22
   IF ~~ THEN DO ~SetGlobal("VP_Brial_Search","LOCALS",1)
 MoveToPointNoInterrupt([2362.1028])
 Face(14)~ EXIT
 END
 
 IF ~~ THEN BEGIN 11
-  SAY ~I can’t think why you should be wanting to speak with Brial, nor do I wish to know <GABBER>. What I do know is that she is not accustomed to speaking with new students, or any students if it comes to that... not since she took her place as a senior mage on the admin side of academy affairs.~
+  SAY @23
   IF ~~ THEN GOTO 12
 END  
    
 IF ~~ THEN BEGIN 12
-  SAY ~If she isn’t in the admin offices, then she is probably to be found in the academy main building, where the seniors have their apartments. I would however advise you not to enter that floor, it is strictly forbidden and severe punishments are handed out to any student found there.~
+  SAY @24
   IF ~~ THEN GOTO 13
 END   
   
 IF ~~ THEN BEGIN 13
-  SAY ~If Brial has business with you then let her come to you... she knows the regulations and should know better than to expect you to risk being found in the senior quarters.~ 
-  IF ~~ THEN REPLY ~Thanks for the warning but it is urgent that I see her, so I guess I am going to have to take the risk.~ GOTO 14
-  IF ~~ THEN REPLY ~I’ll bear in mind what you have said Jiri, but if I don’t find her, the consequences could be more dire than any punishment that I might suffer for being discovered on the senior’s private floor.~ GOTO 14
-  IF ~~ THEN REPLY ~In that case, I am sure you will appreciate the need to keep it to yourself that I am looking for her, at least until I find her and she can vouch for me.~ GOTO 14
+  SAY @25 
+  IF ~~ THEN REPLY @26 GOTO 14
+  IF ~~ THEN REPLY @27 GOTO 14
+  IF ~~ THEN REPLY @28 GOTO 14
 END
 
 IF ~~ THEN BEGIN 14
-  SAY ~By rights I should report this matter and let Brial herself verify that she wishes to speak with you. However, things being as they are at the moment, I suspect that by the time I actually find any one who is remotely interested, you’ll have either found her yourself or you’ll have been caught looking.~
+  SAY @29
   IF ~~ THEN GOTO 15
 END   
   
 IF ~~ THEN BEGIN 15
-  SAY ~So I’ll forget this conversation happened, but please take care <GABBER> and if I should run into Brial, I’ll be sure to tell her that you are looking for her. Farewell.~
+  SAY @30
   IF ~~ THEN DO ~SetGlobal("VP_Brial_Search","LOCALS",1)
 MoveToPointNoInterrupt([2362.1028])
 Face(14)~ EXIT
 END
 
 IF ~~ THEN BEGIN 16
-  SAY ~You too <GABBER>, you feel the sense... an air of secrecy, of something being right under our noses, whilst at the same time being deliberately kept just out of our sight. Whatever it is, I’ve no doubt we students will be the last to know what, if anything, is going on.~
+  SAY @31
   IF ~~ THEN GOTO 17
 END   
   
 IF ~~ THEN BEGIN 17
-  SAY ~Back to your question... Brial? I can’t imagine why anyone should be sending you to Brial, she no longer bothers herself with student affairs, so  you’ll excuse me for asking, but who has sent you to find her?~
-  IF ~~ THEN REPLY ~I have a message for her from RayVon.~ GOTO 6
-  IF ~~ THEN REPLY ~She has some information that is vital to me, I need to find her quickly.~ GOTO 11
-  IF ~~ THEN REPLY ~She sent a message that she wanted to see me but the messenger forgot to tell me where to find her.~ GOTO 11
+  SAY @32
+  IF ~~ THEN REPLY @13 GOTO 6
+  IF ~~ THEN REPLY @14 GOTO 11
+  IF ~~ THEN REPLY @15 GOTO 11
 END
 
 IF ~!NumberOfTimesTalkedTo(0)
 Global("VP_Brial_Search","LOCALS",0)
 !PartyHasItem("TuLette4")~ THEN BEGIN 18
-  SAY ~Good day to you, <GABBER>.~
+  SAY @7
   IF ~~ THEN EXIT
 END
