--- NeJE\dlg\VPARAN.D	Fri Feb 01 13:37:06 2008
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\VPARAN.D	Tue Apr 09 19:37:46 2019
@@ -1,49 +1,49 @@
 BEGIN ~VPARAN~
 
 IF ~Global("VP_Sime_Aran","GLOBAL",1)~ THEN BEGIN 0
-  SAY ~A pearl to you, dear. I have to say, you should act with more caution. It is common knowledge in Athkatla now that you are back and are travelling with <CHARNAME>. There is also a more disturbing rumor, Sime. That’s why I came. I need to talk to <CHARNAME> in private. A man-to-man talk, you understand? Leave us daughter.~
+  SAY @0
   IF ~~ THEN EXTERN ~SIMEJ~ 40
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~I told you to leave us, Sime.~
-  IF ~~ THEN REPLY ~What in Nine Hells is it? Anything you have to say to me you can say in the presence of your daughter!~ EXTERN ~SIMEJ~ 41
-  IF ~~ THEN REPLY ~Ahem, Sime, let’s not aggravate your father.~ EXTERN ~SIMEJ~ 42
-  IF ~~ THEN REPLY ~Do not worry, love, I can handle it. You can stay with the rest of the group and I’ll talk to Aran.~ EXTERN ~SIMEJ~ 42
+  SAY @1
+  IF ~~ THEN REPLY @2 EXTERN ~SIMEJ~ 41
+  IF ~~ THEN REPLY @3 EXTERN ~SIMEJ~ 42
+  IF ~~ THEN REPLY @4 EXTERN ~SIMEJ~ 42
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~Now, son...~
-  IF ~~ THEN REPLY ~Son? I have heard that my mother heated it up with Bhaal, the Lord of Murder to get me. I do not recall Aran Linvail mentioned as my father.~ GOTO 3
-  IF ~~ THEN REPLY ~Yes, Sir!~ GOTO 4
-  IF ~~ THEN REPLY ~(*Look at him patiently and wait for him to continue.*)~ GOTO 4
+  SAY @5
+  IF ~~ THEN REPLY @6 GOTO 3
+  IF ~~ THEN REPLY @7 GOTO 4
+  IF ~~ THEN REPLY @8 GOTO 4
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~I appreciate a good joke, yes. But I have not come all the way here to humor some pup. I might not be Bhaal, but I happen to do random jobs for the Lord of Murder and you may consider me his... associate. If you will not shut up and listen what I have to say I’ll start with shoving your sharp tongue deep into your throat until you choke on your own wit, son.~
-  IF ~~ THEN REPLY ~Yes, Sir!~ GOTO 4
-  IF ~~ THEN REPLY ~You are not scaring me. I can take on Bhaal himself, let alone his associate.~ GOTO 6
-  IF ~~ THEN REPLY ~Sir, your wrath is undeserved. I love your daughter with all my heart and I have the most honorable intentions...~ GOTO 7
+  SAY @9
+  IF ~~ THEN REPLY @7 GOTO 4
+  IF ~~ THEN REPLY @10 GOTO 6
+  IF ~~ THEN REPLY @11 GOTO 7
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~So, son it reached me that my daughter fancies you. Do not bother to deny it, you are involved. I promised Sime’s mother, gods bless her noble soul, to protect our daughter before she died.  Son, should any harm come to my daughter, I’ll have you roasted alive. Do you understand?~
-  IF ~~ THEN REPLY ~Yes, Sir!~ GOTO 5
-  IF ~~ THEN REPLY ~Aran, are you trying to threaten me? If so, you failed. I can take on any number of the likes of you!~ GOTO 6
-  IF ~~ THEN REPLY ~Sir, these threats are misdirected. I love your daughter with all my heart and I have the most honorable intentions... I only want what’s best for her.~ GOTO 7
+  SAY @12
+  IF ~~ THEN REPLY @7 GOTO 5
+  IF ~~ THEN REPLY @13 GOTO 6
+  IF ~~ THEN REPLY @14 GOTO 7
 END
 
 IF ~~ THEN BEGIN 5
-  SAY ~I am glad we had this little chat, son... (*Pats your cheek*)... You are not a wordy one, are you? Those silent, they are said to be the most dangerous ones. I think I’ll give you a taste of unpleasant things that may happen if you get me cross. Just in case, you know.~
+  SAY @15
   IF ~~ THEN EXTERN ~SIMEJ~ 43
 END
 
 IF ~~ THEN BEGIN 6
-  SAY ~I fail to understand what my daughter sees in a fop like you. I hope she comes back to her senses and leaves you. Anyway, son, should any harm come to my daughter, I’ll have you roasted alive. And since you are such a joker, I’ll give you a taste of unpleasant things that may happen to you if you get me cross.~
+  SAY @16
   IF ~~ THEN EXTERN ~SIMEJ~ 43
 END
 
 IF ~~ THEN BEGIN 7
-  SAY ~Honorable intentions? Oh, I forget, you are from Candlekeep, my young friend. Flowery language apart you seem like a decent fellow. I am glad we had this little chat, and I will rest easier, knowing that my Sime is well cared for. Here, son.  This is a ring I once gave to her mother. I paid in blood to get it back from those who killed my wife, hoping that a day would come to give it to my little daughter.  Sime, love, come here, say good-bye to me!  I am leaving.~
+  SAY @17
   IF ~~ THEN EXTERN ~SIMEJ~ 44
 END
