--- TS\bgtnodlg\aBAERIE.D	Thu Jan 06 17:51:18 2005
+++ C:\BWP Patchstudio\patched files\TS\bgtnodlg\aBAERIE.D	Thu May 05 10:45:23 2016
@@ -1,142 +1,142 @@
 APPEND ~BAERIE~
 
 IF ~~ THEN BEGIN TS449
-  SAY ~Yes... yes. Sorry. I did not mean to disturb anyone. I was just thinking of my people...~
+  SAY @0
   IF ~~ THEN EXTERN ~BKACHI~ 46
 END
 
 IF ~~ THEN BEGIN TS450
-  SAY ~My native lands... my native element is the sky. I miss it too. I can see it from here, from the hard ground and I yearn to be embraced by soft warm air. Kachiko, if you only knew how wonderful it is to be able to fly...~
+  SAY @1
   IF ~~ THEN EXTERN ~BKACHI~ 47
 END
 
 IF ~~ THEN BEGIN TS451
-  SAY ~I never thought that anyone could be envious of my fate...~
+  SAY @2
   IF ~~ THEN EXTERN ~BKACHI~ 48
 END
 
 IF ~~ THEN BEGIN TS452
-  SAY ~Me... strong? Kachiko, nobody ever called me strong before... In fact, they usually call me a freak... I shall think about what you said.~
+  SAY @3
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN TS453
-  SAY ~(*sighs*)~
+  SAY @4
   IF ~~ THEN EXTERN ~BVALYGA~ TS508
 END
 
 IF ~~ THEN BEGIN TS454
-  SAY ~I was just thinking of my parents and how I used to fly with them as a little girl. Oh Valygar, you must know what its like to be ripped away from a life you loved.~
+  SAY @5
   IF ~~ THEN EXTERN ~BVALYGA~ TS509
 END
 
 IF ~~ THEN BEGIN TS455
-  SAY ~I'm sorry Valygar I didn't mean to awaken painful memories for you.~
+  SAY @6
   IF ~~ THEN EXTERN ~BVALYGA~ TS510
 END
 
 IF ~~ THEN BEGIN TS456
-  SAY ~I... I do not know... It is all so sad... I feel so... lonely.~
+  SAY @7
   IF ~~ THEN EXTERN ~BVALYGA~ TS512
 END
 
 IF ~~ THEN BEGIN TS457
-  SAY ~I miss Uncle Quayle.~
+  SAY @8
   IF ~~ THEN EXTERN ~BVALYGA~ TS513
 END
 
 IF ~~ THEN BEGIN TS458
-  SAY ~I... do not want to go back. I do not... I do not miss the circus. That was a cruel place where... everyone was ready to do... just about anything for a round of applause. I miss... my Uncle. I wish... I wish I could meet someone as kind and... caring as him... during our travels. Someone who'd care for me... like you care for <CHARNAME>.~
+  SAY @9
   IF ~~ THEN EXTERN ~BVALYGA~ TS514
 END
 
 IF ~~ THEN BEGIN TS459
-  SAY ~(*blush*) I did not... mean you... I must have put it the wrong way, as usual. I... know that you and <CHARNAME>... Well, I do not... know... I just thought that... Oh, that's so embarrassing! I am so clumsy...~
+  SAY @10
   IF ~~ THEN EXTERN ~BVALYGA~ TS515
 END
 
 IF ~~ THEN BEGIN TS460
-  SAY ~My face is not... not sour!~
+  SAY @11
   IF ~~ THEN EXTERN ~BSIME~ 5
 END
 
 IF ~~ THEN BEGIN TS461
-  SAY ~You mean... you mean I look... ugly?~
+  SAY @12
   IF ~~ THEN EXTERN ~BSIME~ 6
 END
 
 IF ~~ THEN BEGIN TS462
-  SAY ~What's wrong with my... my face?~
+  SAY @13
   IF ~~ THEN EXTERN ~BSIME~ 7
 END
 
 IF ~~ THEN BEGIN TS463
-  SAY ~Oh...*sniffs*~
+  SAY @14
   IF ~~ THEN EXTERN ~BSIME~ 8
 END
 
 IF ~~ THEN BEGIN TS464
-  SAY ~I... I... was trying not to fall down...~
+  SAY @15
   IF ~~ THEN EXTERN ~BSIME~ 10
 END
 
 IF ~~ THEN BEGIN TS465
-  SAY ~For me... for me it is, Sime. Avariel are not accustomed to prolonged walks like this. Avariel fly...~
+  SAY @16
   IF ~~ THEN EXTERN ~BSIME~ 11
 END
 
 IF ~~ THEN BEGIN TS466
-  SAY ~In a cage... where I could not lay down, let alone walk. I... I have never walked this much in my whole life.~
+  SAY @17
   IF ~~ THEN EXTERN ~BSIME~ 12
 END
 
 IF ~~ THEN BEGIN TS467
-  SAY ~I... I know... Do not... do not tell anyone, please?~
+  SAY @18
   IF ~~ THEN EXTERN ~BSIME~ 13
 END
 
 IF ~~ THEN BEGIN TS468
-  SAY ~They'd laugh... They always laugh...~
+  SAY @19
   IF ~~ THEN EXTERN ~BSIME~ 14
 END
 
 IF ~~ THEN BEGIN TS469
-  SAY ~Thank you, Sime. Now if only someone could clean this place up... It is so... dirty and dusty!~
+  SAY @20
   IF ~~ THEN EXTERN ~BSIME~ 15
 END
 
 IF ~~ THEN BEGIN TS470
-  SAY ~You... do not need to check me. I... do what I want.~
+  SAY @21
   IF ~~ THEN EXTERN ~BSIME~ 563
 END
 
 IF ~~ THEN BEGIN TS471
-  SAY ~It is empty.~
+  SAY @22
   IF ~~ THEN EXTERN ~BSIME~ 564
 END
 
 IF ~~ THEN BEGIN TS472
-  SAY ~Here, catch it if you can... (*She throws the vial to Sime and tries to run away, crying).~
+  SAY @23
   IF ~~ THEN EXTERN ~BSIME~ 565
 END
 
 IF ~~ THEN BEGIN TS473
-  SAY ~I... do not need talking to, Sime. I have enough of that from <CHARNAME>.~
+  SAY @24
   IF ~~ THEN EXTERN ~BSIME~ 572
 END
 
 IF ~~ THEN BEGIN TS474
-  SAY ~Help me? You... want to help me? How can you help me? Can you make <CHARNAME> to love me?~
+  SAY @25
   IF ~~ THEN EXTERN ~BSIME~ 573
 END
 
 IF ~~ THEN BEGIN TS475
-  SAY ~You do not... understand.~
+  SAY @26
   IF ~~ THEN EXTERN ~BSIME~ 574
 END
 
 IF ~~ THEN BEGIN TS476
-  SAY ~You... are wrong. I will never... love anyone, but <CHARNAME>!~
+  SAY @27
   IF ~~ THEN EXTERN ~BSIME~ 575
 END
 END
