--- TS\bgtdlg\aBANOMEN.D	Sun Sep 20 04:16:26 2009
+++ C:\BWP Patchstudio\patched files\TS\bgtdlg\aBANOMEN.D	Thu May 05 10:43:42 2016
@@ -1,219 +1,211 @@
 APPEND ~BANOMEN~
 
 IF ~~ THEN BEGIN TS587
-  SAY ~Ahem, Yoshimo, I had... accidentally... noticed that you are writing something. Well, poetry. So you have poetry in Kara-Tur?~
+  SAY @0
   IF ~~ THEN EXTERN ~BYOSHIM~ TS100
 END
 
 IF ~~ THEN BEGIN TS588
-  SAY ~That’s a good start... was it what you were writing for the past fortnight?  Keep up good work, Yoshimo and you might actually finish it.~
+  SAY @1
   IF ~~ THEN EXTERN ~BYOSHIM~ TS101
 END
 
 IF ~~ THEN BEGIN TS589
-  SAY ~Oh, you are working for dwarven market? New style? Short poetry for short people?~
+  SAY @2
   IF ~~ THEN EXTERN ~BKACHI~ 16
 END
 
 IF ~~ THEN BEGIN TS590
-  SAY ~Kachiko, the poetry is a dove which whispers tender desires to the Lady of the heart... Not a childish line and a half... If I may offer you a real madrigal:
- 
-She walks in beauty, like the night 
-Of cloudless climes and starry skies;
-And all that’s best of dark and bright
-Meet in her aspect and her eyes:
-Thus mellow’d to that tender light
-Which heaven to gaudy day denies...
-~
+  SAY @3
   IF ~~ THEN EXTERN ~BKACHI~ 17
 END
 
 IF ~~ THEN BEGIN TS591
-  SAY ~The judge is not objective! My star, come, tell what you think?~
-  IF ~~ THEN REPLY ~What, the poem you wrote about Kachiko? Well, I’d say I agree with her, it’s not the greatest...~ GOTO TS592
+  SAY @4
+  IF ~~ THEN REPLY @5 GOTO TS592
 END
 
 IF ~~ THEN BEGIN TS592
-  SAY ~All that nights I spend writing it for you... all wasted...~
-  IF ~~ THEN REPLY ~On the second thought, may we call it a tie, Kachiko?~ EXTERN ~BKACHI~ 18
+  SAY @6
+  IF ~~ THEN REPLY @7 EXTERN ~BKACHI~ 18
 END
 
 IF ~~ THEN BEGIN TS593
-  SAY ~A tie? A TIE?~
+  SAY @8
   IF ~~ THEN EXTERN ~BYOSHIM~ TS103
 END
 
 IF ~~ THEN BEGIN TS594
-  SAY ~Valygar, if she needed anybody’s help she’d ask for it.~
+  SAY @9
   IF ~~ THEN EXTERN ~BVALYGA~ TS174
 END
 
 IF ~~ THEN BEGIN TS595
-  SAY ~Verily, Valygar, I know little of love beside what the tales say. I would speak to an older comrade in the Order, but I thought you might be willing to listen. Besides, the girl I like, well, she is very special... It’s <CHARNAME> and with her heritage, I am not sure the Order would look benevolently on our relationship...~
+  SAY @10
   IF ~~ THEN EXTERN ~BVALYGA~ TS379
 END
 
 IF ~~ THEN BEGIN TS596
-  SAY ~See my problem? Of all fair ladies I love the one who would never be accepted by the Order as my bride...~
+  SAY @11
   IF ~~ THEN EXTERN ~BVALYGA~ TS380
 END
 
 IF ~~ THEN BEGIN TS597
-  SAY ~Why are you so upset Valygar? I merely...~
+  SAY @12
   IF ~~ THEN EXTERN ~BVALYGA~ TS382
 END
 
 IF ~~ THEN BEGIN TS598
-  SAY ~Have you never met a nice girl?~
+  SAY @13
   IF ~~ THEN EXTERN ~BVALYGA~ TS383
 END
 
 IF ~~ THEN BEGIN TS599
-  SAY ~I think I am luckier than you then. <CHARNAME> is a woman of rare virtue...~
+  SAY @14
   IF ~~ THEN EXTERN ~BVALYGA~ TS384
 END
 
 IF ~~ THEN BEGIN TS600
-  SAY ~You should try. She went mad, dragging us all over the place instead of saving the elven city. Many lives were forsaken to save yours. Verily, a worthy exchange! It would seem that you have some special value in the eyes of <CHARNAME>.~
+  SAY @15
   IF ~~ THEN EXTERN ~BVALYGA~ TS386
 END
 
 IF ~~ THEN BEGIN TS601
-  SAY ~Is that true <CHARNAME>? Would you do it, say, for me?~
-  IF ~~ THEN REPLY ~Yes I would. Now cool down, you are acting irrationally.~ GOTO TS602
-  IF ~~ THEN REPLY ~I am not going to answer that. Think what you will, Anomen, I just do not care.~ EXTERN ~BVALYGA~ TS389
-  IF ~~ THEN REPLY ~No. I would have left you to rot in the crypt. ~ GOTO TS604
+  SAY @16
+  IF ~~ THEN REPLY @17 GOTO TS602
+  IF ~~ THEN REPLY @18 EXTERN ~BVALYGA~ TS389
+  IF ~~ THEN REPLY @19 GOTO TS604
 END
 
 IF ~~ THEN BEGIN TS602
-  SAY ~I... I was... I am sorry my Lady.~
-  IF ~~ THEN REPLY ~You should apologize to Valygar, not to me...~ GOTO TS603
-  IF ~~ THEN REPLY ~Apology accepted. Now, let’s all calm down and do our job.~ EXTERN ~BVALYGA~ TS388
+  SAY @20
+  IF ~~ THEN REPLY @21 GOTO TS603
+  IF ~~ THEN REPLY @22 EXTERN ~BVALYGA~ TS388
 END
 
 IF ~~ THEN BEGIN TS603
-  SAY ~Oh, that? Well, Valygar, I must say I had been ill-tempered and said what I should have not...~
+  SAY @23
   IF ~~ THEN EXTERN ~BVALYGA~ TS387
 END
 
 IF ~~ THEN BEGIN TS604
-  SAY ~Oh, I see... Well, then I’ll relieve you from the necessity of being in my company. Fare thee well.~
+  SAY @24
   IF ~~ THEN DO ~LeaveParty()
 EscapeArea()~ EXIT
 END
 
 IF ~~ THEN BEGIN TS605
-  SAY ~Somehow I do not think so. She cares not if I would turn into a stone or a turnip... or a vampire.~
+  SAY @25
   IF ~~ THEN EXTERN ~BVALYGA~ TS390
 END
 
 IF ~~ THEN BEGIN TS606
-  SAY ~Indeed, it would serve me well. Looks like <CHARNAME> is into the strong and silent type.~
+  SAY @26
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN TS607
-  SAY ~I have a strange suspicion, ranger. I think you are trying to...~
+  SAY @27
   IF ~~ THEN EXTERN ~BVALYGA~ TS462
 END
 
 IF ~~ THEN BEGIN TS608
-  SAY ~You know what!~
+  SAY @28
   IF ~~ THEN EXTERN ~BVALYGA~ TS463
 END
 
 IF ~~ THEN BEGIN TS609
-  SAY ~Laughing! Sure, you can laugh! It’s my heart that is bleeding!~
+  SAY @29
   IF ~~ THEN EXTERN ~BVALYGA~ TS464
 END
 
 IF ~~ THEN BEGIN TS610
-  SAY ~I want NOTHING from you!~
+  SAY @30
   IF ~~ THEN EXTERN ~BVALYGA~ TS465
 END
 
 IF ~~ THEN BEGIN TS611
-  SAY ~I am not through with you yet! You cannot just walk away!~
+  SAY @31
   IF ~~ THEN EXTERN ~BVALYGA~ TS466
 END
 
 IF ~~ THEN BEGIN TS612
-  SAY ~I have seen you together... Do not try to deny it! You hold her hand! You who have told me endlessly how uninterested you are in ladies! I was so blind. I let you to steal <CHARNAME> from me...~
+  SAY @32
   IF ~~ THEN EXTERN ~BVALYGA~ TS468
 END
 
 IF ~~ THEN BEGIN TS613
-  SAY ~Spend time indeed... Travel to Zakhara - that’s going to give you two a lot of time to kill.~
+  SAY @33
   IF ~~ THEN EXTERN ~BVALYGA~ TS469
 END
 
 IF ~~ THEN BEGIN TS614
-  SAY ~I should have done that long ago, instead of trusting her, instead of trusting you. It is my own fault that the one I love and you, whom I called my friend, have broken my life. I was too carefree. Now they all look at me with pity and say nice words. But my heart is the vulture’s food. Nothing hurts more than treachery.~
+  SAY @34
   IF ~~ THEN EXTERN ~BVALYGA~ TS470
 END
 
 IF ~~ THEN BEGIN TS615
-  SAY ~How dare you!~
+  SAY @35
   IF ~~ THEN EXTERN ~BVALYGA~ TS471
 END
 
 IF ~~ THEN BEGIN TS616
-  SAY ~Oh, now it’s my own fault, is not it?~
+  SAY @36
   IF ~~ THEN EXTERN ~BVALYGA~ TS472
 END
 
 IF ~~ THEN BEGIN TS617
-  SAY ~I see... May be she would have more to say?~
+  SAY @37
   IF ~~ THEN EXTERN ~BVALYGA~ TS473
 END
 
 IF ~~ THEN BEGIN TS618
-  SAY ~Are you threatening me?~
+  SAY @38
   IF ~~ THEN EXTERN ~BVALYGA~ TS474
 END
 
 IF ~~ THEN BEGIN TS619
-  SAY ~I care not what you told me. My conversation with her is a private one.~
+  SAY @39
   IF ~~ THEN EXTERN ~BVALYGA~ TS551
 END
 
 IF ~~ THEN BEGIN TS620
-  SAY ~Jealous? Of course you are jealous! She loves me, not you!~
+  SAY @40
   IF ~~ THEN EXTERN ~BVALYGA~ TS552
 END
 
 IF ~~ THEN BEGIN TS621
-  SAY ~What! I love her more than my own life, and I will hurt her? You are going queer, ranger! I just need to talk to her. <CHARNAME>, I beg you to tell me what is going on... Do you like Valygar better than I?~
-  IF ~~ THEN REPLY ~That’s enough. Valygar, you have some sort of idea-fix that everyone tries to hurt me and you are the one to save me. Let me remind you, that I am quite capable of taking care of myself. Besides I do not desire to be protected from the love of my life, Anomen... (*give Anomen a sweet smile*)~ EXTERN ~BVALYGA~ TS553
-  IF ~~ THEN REPLY ~I think I know the way to solve this conflict: I care for neither of you. But I consider both of you my friends. So do make up and let’s forget about this silly conversation.~ EXTERN ~BVALYGA~ TS554
-  IF ~~ THEN REPLY ~Ahem... Anomen. I do not know how you came up with the notion that I like you... I am sorry if I have given you a false hope. I prefer you not to intervene into my relationship with Valygar. ~ EXTERN ~BVALYGA~ TS556
-  IF ~~ THEN REPLY ~(*grin... and stay silent waiting for the next scene*)~ EXTERN ~BVALYGA~ TS558
-  IF ~~ THEN REPLY ~(*cry helplessly*) Fools, I... I love both of you. Please, do not make me choose...~ EXTERN ~BVALYGA~ TS560
+  SAY @41
+  IF ~~ THEN REPLY @42 EXTERN ~BVALYGA~ TS553
+  IF ~~ THEN REPLY @43 EXTERN ~BVALYGA~ TS554
+  IF ~~ THEN REPLY @44 EXTERN ~BVALYGA~ TS556
+  IF ~~ THEN REPLY @45 EXTERN ~BVALYGA~ TS558
+  IF ~~ THEN REPLY @46 EXTERN ~BVALYGA~ TS560
 END
 
 IF ~~ THEN BEGIN TS622
-  SAY ~I cannot believe my own ears!~
+  SAY @47
   IF ~~ THEN EXTERN ~BVALYGA~ TS555
 END
 
 IF ~~ THEN BEGIN TS623
-  SAY ~I do not want to hear another word! I have to be alone now.~
+  SAY @48
   IF ~~ THEN DO ~ChangeAlignment(Myself,CHAOTIC_NEUTRAL)~ EXIT
 END
 
 IF ~~ THEN BEGIN TS624
-  SAY ~I do not want to hear another word! I have to be alone now.~
+  SAY @48
   IF ~~ THEN DO ~LeaveParty()
 EscapeArea()~ EXIT
 END
 
 IF ~~ THEN BEGIN TS625
-  SAY ~Let me go talk to her! Or I swear I will kill you!~
+  SAY @49
   IF ~~ THEN EXTERN ~BVALYGA~ TS559
 END
 
 IF ~~ THEN BEGIN TS626
-  SAY ~How dare you are...~
+  SAY @50
   IF ~~ THEN DO ~SetGlobal("AnomenValygarFight","GLOBAL",1)
 SetLeavePartyDialogueFile()
 LeaveParty()
@@ -223,47 +215,47 @@
 END
 
 IF ~~ THEN BEGIN TS627
-  SAY ~Nay my Lady, not in the least.~
+  SAY @51
   IF ~~ THEN EXTERN ~BSIME~ 40
 END
 
 IF ~~ THEN BEGIN TS628
-  SAY ~You should my Lady, you should.~
+  SAY @52
   IF ~~ THEN EXTERN ~BSIME~ 41
 END
 
 IF ~~ THEN BEGIN TS629
-  SAY ~It is most hurtful to hear such harsh words from a noble born Lady.~
+  SAY @53
   IF ~~ THEN EXTERN ~BSIME~ 42
 END
 
 IF ~~ THEN BEGIN TS630
-  SAY ~But surely, your Lady mother was of a high station...~
+  SAY @54
   IF ~~ THEN EXTERN ~BSIME~ 43
 END
 
 IF ~~ THEN BEGIN TS631
-  SAY ~Mayhaps you are a Princess of some enchanted place and you know not of it, my Lady?~
+  SAY @55
   IF ~~ THEN EXTERN ~BSIME~ 44
 END
 
 IF ~~ THEN BEGIN TS632
-  SAY ~My Lady, you are an observant friend. I am worried of my feeling about our company.~
+  SAY @56
   IF ~~ THEN EXTERN ~BSIME~ 46
 END
 
 IF ~~ THEN BEGIN TS633
-  SAY ~Quite the opposite, my dear Sime, quite the opposite...~
+  SAY @57
   IF ~~ THEN EXTERN ~BSIME~ 47
 END
 
 IF ~~ THEN BEGIN TS634
-  SAY ~Our group is lacking discipline and the purpose. Yet I like it. What would the Prelate of the Order say if he would learn that I am not insistent enough on proper procedures? When are we to undertake the heroic deeds fit for a paladin of the Order?~
+  SAY @58
   IF ~~ THEN EXTERN ~BSIME~ 48
 END
 
 IF ~~ THEN BEGIN TS635
-  SAY ~Aye, you speak truthfully, my Lady. And that was "the pool of right"... I suppose I strike evil in the name of the Order and I am pleased with it. Menial tasks or no, we do good things. But the lack of discipline and insubordination does not sit well with me!~
+  SAY @59
   IF ~~ THEN EXTERN ~BSIME~ 49
 END
 END
