--- TS\bgtdlg\aBCERND.D	Wed Nov 17 10:09:53 2004
+++ C:\BWP Patchstudio\patched files\TS\bgtdlg\aBCERND.D	Thu May 05 10:43:42 2016
@@ -1,47 +1,47 @@
 APPEND ~BCERND~
 
 IF ~~ THEN BEGIN TS100
-  SAY ~What's your heart telling you on that account, ranger?~
+  SAY @0
   IF ~~ THEN EXTERN ~BSIME~ 167
 END
 
 IF ~~ THEN BEGIN TS101
-  SAY ~You sound very sure. Why ask me then, Sime?~
+  SAY @1
   IF ~~ THEN EXTERN ~BSIME~ 168
 END
 
 IF ~~ THEN BEGIN TS102
-  SAY ~I know that every living thing has dreams, Sime. The Oak Father granted me a rare spell once and I... I was fortunate enough to see the dreams of a beast then.~
+  SAY @2
   IF ~~ THEN EXTERN ~BSIME~ 169
 END
 
 IF ~~ THEN BEGIN TS103
-  SAY ~A spotted lion. A strange and wonderful thing are the beast's dreams, Sime. Wild and so... unrestricted.~
+  SAY @3
   IF ~~ THEN EXTERN ~BSIME~ 170
 END
 
 IF ~~ THEN BEGIN TS104
-  SAY ~(*shakes his head*) No, Sime. They cannot be put in words, those images. Pray to have an experience like that for yourself.~
+  SAY @4
   IF ~~ THEN EXTERN ~BSIME~ 171
 END
 
 IF ~~ THEN BEGIN TS105
-  SAY ~Yes. They are expecting me back at the Grove.~
+  SAY @5
   IF ~~ THEN EXTERN ~BSIME~ 173
 END
 
 IF ~~ THEN BEGIN TS106
-  SAY ~All things happen in due course in nature. My part now is to follow Bhaal's child.~
+  SAY @6
   IF ~~ THEN EXTERN ~BSIME~ 174
 END
 
 IF ~~ THEN BEGIN TS107
-  SAY ~Demigods are strange creatures, Sime. I know that Bhaal's blood torments <CHARNAME>. If I am not mistaken, events mysterious and terrible are brewing... The fate of the greater balance depends on how well <CHARNAME> can maintain the balance of <PRO_HISHER> soul. Someone who understands the ways of Balance needs to be watching after <PRO_HIMHER>.~
+  SAY @7
   IF ~~ THEN EXTERN ~BSIME~ 175
 END
 
 IF ~~ THEN BEGIN TS108
-  SAY ~I would, Sime. I yearn to return to the Grove and be with him. But the very fate of my child depends on the actions of <CHARNAME>. Balance must be preserved...~
+  SAY @8
   IF ~~ THEN EXTERN ~BSIME~ 176
 END
 END
