--- TS\bgtnodlg\aBEDWIN.D	Wed Nov 17 10:14:32 2004
+++ C:\BWP Patchstudio\patched files\TS\bgtnodlg\aBEDWIN.D	Thu May 05 10:45:26 2016
@@ -1,109 +1,109 @@
 APPEND ~BEDWIN~
 
 IF ~~ THEN BEGIN TS106
-  SAY ~I do not expect a group of unruly imbeciles to even start to understand my grandeur and finesse. I wish no stupid statements like that in the future from you, wench.~
+  SAY @0
   IF ~~ THEN EXTERN ~BKACHI~ 56
 END
 
 IF ~~ THEN BEGIN TS107
-  SAY ~Or I'll destroy you, or yours precious samurai, or both of you (*yawns*).~
+  SAY @1
   IF ~~ THEN EXTERN ~BYOSHIM~ TS107
 END
 
 IF ~~ THEN BEGIN TS108
-  SAY ~Fortunately to me, and unfortunately to you, we won't meet again (*finally, I get rid of these fools*).~
+  SAY @2
   IF ~~ THEN DO ~LeaveParty()
 EscapeArea()~ EXIT
 END
 
 IF ~~ THEN BEGIN TS109
-  SAY ~You will suffer yet...~
+  SAY @3
   IF ~~ THEN DO ~LeaveParty()
 Enemy()~ EXIT
 END
 
 IF ~~ THEN BEGIN TS110
-  SAY ~Yes, pretty face, I had noticed you and that should put a smile on any maiden's face. (This one is smarter than she looks. She appreciates me and shows appropriate reverence.)~
+  SAY @4
   IF ~~ THEN EXTERN ~BSIME~ 139
 END
 
 IF ~~ THEN BEGIN TS111
-  SAY ~I am coming to the conclusion that mutual attraction is growing between the most attractive and intelligent person in this pathetic troupe and his devoted maid. (Should I have included some compliment for the wench? No, that would be giving her too much. She is gasping in anticipation already.)~
+  SAY @5
   IF ~~ THEN EXTERN ~BSIME~ 140
 END
 
 IF ~~ THEN BEGIN TS112
-  SAY ~Sime, you had better sit down, least your happiness takes you off your feet. (Amazing imbecile. Comes of roaming the wilds in the company of beasts, no doubt. I should be more direct with the likes of her. But she can be educated, I hope.)~
+  SAY @6
   IF ~~ THEN EXTERN ~BSIME~ 141
 END
 
 IF ~~ THEN BEGIN TS113
-  SAY ~Sime, I had devised that you, poor soul, are deeply taken by my superior charms. While it is clear that you are inferior to me I am willing to condescend and bring the relationship to consummation tonight.~
+  SAY @7
   IF ~~ THEN EXTERN ~BSIME~ 142
 END
 
 IF ~~ THEN BEGIN TS114
-  SAY ~Fool woman, my look and style are perfect. How dare you to suggest an imbalance? (And I thought she had a grain of sense!)~
+  SAY @8
   IF ~~ THEN EXTERN ~BSIME~ 144
 END
 
 IF ~~ THEN BEGIN TS115
-  SAY ~You do not suggest seriously that I might have pay attention to anything tree-hugging rabble have to say?~
+  SAY @9
   IF ~~ THEN EXTERN ~BSIME~ 145
 END
 
 IF ~~ THEN BEGIN TS116
-  SAY ~I do not see how it is relevant to the kiss you were ordered to give me, but indeed I have an intelligent concept of the world's balance (do I even dare to hope that someone can understand a scheme so perfectly devised?)~
+  SAY @10
   IF ~~ THEN EXTERN ~BSIME~ 146
 END
 
 IF ~~ THEN BEGIN TS117
-  SAY ~I can see that you are not as hopelessly foolish as I originally thought (I would have to use my charm, the wench is stubborn) but you should not be afraid of my greatness. I am a red-blooded male as well as an intellectual pinnacle of mankind.~
+  SAY @11
   IF ~~ THEN EXTERN ~BSIME~ 147
 END
 
 IF ~~ THEN BEGIN TS118
-  SAY ~Elves? An outdated and almost extinct nation. The only true power lies with humans.~
+  SAY @12
   IF ~~ THEN EXTERN ~BSIME~ 148
 END
 
 IF ~~ THEN BEGIN TS119
-  SAY ~You had the most annoying outburst of base emotions when I have spoken to you last time. (Must I explain everything to those imbeciles?) I am interested to know if you saw the truth of my words and are ready to apologize for your unspeakable behavior?~
+  SAY @13
   IF ~~ THEN EXTERN ~BSIME~ 150
 END
 
 IF ~~ THEN BEGIN TS120
-  SAY ~I fail to understand how you can fight off your attraction to me for so long. It must be very painful to be away from my gorgeous body and not to be a receptacle of my nearly divine wisdom.~
+  SAY @14
   IF ~~ THEN EXTERN ~BSIME~ 151
 END
 
 IF ~~ THEN BEGIN TS121
-  SAY ~I have suspected that you with your narrow mind were interested in some stupid joke. (I have a long memory and I shall have revenge.)~
+  SAY @15
   IF ~~ THEN EXTERN ~BSIME~ 152
 END
 
 IF ~~ THEN BEGIN TS122
-  SAY ~(Is it not amazing how their attitude can change.) I am coming, wench.~
+  SAY @16
   IF ~~ THEN EXTERN ~BSIME~ 154
 END
 
 IF ~~ THEN BEGIN TS123
-  SAY ~Here I am, in all my glorious beauty... What is it? Why are you dressed? And armored? And you hold your mace? Sime, my love...~
+  SAY @17
   IF ~~ THEN EXTERN ~BSIME~ 155
 END
 
 IF ~~ THEN BEGIN TS124
-  SAY ~Only for the love of you my queen, my dear elven queen... You would not hurt your humble slave, will you? I am naked and defenseless before you... (*tries to cast a fireball*)~
+  SAY @18
   IF ~~ THEN EXTERN ~BSIME~ 156
 END
 
 IF ~~ THEN BEGIN TS125
-  SAY ~But my sweet charming elven beauty... (How many more humiliation the wench wants me to go through?)~
+  SAY @19
   IF ~~ THEN EXTERN ~BSIME~ 157
 END
 
 IF ~~ THEN BEGIN TS126
-  SAY ~I... shall... leave Sime alone!~
+  SAY @20
   IF ~~ THEN EXTERN ~BSIME~ 158
 END
 END
