--- TS\bgtdlg\aBHAERDA.D	Wed Nov 17 10:16:32 2004
+++ C:\BWP Patchstudio\patched files\TS\bgtdlg\aBHAERDA.D	Thu May 05 10:43:43 2016
@@ -1,164 +1,137 @@
 APPEND ~BHAERDA~
 
 IF ~~ THEN BEGIN TS165
-  SAY ~Wonderful since butterflies of friendly words went flying from the rose petals of your lips, to nest on my eager ear...~
+  SAY @0
   IF ~~ THEN EXTERN ~BKACHI~ 37
 END
 
 IF ~~ THEN BEGIN TS166
-  SAY ~An actor is as many entities as he recreates for the great gods of THEATER, yet I also have had many experiences in my life, which made me a poet... Kachiko, if I only could draw for you the many visions of mystery and grandeur I had witnessed in my travels! The half-ruined palaces veiled by the clouds of multicolored vapors, surrounded by ocean of melting mercury and hidden between the planes... People of other spheres covered in radiance... Females painted by henna from head to toe, dancing and singing in ecstasy...~
+  SAY @1
   IF ~~ THEN EXTERN ~BYOSHIM~ TS105
 END
 
 IF ~~ THEN BEGIN TS167
-  SAY ~I have seen it, Yoshimo... In fact, I have seen it too many times and I am bored of the sight. Kachiko, may be we shall chat another time? Maybe in more... eh... private place?~
+  SAY @2
   IF ~~ THEN EXTERN ~BKACHI~ 38
 END
 
 IF ~~ THEN BEGIN TS168
-  SAY ~Maybe not, then. A pity, my butterfly, a pity.~
+  SAY @3
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN TS169
-  SAY ~I easily believe it, my seagull. Your voice is melodic and your steps are light. You were born to be a minstrel.~
+  SAY @4
   IF ~~ THEN EXTERN ~BSIME~ 178
 END
 
 IF ~~ THEN BEGIN TS170
-  SAY ~You sound sure.~
+  SAY @5
   IF ~~ THEN EXTERN ~BSIME~ 179
 END
 
 IF ~~ THEN BEGIN TS171
-  SAY ~Might I assume that my little seagull made her own plans?~
+  SAY @6
   IF ~~ THEN EXTERN ~BSIME~ 180
 END
 
 IF ~~ THEN BEGIN TS172
-  SAY ~I am eager to hear the rest, Sime, though I foresee that the rude crowd did not appreciate your talent and grace.~
+  SAY @7
   IF ~~ THEN EXTERN ~BSIME~ 181
 END
 
 IF ~~ THEN BEGIN TS173
-  SAY ~How did the adventure end? Did your father come for you? Or...?~
+  SAY @8
   IF ~~ THEN EXTERN ~BSIME~ 182
 END
 
 IF ~~ THEN BEGIN TS174
-  SAY ~He was not?~
+  SAY @9
   IF ~~ THEN EXTERN ~BSIME~ 183
 END
 
 IF ~~ THEN BEGIN TS175
-  SAY ~My sweetest Lady, I am just dreamy because
-
-I met a lady in the meads
-Full beautiful, a faery's child;
-Her hair was long, her foot was light,
-And her eyes were wild.~
+  SAY @10
   IF ~~ THEN EXTERN ~BSIME~ 185
 END
 
 IF ~~ THEN BEGIN TS176
-  SAY ~One cannot get used to beauty.~
+  SAY @11
   IF ~~ THEN EXTERN ~BSIME~ 186
 END
 
 IF ~~ THEN BEGIN TS177
-  SAY ~The blood of a fiend is in my veins Sime. Hot, dark blood. It burns stronger than fire. A simple torch cannot harm me.~
+  SAY @12
   IF ~~ THEN EXTERN ~BSIME~ 188
 END
 
 IF ~~ THEN BEGIN TS178
-  SAY ~My Lady will not allow a poet romantic exaggeration?~
+  SAY @13
   IF ~~ THEN EXTERN ~BSIME~ 189
 END
 
 IF ~~ THEN BEGIN TS179
-  SAY ~Oh, on the contrary I am grateful. I needed to be interrupted. The structured and coherent thought is an enemy of chaos. But yours is a timely arrival for I was thinking of you.~
+  SAY @14
   IF ~~ THEN EXTERN ~BSIME~ 191
 END
 
 IF ~~ THEN BEGIN TS180
-  SAY ~Are you a demon, Sime? That would make you a most appropriate mate for a member of the Doomguard such as I.~
+  SAY @15
   IF ~~ THEN EXTERN ~BSIME~ 192
 END
 
 IF ~~ THEN BEGIN TS181
-  SAY ~Oh, may I be allowed to continue?~
+  SAY @16
   IF ~~ THEN EXTERN ~BSIME~ 193
 END
 
 IF ~~ THEN BEGIN TS182
-  SAY ~
-
-I was made that way.
-When I wish to laugh
-I laugh till it hurts.
-
-I love the one who loves me
-And is it really my fault
-that it is not the same man
-I love every night?
-
-~
+  SAY @17
   IF ~~ THEN EXTERN ~BSIME~ 194
 END
 
 IF ~~ THEN BEGIN TS183
-  SAY ~As my Lady demands. Let me find a melody sweet enough to engage the interest of the seagull.
-
-I bring you with reverent hands
-The books of my countless dreams,
-White woman that passion has worn 
-As the tide wears the dove-gray sands, 
-And with heart much older than the horn 
-That is brimmed from the pale fire of time: 
-White woman with numberless dreams,
- I bring you my passionate rhyme.
-
-~
+  SAY @18
   IF ~~ THEN EXTERN ~BSIME~ 196
 END
 
 IF ~~ THEN BEGIN TS184
-  SAY ~Why Sime? Your heart will not listen tonight to a poet's plea for love?~
+  SAY @19
   IF ~~ THEN EXTERN ~BSIME~ 197
 END
 
 IF ~~ THEN BEGIN TS185
-  SAY ~Have you heard elvenkind sing before, Sime?~
+  SAY @20
   IF ~~ THEN EXTERN ~BSIME~ 199
 END
 
 IF ~~ THEN BEGIN TS186
-  SAY ~I shall tell you a secret, Sime. About elven voices.~
+  SAY @21
   IF ~~ THEN EXTERN ~BSIME~ 200
 END
 
 IF ~~ THEN BEGIN TS187
-  SAY ~An elf sings the prettiest after a night of passionate lovemaking. Care to find out for yourself?~
+  SAY @22
   IF ~~ THEN EXTERN ~BSIME~ 201
 END
 
 IF ~~ THEN BEGIN TS188
-  SAY ~The possibility of rejection is part of the game, seagull.~
+  SAY @23
   IF ~~ THEN EXTERN ~BSIME~ 203
 END
 
 IF ~~ THEN BEGIN TS189
-  SAY ~Oh, but the ways of love are often complicated. I will have hope so long as I draw breath.~
+  SAY @24
   IF ~~ THEN EXTERN ~BSIME~ 204
 END
 
 IF ~~ THEN BEGIN TS190
-  SAY ~Is it my hair color?~
+  SAY @25
   IF ~~ THEN EXTERN ~BSIME~ 205
 END
 
 IF ~~ THEN BEGIN TS191
-  SAY ~That's a relief. You could not begin to imagine how I suffer over it!~
+  SAY @26
   IF ~~ THEN EXTERN ~BSIME~ 206
 END
 END
