--- TS\bgtdlg\aBKORGAN.D	Thu Jan 06 17:45:25 2005
+++ C:\BWP Patchstudio\patched files\TS\bgtdlg\aBKORGAN.D	Thu May 05 10:43:46 2016
@@ -1,52 +1,52 @@
 APPEND ~BKORGAN~
 
 IF ~~ THEN BEGIN TS134
-  SAY ~Shut your mouth, girl afore I shut it with me fist.~
+  SAY @0
   IF ~~ THEN EXTERN ~BSIME~ 17
 END
 
 IF ~~ THEN BEGIN TS135
-  SAY ~Blasted fool! Think you can spit on me cause I take care of meself and not burden me head with worries of nonsense! I remember there was that priest of Ilmater, who thought to convert me...~
+  SAY @1
   IF ~~ THEN EXTERN ~BSIME~ 18
 END
 
 IF ~~ THEN BEGIN TS136
-  SAY ~A dimwit more like. I hit low times and I told him, I'd go to his darned church and sleep there. ~
+  SAY @2
   IF ~~ THEN EXTERN ~BSIME~ 19
 END
 
 IF ~~ THEN BEGIN TS137
-  SAY ~I did not stay. I slept that night in the best inn!~
+  SAY @3
   IF ~~ THEN EXTERN ~BSIME~ 20
 END
 
 IF ~~ THEN BEGIN TS138
-  SAY ~Some miserly Lord came by with a load of coin to donate to the priests. His sins were plugging him. So the idiot decided to give, not to take. I persuaded him to give... to me that is. Low times were over for Korgan. And he went straight to his god's arms, just as he desired.~
+  SAY @4
   IF ~~ THEN EXTERN ~BSIME~ 21
 END
 
 IF ~~ THEN BEGIN TS139
-  SAY ~I love when they fight! Come, girl you want it, do not you?~
+  SAY @5
   IF ~~ THEN EXTERN ~BSIME~ 23
 END
 
 IF ~~ THEN BEGIN TS140
-  SAY ~A man cannot go fighting without his ale, fool girl. Now, enough talking!~
+  SAY @6
   IF ~~ THEN EXTERN ~BSIME~ 24
 END
 
 IF ~~ THEN BEGIN TS141
-  SAY ~Ooh! Yor hit a dwarf on the head! Ha-Ha-Ha! Yor have a lot to learn!~
+  SAY @7
   IF ~~ THEN EXTERN ~BSIME~ 25
 END
 
 IF ~~ THEN BEGIN TS142
-  SAY ~Hot stuff!~
+  SAY @8
   IF ~~ THEN EXTERN ~BSIME~ 26
 END
 
 IF ~~ THEN BEGIN TS143
-  SAY ~Fast learner! Ouch! That hurts! You're gonna suffer... Where is me axe? (*Tumbles over, hits the floor and starts snoring.*)~
+  SAY @9
   IF ~~ THEN EXTERN ~BSIME~ 271
 END
 END
