--- TS\bgtdlg\aBNALIA.D	Wed Nov 17 12:27:11 2004
+++ C:\BWP Patchstudio\patched files\TS\bgtdlg\aBNALIA.D	Thu May 05 10:43:49 2016
@@ -1,87 +1,87 @@
 APPEND ~BNALIA~
 
 IF ~~ THEN BEGIN TS51
-  SAY ~No, Kachiko, how can you say so? I hate this dress! My aunt made me wear it when I was in the Castle. I had to hurry when trolls attacked... So I could not change. Your clothes are simple, yet so elegant. The cut, the embroidery! The color makes you look like... like...~
+  SAY @0
   IF ~~ THEN EXTERN ~BYOSHIM~ TS104
 END
 
 IF ~~ THEN BEGIN TS52
-  SAY ~May be... Or peach... Believe me, Kachiko; many ladies in Athkatla would sell their soul to appear as young and beautiful as you... But they cannot, these painted and perfumed snobs!~
+  SAY @1
   IF ~~ THEN EXTERN ~BKACHI~ 30
 END
 
 IF ~~ THEN BEGIN TS53
-  SAY ~A poor woman working endless hours, loosing her eyesight, her back forever bent, and her fingers disfigured by needles, sewed this dress... No, Kachiko, I prefer your simple and honest dress and I hate mine.~
+  SAY @2
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN TS54
-  SAY ~My aunt is known in the certain circles of Athkatla, but I would not call her infamous, Valygar.~
+  SAY @3
   IF ~~ THEN EXTERN ~BVALYGA~ TS539
 END
 
 IF ~~ THEN BEGIN TS55
-  SAY ~No! Of course not! I could not imagine that Auntie even knows where Slums are.~
+  SAY @4
   IF ~~ THEN EXTERN ~BVALYGA~ TS540
 END
 
 IF ~~ THEN BEGIN TS56
-  SAY ~What... what are you talking about, Valygar?~
+  SAY @5
   IF ~~ THEN EXTERN ~BVALYGA~ TS541
 END
 
 IF ~~ THEN BEGIN TS57
-  SAY ~This cannot be...~
+  SAY @6
   IF ~~ THEN EXTERN ~BVALYGA~ TS542
 END
 
 IF ~~ THEN BEGIN TS58
-  SAY ~Did he... did he die?~
+  SAY @7
   IF ~~ THEN EXTERN ~BVALYGA~ TS543
 END
 
 IF ~~ THEN BEGIN TS59
-  SAY ~Is this even true?~
+  SAY @8
   IF ~~ THEN EXTERN ~BVALYGA~ TS544
 END
 
 IF ~~ THEN BEGIN TS60
-  SAY ~We still should try to support less fortunate. Why would not we donate some money to a temple of Ilmater if we do not have time for helping out ourselves?~
+  SAY @9
   IF ~~ THEN EXTERN ~BVALYGA~ TS546
 END
 
 IF ~~ THEN BEGIN TS61
-  SAY ~I love the Keep, Sime. I spent all my childhood there and I have many fond memories of my relatives and friends coming to visit us there. But of course I cannot help but to feel guilty that my family owns such an excessive space when the less fortunate have no roof over their head.~
+  SAY @10
   IF ~~ THEN EXTERN ~BSIME~ 74
 END
 
 IF ~~ THEN BEGIN TS62
-  SAY ~As a matter of fact I have very definite plans for the Keep. In the Northern wing I shall create a school for the village kids and I plan to re-arrange the whole of the first floor into a charitable hospital. I have corresponded with few priests of Ilmater and few missionaries are willing to help me to start both the school and the hospital.~
+  SAY @11
   IF ~~ THEN EXTERN ~BSIME~ 75
 END
 
 IF ~~ THEN BEGIN TS63
-  SAY ~I love the Keep, Sime. I spent all my childhood there and I have many fond memories of my relatives and friends coming to visit us there.~
+  SAY @12
   IF ~~ THEN EXTERN ~BSIME~ 77
 END
 
 IF ~~ THEN BEGIN TS64
-  SAY ~I do not really mourn the material loss, Sime. What bothers me is that my plans for the Keep were ruined. In the Northern wing I wanted to create a school for the village kids and I planned to re-arrange the whole of the first floor into a charitable hospital. Now none of it's going to happen!~
+  SAY @13
   IF ~~ THEN EXTERN ~BSIME~ 78
 END
 
 IF ~~ THEN BEGIN TS65
-  SAY ~I do not quite recognize you, Sime. The crimson cloak, jewelry, and I if I am not mistaken some make up...~
+  SAY @14
   IF ~~ THEN EXTERN ~BSIME~ 80
 END
 
 IF ~~ THEN BEGIN TS66
-  SAY ~Which one would that be? I reckoned of none.~
+  SAY @15
   IF ~~ THEN EXTERN ~BSIME~ 81
 END
 
 IF ~~ THEN BEGIN TS67
-  SAY ~* Laughs * They take you for an empty headed noble, do they not?~
+  SAY @16
   IF ~~ THEN EXTERN ~BSIME~ 82
 END
 END
