--- TS\bgtnodlg\aBODHIAM.D	Sat Feb 02 19:57:29 2008
+++ C:\BWP Patchstudio\patched files\TS\bgtnodlg\aBODHIAM.D	Thu May 05 10:45:32 2016
@@ -12,7 +12,7 @@
 APPEND BODHIAMB
 
 IF ~~ THEN BEGIN TS20
-  SAY ~You follow in the hopes of retrieving something dear to you. I say that the longer you keep this up, the more you will lose. Consider the somber Valygar. It is so disappointing to lose something, which you were only starting to dream of...~
+  SAY @0
   IF ~OR(2)
 !InPartyAllowDead("Valygar")
 Dead("Valygar")~ THEN GOTO 6
@@ -21,21 +21,19 @@
 END
 
 IF ~~ THEN BEGIN TS21
-  SAY ~Harm as in bodily injury and death? No, Valygar. That would be too crude... I am not a student of the art of torture, but a professor... Follow me further, <CHARNAME>, and you will lose more than you can imagine. Starting now!~
+  SAY @1
   IF ~~ THEN DO ~ClearAllActions()
 StartCutSceneMode()
-StartCutScene("VPCut53e")~ JOURNAL ~The Final Battle with Bodhi.
-
-Bodhi appeared in an ambush shortly before I reached her haven, warning me that she was aware of my pursuit and stealing from me my trusted friend: Valygar.  Before she vanished and unleashed more of her undead creations upon me, Bodhi warned me that if I continue I will lose even more than I thought possible.  I cannot give up on the Rhynn Lanthorn... but now the life and immortal soul of the one I love is at stake, as well! I love? Yes, of cause I love Valygar. As a friend... oh, I found the best possible moment to get entangled in my own feelings! I should get going, no matter how hurt I am; I should get going.~ EXIT
+StartCutScene("VPCut53e")~ JOURNAL @2 EXIT
 END
 
 IF ~~ THEN BEGIN TS22
-  SAY ~You follow in the hopes of retrieving something dear to you. I say that the longer you keep this up, the more you will lose.~
+  SAY @3
   IF ~~ THEN GOTO TS23
 END
 
 IF ~~ THEN BEGIN TS23
-  SAY ~Consider the thief’s daughter by your side. It is so disappointing to lose something, which you were only starting to dream of...~
+  SAY @4
   IF ~OR(2)
 !InPartyAllowDead("Sime")
 Dead("Sime")~ THEN GOTO 6
@@ -44,11 +42,9 @@
 END
 
 IF ~~ THEN BEGIN TS24
-  SAY ~You misunderstand me completely, Sime. I am going to leave him his pathetic life. But yours I shall have, Linvails’ daughter. Follow me further <CHARNAME>, and you will lose more than you can imagine. Starting now!~
+  SAY @5
   IF ~~ THEN DO ~ClearAllActions()
 StartCutSceneMode()
-StartCutScene("VPCut53f")~ JOURNAL ~The Final Battle with Bodhi.
-
-Bodhi appeared in an ambush shortly before I reached her haven, warning me that she was aware of my pursuit and stealing from me my love, my dear Sime. Before she vanished and unleashed more of her undead creations upon me, Bodhi warned me that if I continue I will lose even more than I thought possible. I cannot give up on the Rhynn Lanthorn... but now the life and immortal soul of the one I love is at stake, as well!  I should get going, no matter how hurt I am; I should get going...~ EXIT
+StartCutScene("VPCut53f")~ JOURNAL @6 EXIT
 END
 END
