--- TS\bgtnodlg\aBSAREV2.D	Fri Feb 08 00:44:04 2008
+++ C:\BWP Patchstudio\patched files\TS\bgtnodlg\aBSAREV2.D	Thu May 05 10:45:32 2016
@@ -1,133 +1,132 @@
 APPEND ~BSAREV25~
 
 IF ~~ THEN BEGIN TS145
-  SAY ~Corthala, there is a discrepancy I cannot help wondering about. You make no secret of your strong aversion of arcane magic, suspecting it corrupts all who wields it without exception.
-~
+  SAY @0
   IF ~~ THEN EXTERN ~BVALYG25~ TS119
 END
 
 IF ~~ THEN BEGIN TS146
-  SAY ~Corthala, I have noticed that you give an increasing portion of your attention to my little sister.~
+  SAY @1
   IF ~~ THEN EXTERN ~BVALYG25~ TS120
 END
 
 IF ~~ THEN BEGIN TS147
-  SAY ~(*chuckle*) I know a bit of problematic family backgrounds myself, ranger. My question is this, then. You would seem to get awfully well along with my little sister, and yet, according to your beliefs, she has no freedom of choice but to be corrupted by her power. This begs a question. What are your intentions with her?~
+  SAY @2
   IF ~~ THEN EXTERN ~BVALYG25~ TS121
 END
 
 IF ~~ THEN BEGIN TS148
-  SAY ~Friend... Perhaps you have willfully closed your eyes from what is evident for anyone observant. That makes me wonder... why do you choose not to see? What it is you intend with my sister?~
+  SAY @3
   IF ~~ THEN EXTERN ~BVALYG25~ TS122
 END
 
 IF ~~ THEN BEGIN TS149
-  SAY ~She is a child of the Lord of Murder, Valygar. So am I. If there is anything to learn from this it is that what we are does not dictate our path. I would like you to think on that.~
+  SAY @4
   IF ~~ THEN EXTERN ~BVALYG25~ TS123
 END
 
 IF ~~ THEN BEGIN TS150
-  SAY ~Your reaction shows me that you are in denial. I will have to think on what that implies.~
+  SAY @5
   IF ~~ THEN EXTERN ~BVALYG25~ TS123
 END
 
 IF ~~ THEN BEGIN TS151
-  SAY ~I was tormented in the Nine Hells for many endless days. I have endured tortures and unimaginable pain. I lost all I had, even the very humanity I was so eager to shake away in my haste to become a god. What I have now is a half-existence... Yet I am happy to have a shadow of life I once had. And it was given to me by my sister.~
+  SAY @6
   IF ~~ THEN EXTERN ~BVALYG25~ TS126
 END
 
 IF ~~ THEN BEGIN TS152
-  SAY ~My sister. My only loyalty in this world. I have seen you gaze at her in a manner I dislike, Valygar. I shall warn you: if you dare harm her in any way, I will come for you and then you will be sorry you were born.~
+  SAY @7
   IF ~~ THEN EXTERN ~BVALYG25~ TS127
 END
 
 IF ~~ THEN BEGIN TS153
-  SAY ~What had happened in Baldur’s Gate is between <CHARNAME> and I. Do not dare to judge Bhaal’s blood, human! You will never understand that even as mortal enemies we are still one. My sister feels the same about this.~
+  SAY @8
   IF ~~ THEN EXTERN ~BVALYG25~ TS128
 END
 
 IF ~~ THEN BEGIN TS154
-  SAY ~She resurrected me and shared her soul with me. That is the true extent of her bond with me. I care not what you think of me. Just stay away from her. If she sheds the smallest tear on your account, I will rip your heart out and mix it with dirt.~
+  SAY @9
   IF ~~ THEN EXTERN ~BVALYG25~ TS129
 END
 
 IF ~~ THEN BEGIN TS155
-  SAY ~Your words are quick and hot. Yet why do you tell me of your love before you tell it to her? Are you lying? Are you hiding something? Are you ashamed of being involved with a Bhaalspawn?~
+  SAY @10
   IF ~~ THEN EXTERN ~BVALYG25~ TS131
 END
 
 IF ~~ THEN BEGIN TS156
-  SAY ~Well, I have no love for you either. But show her any disrespect and I will...~
+  SAY @11
   IF ~~ THEN EXTERN ~BVALYG25~ TS132
 END
 
 IF ~~ THEN BEGIN TS157
-  SAY ~Yes. But fear not, ranger, my threats are not idle. I can go from word to deed in an instant and I will make every one of my words to come true.~
+  SAY @12
   IF ~~ THEN EXTERN ~BVALYG25~ TS133
 END
 
 IF ~~ THEN BEGIN TS158
-  SAY ~Ha-ha-ha. I know you are not a boy to be so easily intimidated.  I also know who you are and what you are exactly. Give me a slightest reason...~
+  SAY @13
   IF ~~ THEN EXTERN ~BVALYG25~ TS134
 END
 
 IF ~~ THEN BEGIN TS159
-  SAY ~You are scowling Corthala, not smiling...~
+  SAY @14
   IF ~~ THEN EXTERN ~BVALYG25~ TS135
 END
 
 IF ~~ THEN BEGIN TS160
-  SAY ~I want to have a word with you in private.~
+  SAY @15
   IF ~~ THEN EXTERN ~BVALYG25~ TS137
 END
 
 IF ~~ THEN BEGIN TS161
-  SAY ~Listen to me then. Know that I have felt the despair of the darkest pit of the Abyss, as well as ridden the soaring wave of euphoria akin that of the highest mountaintop of Celestia. I have left the fields of battle awash with blood, the joy of battle still surging in my veins like the poison of dead god in my soul in the height of my madness.~
+  SAY @16
   IF ~~ THEN GOTO TS162
 END
 
 IF ~~ THEN BEGIN TS162
-  SAY ~ And there is one thing in this world arousing all the passion in me more than anything else. It is my little sister, she who gave me a portion of her soul that I shall now treasure forever. And I just thought it... prudent to inform you that I might get extremely upset if I found that someone was playing with her affections.~
+  SAY @17
   IF ~~ THEN GOTO TS163
 END
 
 IF ~~ THEN BEGIN TS163
-  SAY ~That’s why I had been so... passionate lately. She has spoken to me, and she is rather... upset about our endless arguments.~
+  SAY @18
   IF ~~ THEN EXTERN ~BVALYG25~ TS138
 END
 
 IF ~~ THEN BEGIN TS164
-  SAY ~(*chuckle*) Aye, there is truth to that. Possibly I got a bit carried away. But remember, whether you are impressed or not, my sister’s welfare matters to me like nothing else.~
+  SAY @19
   IF ~~ THEN EXTERN ~BVALYG25~ TS139
 END
 
 IF ~~ THEN BEGIN TS165
-  SAY ~You would be chilled deeper, to the very core of your trembling heart should I unveil the terrors and mysteries of the Abyss. You have not seen enough to be truly afraid, Sime. The Pocket Plain our ingenious <CHARNAME> has conjured is a cozy place, trust me.~
+  SAY @20
   IF ~~ THEN EXTERN ~BSIME~ TSTOB1
 END
 
 IF ~~ THEN BEGIN TS166
-  SAY ~Am I terrible to look upon for a young girl like you?~
+  SAY @21
   IF ~~ THEN EXTERN ~BSIME~ TSTOB2
 END
 
 IF ~~ THEN BEGIN TS167
-  SAY ~The attractiveness of a sin, is not it? Women! To make a perfect one it takes all things the goodly gods can bestow and a drop of a demon’s blood.~
+  SAY @22
   IF ~~ THEN EXTERN ~BSIME~ TSTOB3
 END
 
 IF ~~ THEN BEGIN TS168
-  SAY ~Neither is true. Fear? I never knew what it is. The moment I knew that I was about to die I was not fearful, I was surprised. Redemption? Is it my place to seek redemption? My hands did what was done, but my father was in command of my heart, not me.~
+  SAY @23
   IF ~~ THEN EXTERN ~BSIME~ TSTOB5
 END
 
 IF ~~ THEN BEGIN TS169
-  SAY ~Because it is better to be a lowly peasant who lives than a dead godling. I had women, gold, wine, and power enough in my life to acquire a taste for them. Nay, not taste, thirst and hunger for more of those things.~
+  SAY @24
   IF ~~ THEN EXTERN ~BSIME~ TSTOB6
 END
 
 IF ~~ THEN BEGIN TS170
-  SAY ~Power, Sime, is a better lover than any woman and it tastes sweeter than wine and fills your hands fuller than gold coins.~
+  SAY @25
   IF ~~ THEN EXTERN ~BSIME~ TSTOB7
 END
 END
