--- TS\bgtdlg\BIMOEN2.D	Wed Sep 14 12:41:11 2005
+++ C:\BWP Patchstudio\patched files\TS\bgtdlg\BIMOEN2.D	Thu May 05 10:44:17 2016
@@ -1,131 +1,131 @@
 BEGIN ~BIMOEN2~
 
 IF ~~ THEN BEGIN 0
-  SAY ~(*shudders*) Yes, Kachiko it was an awful place, run by an insane madman... I thought that I would never be able to sleep peacefully again... ~
+  SAY @0
   IF ~~ THEN EXTERN ~BKACHI~ 59
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~My wounds are deep and my pain is not passing. I just cannot be my old self again. But I am grateful for your kind words. We... we should catch up with the rest of the group, Kachiko.~
+  SAY @1
   IF ~~ THEN EXTERN ~BKACHI~ 60
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~And both are absolutely gorgeous goddesses! And, Valygar, you are leaving out the most interesting part. Which one of us you like the better?~
+  SAY @2
   IF ~~ THEN EXTERN ~BVALYGA~ TS326
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~(*laughs*) You do not have to answer. But I am not just gorgeous I am also generous. I will tell you something you will probably be interested in hearing. The lights were bad in Spellhold, but I think I spotted <CHARNAME> blush when she mumbled: "Valygar, the ranger, my new trusted friend..."~
+  SAY @3
   IF ~~ THEN EXTERN ~BVALYGA~ TS327
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~Nope, I just made it up. I wanted to check my theory. I think you got a thing for her...~
+  SAY @4
   IF ~~ THEN EXTERN ~BVALYGA~ TS328
 END
 
 IF ~~ THEN BEGIN 5
-  SAY ~Oh, really? How exciting, I know something, which has not happened yet...~
+  SAY @5
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN 6
-  SAY ~Nope, Sime. I'm fine at last. Just tryin' to shake off the nightmares. I was just thinkin' on how many new faces are around. Yours for instance.~
+  SAY @6
   IF ~~ THEN EXTERN ~BSIME~ 110
 END
 
 IF ~~ THEN BEGIN 7
-  SAY ~Must have been complaining about <PRO_HISHER> "little troublemaker" as usual. <CHARNAME> never noticed that I grew up, I am afraid.~
+  SAY @7
   IF ~~ THEN EXTERN ~BSIME~ 111
 END
 
 IF ~~ THEN BEGIN 8
-  SAY ~Jaheira says I look silly when I smile. Well, actually her exact words were "sillier than usual".~
+  SAY @8
   IF ~~ THEN EXTERN ~BSIME~ 112
 END
 
 IF ~~ THEN BEGIN 9
-  SAY ~(*pssst*) I do not steal, Sime, I just look. Don't ya wanna know what each of them might be carrying around in secret?~
+  SAY @9
   IF ~~ THEN EXTERN ~BSIME~ 114
 END
 
 IF ~~ THEN BEGIN 10
-  SAY ~Aerie carries a mirror and... blush!~
+  SAY @10
   IF ~~ THEN EXTERN ~BSIME~ 115
 END
 
 IF ~~ THEN BEGIN 11
-  SAY ~Korgan carries a a book entitled "How to Make Friends and Influence Other People".~
+  SAY @11
   IF ~~ THEN EXTERN ~BSIME~ 116
 END
 
 IF ~~ THEN BEGIN 12
-  SAY ~Guess what our somber ranger carries? A laced lady's glove!~
+  SAY @12
   IF ~~ THEN EXTERN ~BSIME~ 117
 END
 
 IF ~~ THEN BEGIN 13
-  SAY ~Nope!~
+  SAY @13
   IF ~~ THEN EXTERN ~BSIME~ 118
 END
 
 IF ~~ THEN BEGIN 14
-  SAY ~Oh, my dear Mystra, just look at that! Anomen has...~
+  SAY @14
   IF ~~ THEN EXTERN ~BSIME~ 119
 END
 
 IF ~~ THEN BEGIN 15
-  SAY ~Cernd has some sort of seeds here. And bark. And pebbles.~
+  SAY @15
   IF ~~ THEN EXTERN ~BSIME~ 120
 END
 
 IF ~~ THEN BEGIN 16
-  SAY ~Jan has a turnip, no two turnips... three... ten...~
+  SAY @16
   IF ~~ THEN EXTERN ~BSIME~ 121
 END
 
 IF ~~ THEN BEGIN 17
-  SAY ~Yoshimo has nothing, but his weapons and this other stuff he is carrying for <CHARNAME>.~
+  SAY @17
   IF ~~ THEN EXTERN ~BSIME~ 122
 END
 
 IF ~~ THEN BEGIN 18
-  SAY ~Kachiko has a lock of dark hair!~
+  SAY @18
   IF ~~ THEN EXTERN ~BSIME~ 123
 END
 
 IF ~~ THEN BEGIN 19
-  SAY ~Nalia has some sort of blueprints. Looks like it is her castle.~
+  SAY @19
   IF ~~ THEN EXTERN ~BSIME~ 124
 END
 
 IF ~~ THEN BEGIN 20
-  SAY ~Keldorn has a pipe and smoking weed. But I have never seen him smoking!~
+  SAY @20
   IF ~~ THEN EXTERN ~BSIME~ 125
 END
 
 IF ~~ THEN BEGIN 21
-  SAY ~Mazzy has high-heeled shoes!~
+  SAY @21
   IF ~~ THEN EXTERN ~BSIME~ 126
 END
 
 IF ~~ THEN BEGIN 22
-  SAY ~Jaheira has a portrait of Khalid.~
+  SAY @22
   IF ~~ THEN EXTERN ~BSIME~ 127
 END
 
 IF ~~ THEN BEGIN 23
-  SAY ~Well, he carries Boo, cheese for Boo, peanuts for Boo and carrots for Boo...~
+  SAY @23
   IF ~~ THEN EXTERN ~BSIME~ 128
 END
 
 IF ~~ THEN BEGIN 24
-  SAY ~"How I Became an Evil Overlord" by Edwin Odesseiron.~
+  SAY @24
   IF ~~ THEN EXTERN ~BSIME~ 129
 END
 
 IF ~~ THEN BEGIN 25
-  SAY ~I am afraid to pick-pocket Viconia. If she wakes up, she'll kill me on spot.~
+  SAY @25
   IF ~~ THEN EXTERN ~BSIME~ 130
 END
