--- TS\fix_f\BIMOEN25.D	Thu Jan 06 17:40:57 2005
+++ C:\BWP Patchstudio\patched files\TS\fix_f\BIMOEN25.D	Thu May 05 10:47:05 2016
@@ -636,51 +636,51 @@
 END
 
 IF ~~ THEN BEGIN TS112
-  SAY ~Marriage? Wow, Valygar are you finally making the move? Common, <CHARNAME>, don't just stand there gaping at me! You are not that elderly yet, my big sis...~
+  SAY @0
   IF ~~ THEN EXTERN ~BVALYG25~ TS103
 END
 
 IF ~~ THEN BEGIN TS113
-  SAY ~Abstract? You can't be abstract about a wedding! Baldur's Gate is the best place I figure, though since you are from Athkatla, may be <CHARNAME> will settle for it. But only the biggest temple will do...~
+  SAY @1
   IF ~~ THEN EXTERN ~BVALYG25~ TS104
 END
 
 IF ~~ THEN BEGIN TS114
-  SAY ~Talos? No! Wouldn't it be appropriate if Bhaal's temples were still open for business, but before we find one, we'd lose too much time and it will be difficult to gather all 500 guests in some dank hole were Bhaalists linger nowadays? Lathander might do.~
-  IF ~~ THEN REPLY ~Imoen, I am afraid you did not get the point: nobody is getting married.~ GOTO TS115
+  SAY @2
+  IF ~~ THEN REPLY @3 GOTO TS115
 END
 
 IF ~~ THEN BEGIN TS115
-  SAY ~Well, if you plan to be an old maid it's your choice. But it could have been so beautiful: a warm day in the summer, flowers everywhere, tons of guests... You would be dressed in a huge white dress... Of course I'd be your maid of honor... I have seen an absolutely gorgeous pink silk! It will suit my hair perfectly and I'd look like million gold pieces... I'd look like a veritable goddess in that dress...~
-  IF ~~ THEN REPLY ~And of cause the eyes of all the cute men in the room centered on you...~ GOTO TS116
+  SAY @4
+  IF ~~ THEN REPLY @5 GOTO TS116
 END
 
 IF ~~ THEN BEGIN TS116
-  SAY ~Yes! Huge crowd, songs, people going delirious with happiness... How sweet and sad... Why you are so stubborn?~
+  SAY @6
   IF ~~ THEN EXTERN ~BVALYG25~ TS105
 END
 
 IF ~~ THEN BEGIN TS117
-  SAY ~I dunno, Valygar. Granted, our mothers had lousy taste in man, that's for sure! But Bhaal's not the only jerk to have had kids who have become good people. I've seen decent folk among the Bhaalspawn in Saradush. We all turned out differently. And with the amount of children Bhaal sired some were bound to be evil or insane...~
+  SAY @7
   IF ~~ THEN GOTO TS118
 END
 
 IF ~~ THEN BEGIN TS118
-  SAY ~<CHARNAME> is moody sometimes and Bhaal's blood is stronger in her than in me or most of the others... yet she is not a murderous berserker. I am very happy to have <CHARNAME> as my sister. I enjoy our friendship and loyalty that shared blood can bring.~
+  SAY @8
   IF ~~ THEN EXTERN ~BVALYG25~ TS107
 END
 
 IF ~~ THEN BEGIN TS119
-  SAY ~Maybe you grow weary of the company of Bhaalspawn and their duties and their endless struggles? Do you want to return back to your solitude in the Umar Hills?~
+  SAY @9
   IF ~~ THEN EXTERN ~BVALYG25~ TS108
 END
 
 IF ~~ THEN BEGIN TS120
-  SAY ~Why?~
+  SAY @10
   IF ~~ THEN EXTERN ~BVALYG25~ TS109
 END
 
 IF ~~ THEN BEGIN TS121
-  SAY ~No other reason?~
+  SAY @11
   IF ~~ THEN EXTERN ~BVALYG25~ TS110
 END
