--- TS\bgtdlg\KACHIKO.D	Tue Apr 05 22:22:21 2016
+++ C:\BWP Patchstudio\patched files\TS\bgtdlg\KACHIKO.D	Thu May 05 10:44:54 2016
@@ -17,78 +17,78 @@
 IF WEIGHT #2 ~NumberOfTimesTalkedTo(0)
 InParty("Yoshimo")
 Global("VP_Kachiko_Meet","GLOBAL",1)~ THEN BEGIN 0
-  SAY ~Ah, here you are, bastard! You thought you could flee far enough? I promised to find you even in Nine Hells! I am Kachiko Nakanishi, prepare to die!~ [KACHI01]
+  SAY @0
   IF ~~ THEN DO ~SetGlobal("VP_Kachiko_Meet","GLOBAL",2)~ EXTERN ~YOSHJ~ TS152
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~Who are these people with you? Are they your new associates? Don’t think you’re so safe. I can deal with all of them if they try to stop me.~ [KACHI02]
-  IF ~~ THEN REPLY ~Do you mind if I interrupt your polemic?~ GOTO 2
-  IF ~~ THEN REPLY ~Shut up, both of you. You, crazy one, are gonna die!~ EXTERN ~YOSHJ~ TS153
-  IF ~~ THEN REPLY ~[Silent]~ EXTERN ~YOSHJ~ TS154
+  SAY @1
+  IF ~~ THEN REPLY @2 GOTO 2
+  IF ~~ THEN REPLY @3 EXTERN ~YOSHJ~ TS153
+  IF ~~ THEN REPLY @4 EXTERN ~YOSHJ~ TS154
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~No, but this is not really your concern.~
-  IF ~~ THEN REPLY ~I am <CHARNAME>. Please, introduce yourself, and maybe we shall come to a peaceful solution.~ GOTO 3
-  IF ~~ THEN REPLY ~I grow tired of explaining to everyone that I only wish to leave this dungeon as soon as possible. Prepare to die, witch!~ EXTERN ~YOSHJ~ TS153
+  SAY @5
+  IF ~~ THEN REPLY @6 GOTO 3
+  IF ~~ THEN REPLY @7 EXTERN ~YOSHJ~ TS153
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~There is no peaceful solution, but... Hmm... May be you really don’t know anything about this snake. I am Kachiko Nakanishi. From Kozakura. Do you know much of your associate, Yoshimo?~
-  IF ~~ THEN REPLY ~No. Please, tell more.~ GOTO 4
+  SAY @8
+  IF ~~ THEN REPLY @9 GOTO 4
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~This... this... man, Yoshimo, standing behind you is accused of my brother’s murder...~
+  SAY @10
   IF ~~ THEN EXTERN ~YOSHJ~ TS156
 END
 
 IF ~~ THEN BEGIN 5
-  SAY ~I won’t listen to your lies.~
+  SAY @11
   IF ~~ THEN EXTERN ~YOSHJ~ TS155
 END
 
 IF ~~ THEN BEGIN 6
-  SAY ~Yoshimo, you sound sincere... Hmm... If what you say is true, we must come back to Kara-Tur and try to convince my family that you’re not the murderer. But if you lied...~
+  SAY @12
   IF ~~ THEN EXTERN ~YOSHJ~ TS159
 END
 
 IF ~~ THEN BEGIN 7
-  SAY ~Why? I will ensure your safety.~
+  SAY @13
   IF ~~ THEN EXTERN ~YOSHJ~ TS160
 END
 
 IF ~~ THEN BEGIN 8
-  SAY ~And I cannot come back to Kara-Tur without Yoshimo... dead or alive.~
+  SAY @14
   IF ~~ THEN EXTERN ~YOSHJ~ TS162
 END
 
 IF ~~ THEN BEGIN 9
-  SAY ~Farewell.~
+  SAY @15
   IF ~~ THEN EXTERN ~YOSHJ~ TS164
 END
 
 IF ~~ THEN BEGIN 10
-  SAY ~Don’t lie to me. You won’t escape again. I’ll follow you, and I’ll watch you. And once you close your deal with <CHARNAME>, you’ll come with me. <CHARNAME>, do you need another companion in your group? My grandfather trained me equally in the spiritual and martial arts. I am not very strong and cannot wear armour, but I am fast and deadly in combat. Perhaps, in the future I may improve my wizard skills, as it is within my ability.~
-  IF ~~ THEN REPLY ~No, Kachiko, I have no room for you right now. Perhaps I will need your help later.~ GOTO 11
-  IF ~~ THEN REPLY ~Of course, I need a good fighter-mage.~ EXTERN ~YOSHJ~ TS165
-  IF ~~ THEN REPLY ~Sorry, I don’t need your help, and I won’t allow you to follow us. Go back to Kara-Tur alone.~ GOTO 12
-  IF ~~ THEN REPLY ~I don’t know you, I don’t need your problems, and I won’t allow you to follow us. I should have killed you right away.~ EXTERN ~YOSHJ~ TS153
+  SAY @16
+  IF ~~ THEN REPLY @17 GOTO 11
+  IF ~~ THEN REPLY @18 EXTERN ~YOSHJ~ TS165
+  IF ~~ THEN REPLY @19 GOTO 12
+  IF ~~ THEN REPLY @20 EXTERN ~YOSHJ~ TS153
 END
 
 IF ~~ THEN BEGIN 11
-  SAY ~I have neither time, nor desire to wait for you in this dreadful barbaric city. But I won’t leave without Yoshimo.~
+  SAY @21
   IF ~~ THEN DO ~SetGlobal("VP_Kachiko_Meet","GLOBAL",3)~ EXTERN ~YOSHJ~ TS163
 END
 
 IF ~~ THEN BEGIN 12
-  SAY ~I won’t come back without Yoshimo.~
+  SAY @22
   IF ~~ THEN EXTERN ~YOSHJ~ TS164
 END
 
 IF ~~ THEN BEGIN 13
-  SAY ~Watch your tongue, ’friend’.~
+  SAY @23
   IF ~~ THEN DO ~SetGlobalTimer("VP_KachikoRomance","GLOBAL",ONE_DAY)
 SetGlobal("VP_Kachiko_Joined","GLOBAL",1)
 JoinParty()~ EXIT
@@ -110,62 +110,62 @@
 IF WEIGHT #1 ~NumberOfTimesTalkedTo(0)
 !InParty("Yoshimo")
 Global("VP_Kachiko_Meet","GLOBAL",1)~ THEN BEGIN 14
-  SAY ~You! Have you seen Yoshimo?  I know he’s around here somewhere!~ [KACHI04]
-  IF ~~ THEN REPLY ~Yes, I saw him not long time ago. Err...who, might I ask, are you? And what’s the matter?~ DO ~SetGlobal("VP_Kachiko_Meet","GLOBAL",2)~ GOTO 15
-  IF ~~ THEN REPLY ~No, I didn’t see him before.~ DO ~SetGlobal("VP_Kachiko_Meet","GLOBAL",2)~ GOTO 21
-  IF ~~ THEN REPLY ~Who is Yoshimo?~ DO ~SetGlobal("VP_Kachiko_Meet","GLOBAL",2)~ GOTO 22
+  SAY @24
+  IF ~~ THEN REPLY @25 DO ~SetGlobal("VP_Kachiko_Meet","GLOBAL",2)~ GOTO 15
+  IF ~~ THEN REPLY @26 DO ~SetGlobal("VP_Kachiko_Meet","GLOBAL",2)~ GOTO 21
+  IF ~~ THEN REPLY @27 DO ~SetGlobal("VP_Kachiko_Meet","GLOBAL",2)~ GOTO 22
 END
 
 IF ~~ THEN BEGIN 15
-  SAY ~I am Kachiko from the clan of Nakanishi. Did you hear about Kara-Tur?~ [KACHI05]
-  IF ~~ THEN REPLY ~Yes. Go on.~ GOTO 16
-  IF ~~ THEN REPLY ~No. Please, explain.~ GOTO 16
+  SAY @28
+  IF ~~ THEN REPLY @29 GOTO 16
+  IF ~~ THEN REPLY @30 GOTO 16
 END
 
 IF ~~ THEN BEGIN 16
-  SAY ~This bastard you met before is accused in the murder of my eldest brother... Hmm, are you mercenaries?~ [KACHI06]
-  IF ~~ THEN REPLY ~No. I am trying just to get out of this damned dungeon.~ GOTO 17
-  IF ~~ THEN REPLY ~Yes, why not? And what might be your proposition?~ GOTO 18
+  SAY @31
+  IF ~~ THEN REPLY @32 GOTO 17
+  IF ~~ THEN REPLY @33 GOTO 18
 END
 
 IF ~~ THEN BEGIN 17
-  SAY ~Could you help me finding Yoshimo and bringing him to justice?~
-  IF ~~ THEN REPLY ~Of course, I’ll help you. You may join my company if you wish.~ DO ~SetGlobal("VP_Kachiko_Joined","GLOBAL",1)
+  SAY @34
+  IF ~~ THEN REPLY @35 DO ~SetGlobal("VP_Kachiko_Joined","GLOBAL",1)
 SetGlobal("VP_YoshimoSearch","GLOBAL",1)
 JoinParty()~ EXIT
-  IF ~~ THEN REPLY ~And what is the profit?~ GOTO 18
-  IF ~~ THEN REPLY ~Sorry, I have no time for another quest. You are on your own.~ GOTO 20
+  IF ~~ THEN REPLY @36 GOTO 18
+  IF ~~ THEN REPLY @37 GOTO 20
 END
 
 IF ~~ THEN BEGIN 18
-  SAY ~I’ll help you in your quests, if you help me to find Yoshimo... dead or alive.~
-  IF ~~ THEN REPLY ~What makes you think that I need your help?~ GOTO 19
-  IF ~~ THEN REPLY ~You have a deal.~ DO ~SetGlobal("VP_Kachiko_Joined","GLOBAL",1)
+  SAY @38
+  IF ~~ THEN REPLY @39 GOTO 19
+  IF ~~ THEN REPLY @40 DO ~SetGlobal("VP_Kachiko_Joined","GLOBAL",1)
 SetGlobal("VP_YoshimoSearch","GLOBAL",1)
 JoinParty()~ EXIT
-  IF ~~ THEN REPLY ~Sorry, I have no time for another quest. You are on your own.~ GOTO 20
+  IF ~~ THEN REPLY @37 GOTO 20
 END
 
 IF ~~ THEN BEGIN 19
-  SAY ~My grandfather trained me equally in the spiritual and martial arts. I am not very strong and cannot wear armour, but I am fast and deadly in combat. Perhaps, in the future I may improve my wizard skills, as it is within my ability.~
-  IF ~~ THEN REPLY ~Well then you convinced me. Join me, and I’ll help you to find Yoshimo.~ DO ~SetGlobal("VP_Kachiko_Joined","GLOBAL",1)
+  SAY @41
+  IF ~~ THEN REPLY @42 DO ~SetGlobal("VP_Kachiko_Joined","GLOBAL",1)
 SetGlobal("VP_YoshimoSearch","GLOBAL",1)
 JoinParty()~ EXIT
-  IF ~~ THEN REPLY ~Sorry, I have no time for another quest. You are on your own.~ GOTO 22
+  IF ~~ THEN REPLY @37 GOTO 22
 END
 
 IF ~~ THEN BEGIN 20
-  SAY ~I’ll find him eventually!  I SWEAR I’ll find him! There has to be somewhere I haven’t looked!~ [KACHI07]
+  SAY @43
   IF ~~ THEN DO ~EscapeArea()~ EXIT
 END
 
 IF ~~ THEN BEGIN 21
-  SAY ~Liar! I know you hide him here.~
+  SAY @44
   IF ~~ THEN DO ~Enemy()~ EXIT
 END
 
 IF ~~ THEN BEGIN 22
-  SAY ~Don’t make a fool of me! I know you hide him here.~
+  SAY @45
   IF ~~ THEN DO ~Enemy()~ EXIT
 END
 
@@ -174,38 +174,38 @@
 ////////////////////////////////////////////////////////////////////////////////////////////////
 
 IF ~~ THEN BEGIN 23
-  SAY ~I think it is a good idea! Better than staying in this barbaric tavern and listening to drunkard sailors!~
+  SAY @46
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",0)
 ActionOverride("Yoshimo",JoinParty())
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN 24
-  SAY ~If that is what you wish.~
+  SAY @47
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",0)
 ActionOverride("Yoshimo",JoinParty())
 JoinParty()~ EXIT
 END
 
 IF WEIGHT #0 ~Global("VP_YoshFollowsKach","GLOBAL",1)~ THEN BEGIN 25
-  SAY ~It is good to see you again. I had hoped you would return. We still have not managed to get a ship. Do you need another party member or two?~
-  IF ~~ THEN REPLY ~Yes.~ DO ~SetGlobal("VP_YoshFollowsKach","GLOBAL",0)~ GOTO 26
-  IF ~~ THEN REPLY ~No.~ GOTO 27
+  SAY @48
+  IF ~~ THEN REPLY @49 DO ~SetGlobal("VP_YoshFollowsKach","GLOBAL",0)~ GOTO 26
+  IF ~~ THEN REPLY @50 GOTO 27
 END
 
 IF ~~ THEN BEGIN 26
-  SAY ~Good.  Not another word, then.~
+  SAY @51
   IF ~~ THEN DO ~SetGlobal("VP_JoinYoshimo","GLOBAL",1)
 SetGlobal("Kicked_Out","LOCALS",0)~ EXTERN ~YOSHP~ TS25
 END
 
 IF ~~ THEN BEGIN 27
-  SAY ~If that is what you wish.~
+  SAY @47
   IF ~~ THEN DO ~SetGlobal("VP_YoshFollowsKach","GLOBAL",1)~ EXIT
 END
 
 //IF WEIGHT #-2 ~Global("VP_Kachiko_Joins1","GLOBAL",1)~ THEN BEGIN 27 // from:
-//  SAY ~I think it is a good idea! Better than staying in this barbaric tavern and listening to drunkard sailors!~
+//  SAY @46
 //  IF ~~ THEN DO ~SetGlobal("VP_Kachiko_Joins1","GLOBAL",0)
 //JoinParty()
 //SetGlobal("Kicked_Out","LOCALS",0)~ EXIT
