--- TS\bgtnodlg\SIME.D	Mon Feb 11 22:03:11 2008
+++ C:\BWP Patchstudio\patched files\TS\bgtnodlg\SIME.D	Thu May 05 10:46:50 2016
@@ -1,39 +1,39 @@
 BEGIN ~SIME~
 
 IF WEIGHT #-2 ~Global("Sime_Meet","GLOBAL",0)~ THEN BEGIN 0
-  SAY ~Hello, <CHARNAME>. I have heard... things about you, and I am quite impressed. I was wondering if you need another companion? I would be honored if you’d allow me to travel with you.~ [SIMEPC02]
-  IF ~~ THEN REPLY ~Well, may be, but I hate talking to strangers...~ DO ~SetGlobal("Sime_Meet","GLOBAL",1)~ GOTO 1
-  IF ~~ THEN REPLY ~No chance... I seek to speak only to Aran and I am in a hurry. Away with you!~ DO ~SetGlobal("Sime_Meet","GLOBAL",1)~ GOTO 2
-  IF ~~ THEN REPLY ~I am sorry, but I have enough people for now. I’d keep you in mind, though...~ DO ~SetGlobal("Sime_Meet","GLOBAL",1)~ GOTO 2
-  IF ~~ THEN REPLY ~May be, but who are you and what are you capable of doing?~ DO ~SetGlobal("Sime_Meet","GLOBAL",1)~ GOTO 1
+  SAY @0
+  IF ~~ THEN REPLY @1 DO ~SetGlobal("Sime_Meet","GLOBAL",1)~ GOTO 1
+  IF ~~ THEN REPLY @2 DO ~SetGlobal("Sime_Meet","GLOBAL",1)~ GOTO 2
+  IF ~~ THEN REPLY @3 DO ~SetGlobal("Sime_Meet","GLOBAL",1)~ GOTO 2
+  IF ~~ THEN REPLY @4 DO ~SetGlobal("Sime_Meet","GLOBAL",1)~ GOTO 1
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~My name is Sime (*lifts her hood up*)... and as you can see I am a half-elf. I was an apprentice of a wise ranger, Darriel of Tanglewood once, and now I am a ranger in my own right... My faith in Mielikki is strong, and I am fortunate to preach her teaching, so you can regard me as a priestess as well. Apart from that there is little to tell...~ [SIMEPC03]
-  IF ~~ THEN REPLY ~I am sorry, but I have enough people for now. I’d keep you in mind, though.~ GOTO 2
-  IF ~~ THEN REPLY ~You sound like a valuable companion, Sime. I am glad that you wish to join in with us. I am sure we have more time to chat later on, but now we should hurry! Come along.~ GOTO 5
-  IF ~~ THEN REPLY ~A ranger? A priestess of Mielikki? What are you doing in the Shadow Thieves compound in the middle of the big city?~ GOTO 3
+  SAY @5
+  IF ~~ THEN REPLY @6 GOTO 2
+  IF ~~ THEN REPLY @7 GOTO 5
+  IF ~~ THEN REPLY @8 GOTO 3
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~Sure, proceed without me if you are so inclined. I will stay here for the time being and you can always count on my help if you change your mind.~ [SIMEPC04]
+  SAY @9
   IF ~~ THEN DO ~SetGlobal("VP_Sime_Is_Waiting","GLOBAL",1)~ EXIT
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~It has nothing to do with my abilities, I assure you. There are... some very personal reasons for me to be here. I would not want to talk about it right now. But there is nothing... dishonest about it. So, would you want me to travel with you?~ [SIMEPC05]
-  IF ~~ THEN REPLY ~I am sorry, but I have enough people for now. I’d keep you in mind, though.~ GOTO 2
-  IF ~~ THEN REPLY ~If you do not trust me, why should I trust you? No, I’d do without you. ~ GOTO 2
-  IF ~~ THEN REPLY ~You sound like a valuable companion, Sime. And I respect your privacy. May be you will tell me your story one day. Now we should hurry! Come along, Sime.~ GOTO 5
+  SAY @10
+  IF ~~ THEN REPLY @6 GOTO 2
+  IF ~~ THEN REPLY @11 GOTO 2
+  IF ~~ THEN REPLY @12 GOTO 5
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~Sure, proceed without me if you are so inclined. I will stay here for the time being and you can always count on my help if you change your mind.~ [SIMEPC04]
+  SAY @9
   IF ~~ THEN DO ~SetGlobal("VP_Sime_Is_Waiting","GLOBAL",2)~ EXIT
 END
 
 IF ~~ THEN BEGIN 5
-  SAY ~Let’s go.~
+  SAY @13
   IF ~~ THEN DO ~SetGlobal("VP_MyFlirt","LOCALS",1)
 RealSetGlobalTimer("VP_SimeFlirtTimer","GLOBAL",1800)
 JoinParty()~ EXIT
