--- TS\bgtdlg\SIMEP.D	Sun Jun 09 01:54:25 2013
+++ C:\BWP Patchstudio\patched files\TS\bgtdlg\SIMEP.D	Thu May 05 10:45:15 2016
@@ -3,32 +3,32 @@
 IF WEIGHT #1 ~NumTimesTalkedToGT(0)
 Global("Kicked_Out","LOCALS",1)
 Global("VP_OnIsland","GLOBAL",0)~ THEN BEGIN 0
-  SAY ~Hey, you have returned? Have a need of Sime, don’t you?~
-  IF ~~ THEN REPLY ~Yes, Sime, you are right. It was foolish of me to refuse your help. Join in!~ GOTO 1
-  IF ~~ THEN REPLY ~No, just wanted to say hi. You are cool, but I am satisfied with my companions for now.~ GOTO 7
+  SAY @0
+  IF ~~ THEN REPLY @1 GOTO 1
+  IF ~~ THEN REPLY @2 GOTO 7
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~Let’s go.~
+  SAY @3
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",0)
 JoinParty()~ EXIT
 END
 
 IF WEIGHT #2 ~Global("VP_OnIsland","GLOBAL",1)~ THEN BEGIN 2
-  SAY ~If you need a hand, I’d gladly come with you, if not - I’ll stay around the ship and keep an eye on the shifty captain for you...~
-  IF ~~ THEN REPLY ~Great, stay with the ship then, I do not fully trust Saemon. It might easily be his own plot to get the cash and avoid a dangerous sailing to Spellhold. Stay right where you are and call if Havarian will be up to no good.~ GOTO 8
-  IF ~~ THEN REPLY ~Actually, it was foolish of me to refuse your help. Join in, Sime.~ GOTO 9
+  SAY @4
+  IF ~~ THEN REPLY @5 GOTO 8
+  IF ~~ THEN REPLY @6 GOTO 9
 END
 
 IF WEIGHT #3 ~NumTimesTalkedToGT(0)
 Global("Kicked_Out","LOCALS",0)~ THEN BEGIN 3
-  SAY ~I guess the time have come for us to part. Are you sure you do not want my company any longer?~
-  IF ~~ THEN REPLY ~Sime, it is better if we go on separately for now.~ GOTO 4
-  IF ~~ THEN REPLY ~Of course I want your company! It was a mistake, Sime, stay with us.~ GOTO 13
+  SAY @7
+  IF ~~ THEN REPLY @8 GOTO 4
+  IF ~~ THEN REPLY @9 GOTO 13
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~Where should we meet if you were to change your mind?~
+  SAY @10
   IF ~!AreaCheck("AR0601")
 !AreaCheck("AR0602")
 !AreaCheck("AR0603")
@@ -78,7 +78,7 @@
 !AreaCheck("AR2905")
 !AreaCheck("AR2906")
 Global("VP_InHalruaa","GLOBAL",0)
-Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY ~Meet us at the Sea Bounty Tavern.~ GOTO 10
+Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY @11 GOTO 10
   IF ~!AreaCheck("AR0301")
 !AreaCheck("AR0302")
 !AreaCheck("AR0303")
@@ -104,20 +104,20 @@
 !AreaCheck("AR0605")
 !AreaCheck("AR0606")
 !InWatchersKeep()
-Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY ~Just wait right here and we’ll come back for you later.~ GOTO 11
+Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY @12 GOTO 11
   IF ~Global("VP_InHalruaa","GLOBAL",0)
-Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY ~I don’t know... If I change my mind, I’ll find you.~ GOTO 12
+Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY @13 GOTO 12
 END
 
 IF WEIGHT #4 ~Global("SimeKorganFight","GLOBAL",2)
 Global("Kicked_Out","LOCALS",0)~ THEN BEGIN 5
-  SAY ~Bah! The fool isn’t worse a fight. Bastard... If you still need my help, I’d gladly come with you, if not - I’ll find my own way...~
-  IF ~~ THEN REPLY ~Sime, it is better if we go on separately for now.~ GOTO 6
-  IF ~~ THEN REPLY ~Join the party again, then.~ GOTO 17
+  SAY @14
+  IF ~~ THEN REPLY @8 GOTO 6
+  IF ~~ THEN REPLY @15 GOTO 17
 END
 
 IF ~~ THEN BEGIN 6
-  SAY ~Where should we meet if you were to change your mind?~
+  SAY @10
   IF ~!AreaCheck("AR0601")
 !AreaCheck("AR0602")
 !AreaCheck("AR0603")
@@ -180,7 +180,7 @@
 !AreaCheck("TT0305")
 !AreaCheck("TT0306")
 !AreaCheck("TT0307")
-!AreaCheck("TT0308")~ THEN REPLY ~Meet us at the Sea Bounty Tavern.~ GOTO 14
+!AreaCheck("TT0308")~ THEN REPLY @11 GOTO 14
   IF ~!AreaCheck("AR0301")
 !AreaCheck("AR0302")
 !AreaCheck("AR0303")
@@ -206,74 +206,74 @@
 !AreaCheck("AR0605")
 !AreaCheck("AR0606")
 !InWatchersKeep()
-Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY ~Just wait right here and we’ll come back for you later.~ GOTO 15
+Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY @12 GOTO 15
   IF ~Global("VP_InHalruaa","GLOBAL",0)
-Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY ~I don’t know... If I change my mind, I’ll find you.~ GOTO 16
+Global("VP_OnIsland","GLOBAL",0)~ THEN REPLY @13 GOTO 16
 END
 
 IF ~~ THEN BEGIN 7
-  SAY ~As you wish, my friend.  See you soon, I hope.~
+  SAY @16
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN 8
-  SAY ~See you soon, then.  Take care of yourself.~
+  SAY @17
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",1)
 LeaveParty()
 EscapeArea()~ EXIT
 END
 
 IF ~~ THEN BEGIN 9
-  SAY ~I thought as much.~
+  SAY @18
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",0)
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN 10
-  SAY ~See you soon, then.  Take care of yourself.~
+  SAY @17
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",1)
 EscapeAreaMove("AR0313",784,313,7)~ EXIT
 END
 
 IF ~~ THEN BEGIN 11
-  SAY ~As you wish, my friend.  See you soon, I hope.~
+  SAY @16
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",1)~ EXIT
 END
 
 IF ~~ THEN BEGIN 12
-  SAY ~I will be staying with my father for a time. You can find me there, if you require my further assistance.~
+  SAY @19
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",1)
 EscapeAreaMove("AR0307",4642,2000,7)~ EXIT
 END
 
 IF ~~ THEN BEGIN 13
-  SAY ~I thought as much.~
+  SAY @18
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",0)
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN 14
-  SAY ~See you soon, then.  Take care of yourself.~
+  SAY @17
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",1)
 SetGlobal("SimeKorganFight","GLOBAL",3)
 EscapeAreaMove("AR0313",784,313,7)~ EXIT
 END
 
 IF ~~ THEN BEGIN 15
-  SAY ~As you wish, my friend.  See you soon, I hope.~
+  SAY @16
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",1)
 SetGlobal("SimeKorganFight","GLOBAL",3)~ EXIT
 END
 
 IF ~~ THEN BEGIN 16
-  SAY ~I will be staying with my father for a time. You can find me there, if you require my further assistance.~
+  SAY @19
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",1)
 SetGlobal("SimeKorganFight","GLOBAL",3)
 EscapeAreaMove("AR0307",4642,2000,7)~ EXIT
 END
 
 IF ~~ THEN BEGIN 17
-  SAY ~I thought as much.~
+  SAY @18
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",0)
 SetGlobal("SimeKorganFight","GLOBAL",3)
 JoinParty()~ EXIT
@@ -281,8 +281,8 @@
 
 IF WEIGHT #0 ~Global("VP_SimeVampire","GLOBAL",5)
 Global("VP_OnIsland","GLOBAL",0)~ THEN BEGIN 18
-  SAY ~Is it you, my love? That is a happy dream. I was yearning to dream of you for so long! You are just like I remember you, and you smile... I want to kiss you so much! Please, do not go away until I do. It is strange to beg a dream not to go away. Amazing how we have so little control over our dreams...~
-  IF ~~ THEN REPLY ~It is no dream, Sime. It is for real. I took you away from Bodhi’s clutches and I brought you back to life. And you think that after all these troubles I will simply leave you? No way, milady, no way! You are stuck with me now.~ DO ~EraseJournalEntry(32084)
+  SAY @20
+  IF ~~ THEN REPLY @21 DO ~EraseJournalEntry(32084)
 EraseJournalEntry(32085)
 EraseJournalEntry(16351)
 EraseJournalEntry(7002)
@@ -298,7 +298,7 @@
 EraseJournalEntry(79170)
 EraseJournalEntry(79181)
 SetGlobal("VP_SimeVampire","GLOBAL",6)~ GOTO 19
-  IF ~~ THEN REPLY ~A dream? Do dreams kiss like that? (*You give Sime a deep kiss*)~ DO ~EraseJournalEntry(32084)
+  IF ~~ THEN REPLY @22 DO ~EraseJournalEntry(32084)
 EraseJournalEntry(32085)
 EraseJournalEntry(16351)
 EraseJournalEntry(7002)
@@ -314,7 +314,7 @@
 EraseJournalEntry(79170)
 EraseJournalEntry(79181)
 SetGlobal("VP_SimeVampire","GLOBAL",6)~ GOTO 20
-  IF ~~ THEN REPLY ~Sime, that’s me, <CHARNAME>! Can you hear me? You are going to be alright! Breathe deeply and here is some Elven water for you to drink! You are alive!~ DO ~EraseJournalEntry(32084)
+  IF ~~ THEN REPLY @23 DO ~EraseJournalEntry(32084)
 EraseJournalEntry(32085)
 EraseJournalEntry(16351)
 EraseJournalEntry(7002)
@@ -333,19 +333,19 @@
 END
 
 IF ~~ THEN BEGIN 19
-  SAY ~I am it seems. I... I am sorry... I am not crying, it is just... well tears pouring out of my eyes. I love you, <CHARNAME>, I love you so much!~
+  SAY @24
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",0)
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN 20
-  SAY ~(*Matches your kiss and embraces you tightly*)~
+  SAY @25
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",0)
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN 21
-  SAY ~Ouch! This holy water is cold! I am awake now, stop pouring it over me!  I am bloody glad to see you... Ahem, sorry, very glad to see you...~
+  SAY @26
   IF ~~ THEN DO ~SetGlobal("Kicked_Out","LOCALS",0)
 JoinParty()~ EXIT
 END
