--- TS\bgtnodlg\WOLF.D	Sat Feb 09 20:45:07 2008
+++ C:\BWP Patchstudio\patched files\TS\bgtnodlg\WOLF.D	Thu May 05 10:47:03 2016
@@ -1,17 +1,17 @@
 BEGIN ~WOLF~
 
 IF ~!CharName("Sime",LastTalkedToBy)~ THEN BEGIN 0
-  SAY ~Veter looks at you with suspicion, and after a moment's inspection, it turns and gives you full view of its rear while it strolls to Sime.~
+  SAY @0
   IF ~~ THEN DO ~MoveToObject("Sime")~ EXIT
 END
 
 IF ~CharName("Sime",LastTalkedToBy)~ THEN BEGIN 1
-  SAY ~Veter looks at Sime, its nose is twitching slightly, and its tail is shuddering...~
-  IF ~~ THEN REPLY ~Veter, come here...~ GOTO 2
+  SAY @1
+  IF ~~ THEN REPLY @2 GOTO 2
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~*Sime strokes Veter's head gently, and it suddenly turns into the figurine.*~
+  SAY @3
   IF ~~ THEN DO ~GiveItemCreate("Wolf","Sime",0,0,0)
 SetGlobal("VP_Veter","GLOBAL",1)
 SetGlobal("VP_VeterRest","GLOBAL",0)
