--- NeJ\Dfiles\DEREVAIN.D	Sun Dec 04 01:07:13 2016
+++ C:\BWP Patchstudio\patched files\NeJ\Dfiles\DEREVAIN.D	Wed Apr 10 10:24:00 2019
@@ -1,247 +1,247 @@
 BEGIN ~DEREVAIN~ 83119 // non-zero flags may indicate non-pausing dialogue
 
 IF ~NumTimesTalkedTo(0)~ THEN BEGIN 0 // from:
-  SAY ~Well met, friend.  I am Erevain Blacksheaf of Evereska.  It has been many moons since I have seen friendly faces.  Despite the dreary nature of our current locale, Easthaven is a welcome sight for these elven eyes.~ 
-  IF ~~ THEN REPLY ~Hello.  I'm <GABBER>.  Tell me, Erevain, have you seen much in your travels?~  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 1
-  IF ~!Race(Player1, ELF)~ THEN REPLY ~An elf, eh?  What in the Nine Hells are you doing up here?~  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 2
-  IF ~Race(Player1, ELF)~ THEN REPLY ~Well met, brother.  How goes your journey?~  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 1
-  IF ~~ THEN REPLY ~What's wrong with the locale?  There's nothing wrong with a little bit of chilly weather.~  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 3
-  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~Greetings. Have you heard any rumors in your travels?~ DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 9
-  IF ~~ THEN REPLY ~Yeah, she's a beaut, isn't she?  Well, I must be off.  Farewell.~  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO EOD
+  SAY @0 
+  IF ~~ THEN REPLY @1  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 1
+  IF ~!Race(Player1, ELF)~ THEN REPLY @2  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 2
+  IF ~Race(Player1, ELF)~ THEN REPLY @3  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 1
+  IF ~~ THEN REPLY @4  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 3
+  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @5 DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 9
+  IF ~~ THEN REPLY @6  DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO EOD
 END
 
 IF ~~ THEN BEGIN 1 // from: 0.2 0.0
-  SAY ~When I set out from Evereska, I was used to a peaceful life.  I had no idea what wonders and horrors the life of adventure would bring to my ever-changing doorstep.  I'm glad I didn't go on the Retreat twenty years ago.~ 
+  SAY @7 
   IF ~CheckStatLT(Player1, 9, CHR)
-CheckStatLT(Player1, 9, INT)~ THEN REPLY ~Hmph!  That's why it's called "adventure," stupid elf-person.~  GOTO 4
-  IF ~~ THEN REPLY ~Horror, eh?  Let me tell you about horror.  Have you ever sliced a man from his belly to his throat?  Or watched a man gasp his last breath as you hold his pulsating heart in your hand?  Well... have you?~ GOTO 5
-  IF ~~ THEN REPLY ~There doesn't seem to be much adventure around here, friend.~  GOTO 7
-  IF ~~ THEN REPLY ~Do you know of any other adventurers in Easthaven?~  GOTO 12
-  IF ~!Race(Player1, ELF)~ THEN REPLY ~What is "the Retreat?"~  GOTO 6
-  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~Twenty years ago? You've been adventuring this whole time?~ GOTO mod_ErevainA1
-  IF ~~ THEN REPLY ~Well, it was nice talking to you, but I must take my leave.  Farewell.~  GOTO EOD
+CheckStatLT(Player1, 9, INT)~ THEN REPLY @8  GOTO 4
+  IF ~~ THEN REPLY @9 GOTO 5
+  IF ~~ THEN REPLY @10  GOTO 7
+  IF ~~ THEN REPLY @11  GOTO 12
+  IF ~!Race(Player1, ELF)~ THEN REPLY @12  GOTO 6
+  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @13 GOTO mod_ErevainA1
+  IF ~~ THEN REPLY @14  GOTO EOD
 END
 
 IF ~~ THEN BEGIN 2 // from: 0.1
-  SAY ~Yes, I am an elf.  I am "up here" because I have decided to explore the world before participating in the Retreat.~ 
-  IF ~!Race(Player1, ELF)~ THEN REPLY ~What is "the Retreat?"~  GOTO 6
-  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY ~Yeah, yeah, yeah.  The Retreat.  I gotcha.  So... hear any good rumors lately?~  GOTO 9
-  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY ~Being a traveler, you must hear a lot of interesting things.  Have you heard anything interesting lately?~  GOTO 9
+  SAY @15 
+  IF ~!Race(Player1, ELF)~ THEN REPLY @12  GOTO 6
+  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY @16  GOTO 9
+  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY @17  GOTO 9
   IF ~CheckStatLT(Player1, 6, CHR)
 CheckStatLT(Player1, 6, INT)
-CheckStatLT(Player1, 6, WIS)~ THEN REPLY ~Ooooh!  Ooooh!  An elf!  Are you a Bladesinger?  Can you wield two weapons at the same time?  Can you dodge Fireballs and sleep upside-down in trees?  Ooooh!  Ooooh!  Tell me you can!~  GOTO 8
+CheckStatLT(Player1, 6, WIS)~ THEN REPLY @18  GOTO 8
   IF ~CheckStatLT(Player1, 9, CHR)
-CheckStatLT(Player1, 9, INT)~ THEN REPLY ~So you came to Icewind Dale?  Elf, you're stupid.  This place is stupid too.~  GOTO 4
-  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~Explore the world? You must have been exploring Faerun for a long time to get so far up North.~ GOTO mod_ErevainA1
-  IF ~~ THEN REPLY ~Good luck with your travels.  It was nice talking to you, but I have to go.  Farewell.~  GOTO EOD
+CheckStatLT(Player1, 9, INT)~ THEN REPLY @19  GOTO 4
+  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @20 GOTO mod_ErevainA1
+  IF ~~ THEN REPLY @21  GOTO EOD
 END
 
 IF ~~ THEN BEGIN 3 // from: 0.3
-  SAY ~Ah, I suppose it's fine.  I'm just used to the sunny climate of my homeland.  This place is like an icy anvil where the hammer of Auril smashes down upon us, leaving the landscape barren and the horizon foreboding in every direction.~ 
-  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY ~Yeah, yeah, yeah.  It's not fun.  I gotcha.  So... hear any good rumors lately?~  GOTO 9
-  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY ~True, true.  Being a traveler, you must hear a lot of interesting things.  Have you heard anything interesting lately?~  GOTO 9
-  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~Where is your homeland? Where do you hail from? ~ GOTO mod_ErevainA3
-  IF ~~ THEN REPLY ~Yeah, I suppose your right.  I have to be getting back to the "icy anvil" myself.  Farewell.~  GOTO EOD
+  SAY @22 
+  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY @23  GOTO 9
+  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY @24  GOTO 9
+  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @25 GOTO mod_ErevainA3
+  IF ~~ THEN REPLY @26  GOTO EOD
 END
 
 IF ~~ THEN BEGIN 4 // from: 10.0 8.1 7.1 6.0 2.4 1.0
-  SAY ~Answer me, Corellon... why do I always have to get the idiots?~ 
-  IF ~CheckStatLT(Player1, 6, INT)~ THEN REPLY ~Who you calling id-jut?!  I are saying fare-well.~  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
-  IF ~CheckStatGT(Player1, 5, INT)~ THEN REPLY ~Yeah?  I'm an idiot?  Well... you're a little pointy-eared tree-hugger!  Elfy!  Ha!  Yeah!  How do you like THAT, Mister Elfy Elf-Face?  Farewell... elfy.~  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
-  IF ~CheckStatGT(Player1, 5, INT)~ THEN REPLY ~Oh, so I'm an idiot, am I?  Why don't you step a little closer and say that?  Let's see what you're made of, pansy.~  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
+  SAY @27 
+  IF ~CheckStatLT(Player1, 6, INT)~ THEN REPLY @28  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
+  IF ~CheckStatGT(Player1, 5, INT)~ THEN REPLY @29  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
+  IF ~CheckStatGT(Player1, 5, INT)~ THEN REPLY @30  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
 END
 
 IF ~~ THEN BEGIN 5 // from: 1.1
-  SAY ~Uh... no.  I can't say that I have.  I think I have to be going now.  It was... nice talking to you, friend.  Sweet water and light laughter... I guess.~ 
-  IF ~~ THEN REPLY ~Walk in darkness, "friend."  Farewell.~  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO EOD
+  SAY @31 
+  IF ~~ THEN REPLY @32  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO EOD
 END
 
 IF ~~ THEN BEGIN 6 // from: 2.0 1.4
-  SAY ~The Retreat is the gradual migration of Tel'Quessir to the isle of Evermeet.  It should only take a few hundred years for all of the People to complete the migration.  Until then, I've decided to explore the rest of Faerun.~ 
+  SAY @33 
   IF ~CheckStatLT(Player1, 8, CHR)
-!Race(Player1, ELF)~ THEN REPLY ~That sounds stupid.  Elves are such dummies.~  GOTO 4
-  IF ~~ THEN REPLY ~Interesting.  Hear any good rumors on your way here?~  GOTO 9
-  IF ~~ THEN REPLY ~Do you know of any other adventurers in Easthaven?~  GOTO 12
-  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~Where is your homeland? Where do you hail from? ~ GOTO mod_ErevainA3
-  IF ~~ THEN REPLY ~I wish you well on your travels.  Farewell.~  GOTO EOD
+!Race(Player1, ELF)~ THEN REPLY @34  GOTO 4
+  IF ~~ THEN REPLY @35  GOTO 9
+  IF ~~ THEN REPLY @11  GOTO 12
+  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @25 GOTO mod_ErevainA3
+  IF ~~ THEN REPLY @36  GOTO EOD
 END
 
 IF ~~ THEN BEGIN 7 // from: 12.1 1.2
-  SAY ~You'd be surprised.  Adventure can leap out at you in the most unlikely form and at the most unexpected time.~ 
-  IF ~~ THEN REPLY ~Hmm.  Thanks for the advice.  Have any other information to share?~  GOTO 9
-  IF ~CheckStatLT(Player1, 12, WIS)~ THEN REPLY ~Yeah, whatever.  Anyone tries to pull one over on me and I'll take 'em out so quickly I'll be a LEGEND in these parts.  Move over heroes, there's a new shire-reeve in town.  I'm so fast I can practically dance on the tip of a sword.  WHOOSH!  WHOOSH!  Watch me go!~  GOTO 4
-  IF ~~ THEN REPLY ~I'll keep that in mind.  Farewell.~  GOTO EOD
+  SAY @37 
+  IF ~~ THEN REPLY @38  GOTO 9
+  IF ~CheckStatLT(Player1, 12, WIS)~ THEN REPLY @39  GOTO 4
+  IF ~~ THEN REPLY @40  GOTO EOD
 END
 
 IF ~~ THEN BEGIN 8 // from: 2.3
-  SAY ~Grrrr.  GRRRRR.  NO! NO! NO!  NOOOOOO!  Why does everyone keep asking me that?!~ 
-  IF ~~ THEN REPLY ~Uh oh.  Farewell.~  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO EOD
-  IF ~~ THEN REPLY ~Pray to your fruity gods, elfboy.  It's time for your head to "Retreat" from your body.~  GOTO 4
+  SAY @41 
+  IF ~~ THEN REPLY @42  DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO EOD
+  IF ~~ THEN REPLY @43  GOTO 4
 END
 
 IF ~~ THEN BEGIN 9 // from: 12.0 10.1 7.0 6.1 3.1 3.0 2.2 2.1
-  SAY ~Well, I haven't heard any rumors lately.  However, when I traveled through Kuldahar, I had a strange feeling about something in that valley.  It's more than just the weather that bothered me.  Something seemed amiss in the mountains, the trees, even the clouds in the sky.  Something was out of place.  Something was... wrong.~ 
-  IF ~CheckStatGT(Player1, 9, WIS)~ THEN REPLY ~Now that you mention it, something does seem out of place.  I can't quite put my finger on it, but... yes.  Something's been odd about the weather.  Be careful in your travels.  Farewell.~  EXIT
-  IF ~CheckStatLT(Player1, 10, WIS)~ THEN REPLY ~Eh?  I haven't noticed anything.  Thanks for talking, though.  Farewell.~  GOTO EOD
-  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~I've seen a lot of fascinating and scary occurences myself traveling in strange parts since I left my home at Candlekeep.~ GOTO 1
+  SAY @44 
+  IF ~CheckStatGT(Player1, 9, WIS)~ THEN REPLY @45  EXIT
+  IF ~CheckStatLT(Player1, 10, WIS)~ THEN REPLY @46  GOTO EOD
+  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @47 GOTO 1
   IF ~Global("VP_Create_Erevain","GLOBAL",1)
-NumInPartyGT(1)~ THEN REPLY  ~This is a strange land to us Erevain. You are much more familiar than us. Would you consider joining us to travel to Kuldahar or wherever adventure leads us?~ GOTO mod_ErevainA2
+NumInPartyGT(1)~ THEN REPLY  @48 GOTO mod_ErevainA2
   IF ~Global("VP_Create_Erevain","GLOBAL",1)
-NumInPartyLT(2)~ THEN REPLY  ~This is a strange land to me Erevain. You are much more familiar than I. Would you consider joining me to travel to Kuldahar or wherever adventure leads us?~ GOTO mod_ErevainA2
+NumInPartyLT(2)~ THEN REPLY  @49 GOTO mod_ErevainA2
   IF ~Global("VP_Create_Erevain","GLOBAL",1)
 Global("Join_Posse", "GLOBAL", 3)
-NumInPartyGT(1)~ THEN REPLY  ~We're planning to join Hrothgar on an expedition to Kuldahar. Your familiarity with that area would be a great boon to us. Would you consider joining us?~ GOTO mod_ErevainA2
+NumInPartyGT(1)~ THEN REPLY  @50 GOTO mod_ErevainA2
   IF ~Global("VP_Create_Erevain","GLOBAL",1)
 Global("Join_Posse", "GLOBAL", 3)
-NumInPartyLT(2)~ THEN REPLY  ~I'm planning to join Hrothgar on an expedition to Kuldahar. Your familiarity with that area would be a great boon to us. Would you consider joining us?~ GOTO mod_ErevainA2
-  IF ~~ THEN REPLY ~Interesting.  Thanks for the advice.  Farewell.~  GOTO EOD
+NumInPartyLT(2)~ THEN REPLY  @51 GOTO mod_ErevainA2
+  IF ~~ THEN REPLY @52  GOTO EOD
 END
 
 IF ~NumTimesTalkedToGT(0)
 Global("Erevain_Jerk", "GLOBAL", 0)~ THEN BEGIN 10 // from:
-  SAY ~Well met, again, friends.~ 
-  IF ~!Race(Player1, ELF)~ THEN REPLY ~I'm not your friend, elfling.~  GOTO 4
-  IF ~~ THEN REPLY ~Greetings.  Have you heard any rumors in your travels?~  GOTO 9
-  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~Hello again Erevain. Tell me, have you seen much in your travels?~ GOTO 1
+  SAY @53 
+  IF ~!Race(Player1, ELF)~ THEN REPLY @54  GOTO 4
+  IF ~~ THEN REPLY @55  GOTO 9
+  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @56 GOTO 1
   IF ~!Race(Player1, ELF)
-Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~What in the Nine Hells is an elf doing up here?~ GOTO 2
+Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @57 GOTO 2
   IF ~Race(Player1, ELF)
-Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~Well met again brother. How goes your journey?~ GOTO 1
-  IF ~~ THEN REPLY ~How do you feel about this weather? It's quite chilly around here.~ GOTO 3
-  IF ~~ THEN REPLY ~Well met, Erevain.  Just thought I'd greet you in passing.  Farewell.~  GOTO EOD
+Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @58 GOTO 1
+  IF ~~ THEN REPLY @59 GOTO 3
+  IF ~~ THEN REPLY @60  GOTO EOD
 END
 
 IF ~NumTimesTalkedToGT(0)
 Global("Erevain_Jerk", "GLOBAL", 1)~ THEN BEGIN 11 // from:
-  SAY ~Oh.  You again.  Leave me alone, please.  I'm not interested in talking to you.~
+  SAY @61
   IF ~Global("VP_Create_Erevain","GLOBAL",1)
-Global("Join_Accalia","GLOBAL",1)~ THEN REPLY ~Hrothgar is about to embark on an expedition to Kuldahar to investigate the evil forces plaguing that region. You should join with us.~ GOTO mod_ErevainB1 
-  IF ~~ THEN REPLY ~Uh... farewell.~  GOTO EOD
+Global("Join_Accalia","GLOBAL",1)~ THEN REPLY @62 GOTO mod_ErevainB1 
+  IF ~~ THEN REPLY @63  GOTO EOD
 END
 
 IF ~~ THEN BEGIN 12 // from: 6.2 1.3
-  SAY ~Yes.  I've heard the locals are forming an expedition into the Kuldahar Pass to investigate some problems there.  I won't be going with them, though I may pass through the area.~ 
-  IF ~~ THEN REPLY ~Interesting.  Have any information to share?~  GOTO 9
-  IF ~~ THEN REPLY ~There doesn't seem to be much adventure around here, friend.~  GOTO 7
-  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY ~Why won't you join them especially if you might go through that are anyway? We should all travel together there.~ GOTO mod_ErevainA2
-  IF ~~ THEN REPLY ~Thanks for the information.  Farewell, Erevain.~  GOTO EOD
+  SAY @64 
+  IF ~~ THEN REPLY @65  GOTO 9
+  IF ~~ THEN REPLY @10  GOTO 7
+  IF ~Global("VP_Create_Erevain","GLOBAL",1)~ THEN REPLY @66 GOTO mod_ErevainA2
+  IF ~~ THEN REPLY @67  GOTO EOD
 END
 
 IF ~~ mod_ErevainA1
-  SAY ~Yes indeed. I've travelled all around Evereska, touched on Anauroch, through some of Comyr and the Dalelands, the Silver Marches, now finally the Spine of the World.~ 
-  IF ~~ THEN REPLY ~Hello. I'm <GABBER>. Tell me, Erevain, have you seen much in your travels?~ DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 1
-  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY ~Yeah, yeah, yeah. Lots of places. I gotcha. So... hear any good rumors lately?~ GOTO 9
-  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY ~Being a traveler, you must hear a lot of interesting things. Have you heard anything interesting lately?~ GOTO 9
-  IF ~~ THEN REPLY ~Do you know of any other travelers in Easthaven?~ GOTO 12
-  IF ~NumInPartyGT(1)~ THEN REPLY ~We're travelers too, hailing from the Sword Coast. Would you consider traveling with us? You would be a great addtion.~ GOTO mod_ErevainA2
-  IF ~NumInPartyLT(2)~ THEN REPLY ~I'm a traveler too, hailing from the Sword Coast. Would you consider traveling with me? You would be a great partner.~ GOTO mod_ErevainA2
+  SAY @68 
+  IF ~~ THEN REPLY @69 DO ~SetGlobal("Erevain_Met","GLOBAL", 1)~ GOTO 1
+  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY @70 GOTO 9
+  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY @71 GOTO 9
+  IF ~~ THEN REPLY @72 GOTO 12
+  IF ~NumInPartyGT(1)~ THEN REPLY @73 GOTO mod_ErevainA2
+  IF ~NumInPartyLT(2)~ THEN REPLY @74 GOTO mod_ErevainA2
   IF ~Global("Join_Posse", "GLOBAL", 3)
-NumInPartyGT(1)~ THEN REPLY ~We're planning to join Hrothgar on an expedition to Kuldahar. We could use an adventurer like yourself. Would you consider joining us?~ GOTO mod_ErevainA2
+NumInPartyGT(1)~ THEN REPLY @75 GOTO mod_ErevainA2
   IF ~Global("Join_Posse", "GLOBAL", 3)
-NumInPartyLT(2)~ THEN REPLY ~I'm planning to join Hrothgar on an expedition to Kuldahar. I could use an adventurer like yourself by my side. Would you consider joining me?~ GOTO mod_ErevainA2
-  IF ~~ THEN REPLY ~Well, that's a lot of interesting places traveled , but I must take my leave. Farewell.~ GOTO EOD
+NumInPartyLT(2)~ THEN REPLY @76 GOTO mod_ErevainA2
+  IF ~~ THEN REPLY @77 GOTO EOD
 END
 
 IF ~~ THEN BEGIN mod_ErevainA2
-  SAY ~I must respectfully decline. I have been used to adventuring alone for the last twenty years. I often avoid battles by sneaking past hostile creatures and cannot reliabily hide many other indviduals.~ 
-  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY ~Fine, be mean like that. But, you better at tell us what rumors you know.~ GOTO 9
-  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY ~I'm sorry to hear that. Can you at least tell us any rumors you know?~ GOTO 9
-  IF ~~ THEN REPLY ~What do you mean avoid battles? Can't you fight?~ GOTO mod_ErevainA4
-  IF ~~ THEN REPLY ~What do you mean sneaking past? Are you a Ranger or Thief?~ GOTO mod_ErevainA5
-  IF ~Global("VP_Erevain_Triple","GLOBAL",1)~ THEN REPLY ~What do you mean hide other individuals? Are you a Mage?~ GOTO mod_ErevainA6
-  IF ~~ THEN REPLY ~It sounds like you're just a coward who can't fight. I can't have cowards join my party.~ GOTO mod_ErevainA4
+  SAY @78 
+  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY @79 GOTO 9
+  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY @80 GOTO 9
+  IF ~~ THEN REPLY @81 GOTO mod_ErevainA4
+  IF ~~ THEN REPLY @82 GOTO mod_ErevainA5
+  IF ~Global("VP_Erevain_Triple","GLOBAL",1)~ THEN REPLY @83 GOTO mod_ErevainA6
+  IF ~~ THEN REPLY @84 GOTO mod_ErevainA4
   IF ~CheckStatLT(Player1, 9, CHR)
-CheckStatLT(Player1, 9, INT)~ THEN REPLY ~Bah. Stupid elves are scared. Scaredy elf!~ GOTO 4
-  IF ~NumInPartyGT(1)~ THEN REPLY ~If you've been adventuring for twenty years, you must be good at it. I'm sure you have many useful skills that could help us survive this harsh land. Please, join us.~ GOTO mod_ErevainA7
-  IF ~NumInPartyLT(2)~ THEN REPLY ~If you've been adventuring for twenty years, you must be good at it. I'm sure you have many useful skills that could help us survive this harsh land. Please, join me.~ GOTO mod_ErevainA8
-  IF ~~ THEN REPLY ~That's unfortunate. You would have made a great adventuring companion. Oh well. Farewell.~ GOTO EOD
+CheckStatLT(Player1, 9, INT)~ THEN REPLY @85 GOTO 4
+  IF ~NumInPartyGT(1)~ THEN REPLY @86 GOTO mod_ErevainA7
+  IF ~NumInPartyLT(2)~ THEN REPLY @87 GOTO mod_ErevainA8
+  IF ~~ THEN REPLY @88 GOTO EOD
 END
 
 IF ~~ THEN BEGIN mod_ErevainA3
-  SAY ~Evereska is my homeland. A region and city adjacent to Comyr and located between the Sword Coast and great Anauroch desert. A great many elves participating in the Retreat are leaving there.~ 
-  IF ~!Race(Player1, ELF)~ THEN REPLY ~What is "the Retreat?"~ GOTO 6
-  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY ~Yeah, yeah, yeah. So... hear any good rumors lately?~ GOTO 9
-  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY ~Fascinating. Being a traveler, you must hear a lot of interesting things. Have you heard anything interesting lately?~ GOTO 9
-  IF ~CheckStatLT(Player1, 9, CHR) CheckStatLT(Player1, 9, INT)~ THEN REPLY ~So you came to Icewind Dale? Elf, you're stupid. This place is stupid too.~ GOTO 4
-  IF ~~ THEN REPLY ~That's certainly a long way away. It was nice talking to you, but I have to go. Farewell.~ GOTO EOD
+  SAY @89 
+  IF ~!Race(Player1, ELF)~ THEN REPLY @12 GOTO 6
+  IF ~CheckStatLT(Player1, 9, CHR)~ THEN REPLY @90 GOTO 9
+  IF ~CheckStatGT(Player1, 8, CHR)~ THEN REPLY @91 GOTO 9
+  IF ~CheckStatLT(Player1, 9, CHR) CheckStatLT(Player1, 9, INT)~ THEN REPLY @92 GOTO 4
+  IF ~~ THEN REPLY @93 GOTO EOD
 END
 
 IF ~~ THEN BEGIN mod_ErevainA4
-  SAY ~Oh, I am a fighter and am quite handy with the longsword as well as the longbow. I have simply learned the limits and dangers of fighting solo and seek to avoid unnecessary conflicts.~ 
-  IF ~NumInPartyGT(1)~ THEN REPLY ~It sounds like you can handle a weapon just fine if need be especially given your long tenure adventuring. Please, join us.~ GOTO mod_ErevainA7
-  IF ~NumInPartyGT(1)~ THEN REPLY ~It sounds like you can handle a weapon just fine if need be especially given your long tenure adventuring. Please, join me.~ GOTO mod_ErevainA8
-  IF ~CheckStatGT(Player1, 8, INT)~ THEN REPLY ~I think you're more likely to run up a tree than be of any real help to us. Good riddance.~ DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
-  IF ~CheckStatLT(Player1, 9, INT)~ THEN REPLY ~If Scared elf not fight then scared elf get eaten. Dumb elf. ~ GOTO 4
- IF ~~ THEN REPLY ~Sorry, I don't think you'd be a good fit here. Farewell.~ GOTO EOD
+  SAY @94 
+  IF ~NumInPartyGT(1)~ THEN REPLY @95 GOTO mod_ErevainA7
+  IF ~NumInPartyGT(1)~ THEN REPLY @96 GOTO mod_ErevainA8
+  IF ~CheckStatGT(Player1, 8, INT)~ THEN REPLY @97 DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
+  IF ~CheckStatLT(Player1, 9, INT)~ THEN REPLY @98 GOTO 4
+ IF ~~ THEN REPLY @99 GOTO EOD
 END
 
 IF ~~ THEN BEGIN mod_ErevainA5
-  SAY ~I prefer to call myself an Infiltrator. I excel in getting to places clandestinely as well as posessing the traditional Thief skills. If I must fight, my strike from the shadows is quite deadly.~ 
-  IF ~~ THEN REPLY ~Sorry, I don't think you'd be a good fit here. Farewell.~ GOTO EOD
-  IF ~NumInPartyGT(1)~ THEN REPLY ~It sounds like you can handle a weapon just fine and offer assistance with your rogue skills. Please, join us.~ GOTO mod_ErevainA7
-  IF ~NumInPartyGT(1)~ THEN REPLY ~It sounds like you can handle a weapon just fine and offer assistance with your rogue skills. Please, join me.~ GOTO mod_ErevainA8
-  IF ~CheckStatGT(Player1, 8, INT)~ THEN REPLY ~I don't like rogues or any of their ilk. Good riddance.~ DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO EOD
-  IF ~CheckStatLT(Player1, 9, INT)~ THEN REPLY ~Elf not stand and fight. Elf stupid and dumb. ~ GOTO 4
+  SAY @100 
+  IF ~~ THEN REPLY @99 GOTO EOD
+  IF ~NumInPartyGT(1)~ THEN REPLY @101 GOTO mod_ErevainA7
+  IF ~NumInPartyGT(1)~ THEN REPLY @102 GOTO mod_ErevainA8
+  IF ~CheckStatGT(Player1, 8, INT)~ THEN REPLY @103 DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO EOD
+  IF ~CheckStatLT(Player1, 9, INT)~ THEN REPLY @104 GOTO 4
 END
 
 IF ~~ THEN BEGIN mod_ErevainA6
-  SAY ~Yes, I am a Mage. I have a variety of spells and special abilities that assist me in going invisible or avoiding detection.~ 
-  IF ~~ THEN REPLY ~Sorry, I don't think you'd be a good fit here. Farewell.~ GOTO EOD
-  IF ~NumInPartyGT(1)~ THEN REPLY ~It sounds like your magic and abilities would be most useful. Please, join us.~ GOTO mod_ErevainA7
-  IF ~NumInPartyGT(1)~ THEN REPLY ~It sounds like your magic and abilities would be most useful. Please, join me.~ GOTO mod_ErevainA8
-  IF ~CheckStatGT(Player1, 8, INT)~ THEN REPLY ~The last thing I want is a helpless mage adding dead weight. Good riddance.~ DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
-  IF ~CheckStatLT(Player1, 9, INT)~ THEN REPLY ~Elf's magic very weak and not good. Dumb elf.~ GOTO 4
+  SAY @105 
+  IF ~~ THEN REPLY @99 GOTO EOD
+  IF ~NumInPartyGT(1)~ THEN REPLY @106 GOTO mod_ErevainA7
+  IF ~NumInPartyGT(1)~ THEN REPLY @107 GOTO mod_ErevainA8
+  IF ~CheckStatGT(Player1, 8, INT)~ THEN REPLY @108 DO ~SetGlobal("Erevain_Jerk","GLOBAL", 1)~ GOTO Break
+  IF ~CheckStatLT(Player1, 9, INT)~ THEN REPLY @109 GOTO 4
 END
 
 IF ~~ THEN BEGIN mod_ErevainA7
-  SAY ~Very well. I shall consider traveling with your group another adventure in of itself. Come, let us explore onward!~
+  SAY @110
   IF ~~ THEN DO ~SetGlobal("VP_Erevain_InParty","GLOBAL",1)
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN mod_ErevainA8
-  SAY ~Very well. I shall consider traveling with you another adventure in of itself. Come, let us explore onward!~
+  SAY @111
   IF ~~ THEN DO ~SetGlobal("VP_Erevain_InParty","GLOBAL",1)
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN mod_ErevainB1
-  SAY ~I might consider joining Hrothgar's expedition for his sake, but you were really rude to me earlier. You need to apologize to me.~ 
-  IF ~CheckStatLT(Player1, 9, INT)~ THEN REPLY ~I'm not saying sorry to stupid elfs. Dumb elfs should apologize to me for being dumb.~ GOTO mod_ErevainB2
-  IF ~CheckStatGT(Player1, 8, INT)~ THEN REPLY ~As I recall. You were quite rude to me. You should apologize to me first.~ GOTO mod_ErevainB2
-  IF ~~ THEN REPLY ~We seem to have gotten off on a bad foot. My apologies Erevain. I hope we can make amends and that you will join us on our expedition~ GOTO mod_ErevainB4
+  SAY @112 
+  IF ~CheckStatLT(Player1, 9, INT)~ THEN REPLY @113 GOTO mod_ErevainB2
+  IF ~CheckStatGT(Player1, 8, INT)~ THEN REPLY @114 GOTO mod_ErevainB2
+  IF ~~ THEN REPLY @115 GOTO mod_ErevainB4
 END
 
 IF ~~ THEN BEGIN mod_ErevainB2
-  SAY ~I am now very sure that joining your expedition is a faulty notion. Best of luck. I shall not be joining you.~
+  SAY @116
   IF ~~ THEN DO ~SetGlobal("Erevain_Jerk","GLOBAL", 2)~ EXIT
 END
 
 IF ~NumTimesTalkedToGT(0)
 Global("Erevain_Jerk", "GLOBAL", 2)~ THEN BEGIN mod_ErevainB3
-  SAY ~I don't want to talk to you and I don't want to travel with you. Leave me be, please.~ 
-  IF ~~ THEN REPLY ~Fine... goodbye.~ EXIT
+  SAY @117 
+  IF ~~ THEN REPLY @118 EXIT
 END
 
 IF ~~ THEN BEGIN mod_ErevainB4
-  SAY ~Very well. Apology accepted. Still, the thought of heading further into this dreary weather even to help others is most distressing. The last thing I need in desolate weather is trouble between my own companions~ 
-  IF ~~ THEN REPLY ~If you're too weak to defend yourself from "trouble" with your companions. Then perhaps you should toughen up. Worrying is for weakinglings.~ GOTO mod_ErevainB2
-  IF ~~ THEN REPLY ~We'll support each other every step of the journey. We have to if we're going to help these people.~ GOTO mod_ErevainB5
-  IF ~~ THEN REPLY ~We have to work together if we're going to get that big reward. I want to be successful just as you do.~ GOTO mod_ErevainB5
+  SAY @119 
+  IF ~~ THEN REPLY @120 GOTO mod_ErevainB2
+  IF ~~ THEN REPLY @121 GOTO mod_ErevainB5
+  IF ~~ THEN REPLY @122 GOTO mod_ErevainB5
 END
 
 IF ~~ THEN BEGIN mod_ErevainB5
-  SAY ~Alright. Let us join Hrothgar and begin the expedition.~
+  SAY @123
   IF ~~ THEN DO ~SetGlobal("VP_Erevain_InParty","GLOBAL",1)
 JoinParty()~ EXIT
 END
 
 IF ~~ THEN BEGIN EOD
-  SAY ~Farewell.~ 
+  SAY @124 
   IF ~~ THEN EXIT
 END
 
 IF ~~ THEN BEGIN Break
-  SAY ~I don't want to talk to you anymore. Leave me be.~ 
+  SAY @125 
   IF ~~ THEN EXIT
 END
