--- NeJ\Dfiles\DVALESTI.D	Sat Dec 20 17:49:02 2014
+++ C:\BWP Patchstudio\patched files\NeJ\Dfiles\DVALESTI.D	Wed Apr 10 10:24:20 2019
@@ -1,145 +1,141 @@
 BEGIN ~DVALESTI~
 
 IF ~NumTimesTalkedTo(0)~ THEN BEGIN 0
-  SAY ~Peace be with you.  I have not seen adventurers in quite some time.  I am called Valestis.  Welcome to the arboretum.~
-  IF ~~ THEN REPLY ~Peace be with you as well, Valestis.  Friendly welcomes are few and far between in the Hand.  What are you doing here?~ GOTO 1
-  IF ~~ THEN REPLY ~Well met, Valestis.  I mean no disrespect, but this doesn't seem to be much of an arboretum.~ GOTO 2
+  SAY @0
+  IF ~~ THEN REPLY @1 GOTO 1
+  IF ~~ THEN REPLY @2 GOTO 2
 END
 
 IF ~~ THEN BEGIN 1
-  SAY ~I stand here and watch, hoping for the day that the plants will grow back.  During the years of peace at the Hand, I cared for the arboretum.  It was a place of indescribable beauty.  Now it lies in ruin, but I am patient.  The day will come when life will grow here again.~
-  IF ~~ THEN REPLY ~How did it end up in such ruin?~ GOTO 3
-  IF ~~ THEN REPLY ~Valestis, I must depart.  Farewell.~ EXIT
+  SAY @3
+  IF ~~ THEN REPLY @4 GOTO 3
+  IF ~~ THEN REPLY @5 EXIT
 END
 
 IF ~~ THEN BEGIN 2
-  SAY ~I take no offense to your words, friend.  You speak the truth.  The arboretum lies in ruins, the plants have long since died -- all the beauty here is gone.  Still, I wait patiently for the day when the cycle of nature begins again and life returns here, even if it takes centuries.~
-  IF ~~ THEN REPLY ~What came to pass that caused the destruction?~ GOTO 3
-  IF ~~ THEN REPLY ~I will leave you to your watch, then.  Farewell.~ EXIT
+  SAY @6
+  IF ~~ THEN REPLY @7 GOTO 3
+  IF ~~ THEN REPLY @8 EXIT
 END
 
 IF ~~ THEN BEGIN 3
-  SAY ~The serene peace we had known for many years was ultimately shattered by war.  When the dark horde came, we found ourselves upon the losing side.~
-  IF ~~ THEN REPLY ~What happened next?~ GOTO 4
+  SAY @9
+  IF ~~ THEN REPLY @10 GOTO 4
 END
 
 IF ~~ THEN BEGIN 4
-  SAY ~While I appreciate what we fought for and the sacrifices that were made, I am not a man of war.  During the last days of the fall, I remained here and watched over the arboretum, the last chamber in the Seldarine's Hand that contained any sort of peace.~
+  SAY @11
   IF ~~ THEN GOTO 5
 END
 
 IF ~~ THEN BEGIN 5
-  SAY ~It was at this time that the Seldarine's Hand was blanketed with the spell that doomed us all.  As my life was drained from me, so was all the beauty within the arboretum.  I watched in horror as every plant and creature died before my eyes.~
+  SAY @12
   IF ~~ THEN DO ~SetGlobal("ArboretumStoryRead","GLOBAL",1)~ GOTO 6
 END
 
 IF ~~ THEN BEGIN 6
-  SAY ~So here is where I stand and watch.  Waiting for the day when life will begin anew within these halls.    ~
-  IF ~~ THEN REPLY ~That is a sad tale, Valestis.  Is there any way we could help you?~ DO ~SetGlobal("ArborQuestGiveN","GLOBAL",1)~ UNSOLVED_JOURNAL ~To restore life to the Arboretum.
-
-We spoke with an elven spirit, Valestis, in the Severed Hand.  He was standing within a ruined Arboretum, and he asked us to help him restore it to its former beauty.  He said if we came across any pure water, any small animals in need of a new home, or seeds that could flourish even in the ruined soil of the chamber's garden, he could use them to restore life to the arboretum.~ GOTO 7
-  IF ~~ THEN REPLY ~I will leave you to your watch, then.  Farewell.~ EXIT
+  SAY @13
+  IF ~~ THEN REPLY @14 DO ~SetGlobal("ArborQuestGiveN","GLOBAL",1)~ UNSOLVED_JOURNAL @15 GOTO 7
+  IF ~~ THEN REPLY @8 EXIT
 END
 
 IF ~~ THEN BEGIN 7
-  SAY ~Perhaps you can.  If in your travels you were to acquire seeds that can flourish in the harshest of places, or perhaps come across a sample of pure water, or even any small animals seeking a new home, I would be able to return some beauty to this place.  Such things no longer exist here in the Hand and can only be found in the rarest of places in the North.~
+  SAY @16
   IF ~PartyHasItem("SEEDS")
-Global("Arbor_Seeds", "GLOBAL", 0)~ THEN REPLY ~I found some seeds that you might be able to use.~ DO ~TakePartyItem("SEEDS")
+Global("Arbor_Seeds", "GLOBAL", 0)~ THEN REPLY @17 DO ~TakePartyItem("SEEDS")
 SetGlobal("Arbor_Seeds", "GLOBAL", 1)
 IncrementGlobal("Arbor_Total","GLOBAL",1)~ GOTO 9
   IF ~PartyHasItem("Water")
-Global("Arbor_Water", "GLOBAL", 0)~ THEN REPLY ~This water might be pure enough for your needs.~ DO ~TakePartyItem("Water")
+Global("Arbor_Water", "GLOBAL", 0)~ THEN REPLY @18 DO ~TakePartyItem("Water")
 SetGlobal("Arbor_Water", "GLOBAL", 1)
 IncrementGlobal("Arbor_Total","GLOBAL",1)~ GOTO 11
   IF ~PartyHasItem("BIRDS")
-Global("Arbor_Birds", "GLOBAL", 0)~ THEN REPLY ~Perhaps you can give these birds a new home.~ DO ~TakePartyItem("BIRDS")
+Global("Arbor_Birds", "GLOBAL", 0)~ THEN REPLY @19 DO ~TakePartyItem("BIRDS")
 SetGlobal("Arbor_Birds", "GLOBAL", 1)
 IncrementGlobal("Arbor_Total","GLOBAL",1)~ GOTO 13
   IF ~PartyHasItem("Squirel")
-Global("Arbor_Squirrels", "GLOBAL", 0)~ THEN REPLY ~These squirrels look like they need a bigger place to live in.~ DO ~TakePartyItem("Squirel")
+Global("Arbor_Squirrels", "GLOBAL", 0)~ THEN REPLY @20 DO ~TakePartyItem("Squirel")
 SetGlobal("Arbor_Squirrels", "GLOBAL", 1)
 IncrementGlobal("Arbor_Total","GLOBAL",1)~ GOTO 15
-  IF ~~ THEN REPLY ~I will see what I can find.  Farewell.~ EXIT
+  IF ~~ THEN REPLY @21 EXIT
 END
 
 IF ~NumTimesTalkedToGT(0)~ THEN BEGIN 8
-  SAY ~Peace be with you.  I see you have returned.~
-  IF ~~ THEN REPLY ~What are you doing here? ~ GOTO 1
-  IF ~Global("ArboretumStoryRead","GLOBAL",1)~ THEN REPLY ~Can we help you restore the arboretum to its former glory?~ GOTO 7
+  SAY @22
+  IF ~~ THEN REPLY @23 GOTO 1
+  IF ~Global("ArboretumStoryRead","GLOBAL",1)~ THEN REPLY @24 GOTO 7
   IF ~Global("ArborQuestGiveN","GLOBAL",1)
-PartyHasItem("SEEDS")Global("Arbor_Seeds", "GLOBAL", 0)~ THEN REPLY ~I found some seeds that you might be able to use.~ DO ~TakePartyItem("SEEDS")
+PartyHasItem("SEEDS")Global("Arbor_Seeds", "GLOBAL", 0)~ THEN REPLY @17 DO ~TakePartyItem("SEEDS")
 SetGlobal("Arbor_Seeds", "GLOBAL", 1)
 IncrementGlobal("Arbor_Total","GLOBAL",1)~ GOTO 9
   IF ~Global("ArborQuestGiveN","GLOBAL",1)PartyHasItem("Water")
-Global("Arbor_Water", "GLOBAL", 0)~ THEN REPLY ~This water might be pure enough for your needs.~ DO ~TakePartyItem("Water")
+Global("Arbor_Water", "GLOBAL", 0)~ THEN REPLY @18 DO ~TakePartyItem("Water")
 SetGlobal("Arbor_Water", "GLOBAL", 1)
 IncrementGlobal("Arbor_Total","GLOBAL",1)~ GOTO 11
   IF ~Global("ArborQuestGiveN","GLOBAL",1)
 PartyHasItem("BIRDS")
-Global("Arbor_Birds", "GLOBAL", 0)~ THEN REPLY ~Perhaps you can give these birds a new home.~ DO ~TakePartyItem("BIRDS")
+Global("Arbor_Birds", "GLOBAL", 0)~ THEN REPLY @19 DO ~TakePartyItem("BIRDS")
 SetGlobal("Arbor_Birds", "GLOBAL", 1)
 IncrementGlobal("Arbor_Total","GLOBAL",1)~ GOTO 13
   IF ~Global("ArborQuestGiveN","GLOBAL",1)
 PartyHasItem("Squirel")
-Global("Arbor_Squirrels", "GLOBAL", 0)~ THEN REPLY ~These squirrels look like they need a bigger place to live in.~ DO ~TakePartyItem("Squirel")
+Global("Arbor_Squirrels", "GLOBAL", 0)~ THEN REPLY @20 DO ~TakePartyItem("Squirel")
 SetGlobal("Arbor_Squirrels", "GLOBAL", 1)
 IncrementGlobal("Arbor_Total","GLOBAL",1)~ GOTO 15
-  IF ~~ THEN REPLY ~I must take my leave.  Farewell.~ EXIT
+  IF ~~ THEN REPLY @25 EXIT
 END
 
 IF ~~ THEN BEGIN 9
-  SAY ~You have found some seeds!  With these, I can return some beauty to this place.  I had almost given up on ever seeing life here.  I thank you.~
-  IF ~!Global("Arbor_Total","GLOBAL",3)~ THEN REPLY ~Think nothing of it.~ DO ~AddexperienceParty(9000)~ GOTO 17
-  IF ~Global("Arbor_Total","GLOBAL",3)~ THEN REPLY ~Think nothing of it.~ DO ~AddexperienceParty(9000)~ GOTO 18
+  SAY @26
+  IF ~!Global("Arbor_Total","GLOBAL",3)~ THEN REPLY @27 DO ~AddexperienceParty(9000)~ GOTO 17
+  IF ~Global("Arbor_Total","GLOBAL",3)~ THEN REPLY @27 DO ~AddexperienceParty(9000)~ GOTO 18
 END
 
 IF ~~ THEN BEGIN 10
-  SAY ~I had almost given up on ever seeing life here.  You have returned some beauty to this place, and I thank you.~
-  IF ~~ THEN REPLY ~You are welcome.~ DO ~AddexperienceParty(6300)~ EXIT
+  SAY @28
+  IF ~~ THEN REPLY @29 DO ~AddexperienceParty(6300)~ EXIT
 END
 
 IF ~~ THEN BEGIN 11
-  SAY ~This water will do nicely!  With this, water will once again flow in the arboretum.  Your actions have brought beauty here once again.  Peace be with you.~
-  IF ~!Global("Arbor_Total","GLOBAL",3)~ THEN REPLY ~Think nothing of it.~ DO ~AddexperienceParty(9000)~ GOTO 17
-  IF ~Global("Arbor_Total","GLOBAL",3)~ THEN REPLY ~Think nothing of it.~ DO ~AddexperienceParty(9000)~ GOTO 18
+  SAY @30
+  IF ~!Global("Arbor_Total","GLOBAL",3)~ THEN REPLY @27 DO ~AddexperienceParty(9000)~ GOTO 17
+  IF ~Global("Arbor_Total","GLOBAL",3)~ THEN REPLY @27 DO ~AddexperienceParty(9000)~ GOTO 18
 END
 
 IF ~~ THEN BEGIN 12
-  SAY ~As you can see, water flows again in the arboretum.  Your actions have brought beauty here once again, and I thank you.  Peace be with you.~
-  IF ~~ THEN REPLY ~To you as well.~ DO ~AddexperienceParty(6300)~ EXIT
+  SAY @31
+  IF ~~ THEN REPLY @32 DO ~AddexperienceParty(6300)~ EXIT
 END
 
 IF ~~ THEN BEGIN 13
-  SAY ~They will like their new home nicely.  Their songs shall serve to brighten the arboretum.  I thank you.~
-  IF ~!Global("Arbor_Total","GLOBAL",3)~ THEN REPLY ~Think nothing of it.~ DO ~AddexperienceParty(9000)~ GOTO 17
-  IF ~Global("Arbor_Total","GLOBAL",3)~ THEN REPLY ~Think nothing of it.~ DO ~AddexperienceParty(9000)~ GOTO 18
+  SAY @33
+  IF ~!Global("Arbor_Total","GLOBAL",3)~ THEN REPLY @27 DO ~AddexperienceParty(9000)~ GOTO 17
+  IF ~Global("Arbor_Total","GLOBAL",3)~ THEN REPLY @27 DO ~AddexperienceParty(9000)~ GOTO 18
 END
 
 IF ~~ THEN BEGIN 14
-  SAY ~They seem to like their new home quite nicely.  Their songs shall serve to brighten the arboretum.  I thank you.~
-  IF ~~ THEN REPLY ~Farewell.~ DO ~AddexperienceParty(6300)~ EXIT
+  SAY @34
+  IF ~~ THEN REPLY @35 DO ~AddexperienceParty(6300)~ EXIT
 END
 
 IF ~~ THEN BEGIN 15
-  SAY ~The arboretum will serve as a pleasant home for them.  Thank you.  Go in peace, my friends.~
-  IF ~!Global("Arbor_Total","GLOBAL",3)~ THEN REPLY ~Think nothing of it.~ DO ~AddexperienceParty(9000)~ GOTO 17
-  IF ~Global("Arbor_Total","GLOBAL",3)~ THEN REPLY ~Think nothing of it.~ DO ~AddexperienceParty(9000)~ GOTO 18
+  SAY @36
+  IF ~!Global("Arbor_Total","GLOBAL",3)~ THEN REPLY @27 DO ~AddexperienceParty(9000)~ GOTO 17
+  IF ~Global("Arbor_Total","GLOBAL",3)~ THEN REPLY @27 DO ~AddexperienceParty(9000)~ GOTO 18
 END
 
 IF ~~ THEN BEGIN 16
-  SAY ~The arboretum will serve as a pleasant home for them.  Thank you.  Go in peace, my friends.~
-  IF ~~ THEN REPLY ~Farewell.~ DO ~AddexperienceParty(6300)~ EXIT
+  SAY @36
+  IF ~~ THEN REPLY @35 DO ~AddexperienceParty(6300)~ EXIT
 END
 
 IF ~~ THEN BEGIN 17
-  SAY ~Oh, and if you come across any other items that could help restore the Arboretum, I would be most grateful.  Farewell, my friends.~
-  IF ~~ THEN REPLY ~Farewell.~ EXIT
+  SAY @37
+  IF ~~ THEN REPLY @35 EXIT
 END
 
 IF ~~ THEN BEGIN 18
-  SAY ~The Arboretum has been returned to its former glory, my friends.  I feel that my work here is done, that it is time to move on.  I thank you.  Farewell.~
-  IF ~~ THEN REPLY ~Farewell, Valestis.~ DO ~DestroySelf()
-EraseJournalEntry(%To restore life to the Arboretum.
-
-We spoke with an elven spirit, Valestis, in the Severed Hand.  He was standing within a ruined Arboretum, and he asked us to help him restore it to its former beauty.  He said if we came across any pure water, any small animals in need of a new home, or seeds that could flourish even in the ruined soil of the chamber's garden, he could use them to restore life to the arboretum.%)~ EXIT
+  SAY @38
+  IF ~~ THEN REPLY @39 DO ~DestroySelf()
+EraseJournalEntry(@15)~ EXIT
 END
