--- NeJ\baf\VP_Mov01.baf	Sat Sep 15 01:10:44 2018
+++ C:\BWP Patchstudio\patched files\NeJ\baf\VP_Mov01.baf	Wed Apr 10 10:30:31 2019
@@ -12,130 +12,48 @@
 		CreateVisualEffectObject("SPCLOUD1",Player5)
 		CreateVisualEffectObject("SPCLOUD1",Player6)
 		SmallWait(7)
-		EraseJournalEntry(~A bottle of wine for Old Jed.
-
-We delivered a bottle of wine to Old Jed, one of the local Easthaven fishermen.~)
-		EraseJournalEntry(~Blade of Aihonen.
-
-We told Jhonen why he was plagued with dreams, and we returned to him his ancestor's blade that Elisia wished to give him.  He was dumbfounded by the gift, and he seemed more than a little sad that she would not be returning to him.  Such sadnesses are the way of people from two worlds.~)
-		EraseJournalEntry(~Blade of Aihonen.
-
-We returned and told Elisia we had delivered the blade to Jhonen.  She was pleased, and she rewarded us for our efforts.~)
-		EraseJournalEntry(~Claim for the Evening Shade.
-
-We showed Aldwin Eidan's signet ring that we had found upstairs, and he confessed to having forged the claim to the Evening Shade... but it didn't seem there was any advantage to us in exposing his deception, so we worked out an arrangement where our band could stay at the Evening Shade for free in exchange for our silence.~)
-		EraseJournalEntry(~Damien's fish.
-
-We were able to help Damien out, and we gave him some fish to take to his father so he wouldn't get in trouble.~)
-		EraseJournalEntry(~Dragon's Eye.
-
-We met the "Lizard King" of Dragon's Eye.  He informed us that the upper caves of the volcano belonged to him.  He also indicated that he was brought to the caves by someone named Yxunomei.  Apparently, the lizard men do the bidding of Yxunomei to feed his people.  They abduct the villagers for food and for Yxunomei.~)
-		EraseJournalEntry(~Dragon's Eye.
-
-We encountered an undead creation today that was used as the "eyes and mouth" of a necromancer.  It attacked us after it had addressed us.~)
-		EraseJournalEntry(~Dragon's Eye.
-
-We ventured down to the fourth level of the Dragon's Eye volcano.  We found a priest of Eldath, the goddess of peace, there.  He told us that he and his students are part of an Eldathyn retreat in Dragon's Eye.  He offered us healing and a place to rest.~)
-		EraseJournalEntry(~Dragon‘s Eye.
-
-We saved a priestess of Ilmater, Egenia, from a group of Talonite priests and trolls.  She came up into Dragon‘s Eye to join in the suffering of the abducted villagers.  She told us that she would wait in Dragon‘s Eye to aid us until we had destroyed the source of evil.~)
-		EraseJournalEntry(~Find the missing people of Kuldahar.
-
-After killing the Lizard King, we encountered a little girl in front of a decline leading to a deeper level of Dragon's Eye.  She thanked us for killing the lizard men and told us that she was going to go home.~)
-		EraseJournalEntry(~Finding a missing caravan.
-
-We told Gaspar the Fishmonger about the lost caravan and showed him the contract from Ilmus Gallaway we'd found in the orc caves.  Once he knew the caravan had been lost, Gaspar set about re-negotiating a contract and insuring that a new caravan of supplies reaches Easthaven.  He thanked us for our trouble, and he paid us a small sum of gold.~)
-		EraseJournalEntry(~Finding a missing caravan.
-
-We found the ambushed caravan bound from Caer-Dinival to Easthaven.  We returned to Hrothgar to tell him of our discovery and of how we tracked the marauding orc raiders back to their lair.  The stout warrior seemed somewhat worried about the orc attack, but was relieved to know that the supplies were still recoverable.~)
-		EraseJournalEntry(~Finding Sheemish, Conlan‘s son.
-
-We spoke to the Kuldahar blacksmith, Conlan, whose son was abducted in the night without a trace. Conlan also mentioned something about the town elder, Arundel. Maybe we should pay him a visit.~)
-		EraseJournalEntry(~Finding Sheemish, Conlan's son.
-
-We spoke to the Kuldahar blacksmith, Conlan, whose son was abducted in the night without a trace.~)
-		EraseJournalEntry(~Finding Sheemish, Conlan's son.
-
-We freed Conlan's son, Sheemish from the depths of Dragon's Eye. We should probably return to Kuldahar to check on him to make sure he got home all right.~)
+		EraseJournalEntry(@0)
+		EraseJournalEntry(@1)
+		EraseJournalEntry(@2)
+		EraseJournalEntry(@3)
+		EraseJournalEntry(@4)
+		EraseJournalEntry(@5)
+		EraseJournalEntry(@6)
+		EraseJournalEntry(@7)
+		EraseJournalEntry(@8)
+		EraseJournalEntry(@9)
+		EraseJournalEntry(@10)
+		EraseJournalEntry(@11)
+		EraseJournalEntry(@12)
+		EraseJournalEntry(@13)
+		EraseJournalEntry(@14)
 		
-		EraseJournalEntry(~Finding the priestess of Auril.
-
-We found the priestess of Auril that Kresselack spoke of, a woman named Lysan. Once she revealed herself, we attacked her.~)
-		EraseJournalEntry(~Get rid of the wolf in Apsel's workshop.
-
-We succeeded in getting rid of the wolf in Apsel's workshop.  Apparently the beast had chewed its way inside to get at the fish bones the old man had stored inside.  Apsel was quite relieved and graciously rewarded us with some gold and carved scrimshaw dagger.~)
-		EraseJournalEntry(~Helping Orrick to retrieve a *mythal*.
-
-Orrick, the mage in Kuldahar, mentioned that he was searching for the ruins of a lost elven outpost rumoured to be located somewhere in the mountains near Kuldahar Pass.  He seemed particularly interested in obtaining any knowledge that pertains to an ancient elven enchantment called a mythal.~)
-		EraseJournalEntry(~Ilmater Clergy in Kuldahar.
-
-We spoke with Sister Calliana, the acting head priest of the Temple of Ilmater in Kuldahar.  She's assumed the role of head priest of the temple ever since Mother Egenia vanished a few weeks ago.  Calliana also mentioned that another member of the Ilmater clergy, a man by the name of Brother Poquelin, passed through Kuldahar recently from all the way from Cormyr and was going to try and get help from the Monastery of the Weeping Lilac in Bryn Shader.  Calliana fears he may have been abducted as well.~)
-		EraseJournalEntry(~Kuldahar Pass.
-
-We met a goblin marshal in Kuldahar Pass who was going mad with pain.  He told us that his head hurt, that something was whispering things to him.  He didn't understand what was wrong with him, but he felt some instinctive urge to "answer" the voice in his head.~)
-		EraseJournalEntry(~Kuldahar Pass.
-
-We met a strange hermit in the valley at the mouth of Kuldahar Pass.  He claimed to have witnessed the avalanche that wiped out the rest of the expedition from Easthaven and nearly killed us as well.  He told us, somewhat reluctantly, that the town of Kuldahar lay to the east of the valley, just beyond an old mill.~)
-		EraseJournalEntry(~Kuldahar Pass.
-
-We found a little boy hiding in a closet in a mill near Kuldahar.  He told us that his father and sister were killed by goblins.  Their bodies were probably the ones we passed on the first floor.~)
-		EraseJournalEntry(~Mirek’s family heirloom.
-
-We were able to retrieve Mirek of Kuldahar’s family heirloom.~)
-		EraseJournalEntry(~Remedy for Ghereg.
-
-We told Ghereg, the ogre in Kuldahar Pass, a recipe that would make his headache go away. He was very grateful, and promptly left.~)
-		EraseJournalEntry(~Disturbances in Kuldahar.
-
-Everard, the head priest of Tempus in Easthaven, confirmed there was a messenger from the northern town of Kuldahar that came to the Temple of Tempus and died shortly after his arrival.  Before his death, the messenger spoke of great disturbances in Kuldahar and asked the people of Easthaven for help. Apparently, Hrothgar is organizing an expedition to Kuldahar to investigate the matter.~)
-		EraseJournalEntry(~Never Ending Journey: Spine Of The World.
-
-Somehow we were teleported from the strange dungeon to the snowy, lakefront community known as Easthaven. Weary and half-frozen from the road, we hastily made our way into the local tavern to find a hot meal and some friendly conversation. Scarcely had we seated ourselves at a table by the hearth when we were approached by a burly warrior who called himself Hrothgar. He seemed friendly enough, but something in his eyes warned us that he wasn‘t one to cross. He politely asked us to come see him at his home, a couple doors west of the tavern, to discuss business of some sort.~)
-		EraseJournalEntry(~Never Ending Journey: Spine Of The World.
-
-So it was that the patchwork militia set off from Easthaven, bound for the troubled village of Kuldahar with the party of strangers in tow. On they traveled, across the windswept tundra of the Dale, through the foothills of the Spine of the World, and upwards on the steep and treacherous trails of the Kuldahar Pass. Eager to seek out the evil that threatened the Pass, they did not expect it to find them first.  High upon the cliffs of the Pass, a band of frost giants had prepared an ambush. Hurling boulders and dislodging massive outcroppings of rock and snow, the giants sparked an avalanche that thundered down the mountainside and crashed down upon the heads of the unsuspecting expedition. Those fortunate enough to survive the avalanche pulled themselves free of the mountain of snow and bodies that now barred the way back to Easthaven. Battered and disheartened by the loss of their comrades, the survivors had little choice but to continue on to Kuldahar... alone.~)
-		EraseJournalEntry(~Search for the Heartstone Gem.
-
-We penetrated the inner sanctum of the Temple of the Forgotten God only to find that someone had gotten their before us. The pedestal that, presumably, had once held the artifact was empty. Bodies lay strewn about the chamber, indicating that the acolytes and their verbeeg guardians had put up fight in the defense of the gem. Among the bodies was the corpse of man garbed in the vestments of a priest of Talona. It is difficult to say what part he might have played in the battle or the resulting theft. Perhaps we should return to Arundel and see what sense he can make of all this.~)
-		EraseJournalEntry(~Search for the Heartstone Gem.
-
-We have agreed to search for a missing druidic artifact called the Heartstone Gem. Apparently, it was stolen from the druids of Kuldahar years ago by one of their own. Arundel believes that it might be located within a temple that lies in a canyon far to the east of Kuldahar. We are to travel to this temple and discover if the Heartstone is indeed there. If it is, we are to return it to Arundel so that he may use its power to divine the location of the true evil behind the disturbances in the pass.~)
-		EraseJournalEntry(~Search for the Heartstone Gem.
-
-Our search for the Heartstone Gem continues. Arundel has managed to piece together the possible location of the Gem from the clues we discovered at the temple. We are to journey to the volcanic caverns known as Dragon‘s Eye and discover the identities of the mysterious thieves who sacked the temple and stole the Gem. Hopefully, the artifact is still in their possession.~)
-		EraseJournalEntry(~Vale of Shadows.
-
-We found the priestess of Auril that Kresselack spoke of, a woman named Lysan. Once she revealed herself, we attacked her.~)
-		EraseJournalEntry(~Vale of Shadows.
-
-We found the priestess of Auril that Kresselack spoke of, a woman named Lysan. She didn't welcome the intrusion, and she attacked us.~)
-		EraseJournalEntry(~Vale of Shadows.
-
-We met the priestess of Auril, and recognized her as a barmaid from Kuldahar. Once she revealed herself, we attacked her.~)
-		EraseJournalEntry(~Vale of Shadows.
-
-We finally spoke with Arundel, the Archdruid of Kuldahar. The old man explained the reason why he sent for help from Easthaven. It seems that there are evil forces stirring in the mountains nearby. He believes this evil is somehow affecting the weather in the pass, causing the ring of warmth that centers around Kuldahar‘s Great Oak to slowly shrink. He also believes that the evil is behind the numerous monster sightings and the mysterious abductions about town. We have agreed to help discover the source of the evil by investigating a nearby vale. Apparently, there have been rumors of undead stirring within some ancient crypts located within this "Vale of Shadows."  We are to travel to the Vale, discover if there is a connection between the undead and the disturbances in Kuldahar, then return to Arundel with whatever information we find.~)
-		EraseJournalEntry(~Vale of Shadows.
-
-We finally spoke with that tree-loving codger who calls himself the Archdruid of Kuldahar. The old druid explained the reason why he sent for help from Easthaven. It seems that there are evil forces stirring in the mountains nearby. He believes this evil is somehow affecting the weather in the pass, causing the ring of warmth that centers around Kuldahar‘s Great Oak to slowly shrink. He also believes that the evil is behind the numerous monster sightings and the mysterious abductions about town. Reluctantly, we have agreed to help by investigating a nearby vale. Apparently, there have been rumors of undead stirring within some ancient crypts located within this "Vale of Shadows."  We are to travel to the Vale, discover if there is a connection between the undead and the disturbances in Kuldahar, then return to Arundel with whatever information we find.~)
-		EraseJournalEntry(~Vale of Shadows.
-
-We spoke with Kresselack‘s spirit, and he claims he is not responsible for the evil that plagues Kuldahar, but he knows who is... since we had already slain the Auril priestess that he wished dead, he was willing to share his knowledge with us. Unfortunately, all Kresselack could tell me was that neither he nor anyone in the Vale of Shadows was responsible for the evil plaguing Kuldahar. It seems we will need to go back to Arundel and inform him of this. Perhaps he will have a clue as to what we should do next.~)
-		EraseJournalEntry(~Vale of Shadows.
-
-We spoke with Kresselack's spirit, and he claims he is not responsible for the evil that plagues Kuldahar... but he claims to know who is. In exchange for an answer, he asked a boon of our party: slay a priestess of Auril who has recently tried to enter his tomb and extinguish the warmth within. Only when she is dead would he give us the answer we seek.~)
-		EraseJournalEntry(~Vale of Shadows.
-
-The spirit of Kresselack asked us to slay an Aurilite priestess whom he thinks intends to destroy his tomb. We have agreed to find this priestess and deal with her but not necessarily slay her. In any case, Kresselack says that he senses her presence within the Vale of Shadows. We should begin our search there.~)
-		EraseJournalEntry(~Vale of Shadows.
-
-We killed Lysan, the Auril priestess, and returned to Kresselack. Unfortunately, all Kresselack could tell us was that neither he or nor anyone in the Vale of Shadows was responsible for the evil plaguing Kuldahar. It seems we will need to go back to Arundel and inform him of this. Perhaps he will have a clue as to what we should do next.~)
-		EraseJournalEntry(~Heartstone Gem.
-
-We gave the Heartstone Gem to Larrel, and he used its power to discover the location of the evil that plagues Kuldahar.  From what the Heartstone was able to divine, the threat lies deep within the dwarven fortress of Dorn‘s Deep.~)
-		EraseJournalEntry(~Severed Hand.
-
-Larrel told us the sad tale of their darkest hour.  In it, he detailed the losing conflict with the dark horde, the desperate plan to use the Mythal, the aftermath of its failed attempt, Labelas‘ anger, and the punishment bestowed upon him.~)
+		EraseJournalEntry(@15)
+		EraseJournalEntry(@16)
+		EraseJournalEntry(@17)
+		EraseJournalEntry(@18)
+		EraseJournalEntry(@19)
+		EraseJournalEntry(@20)
+		EraseJournalEntry(@21)
+		EraseJournalEntry(@22)
+		EraseJournalEntry(@23)
+		EraseJournalEntry(@24)
+		EraseJournalEntry(@25)
+		EraseJournalEntry(@26)
+		EraseJournalEntry(@27)
+		EraseJournalEntry(@28)
+		EraseJournalEntry(@29)
+		EraseJournalEntry(@30)
+		EraseJournalEntry(@31)
+		EraseJournalEntry(@32)
+		EraseJournalEntry(@33)
+		EraseJournalEntry(@34)
+		EraseJournalEntry(@35)
+		EraseJournalEntry(@36)
+		EraseJournalEntry(@37)
+		EraseJournalEntry(@38)
+		EraseJournalEntry(@39)
+		EraseJournalEntry(@40)
 		FadeToColor([30.0],0)
 		Wait(2)
 //		TextScreen("SCENE08")
