--- Bdash\v170\BOTSMITH.d	Fri Apr 29 15:39:56 2016
+++ C:\BWP Patchstudio\patched files\Bdash\v170\BOTSMITH.d	Wed May 04 07:21:04 2016
@@ -5,18 +5,18 @@
 APPEND ~BOTSMITH~
 
 IF ~~ THEN BEGIN N1
-  SAY ~Ooo!  Such a rare and shining sword...~ 
+  SAY @0 
   IF ~!PartyHasItem("JHOSWD2")~ THEN GOTO N2
   IF ~PartyHasItem("JHOSWD2")~ THEN GOTO N3
 END
 
 IF ~~ THEN BEGIN N2
-  SAY ~Has recipe for polishing your Vorpal even more. I'll give you one sword for two. Needing Restored Blade of Aihonen, though, oh yes. No idea where to find one, though.~
+  SAY @1
   IF ~~ THEN GOTO 9
 END
 
 IF ~~ THEN BEGIN N3
-  SAY ~Has recipe for polishing your Vorpal even more. Is combined with Restored Blade of Aihonen. That and 50,000 gp and I will do it for you. Want you that?~
+  SAY @2
   IF ~PartyGoldLT(50000)~ THEN REPLY #66594 /* ~I don't have that much money.~ */ GOTO 10
   IF ~PartyGoldGT(49999)~ THEN REPLY #66595 /* ~Sure... put them together.~ */ DO ~SetGlobal("ItemMaker","GLOBAL",99)
 TakePartyGold(50000)
