--- Setup-SeluneArmoury.tp2	Sun Aug 03 15:43:04 2008
+++ C:\BWP Patchstudio\patched Text files\SeluneArmoury\Setup-SeluneArmoury.tp2	Sun Aug 10 05:43:29 2008
@@ -1,246 +1,79 @@
 BACKUP ~SeluneArmoury\backup~
 AUTHOR ~cuteomen@gmail.com~
-BEGIN ~Selune's Armoury v2~
+
+AUTO_TRA ~SeluneArmoury\%s~
+
+LANGUAGE ~English~
+         ~English~
+         ~SeluneArmoury\English\Setup.tra~
+
+LANGUAGE ~Deutsch~
+         ~German~
+         ~SeluneArmoury\German\Setup.tra~
+
+BEGIN @0
 
 COPY_EXISTING ~doghma.sto~ ~override/doghma.sto~
 ADD_STORE_ITEM "MGIHELM1" #10 #0 #0 ~IDENTIFIED~ #2
 COPY ~SeluneArmoury/Items/MGIHELM1.itm~ ~override/MGIHELM1.itm~
-SAY NAME1 ~Helmet of Wisdom~
-SAY NAME2 ~Helmet of Wisdom~
-SAY UNIDENTIFIED_DESC ~Helmet of Wisdom: "Common Sense"
-This helm was crafted by faithful blacksmiths to aid apprentice clerics and squire holy knights in their spiritual and religious training.
-
-STATISTICS:
-
-Bonus +1 to Wisdom.
-Weight: 4
-Usable By:
- Clerics (single, dual and multi-class)
- Paladins~
-SAY DESC ~Helmet of Wisdom: "Common Sense"
-This helm was crafted by faithful blacksmiths to aid apprentice clerics and squire holy knights in their spiritual and religious training.
-
-STATISTICS:
-
-Bonus +1 to Wisdom.
-Weight: 4
-Usable By:
- Clerics (single, dual and multi-class)
- Paladins~
+SAY NAME1 @1
+SAY NAME2 @1
+SAY UNIDENTIFIED_DESC @2
+SAY DESC @2
 
 COPY_EXISTING ~roger.sto~ ~override/roger.sto~
 ADD_STORE_ITEM "MGIHELM2" #10 #0 #0 ~IDENTIFIED~ #1
 COPY ~SeluneArmoury/Items/MGIHELM2.itm~ ~override/MGIHELM2.itm~
-SAY NAME1 ~Ioun Stone of Alertness~
-SAY NAME2 ~Ioun Stone of Alertness~
-SAY UNIDENTIFIED_DESC ~Ioun Stone of Alertness: "Burglar's Second Eye"
-Smuggled into Amn by the Shadow Thieves, these special ioun stones are specifically crafted for burglars and robbers who prefer to work alone. Wearing this stone above their heads, they find their senses heightened, able to keep an eye out for illusions and traps during a job.
-
-STATISTICS:
-
-Detect Illusions bonus: 15%
-Find Traps bonus: 20%
-Weight: 2
-Usable By:
- Thieves (single, dual and multi-class)~
-SAY DESC ~Ioun Stone of Alertness: "Burglar's Second Eye"
-Smuggled into Amn by the Shadow Thieves, these special ioun stones are specifically crafted for burglars and robbers who prefer to work alone. Wearing this stone above their heads, they find their senses heightened, able to keep an eye out for illusions and traps during a job.
-
-STATISTICS:
-
-Detect Illusions bonus: 15%
-Find Traps bonus: 20%
-Weight: 2
-Usable By:
- Thieves (single, dual and multi-class)~
+SAY NAME1 @3
+SAY NAME2 @3
+SAY UNIDENTIFIED_DESC @4
+SAY DESC @4
 
 COPY_EXISTING ~bernard2.sto~ ~override/bernard2.sto~
 ADD_STORE_ITEM "MGILEAT1" #10 #0 #0 ~IDENTIFIED~ #2
 COPY ~SeluneArmoury/Items/MGILEAT1.itm~ ~override/MGILEAT1.itm~
-SAY NAME1 ~Cured Leaf Hauberk +3~
-SAY NAME2 ~Cured Leaf Hauberk +3~
-SAY UNIDENTIFIED_DESC ~Studded Leather +3: Cured Leaf Hauberk
-These suits of leather armour are created of soft comfortable leather, with fresh cured leaves attached to them by means of magical wax. This combination makes for a tight fit, yet it provides a decent protection against attacks. Only woodsmen and druids would wear these kind of items.
-
-STATISTICS:
-
-Increase casting speed factor: 1
-Armor Class: 5
-Weight: 5
-Requires: 6 Strength
-Usable By:
- Rangers
- Druids~
-SAY DESC ~Studded Leather +3: Cured Leaf Hauberk
-These suits of leather armour are created of soft comfortable leather, with fresh cured leaves attached to them by means of magical wax. This combination makes for a tight fit, yet it provides a decent protection against attacks. Only woodsmen and druids would wear these kind of items.
-
-STATISTICS:
-
-Increase casting speed factor: 1
-Armor Class: 5
-Weight: 5
-Requires: 6 Strength
-Usable By:
- Rangers
- Druids~
+SAY NAME1 @5
+SAY NAME2 @5
+SAY UNIDENTIFIED_DESC @6
+SAY DESC @6
 
 COPY_EXISTING ~ribald.sto~ ~override/ribald.sto~
 ADD_STORE_ITEM "MGIBRAC1" #10 #0 #0 ~IDENTIFIED~ #1
 COPY ~SeluneArmoury/Items/MGIBRAC1.itm~ ~override/MGIBRAC1.itm~
-SAY NAME1 ~Etherbrand Bracers~
-SAY NAME2 ~Etherbrand Bracers~
-SAY UNIDENTIFIED_DESC ~Etherbrand Bracers
-Etherbrand items were commonly used in the old days, and are now a rarity. This is because Etherbrand items have possibly dangerous side-effects when equipped, making the owner think twice on whether the advantages are worth the disadvantages. This set of bracers gives the user a greater number of spells to memorize, yet slows them down in the process.
-
-STATISTICS:
-
-Bonus Wizard Spells: First level (one), Second level (one), Third level (one)
-Penalty: Slow effect
-Weight: 2
-Usable by:
- Mages~
-SAY DESC ~Etherbrand Bracers
-Etherbrand items were commonly used in the old days, and are now a rarity. This is because Etherbrand items have possibly dangerous side-effects when equipped, making the owner think twice on whether the advantages are worth the disadvantages. This set of bracers gives the user a greater number of spells to memorize, yet slows them down in the process.
-
-STATISTICS:
-
-Bonus Wizard Spells: First level (one), Second level (one), Third level (one)
-Penalty: Slow effect
-Weight: 2
-Usable by:
- Mages~
+SAY NAME1 @7
+SAY NAME2 @7
+SAY UNIDENTIFIED_DESC @8
+SAY DESC @8
 
 
 COPY_EXISTING ~ribald.sto~ ~override/ribald.sto~
 ADD_STORE_ITEM "MGIPLAT1" #10 #0 #0 ~IDENTIFIED~ #1
 COPY ~SeluneArmoury/Items/MGIPLAT1.itm~ ~override/MGIPLAT1.itm~
-SAY NAME1 ~Restless Spirit +1~
-SAY NAME2 ~Restless Spirit +1~
-SAY UNIDENTIFIED_DESC ~Full Plate +1: "Restless Spirit"
-This suit of armour was once in the possession of a restless warrior spirit. Part of its ethereal nature remains, as it seems to protect against various forms of magic damage.
-
-STATISTICS:
-
-Magic damage resistance: 15%
-Armor Class: 0
-Weight: 45
-Requires: 16 Strength
-Not Usable By:
- Bard
- Druid
- Mage
- Thief
- Monk~
-SAY DESC ~Full Plate +1: "Restless Spirit"
-This suit of armour was once in the possession of a restless warrior spirit. Part of its ethereal nature remains, as it seems to protect against various forms of magic damage.
-
-STATISTICS:
-
-Magic damage resistance: 15%
-Armor Class: 0
-Weight: 45
-Requires: 16 Strength
-Not Usable By:
- Bard
- Druid
- Mage
- Thief
- Monk~
+SAY NAME1 @9
+SAY NAME2 @9
+SAY UNIDENTIFIED_DESC @10
+SAY DESC @10
 
 COPY_EXISTING ~dshop01.sto~ ~override/dshop01.sto~
 ADD_STORE_ITEM "MGIROBE1" #10 #0 #0 ~IDENTIFIED~ #6
 COPY ~SeluneArmoury/Items/MGIROBE1.itm~ ~override/MGIROBE1.itm~
-SAY NAME1 ~Casual Robes~
-SAY NAME2 ~Casual Robes~
-SAY UNIDENTIFIED_DESC ~Casual Robes
-Casual robes are light and offer protection only slightly surpassing that of cloth, but offer greater freedom than any other armour. As such, it is an ideal fit for monks, bards and mages.
-
-STATISTICS:
-
-Bonus +1 to Dexterity
-Armor Class: 9
-Weight: 4
-Usable By:
- Bard
- Monk~
-SAY DESC ~Casual Robes
-Casual robes are light and offer protection only slightly surpassing that of cloth, but offer greater freedom than any other armour. As such, it is an ideal fit for monks, bards and mages.
-
-STATISTICS:
-
-Bonus +1 to Dexterity
-Armor Class: 9
-Weight: 4
-Usable By:
- Bard
- Monk~
+SAY NAME1 @11
+SAY NAME2 @11
+SAY UNIDENTIFIED_DESC @12
+SAY DESC @12
  
 COPY_EXISTING ~temhelm.sto~ ~override/temhelm.sto~
 ADD_STORE_ITEM "MGISHLD1" #10 #0 #0 ~IDENTIFIED~ #1
 COPY ~SeluneArmoury/Items/MGISHLD1.itm~ ~override/MGISHLD1.itm~
-SAY NAME1 ~Featherweight Shield~
-SAY NAME2 ~Featherweight Shield~
-SAY UNIDENTIFIED_DESC ~Medium Shield +2: "Featherweight"
-This shield radiates an aura of air elemental magic, making it light as a feather and thus easy to use, even for ones who normally lack the physical strength to use medium shield.
-
-STATISTICS:
-
-Armor Class Bonus: 3
-Weight: 1
-Requires: 8 Strength
-Not Usable By:
- Bard
- Druid
- Mage
- Thief
- Monk~
-SAY DESC ~Medium Shield +2: "Featherweight"
-This shield radiates an aura of air elemental magic, making it light as a feather and thus easy to use, even for ones who normally lack the physical strength to use medium shield.
-
-STATISTICS:
-
-Armor Class Bonus: 3
-Weight: 1
-Requires: 8 Strength
-Not Usable By:
- Bard
- Druid
- Mage
- Thief
- Monk~
+SAY NAME1 @13
+SAY NAME2 @13
+SAY UNIDENTIFIED_DESC @14
+SAY DESC @14
 
 COPY_EXISTING ~temlath.sto~ ~override/temlath.sto~
 ADD_STORE_ITEM "MGISHLD2" #10 #0 #0 ~IDENTIFIED~ #1
 COPY ~SeluneArmoury/Items/MGISHLD2.itm~ ~override/MGISHLD2.itm~
-SAY NAME1 ~Shield of Fortitude~
-SAY NAME2 ~Shield of Fortitude~
-SAY UNIDENTIFIED_DESC ~Small Shield +2: "Shield of Fortitude"
-This is a relatively simple shield that holds powerful enchantments that make the wearer more hardy and less susceptible to poison and disease.
-
-STATISTICS:
-
-Bonus +1 to Constitution
-Armor Class Bonus: 3
-Weight: 3
-Requires: 4 Strength
-Not Usable By:
- Bard
- Druid
- Mage
- Thief
- Monk~
-SAY DESC ~Small Shield +2: "Shield of Fortitude"
-This is a relatively simple shield that holds powerful enchantments that make the wearer more hardy and less susceptible to poison and disease.
-
-STATISTICS:
-
-Bonus +1 to Constitution
-Armor Class Bonus: 3
-Weight: 3
-Requires: 4 Strength
-Not Usable By:
- Bard
- Druid
- Mage
- Thief
- Monk~
\ No newline at end of file
+SAY NAME1 @15
+SAY NAME2 @15
+SAY UNIDENTIFIED_DESC @16
+SAY DESC @16
\ No newline at end of file
